module.exports = {
  'SO:0000000': {
    'name': 'Sequence_Ontology'
  },
  'SO:0000001': {
    'name': 'region',
    'def': 'A sequence_feature with an extent greater than zero. A nucleotide region is composed of bases and a polypeptide region is composed of amino acids. [SO:ke]'
  },
  'SO:0000002': {
    'name': 'sequence_secondary_structure',
    'def': 'A folded sequence. [SO:ke]'
  },
  'SO:0000003': {
    'name': 'G_quartet',
    'def': 'G-quartets are unusual nucleic acid structures consisting of a planar arrangement where each guanine is hydrogen bonded by hoogsteen pairing to another guanine in the quartet. [http://www.ncbi.nlm.nih.gov/pubmed/7919797?dopt=Abstract]'
  },
  'SO:0000004': {
    'name': 'interior_coding_exon',
    'def': 'A coding exon that is not the most 3-prime or the most 5-prime in a given transcript. []'
  },
  'SO:0000005': {
    'name': 'satellite_DNA',
    'def': 'The many tandem repeats (identical or related) of a short basic repeating unit; many have a base composition or other property different from the genome average that allows them to be separated from the bulk (main band) genomic DNA. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000006': {
    'name': 'PCR_product',
    'def': 'A region amplified by a PCR reaction. [SO:ke]'
  },
  'SO:0000007': {
    'name': 'read_pair',
    'def': 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert. [SO:ls]'
  },
  'SO:0000008': {
    'name': 'gene_sensu_your_favorite_organism'
  },
  'SO:0000009': {
    'name': 'gene_class'
  },
  'SO:0000010': {
    'name': 'protein_coding',
    'def': 'A gene which, when transcribed, can be translated into a protein. []'
  },
  'SO:0000011': {
    'name': 'non_protein_coding',
    'def': 'A gene which can be transcribed, but will not be translated into a protein. []'
  },
  'SO:0000012': {
    'name': 'scRNA_primary_transcript',
    'def': 'The primary transcript of any one of several small cytoplasmic RNA molecules present in the cytoplasm and sometimes nucleus of a Eukaryote. [http://www.ebi.ac.uk/embl/WebFeat/align/scRNA_s.html]'
  },
  'SO:0000013': {
    'name': 'scRNA',
    'def': 'A small non coding RNA sequence, present in the cytoplasm. [SO:ke]'
  },
  'SO:0000014': {
    'name': 'INR_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters required for the correct positioning of the polymerase for the start of transcription. Overlaps the TSS. The mammalian consensus sequence is YYAN(T|A)YY; the Drosophila consensus sequence is TCA(G|T)t(T|C). In each the A is at position +1 with respect to the TSS. Functionally similar to the TATA box element. [PMID:12651739, PMID:16858867]'
  },
  'SO:0000015': {
    'name': 'DPE_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters; Positioned from +28 to +32 with respect to the TSS (+1). Experimental results suggest that the DPE acts in conjunction with the INR_motif to provide a binding site for TFIID in the absence of a TATA box to mediate transcription of TATA-less promoters. Consensus sequence (A|G)G(A|T)(C|T)(G|A|C). [PMID:12515390, PMID:12537576, PMID:12651739, PMID:16858867]'
  },
  'SO:0000016': {
    'name': 'BREu_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements at -37 to -32 with respect to the TSS (+1). Consensus sequence is (G|C)(G|C)(G|A)CGCC. Binds TFIIB. [PMID:12651739, PMID:16858867]'
  },
  'SO:0000017': {
    'name': 'PSE_motif',
    'def': 'A sequence element characteristic of the promoters of snRNA genes transcribed by RNA polymerase II or by RNA polymerase III. Located between -45 and -60 relative to the TSS. The human PSE_motif consensus sequence is TCACCNTNA(C|G)TNAAAAG(T|G). The basal transcription factor, snRNA-activating protein complex (SNAPc), binds the PSE_motif and is required for the transcription of both RNA polymerase II and III transcribed small-nuclear RNA genes. [PMID:11390411, PMID:12621023, PMID:12651739, PMID:23166507, PMID:8339931]'
  },
  'SO:0000018': {
    'name': 'linkage_group',
    'def': 'A group of loci that can be grouped in a linear order representing the different degrees of linkage among the genes concerned. [ISBN:038752046]'
  },
  'SO:0000020': {
    'name': 'RNA_internal_loop',
    'def': 'A region of double stranded RNA where the bases do not conform to WC base pairing. The loop is closed on both sides by canonical base pairing. If the interruption to base pairing occurs on one strand only, it is known as a bulge. [SO:ke]'
  },
  'SO:0000021': {
    'name': 'asymmetric_RNA_internal_loop',
    'def': 'An internal RNA loop where one of the strands includes more bases than the corresponding region on the other strand. [SO:ke]'
  },
  'SO:0000022': {
    'name': 'A_minor_RNA_motif',
    'def': 'A region forming a motif, composed of adenines, where the minor groove edges are inserted into the minor groove of another helix. [SO:ke]'
  },
  'SO:0000023': {
    'name': 'K_turn_RNA_motif',
    'def': 'The kink turn (K-turn) is an RNA structural motif that creates a sharp (~120 degree) bend between two continuous helices. [SO:ke]'
  },
  'SO:0000024': {
    'name': 'sarcin_like_RNA_motif',
    'def': 'A loop in ribosomal RNA containing the sites of attack for ricin and sarcin. [http://www.ncbi.nlm.nih.gov/pubmed/7897662]'
  },
  'SO:0000025': {
    'name': 'symmetric_RNA_internal_loop',
    'def': 'An internal RNA loop where the extent of the loop on both stands is the same size. [SO:ke]'
  },
  'SO:0000026': {
    'name': 'RNA_junction_loop'
  },
  'SO:0000027': {
    'name': 'RNA_hook_turn'
  },
  'SO:0000028': {
    'name': 'base_pair',
    'def': 'Two bases paired opposite each other by hydrogen bonds creating a secondary structure. []'
  },
  'SO:0000029': {
    'name': 'WC_base_pair',
    'def': 'The canonical base pair, where two bases interact via WC edges, with glycosidic bonds oriented cis relative to the axis of orientation. [PMID:12177293]'
  },
  'SO:0000030': {
    'name': 'sugar_edge_base_pair',
    'def': 'A type of non-canonical base-pairing. [PMID:12177293]'
  },
  'SO:0000031': {
    'name': 'aptamer',
    'def': 'DNA or RNA molecules that have been selected from random pools based on their ability to bind other molecules. [http://aptamer.icmb.utexas.edu]'
  },
  'SO:0000032': {
    'name': 'DNA_aptamer',
    'def': 'DNA molecules that have been selected from random pools based on their ability to bind other molecules. [http:aptamer.icmb.utexas.edu]'
  },
  'SO:0000033': {
    'name': 'RNA_aptamer',
    'def': 'RNA molecules that have been selected from random pools based on their ability to bind other molecules. [http://aptamer.icmb.utexas.edu]'
  },
  'SO:0000034': {
    'name': 'morpholino_oligo',
    'def': 'Morpholino oligos are synthesized from four different Morpholino subunits, each of which contains one of the four genetic bases (A, C, G, T) linked to a 6-membered morpholine ring. Eighteen to 25 subunits of these four subunit types are joined in a specific order by non-ionic phosphorodiamidate intersubunit linkages to give a Morpholino. [http://www.gene-tools.com/]'
  },
  'SO:0000035': {
    'name': 'riboswitch',
    'def': "A riboswitch is a part of an mRNA that can act as a direct sensor of small molecules to control their own expression. A riboswitch is a cis element in the 5' end of an mRNA, that acts as a direct sensor of metabolites. [PMID:2820954]"
  },
  'SO:0000036': {
    'name': 'matrix_attachment_site',
    'def': 'A DNA region that is required for the binding of chromatin to the nuclear matrix. [SO:ma]'
  },
  'SO:0000037': {
    'name': 'locus_control_region',
    'def': 'A DNA region that includes DNAse hypersensitive sites located near a gene that confers the high-level, position-independent, and copy number-dependent expression to that gene. [SO:ma]'
  },
  'SO:0000038': {
    'name': 'match_set',
    'def': 'A collection of match parts. [SO:ke]'
  },
  'SO:0000039': {
    'name': 'match_part',
    'def': 'A part of a match, for example an hsp from blast is a match_part. [SO:ke]'
  },
  'SO:0000040': {
    'name': 'genomic_clone',
    'def': 'A clone of a DNA region of a genome. [SO:ma]'
  },
  'SO:0000041': {
    'name': 'sequence_operation',
    'def': 'An operation that can be applied to a sequence, that results in a change. [SO:ke]'
  },
  'SO:0000042': {
    'name': 'pseudogene_attribute',
    'def': 'An attribute of a pseudogene (SO:0000336). [SO:ma]'
  },
  'SO:0000043': {
    'name': 'processed_pseudogene',
    'def': 'A pseudogene created via retrotranposition of the mRNA of a functional protein-coding parent gene followed by accumulation of deleterious mutations lacking introns and promoters, often including a polyA tail. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0000044': {
    'name': 'pseudogene_by_unequal_crossing_over',
    'def': 'A pseudogene caused by unequal crossing over at recombination. [SO:ke]'
  },
  'SO:0000045': {
    'name': 'delete',
    'def': 'To remove a subsection of sequence. [SO:ke]'
  },
  'SO:0000046': {
    'name': 'insert',
    'def': 'To insert a subsection of sequence. [SO:ke]'
  },
  'SO:0000047': {
    'name': 'invert',
    'def': 'To invert a subsection of sequence. [SO:ke]'
  },
  'SO:0000048': {
    'name': 'substitute',
    'def': 'To substitute a subsection of sequence for another. [SO:ke]'
  },
  'SO:0000049': {
    'name': 'translocate',
    'def': 'To translocate a subsection of sequence. [SO:ke]'
  },
  'SO:0000050': {
    'name': 'gene_part',
    'def': 'A part of a gene, that has no other route in the ontology back to region. This concept is necessary for logical inference as these parts must have the properties of region. It also allows us to associate all the parts of genes with a gene. [SO:ke]'
  },
  'SO:0000051': {
    'name': 'probe',
    'def': 'A DNA sequence used experimentally to detect the presence or absence of a complementary nucleic acid. [SO:ma]'
  },
  'SO:0000052': {
    'name': 'assortment_derived_deficiency'
  },
  'SO:0000053': {
    'name': 'sequence_variant_affecting_regulatory_region',
    'def': 'A sequence_variant_effect which changes the regulatory region of a gene. [SO:ke]'
  },
  'SO:0000054': {
    'name': 'aneuploid',
    'def': 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number. [SO:ke]'
  },
  'SO:0000055': {
    'name': 'hyperploid',
    'def': 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as extra chromosomes are present. [SO:ke]'
  },
  'SO:0000056': {
    'name': 'hypoploid',
    'def': 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as some chromosomes are missing. [SO:ke]'
  },
  'SO:0000057': {
    'name': 'operator',
    'def': 'A regulatory element of an operon to which activators or repressors bind thereby effecting translation of genes in that operon. [SO:ma]'
  },
  'SO:0000058': {
    'name': 'assortment_derived_aneuploid'
  },
  'SO:0000059': {
    'name': 'nuclease_binding_site',
    'def': 'A binding site that, of a nucleotide molecule, that interacts selectively and non-covalently with polypeptide residues of a nuclease. [SO:cb]'
  },
  'SO:0000060': {
    'name': 'compound_chromosome_arm',
    'def': 'One arm of a compound chromosome. []'
  },
  'SO:0000061': {
    'name': 'restriction_enzyme_binding_site',
    'def': 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues of a restriction enzyme. [SO:cb]'
  },
  'SO:0000062': {
    'name': 'deficient_intrachromosomal_transposition',
    'def': 'An intrachromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining. [FB:reference_manual]'
  },
  'SO:0000063': {
    'name': 'deficient_interchromosomal_transposition',
    'def': 'An interchromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining. [SO:ke]'
  },
  'SO:0000064': {
    'name': 'gene_by_transcript_attribute'
  },
  'SO:0000065': {
    'name': 'free_chromosome_arm',
    'def': 'A chromosome structure variation whereby an arm exists as an individual chromosome element. [SO:ke]'
  },
  'SO:0000066': {
    'name': 'gene_by_polyadenylation_attribute'
  },
  'SO:0000067': {
    'name': 'gene_to_gene_feature'
  },
  'SO:0000068': {
    'name': 'overlapping',
    'def': 'An attribute describing a gene that has a sequence that overlaps the sequence of another gene. [SO:ke]'
  },
  'SO:0000069': {
    'name': 'inside_intron',
    'def': 'An attribute to describe a gene when it is located within the intron of another gene. [SO:ke]'
  },
  'SO:0000070': {
    'name': 'inside_intron_antiparallel',
    'def': 'An attribute to describe a gene when it is located within the intron of another gene and on the opposite strand. [SO:ke]'
  },
  'SO:0000071': {
    'name': 'inside_intron_parallel',
    'def': 'An attribute to describe a gene when it is located within the intron of another gene and on the same strand. [SO:ke]'
  },
  'SO:0000072': {
    'name': 'end_overlapping_gene'
  },
  'SO:0000073': {
    'name': 'five_prime_three_prime_overlap',
    'def': "An attribute to describe a gene when the five prime region overlaps with another gene's 3' region. [SO:ke]"
  },
  'SO:0000074': {
    'name': 'five_prime_five_prime_overlap',
    'def': "An attribute to describe a gene when the five prime region overlaps with another gene's five prime region. [SO:ke]"
  },
  'SO:0000075': {
    'name': 'three_prime_three_prime_overlap',
    'def': "An attribute to describe a gene when the 3' region overlaps with another gene's 3' region. [SO:ke]"
  },
  'SO:0000076': {
    'name': 'three_prime_five_prime_overlap',
    'def': "An attribute to describe a gene when the 3' region overlaps with another gene's 5' region. [SO:ke]"
  },
  'SO:0000077': {
    'name': 'antisense',
    'def': 'A region sequence that is complementary to a sequence of messenger RNA. [SO:ke]'
  },
  'SO:0000078': {
    'name': 'polycistronic_transcript',
    'def': 'A transcript that is polycistronic. [SO:xp]'
  },
  'SO:0000079': {
    'name': 'dicistronic_transcript',
    'def': 'A transcript that is dicistronic. [SO:ke]'
  },
  'SO:0000080': {
    'name': 'operon_member',
    'def': 'A gene that is a member of an operon, which is a set of genes transcribed together as a unit. []'
  },
  'SO:0000081': {
    'name': 'gene_array_member'
  },
  'SO:0000082': {
    'name': 'processed_transcript_attribute'
  },
  'SO:0000083': {
    'name': 'macronuclear_sequence',
    'def': 'DNA belonging to the macronuclei of ciliates. []'
  },
  'SO:0000084': {
    'name': 'micronuclear_sequence',
    'def': 'DNA belonging to the micronuclei of a cell. []'
  },
  'SO:0000085': {
    'name': 'gene_by_genome_location'
  },
  'SO:0000086': {
    'name': 'gene_by_organelle_of_genome'
  },
  'SO:0000087': {
    'name': 'nuclear_gene',
    'def': 'A gene from nuclear sequence. [SO:xp]'
  },
  'SO:0000088': {
    'name': 'mt_gene',
    'def': 'A gene located in mitochondrial sequence. [SO:xp]'
  },
  'SO:0000089': {
    'name': 'kinetoplast_gene',
    'def': 'A gene located in kinetoplast sequence. [SO:xp]'
  },
  'SO:0000090': {
    'name': 'plastid_gene',
    'def': 'A gene from plastid sequence. [SO:xp]'
  },
  'SO:0000091': {
    'name': 'apicoplast_gene',
    'def': 'A gene from apicoplast sequence. [SO:xp]'
  },
  'SO:0000092': {
    'name': 'ct_gene',
    'def': 'A gene from chloroplast sequence. [SO:xp]'
  },
  'SO:0000093': {
    'name': 'chromoplast_gene',
    'def': 'A gene from chromoplast_sequence. [SO:xp]'
  },
  'SO:0000094': {
    'name': 'cyanelle_gene',
    'def': 'A gene from cyanelle sequence. [SO:xp]'
  },
  'SO:0000095': {
    'name': 'leucoplast_gene',
    'def': 'A plastid gene from leucoplast sequence. [SO:xp]'
  },
  'SO:0000096': {
    'name': 'proplastid_gene',
    'def': 'A gene from proplastid sequence. [SO:ke]'
  },
  'SO:0000097': {
    'name': 'nucleomorph_gene',
    'def': 'A gene from nucleomorph sequence. [SO:xp]'
  },
  'SO:0000098': {
    'name': 'plasmid_gene',
    'def': 'A gene from plasmid sequence. [SO:xp]'
  },
  'SO:0000099': {
    'name': 'proviral_gene',
    'def': 'A gene from proviral sequence. [SO:xp]'
  },
  'SO:0000100': {
    'name': 'endogenous_retroviral_gene',
    'def': 'A proviral gene with origin endogenous retrovirus. [SO:xp]'
  },
  'SO:0000101': {
    'name': 'transposable_element',
    'def': 'A transposon or insertion sequence. An element that can insert in a variety of DNA sequences. [http://www.sci.sdsu.edu/~smaloy/Glossary/T.html]'
  },
  'SO:0000102': {
    'name': 'expressed_sequence_match',
    'def': 'A match to an EST or cDNA sequence. [SO:ke]'
  },
  'SO:0000103': {
    'name': 'clone_insert_end',
    'def': 'The end of the clone insert. [SO:ke]'
  },
  'SO:0000104': {
    'name': 'polypeptide',
    'def': 'A sequence of amino acids linked by peptide bonds which may lack appreciable tertiary structure and may not be liable to irreversible denaturation. [SO:ma]'
  },
  'SO:0000105': {
    'name': 'chromosome_arm',
    'def': 'A region of the chromosome between the centromere and the telomere. Human chromosomes have two arms, the p arm (short) and the q arm (long) which are separated from each other by the centromere. [http://www.medterms.com/script/main/art.asp?articlekey=5152]'
  },
  'SO:0000106': {
    'name': 'non_capped_primary_transcript'
  },
  'SO:0000107': {
    'name': 'sequencing_primer',
    'def': 'A single stranded oligo used for polymerase chain reaction. []'
  },
  'SO:0000108': {
    'name': 'mRNA_with_frameshift',
    'def': 'An mRNA with a frameshift. [SO:xp]'
  },
  'SO:0000109': {
    'name': 'sequence_variant_obs',
    'def': 'A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration. [SO:ke]'
  },
  'SO:0000110': {
    'name': 'sequence_feature',
    'def': 'Any extent of continuous biological sequence. [LAMHDI:mb, SO:ke]'
  },
  'SO:0000111': {
    'name': 'transposable_element_gene',
    'def': 'A gene encoded within a transposable element. For example gag, int, env and pol are the transposable element genes of the TY element in yeast. [SO:ke]'
  },
  'SO:0000112': {
    'name': 'primer',
    'def': 'An oligo to which new deoxyribonucleotides can be added by DNA polymerase. [SO:ke]'
  },
  'SO:0000113': {
    'name': 'proviral_region',
    'def': 'A viral sequence which has integrated into a host genome. [SO:ke]'
  },
  'SO:0000114': {
    'name': 'methylated_cytosine',
    'def': 'A methylated deoxy-cytosine. [SO:ke]'
  },
  'SO:0000115': {
    'name': 'transcript_feature'
  },
  'SO:0000116': {
    'name': 'edited',
    'def': 'An attribute describing a sequence that is modified by editing. [SO:ke]'
  },
  'SO:0000117': {
    'name': 'transcript_with_readthrough_stop_codon'
  },
  'SO:0000118': {
    'name': 'transcript_with_translational_frameshift',
    'def': 'A transcript with a translational frameshift. [SO:xp]'
  },
  'SO:0000119': {
    'name': 'regulated',
    'def': 'An attribute to describe a sequence that is regulated. [SO:ke]'
  },
  'SO:0000120': {
    'name': 'protein_coding_primary_transcript',
    'def': 'A primary transcript that, at least in part, encodes one or more proteins. [SO:ke]'
  },
  'SO:0000121': {
    'name': 'forward_primer',
    'def': 'A single stranded oligo used for polymerase chain reaction. [http://mged.sourceforge.net/ontologies/MGEDontology.php]'
  },
  'SO:0000122': {
    'name': 'RNA_sequence_secondary_structure',
    'def': 'A folded RNA sequence. [SO:ke]'
  },
  'SO:0000123': {
    'name': 'transcriptionally_regulated',
    'def': 'An attribute describing a gene that is regulated at transcription. [SO:ma]'
  },
  'SO:0000124': {
    'name': 'transcriptionally_constitutive',
    'def': 'Expressed in relatively constant amounts without regard to cellular environmental conditions such as the concentration of a particular substrate. [SO:ke]'
  },
  'SO:0000125': {
    'name': 'transcriptionally_induced',
    'def': 'An inducer molecule is required for transcription to occur. [SO:ke]'
  },
  'SO:0000126': {
    'name': 'transcriptionally_repressed',
    'def': 'A repressor molecule is required for transcription to stop. [SO:ke]'
  },
  'SO:0000127': {
    'name': 'silenced_gene',
    'def': 'A gene that is silenced. [SO:xp]'
  },
  'SO:0000128': {
    'name': 'gene_silenced_by_DNA_modification',
    'def': 'A gene that is silenced by DNA modification. [SO:xp]'
  },
  'SO:0000129': {
    'name': 'gene_silenced_by_DNA_methylation',
    'def': 'A gene that is silenced by DNA methylation. [SO:xp]'
  },
  'SO:0000130': {
    'name': 'post_translationally_regulated',
    'def': 'An attribute describing a gene that is regulated after it has been translated. [SO:ke]'
  },
  'SO:0000131': {
    'name': 'translationally_regulated',
    'def': 'An attribute describing a gene that is regulated as it is translated. [SO:ke]'
  },
  'SO:0000132': {
    'name': 'reverse_primer',
    'def': 'A single stranded oligo used for polymerase chain reaction. [http://mged.sourceforge.net/ontologies/MGEDontology.php]'
  },
  'SO:0000133': {
    'name': 'epigenetically_modified',
    'def': 'This attribute describes a gene where heritable changes other than those in the DNA sequence occur. These changes include: modification to the DNA (such as DNA methylation, the covalent modification of cytosine), and post-translational modification of histones. [SO:ke]'
  },
  'SO:0000134': {
    'name': 'genomically_imprinted',
    'def': 'Imprinted genes are epigenetically modified genes that are expressed monoallelically according to their parent of origin. [SO:ke]'
  },
  'SO:0000135': {
    'name': 'maternally_imprinted',
    'def': 'The maternal copy of the gene is modified, rendering it transcriptionally silent. [SO:ke]'
  },
  'SO:0000136': {
    'name': 'paternally_imprinted',
    'def': 'The paternal copy of the gene is modified, rendering it transcriptionally silent. [SO:ke]'
  },
  'SO:0000137': {
    'name': 'allelically_excluded',
    'def': 'Allelic exclusion is a process occurring in diploid organisms, where a gene is inactivated and not expressed in that cell. [SO:ke]'
  },
  'SO:0000138': {
    'name': 'gene_rearranged_at_DNA_level',
    'def': 'An epigenetically modified gene, rearranged at the DNA level. [SO:xp]'
  },
  'SO:0000139': {
    'name': 'ribosome_entry_site',
    'def': 'Region in mRNA where ribosome assembles. [SO:ke]'
  },
  'SO:0000140': {
    'name': 'attenuator',
    'def': 'A sequence segment located within the five prime end of an mRNA that causes premature termination of translation. [SO:as]'
  },
  'SO:0000141': {
    'name': 'terminator',
    'def': 'The sequence of DNA located either at the end of the transcript that causes RNA polymerase to terminate transcription. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000142': {
    'name': 'DNA_sequence_secondary_structure',
    'def': 'A folded DNA sequence. [SO:ke]'
  },
  'SO:0000143': {
    'name': 'assembly_component',
    'def': 'A region of known length which may be used to manufacture a longer region. [SO:ke]'
  },
  'SO:0000144': {
    'name': 'primary_transcript_attribute'
  },
  'SO:0000145': {
    'name': 'recoded_codon',
    'def': 'A codon that has been redefined at translation. The redefinition may be as a result of translational bypass, translational frameshifting or stop codon readthrough. [SO:xp]'
  },
  'SO:0000146': {
    'name': 'capped',
    'def': "An attribute describing when a sequence, usually an mRNA is capped by the addition of a modified guanine nucleotide at the 5' end. [SO:ke]"
  },
  'SO:0000147': {
    'name': 'exon',
    'def': 'A region of the transcript sequence within a gene which is not removed from the primary RNA transcript by RNA splicing. [SO:ke]'
  },
  'SO:0000148': {
    'name': 'supercontig',
    'def': "One or more contigs that have been ordered and oriented using end-read information. Contains gaps that are filled with N's. [SO:ls]"
  },
  'SO:0000149': {
    'name': 'contig',
    'def': "A contiguous sequence derived from sequence assembly. Has no gaps, but may contain N's from unavailable bases. [SO:ls]"
  },
  'SO:0000150': {
    'name': 'read',
    'def': 'A sequence obtained from a single sequencing experiment. Typically a read is produced when a base calling program interprets information from a chromatogram trace file produced from a sequencing machine. [SO:rd]'
  },
  'SO:0000151': {
    'name': 'clone',
    'def': 'A piece of DNA that has been inserted in a vector so that it can be propagated in a host bacterium or some other organism. [SO:ke]'
  },
  'SO:0000152': {
    'name': 'YAC',
    'def': 'Yeast Artificial Chromosome, a vector constructed from the telomeric, centromeric, and replication origin sequences needed for replication in yeast cells. [SO:ma]'
  },
  'SO:0000153': {
    'name': 'BAC',
    'def': 'Bacterial Artificial Chromosome, a cloning vector that can be propagated as mini-chromosomes in a bacterial host. [SO:ma]'
  },
  'SO:0000154': {
    'name': 'PAC',
    'def': 'The P1-derived artificial chromosome are DNA constructs that are derived from the DNA of P1 bacteriophage. They can carry large amounts (about 100-300 kilobases) of other sequences for a variety of bioengineering purposes. It is one type of vector used to clone DNA fragments (100- to 300-kb insert size; average, 150 kb) in Escherichia coli cells. [http://en.wikipedia.org/wiki/P1-derived_artificial_chromosome]'
  },
  'SO:0000155': {
    'name': 'plasmid',
    'def': 'A self replicating, using the hosts cellular machinery, often circular nucleic acid molecule that is distinct from a chromosome in the organism. [SO:ma]'
  },
  'SO:0000156': {
    'name': 'cosmid',
    'def': 'A cloning vector that is a hybrid of lambda phages and a plasmid that can be propagated as a plasmid or packaged as a phage,since they retain the lambda cos sites. [SO:ma]'
  },
  'SO:0000157': {
    'name': 'phagemid',
    'def': 'A plasmid which carries within its sequence a bacteriophage replication origin. When the host bacterium is infected with "helper" phage, a phagemid is replicated along with the phage DNA and packaged into phage capsids. [SO:ma]'
  },
  'SO:0000158': {
    'name': 'fosmid',
    'def': 'A cloning vector that utilizes the E. coli F factor. [SO:ma]'
  },
  'SO:0000159': {
    'name': 'deletion',
    'def': 'The point at which one or more contiguous nucleotides were excised. [SO:ke]'
  },
  'SO:0000160': {
    'name': 'lambda_clone',
    'def': 'A linear clone derived from lambda bacteriophage. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome. [ISBN:0-1767-2380-8]'
  },
  'SO:0000161': {
    'name': 'methylated_adenine',
    'def': 'A modified base in which adenine has been methylated. [SO:ke]'
  },
  'SO:0000162': {
    'name': 'splice_site',
    'def': 'Consensus region of primary transcript bordering junction of splicing. A region that overlaps exactly 2 base and adjacent_to splice_junction. [SO:cjm, SO:ke]'
  },
  'SO:0000163': {
    'name': 'five_prime_cis_splice_site',
    'def': "Intronic 2 bp region bordering the exon, at the 5' edge of the intron. A splice_site that is downstream_adjacent_to exon and starts intron. [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html, SO:cjm, SO:ke]"
  },
  'SO:0000164': {
    'name': 'three_prime_cis_splice_site',
    'def': "Intronic 2 bp region bordering the exon, at the 3' edge of the intron. A splice_site that is upstream_adjacent_to exon and finishes intron. [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html, SO:cjm, SO:ke]"
  },
  'SO:0000165': {
    'name': 'enhancer',
    'def': 'A cis-acting sequence that increases the utilization of (some) eukaryotic promoters, and can function in either orientation and in any location (upstream or downstream) relative to the promoter. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000166': {
    'name': 'enhancer_bound_by_factor',
    'def': 'An enhancer bound by a factor. [SO:xp]'
  },
  'SO:0000167': {
    'name': 'promoter',
    'def': 'A regulatory_region composed of the TSS(s) and binding sites for TF_complexes of the core transcription machinery. A region (DNA) to which RNA polymerase binds, to begin transcription. [SO:regcreative]'
  },
  'SO:0000168': {
    'name': 'restriction_enzyme_cut_site',
    'def': 'A specific nucleotide sequence of DNA at or near which a particular restriction enzyme cuts the DNA. [SO:ma]'
  },
  'SO:0000169': {
    'name': 'RNApol_I_promoter',
    'def': 'A DNA sequence in eukaryotic DNA to which RNA polymerase I binds, to begin transcription. [SO:ke]'
  },
  'SO:0000170': {
    'name': 'RNApol_II_promoter',
    'def': 'A DNA sequence in eukaryotic DNA to which RNA polymerase II binds, to begin transcription. [SO:ke]'
  },
  'SO:0000171': {
    'name': 'RNApol_III_promoter',
    'def': 'A DNA sequence in eukaryotic DNA to which RNA polymerase III binds, to begin transcription. [SO:ke]'
  },
  'SO:0000172': {
    'name': 'CAAT_signal',
    'def': 'Part of a conserved sequence located about 75-bp upstream of the start point of eukaryotic transcription units which may be involved in RNA polymerase binding; consensus=GG(C|T)CAATCT. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000173': {
    'name': 'GC_rich_promoter_region',
    'def': 'A conserved GC-rich region located upstream of the start point of eukaryotic transcription units which may occur in multiple copies or in either orientation; consensus=GGGCGG. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000174': {
    'name': 'TATA_box',
    'def': 'A conserved AT-rich septamer found about 25-bp before the start point of many eukaryotic RNA polymerase II transcript units; may be involved in positioning the enzyme for correct initiation; consensus=TATA(A|T)A(A|T). [http://www.insdc.org/files/feature_table.html, PMID:16858867]'
  },
  'SO:0000175': {
    'name': 'minus_10_signal',
    'def': 'A conserved region about 10-bp upstream of the start point of bacterial transcription units which may be involved in binding RNA polymerase; consensus=TAtAaT. This region is associated with sigma factor 70. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000176': {
    'name': 'minus_35_signal',
    'def': 'A conserved hexamer about 35-bp upstream of the start point of bacterial transcription units; consensus=TTGACa or TGTTGACA. This region is associated with sigma factor 70. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000177': {
    'name': 'cross_genome_match',
    'def': 'A nucleotide match against a sequence from another organism. [SO:ma]'
  },
  'SO:0000178': {
    'name': 'operon',
    'def': 'The DNA region of a group of adjacent genes whose transcription is coordinated onÂ one or several mutually overlapping transcription units transcribed in the same direction and sharing at least one gene. [SO:ma]'
  },
  'SO:0000179': {
    'name': 'clone_insert_start',
    'def': 'The start of the clone insert. [SO:ke]'
  },
  'SO:0000180': {
    'name': 'retrotransposon',
    'def': 'A transposable element that is incorporated into a chromosome by a mechanism that requires reverse transcriptase. [http://www.dddmag.com/Glossary.aspx#r]'
  },
  'SO:0000181': {
    'name': 'translated_nucleotide_match',
    'def': 'A match against a translated sequence. [SO:ke]'
  },
  'SO:0000182': {
    'name': 'DNA_transposon',
    'def': 'A transposon where the mechanism of transposition is via a DNA intermediate. [SO:ke]'
  },
  'SO:0000183': {
    'name': 'non_transcribed_region',
    'def': 'A region of the gene which is not transcribed. [SO:ke]'
  },
  'SO:0000184': {
    'name': 'U2_intron',
    'def': 'A major type of spliceosomal intron spliced by the U2 spliceosome, that includes U1, U2, U4/U6 and U5 snRNAs. [PMID:9428511]'
  },
  'SO:0000185': {
    'name': 'primary_transcript',
    'def': 'A transcript that in its initial state requires modification to be functional. [SO:ma]'
  },
  'SO:0000186': {
    'name': 'LTR_retrotransposon',
    'def': 'A retrotransposon flanked by long terminal repeat sequences. [SO:ke]'
  },
  'SO:0000187': {
    'name': 'repeat_family',
    'def': 'A group of characterized repeat sequences. [SO:ke]'
  },
  'SO:0000188': {
    'name': 'intron',
    'def': 'A region of a primary transcript that is transcribed, but removed from within the transcript by splicing together the sequences (exons) on either side of it. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000189': {
    'name': 'non_LTR_retrotransposon',
    'def': 'A retrotransposon without long terminal repeat sequences. [SO:ke]'
  },
  'SO:0000190': {
    'name': 'five_prime_intron',
    'def': 'An intron that is the most 5-prime in a given transcript. []'
  },
  'SO:0000191': {
    'name': 'interior_intron',
    'def': 'An intron that is not the most 3-prime or the most 5-prime in a given transcript. []'
  },
  'SO:0000192': {
    'name': 'three_prime_intron',
    'def': 'An intron that is the most 3-prime in a given transcript. []'
  },
  'SO:0000193': {
    'name': 'RFLP_fragment',
    'def': 'A DNA fragment used as a reagent to detect the polymorphic genomic loci by hybridizing against the genomic DNA digested with a given restriction enzyme. [GOC:pj]'
  },
  'SO:0000194': {
    'name': 'LINE_element',
    'def': "A dispersed repeat family with many copies, each from 1 to 6 kb long. New elements are generated by retroposition of a transcribed copy. Typically the LINE contains 2 ORF's one of which is reverse transcriptase, and 3'and 5' direct repeats. [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html]"
  },
  'SO:0000195': {
    'name': 'coding_exon',
    'def': "An exon whereby at least one base is part of a codon (here, 'codon' is inclusive of the stop_codon). [SO:ke]"
  },
  'SO:0000196': {
    'name': 'five_prime_coding_exon_coding_region',
    'def': 'The sequence of the five_prime_coding_exon that codes for protein. [SO:cjm]'
  },
  'SO:0000197': {
    'name': 'three_prime_coding_exon_coding_region',
    'def': 'The sequence of the three_prime_coding_exon that codes for protein. [SO:cjm]'
  },
  'SO:0000198': {
    'name': 'noncoding_exon',
    'def': 'An exon that does not contain any codons. [SO:ke]'
  },
  'SO:0000199': {
    'name': 'translocation',
    'def': 'A region of nucleotide sequence that has translocated to a new position. The observed adjacency of two previously separated regions. [NCBI:th, SO:ke]'
  },
  'SO:0000200': {
    'name': 'five_prime_coding_exon',
    'def': "The 5' most coding exon. [SO:ke]"
  },
  'SO:0000201': {
    'name': 'interior_exon',
    'def': "An exon that is bounded by 5' and 3' splice sites. [PMID:10373547]"
  },
  'SO:0000202': {
    'name': 'three_prime_coding_exon',
    'def': 'The coding exon that is most 3-prime on a given transcript. [SO:ma]'
  },
  'SO:0000203': {
    'name': 'UTR',
    'def': 'Messenger RNA sequences that are untranslated and lie five prime or three prime to sequences which are translated. [SO:ke]'
  },
  'SO:0000204': {
    'name': 'five_prime_UTR',
    'def': "A region at the 5' end of a mature transcript (preceding the initiation codon) that is not translated into a protein. [http://www.insdc.org/files/feature_table.html]"
  },
  'SO:0000205': {
    'name': 'three_prime_UTR',
    'def': "A region at the 3' end of a mature transcript (following the stop codon) that is not translated into a protein. [http://www.insdc.org/files/feature_table.html]"
  },
  'SO:0000206': {
    'name': 'SINE_element',
    'def': 'A repetitive element, a few hundred base pairs long, that is dispersed throughout the genome. A common human SINE is the Alu element. [SO:ke]'
  },
  'SO:0000207': {
    'name': 'simple_sequence_length_variation',
    'def': 'SSLP are a kind of sequence alteration where the number of repeated sequences in intergenic regions may differ. [SO:ke]'
  },
  'SO:0000208': {
    'name': 'terminal_inverted_repeat_element',
    'def': 'A DNA transposable element defined as having termini with perfect, or nearly perfect short inverted repeats, generally 10 - 40 nucleotides long. [http://www.genetics.org/cgi/reprint/156/4/1983.pdf]'
  },
  'SO:0000209': {
    'name': 'rRNA_primary_transcript',
    'def': 'A primary transcript encoding a ribosomal RNA. [SO:ke]'
  },
  'SO:0000210': {
    'name': 'tRNA_primary_transcript',
    'def': 'A primary transcript encoding a transfer RNA (SO:0000253). [SO:ke]'
  },
  'SO:0000211': {
    'name': 'alanine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding alanyl tRNA. [SO:ke]'
  },
  'SO:0000212': {
    'name': 'arginine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding arginyl tRNA (SO:0000255). [SO:ke]'
  },
  'SO:0000213': {
    'name': 'asparagine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding asparaginyl tRNA (SO:0000256). [SO:ke]'
  },
  'SO:0000214': {
    'name': 'aspartic_acid_tRNA_primary_transcript',
    'def': 'A primary transcript encoding aspartyl tRNA (SO:0000257). [SO:ke]'
  },
  'SO:0000215': {
    'name': 'cysteine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding cysteinyl tRNA (SO:0000258). [SO:ke]'
  },
  'SO:0000216': {
    'name': 'glutamic_acid_tRNA_primary_transcript',
    'def': 'A primary transcript encoding glutaminyl tRNA (SO:0000260). [SO:ke]'
  },
  'SO:0000217': {
    'name': 'glutamine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding glutamyl tRNA (SO:0000260). [SO:ke]'
  },
  'SO:0000218': {
    'name': 'glycine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding glycyl tRNA (SO:0000263). [SO:ke]'
  },
  'SO:0000219': {
    'name': 'histidine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding histidyl tRNA (SO:0000262). [SO:ke]'
  },
  'SO:0000220': {
    'name': 'isoleucine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding isoleucyl tRNA (SO:0000263). [SO:ke]'
  },
  'SO:0000221': {
    'name': 'leucine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding leucyl tRNA (SO:0000264). [SO:ke]'
  },
  'SO:0000222': {
    'name': 'lysine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding lysyl tRNA (SO:0000265). [SO:ke]'
  },
  'SO:0000223': {
    'name': 'methionine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding methionyl tRNA (SO:0000266). [SO:ke]'
  },
  'SO:0000224': {
    'name': 'phenylalanine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding phenylalanyl tRNA (SO:0000267). [SO:ke]'
  },
  'SO:0000225': {
    'name': 'proline_tRNA_primary_transcript',
    'def': 'A primary transcript encoding prolyl tRNA (SO:0000268). [SO:ke]'
  },
  'SO:0000226': {
    'name': 'serine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding seryl tRNA (SO:000269). [SO:ke]'
  },
  'SO:0000227': {
    'name': 'threonine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding threonyl tRNA (SO:000270). [SO:ke]'
  },
  'SO:0000228': {
    'name': 'tryptophan_tRNA_primary_transcript',
    'def': 'A primary transcript encoding tryptophanyl tRNA (SO:000271). [SO:ke]'
  },
  'SO:0000229': {
    'name': 'tyrosine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding tyrosyl tRNA (SO:000272). [SO:ke]'
  },
  'SO:0000230': {
    'name': 'valine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding valyl tRNA (SO:000273). [SO:ke]'
  },
  'SO:0000231': {
    'name': 'snRNA_primary_transcript',
    'def': 'A primary transcript encoding a small nuclear RNA (SO:0000274). [SO:ke]'
  },
  'SO:0000232': {
    'name': 'snoRNA_primary_transcript',
    'def': 'A primary transcript encoding one or more small nucleolar RNAs (SO:0000275). [SO:ke]'
  },
  'SO:0000233': {
    'name': 'mature_transcript',
    'def': "A transcript which has undergone the necessary modifications, if any, for its function. In eukaryotes this includes, for example, processing of introns, cleavage, base modification, and modifications to the 5' and/or the 3' ends, other than addition of bases. In bacteria functional mRNAs are usually not modified. [SO:ke]"
  },
  'SO:0000234': {
    'name': 'mRNA',
    'def': 'Messenger RNA is the intermediate molecule between DNA and protein. It includes UTR and coding sequences. It does not contain introns. [SO:ma]'
  },
  'SO:0000235': {
    'name': 'TF_binding_site',
    'def': 'A DNA site where a transcription factor binds. [SO:ke]'
  },
  'SO:0000236': {
    'name': 'ORF',
    'def': 'The in-frame interval between the stop codons of a reading frame which when read as sequential triplets, has the potential of encoding a sequential string of amino acids. TER(NNN)nTER. [SGD:rb, SO:ma]'
  },
  'SO:0000237': {
    'name': 'transcript_attribute',
    'def': 'An attribute describing a transcript. []'
  },
  'SO:0000238': {
    'name': 'foldback_element',
    'def': 'A transposable element with extensive secondary structure, characterized by large modular imperfect long inverted repeats. [http://www.genetics.org/cgi/reprint/156/4/1983.pdf]'
  },
  'SO:0000239': {
    'name': 'flanking_region',
    'def': 'The sequences extending on either side of a specific region. [SO:ke]'
  },
  'SO:0000240': {
    'name': 'chromosome_variation',
    'def': 'A deviation in chromosome structure or number. []'
  },
  'SO:0000241': {
    'name': 'internal_UTR',
    'def': "A UTR bordered by the terminal and initial codons of two CDSs in a polycistronic transcript. Every UTR is either 5', 3' or internal. [SO:cjm]"
  },
  'SO:0000242': {
    'name': 'untranslated_region_polycistronic_mRNA',
    'def': "The untranslated sequence separating the 'cistrons' of multicistronic mRNA. [SO:ke]"
  },
  'SO:0000243': {
    'name': 'internal_ribosome_entry_site',
    'def': 'Sequence element that recruits a ribosomal subunit to internal mRNA for translation initiation. [SO:ke]'
  },
  'SO:0000244': {
    'name': 'four_cutter_restriction_site'
  },
  'SO:0000245': {
    'name': 'mRNA_by_polyadenylation_status'
  },
  'SO:0000246': {
    'name': 'polyadenylated',
    'def': "A attribute describing the addition of a poly A tail to the 3' end of a mRNA molecule. [SO:ke]"
  },
  'SO:0000247': {
    'name': 'mRNA_not_polyadenylated'
  },
  'SO:0000248': {
    'name': 'sequence_length_alteration',
    'def': 'A kind of kind of sequence alteration where the copies of a region present varies across a population. [SO:ke]'
  },
  'SO:0000249': {
    'name': 'six_cutter_restriction_site'
  },
  'SO:0000250': {
    'name': 'modified_RNA_base_feature',
    'def': 'A post_transcriptionally modified base. [SO:ke]'
  },
  'SO:0000251': {
    'name': 'eight_cutter_restriction_site'
  },
  'SO:0000252': {
    'name': 'rRNA',
    'def': 'rRNA is an RNA component of a ribosome that can provide both structural scaffolding and catalytic activity. [http://www.insdc.org/files/feature_table.html, ISBN:0198506732]'
  },
  'SO:0000253': {
    'name': 'tRNA',
    'def': "Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3' end, to which the tRNA's corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and 'wobble' base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00005, ISBN:0198506732]"
  },
  'SO:0000254': {
    'name': 'alanyl_tRNA',
    'def': "A tRNA sequence that has an alanine anticodon, and a 3' alanine binding region. [SO:ke]"
  },
  'SO:0000255': {
    'name': 'rRNA_small_subunit_primary_transcript',
    'def': 'A primary transcript encoding a small ribosomal subunit RNA. [SO:ke]'
  },
  'SO:0000256': {
    'name': 'asparaginyl_tRNA',
    'def': "A tRNA sequence that has an asparagine anticodon, and a 3' asparagine binding region. [SO:ke]"
  },
  'SO:0000257': {
    'name': 'aspartyl_tRNA',
    'def': "A tRNA sequence that has an aspartic acid anticodon, and a 3' aspartic acid binding region. [SO:ke]"
  },
  'SO:0000258': {
    'name': 'cysteinyl_tRNA',
    'def': "A tRNA sequence that has a cysteine anticodon, and a 3' cysteine binding region. [SO:ke]"
  },
  'SO:0000259': {
    'name': 'glutaminyl_tRNA',
    'def': "A tRNA sequence that has a glutamine anticodon, and a 3' glutamine binding region. [SO:ke]"
  },
  'SO:0000260': {
    'name': 'glutamyl_tRNA',
    'def': "A tRNA sequence that has a glutamic acid anticodon, and a 3' glutamic acid binding region. [SO:ke]"
  },
  'SO:0000261': {
    'name': 'glycyl_tRNA',
    'def': "A tRNA sequence that has a glycine anticodon, and a 3' glycine binding region. [SO:ke]"
  },
  'SO:0000262': {
    'name': 'histidyl_tRNA',
    'def': "A tRNA sequence that has a histidine anticodon, and a 3' histidine binding region. [SO:ke]"
  },
  'SO:0000263': {
    'name': 'isoleucyl_tRNA',
    'def': "A tRNA sequence that has an isoleucine anticodon, and a 3' isoleucine binding region. [SO:ke]"
  },
  'SO:0000264': {
    'name': 'leucyl_tRNA',
    'def': "A tRNA sequence that has a leucine anticodon, and a 3' leucine binding region. [SO:ke]"
  },
  'SO:0000265': {
    'name': 'lysyl_tRNA',
    'def': "A tRNA sequence that has a lysine anticodon, and a 3' lysine binding region. [SO:ke]"
  },
  'SO:0000266': {
    'name': 'methionyl_tRNA',
    'def': "A tRNA sequence that has a methionine anticodon, and a 3' methionine binding region. [SO:ke]"
  },
  'SO:0000267': {
    'name': 'phenylalanyl_tRNA',
    'def': "A tRNA sequence that has a phenylalanine anticodon, and a 3' phenylalanine binding region. [SO:ke]"
  },
  'SO:0000268': {
    'name': 'prolyl_tRNA',
    'def': "A tRNA sequence that has a proline anticodon, and a 3' proline binding region. [SO:ke]"
  },
  'SO:0000269': {
    'name': 'seryl_tRNA',
    'def': "A tRNA sequence that has a serine anticodon, and a 3' serine binding region. [SO:ke]"
  },
  'SO:0000270': {
    'name': 'threonyl_tRNA',
    'def': "A tRNA sequence that has a threonine anticodon, and a 3' threonine binding region. [SO:ke]"
  },
  'SO:0000271': {
    'name': 'tryptophanyl_tRNA',
    'def': "A tRNA sequence that has a tryptophan anticodon, and a 3' tryptophan binding region. [SO:ke]"
  },
  'SO:0000272': {
    'name': 'tyrosyl_tRNA',
    'def': "A tRNA sequence that has a tyrosine anticodon, and a 3' tyrosine binding region. [SO:ke]"
  },
  'SO:0000273': {
    'name': 'valyl_tRNA',
    'def': "A tRNA sequence that has a valine anticodon, and a 3' valine binding region. [SO:ke]"
  },
  'SO:0000274': {
    'name': 'snRNA',
    'def': 'A small nuclear RNA molecule involved in pre-mRNA splicing and processing. [http://www.insdc.org/files/feature_table.html, PMID:11733745, WB:ems]'
  },
  'SO:0000275': {
    'name': 'snoRNA',
    'def': 'Small nucleolar RNAs (snoRNAs) are short non-coding RNAs enriched in the nucleolus as components of small nucleolar ribonucleoproteins. They guide ribose methylation and pseudouridylation of rRNAs and snRNAs, and a subgroup regulate excision of rRNAs from rRNA precursor transcripts. snoRNAs may also guide rRNA acetylation and tRNA methylation, and regulate mRNA abundance and alternative splicing. [GOC:kgc, PMID:31828325]'
  },
  'SO:0000276': {
    'name': 'miRNA',
    'def': 'Small, ~22-nt, RNA molecule that is the endogenous transcript of a miRNA gene (or the product of other non coding RNA genes). Micro RNAs are produced from precursor molecules (SO:0001244) that can form local hairpin structures, which ordinarily are processed (usually via the Dicer pathway) such that a single miRNA molecule accumulates from one arm of a hairpin precursor molecule. Micro RNAs may trigger the cleavage of their target molecules or act as translational repressors. [PMID:11081512, PMID:12592000]'
  },
  'SO:0000277': {
    'name': 'bound_by_factor',
    'def': 'An attribute describing a sequence that is bound by another molecule. [SO:ke]'
  },
  'SO:0000278': {
    'name': 'transcript_bound_by_nucleic_acid',
    'def': 'A transcript that is bound by a nucleic acid. [SO:xp]'
  },
  'SO:0000279': {
    'name': 'transcript_bound_by_protein',
    'def': 'A transcript that is bound by a protein. [SO:xp]'
  },
  'SO:0000280': {
    'name': 'engineered_gene',
    'def': 'A gene that is engineered. [SO:xp]'
  },
  'SO:0000281': {
    'name': 'engineered_foreign_gene',
    'def': 'A gene that is engineered and foreign. [SO:xp]'
  },
  'SO:0000282': {
    'name': 'mRNA_with_minus_1_frameshift',
    'def': 'An mRNA with a minus 1 frameshift. [SO:xp]'
  },
  'SO:0000283': {
    'name': 'engineered_foreign_transposable_element_gene',
    'def': 'A transposable_element that is engineered and foreign. [SO:xp]'
  },
  'SO:0000284': {
    'name': 'type_I_enzyme_restriction_site',
    'def': 'The recognition site is bipartite and interrupted. [http://www.promega.com]'
  },
  'SO:0000285': {
    'name': 'foreign_gene',
    'def': 'A gene that is foreign. [SO:xp]'
  },
  'SO:0000286': {
    'name': 'long_terminal_repeat',
    'def': 'A sequence directly repeated at both ends of a defined sequence, of the sort typically found in retroviruses. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000287': {
    'name': 'fusion_gene',
    'def': 'A gene that is a fusion. [SO:xp]'
  },
  'SO:0000288': {
    'name': 'engineered_fusion_gene',
    'def': 'A fusion gene that is engineered. [SO:xp]'
  },
  'SO:0000289': {
    'name': 'microsatellite',
    'def': 'A repeat_region containing repeat_units of 2 to 10 bp repeated in tandem. [http://www.informatics.jax.org/silver/glossary.shtml, NCBI:th]'
  },
  'SO:0000290': {
    'name': 'dinucleotide_repeat_microsatellite_feature',
    'def': 'A region of a repeating dinucleotide sequence (two bases). []'
  },
  'SO:0000291': {
    'name': 'trinucleotide_repeat_microsatellite_feature',
    'def': 'A region of a repeating trinucleotide sequence (three bases). []'
  },
  'SO:0000292': {
    'name': 'repetitive_element'
  },
  'SO:0000293': {
    'name': 'engineered_foreign_repetitive_element',
    'def': 'A repetitive element that is engineered and foreign. [SO:xp]'
  },
  'SO:0000294': {
    'name': 'inverted_repeat',
    'def': 'The sequence is complementarily repeated on the opposite strand. It is a palindrome, and it may, or may not be hyphenated. Examples: GCTGATCAGC, or GCTGA-----TCAGC. [SO:ke]'
  },
  'SO:0000295': {
    'name': 'U12_intron',
    'def': 'A type of spliceosomal intron spliced by the U12 spliceosome, that includes U11, U12, U4atac/U6atac and U5 snRNAs. [PMID:9428511]'
  },
  'SO:0000296': {
    'name': 'origin_of_replication',
    'def': 'A region of nucleic acid from which replication initiates; includes sequences that are recognized by replication proteins, the site from which the first separation of complementary strands occurs, and specific replication start sites. [http://www.insdc.org/files/feature_table.html, NCBI:cf]'
  },
  'SO:0000297': {
    'name': 'D_loop',
    'def': 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000298': {
    'name': 'recombination_feature',
    'def': 'A feature where there has been exchange of genetic material in the event of mitosis or meiosis []'
  },
  'SO:0000299': {
    'name': 'specific_recombination_site',
    'def': 'A location where recombination or occurs during mitosis or meiosis. []'
  },
  'SO:0000300': {
    'name': 'recombination_feature_of_rearranged_gene',
    'def': 'A location where a gene is rearranged due to recombination during mitosis or meiosis. []'
  },
  'SO:0000301': {
    'name': 'vertebrate_immune_system_gene_recombination_feature',
    'def': 'A feature where recombination has occurred for the purpose of generating a diversity in the immune system. []'
  },
  'SO:0000302': {
    'name': 'J_gene_recombination_feature',
    'def': "Recombination signal including J-heptamer, J-spacer and J-nonamer in 5' of J-region of a J-gene or J-sequence. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000303': {
    'name': 'clip',
    'def': 'Part of the primary transcript that is clipped off during processing. [SO:ke]'
  },
  'SO:0000304': {
    'name': 'type_II_enzyme_restriction_site',
    'def': 'The recognition site is either palindromic, partially palindromic or an interrupted palindrome. Cleavage occurs within the recognition site. [http://www.promega.com]'
  },
  'SO:0000305': {
    'name': 'modified_DNA_base',
    'def': 'A modified nucleotide, i.e. a nucleotide other than A, T, C. G. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000306': {
    'name': 'methylated_DNA_base_feature',
    'def': 'A nucleotide modified by methylation. [SO:ke]'
  },
  'SO:0000307': {
    'name': 'CpG_island',
    'def': "Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5' ends of genes. [SO:rd]"
  },
  'SO:0000308': {
    'name': 'sequence_feature_locating_method'
  },
  'SO:0000309': {
    'name': 'computed_feature'
  },
  'SO:0000310': {
    'name': 'predicted_ab_initio_computation'
  },
  'SO:0000311': {
    'name': 'computed_feature_by_similarity',
    'def': '. [SO:ma]'
  },
  'SO:0000312': {
    'name': 'experimentally_determined',
    'def': 'Attribute to describe a feature that has been experimentally verified. [SO:ke]'
  },
  'SO:0000313': {
    'name': 'stem_loop',
    'def': 'A double-helical region of nucleic acid formed by base-pairing between adjacent (inverted) complementary sequences. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000314': {
    'name': 'direct_repeat',
    'def': 'A repeat where the same sequence is repeated in the same direction. Example: GCTGA-followed by-GCTGA. [SO:ke]'
  },
  'SO:0000315': {
    'name': 'TSS',
    'def': 'The first base where RNA polymerase begins to synthesize the RNA transcript. [SO:ke]'
  },
  'SO:0000316': {
    'name': 'CDS',
    'def': 'A contiguous sequence which begins with, and includes, a start codon and ends with, and includes, a stop codon. [SO:ma]'
  },
  'SO:0000317': {
    'name': 'cDNA_clone',
    'def': 'Complementary DNA; A piece of DNA copied from an mRNA and spliced into a vector for propagation in a suitable host. [http://seqcore.brcf.med.umich.edu/doc/educ/dnapr/mbglossary/mbgloss.html]'
  },
  'SO:0000318': {
    'name': 'start_codon',
    'def': 'First codon to be translated by a ribosome. [SO:ke]'
  },
  'SO:0000319': {
    'name': 'stop_codon',
    'def': 'In mRNA, a set of three nucleotides that indicates the end of information for protein synthesis. [SO:ke]'
  },
  'SO:0000320': {
    'name': 'intronic_splice_enhancer',
    'def': 'Sequences within the intron that modulate splice site selection for some introns. [SO:ke]'
  },
  'SO:0000321': {
    'name': 'mRNA_with_plus_1_frameshift',
    'def': 'An mRNA with a plus 1 frameshift. [SO:ke]'
  },
  'SO:0000322': {
    'name': 'nuclease_hypersensitive_site',
    'def': 'A region of nucleotide sequence targeted by a nuclease enzyme that is found cleaved more than would be expected by chance. []'
  },
  'SO:0000323': {
    'name': 'coding_start',
    'def': 'The first base to be translated into protein. [SO:ke]'
  },
  'SO:0000324': {
    'name': 'tag',
    'def': 'A nucleotide sequence that may be used to identify a larger sequence. [SO:ke]'
  },
  'SO:0000325': {
    'name': 'rRNA_large_subunit_primary_transcript',
    'def': 'A primary transcript encoding a large ribosomal subunit RNA. [SO:ke]'
  },
  'SO:0000326': {
    'name': 'SAGE_tag',
    'def': 'A short diagnostic sequence tag, serial analysis of gene expression (SAGE), that allows the quantitative and simultaneous analysis of a large number of transcripts. [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=7570003&dopt=Abstract]'
  },
  'SO:0000327': {
    'name': 'coding_end',
    'def': 'The last base to be translated into protein. It does not include the stop codon. [SO:ke]'
  },
  'SO:0000328': {
    'name': 'microarray_oligo',
    'def': 'A DNA sequence used experimentally to detect the presence or absence of a complementary nucleic acid. []'
  },
  'SO:0000329': {
    'name': 'mRNA_with_plus_2_frameshift',
    'def': 'An mRNA with a plus 2 frameshift. [SO:xp]'
  },
  'SO:0000330': {
    'name': 'conserved_region',
    'def': 'Region of sequence similarity by descent from a common ancestor. [SO:ke]'
  },
  'SO:0000331': {
    'name': 'STS',
    'def': 'Short (typically a few hundred base pairs) DNA sequence that has a single occurrence in a genome and whose location and base sequence are known. [http://www.biospace.com]'
  },
  'SO:0000332': {
    'name': 'coding_conserved_region',
    'def': 'Coding region of sequence similarity by descent from a common ancestor. [SO:ke]'
  },
  'SO:0000333': {
    'name': 'exon_junction',
    'def': 'The boundary between two exons in a processed transcript. [SO:ke]'
  },
  'SO:0000334': {
    'name': 'nc_conserved_region',
    'def': 'Non-coding region of sequence similarity by descent from a common ancestor. [SO:ke]'
  },
  'SO:0000335': {
    'name': 'mRNA_with_minus_2_frameshift',
    'def': 'A mRNA with a minus 2 frameshift. [SO:ke]'
  },
  'SO:0000336': {
    'name': 'pseudogene',
    'def': 'A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their "normal" paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its "normal" paralog). [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html]'
  },
  'SO:0000337': {
    'name': 'RNAi_reagent',
    'def': 'A double stranded RNA duplex, at least 20bp long, used experimentally to inhibit gene function by RNA interference. [SO:rd]'
  },
  'SO:0000338': {
    'name': 'MITE',
    'def': 'A highly repetitive and short (100-500 base pair) transposable element with terminal inverted repeats (TIR) and target site duplication (TSD). MITEs do not encode proteins. [http://www.pnas.org/cgi/content/full/97/18/10083]'
  },
  'SO:0000339': {
    'name': 'recombination_hotspot',
    'def': 'A region in a genome which promotes recombination. [SO:rd]'
  },
  'SO:0000340': {
    'name': 'chromosome',
    'def': 'Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication. [SO:ma]'
  },
  'SO:0000341': {
    'name': 'chromosome_band',
    'def': 'A cytologically distinguishable feature of a chromosome, often made visible by staining, and usually alternating light and dark. [SO:ma]'
  },
  'SO:0000342': {
    'name': 'site_specific_recombination_target_region',
    'def': 'A region specifically recognised by a recombinase where recombination can occur during mitosis or meiosis. []'
  },
  'SO:0000343': {
    'name': 'match',
    'def': 'A region of sequence, aligned to another sequence with some statistical significance, using an algorithm such as BLAST or SIM4. [SO:ke]'
  },
  'SO:0000344': {
    'name': 'splice_enhancer',
    'def': 'Region of a transcript that regulates splicing. [SO:ke]'
  },
  'SO:0000345': {
    'name': 'EST',
    'def': 'A tag produced from a single sequencing read from a cDNA clone or PCR product; typically a few hundred base pairs long. [SO:ke]'
  },
  'SO:0000346': {
    'name': 'loxP_site',
    'def': 'Cre-Recombination target sequence. []'
  },
  'SO:0000347': {
    'name': 'nucleotide_match',
    'def': 'A match against a nucleotide sequence. [SO:ke]'
  },
  'SO:0000348': {
    'name': 'nucleic_acid',
    'def': 'An attribute describing a sequence consisting of nucleobases bound to repeating units. The forms found in nature are deoxyribonucleic acid (DNA), where the repeating units are 2-deoxy-D-ribose rings connected to a phosphate backbone, and ribonucleic acid (RNA), where the repeating units are D-ribose rings connected to a phosphate backbone. [CHEBI:33696, RSC:cb]'
  },
  'SO:0000349': {
    'name': 'protein_match',
    'def': 'A match against a protein sequence. [SO:ke]'
  },
  'SO:0000350': {
    'name': 'FRT_site',
    'def': 'An inversion site found on the Saccharomyces cerevisiae 2 micron plasmid. [SO:ma]'
  },
  'SO:0000351': {
    'name': 'synthetic_sequence',
    'def': 'An attribute to decide a sequence of nucleotides, nucleotide analogs, or amino acids that has been designed by an experimenter and which may, or may not, correspond with any natural sequence. [SO:ma]'
  },
  'SO:0000352': {
    'name': 'DNA',
    'def': 'An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a 2-deoxy-D-ribose ring connected to a phosphate backbone. [RSC:cb]'
  },
  'SO:0000353': {
    'name': 'sequence_assembly',
    'def': 'A sequence of nucleotides that has been algorithmically derived from an alignment of two or more different sequences. [SO:ma]'
  },
  'SO:0000354': {
    'name': 'group_1_intron_homing_endonuclease_target_region',
    'def': 'A region of intronic nucleotide sequence targeted by a nuclease enzyme. [SO:ke]'
  },
  'SO:0000355': {
    'name': 'haplotype_block',
    'def': 'A region of the genome which is co-inherited as the result of the lack of historic recombination within it. [SO:ma]'
  },
  'SO:0000356': {
    'name': 'RNA',
    'def': 'An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a D-ribose ring connected to a phosphate backbone. [RSC:cb]'
  },
  'SO:0000357': {
    'name': 'flanked',
    'def': 'An attribute describing a region that is bounded either side by a particular kind of region. [SO:ke]'
  },
  'SO:0000359': {
    'name': 'floxed',
    'def': 'An attribute describing sequence that is flanked by Lox-P sites. [SO:ke]'
  },
  'SO:0000360': {
    'name': 'codon',
    'def': 'A set of (usually) three nucleotide bases in a DNA or RNA sequence, which together code for a unique amino acid or the termination of translation and are contained within the CDS. [SO:ke]'
  },
  'SO:0000361': {
    'name': 'FRT_flanked',
    'def': 'An attribute to describe sequence that is flanked by the FLP recombinase recognition site, FRT. [SO:ke]'
  },
  'SO:0000362': {
    'name': 'invalidated_by_chimeric_cDNA',
    'def': 'A cDNA clone constructed from more than one mRNA. Usually an experimental artifact. [SO:ma]'
  },
  'SO:0000363': {
    'name': 'floxed_gene',
    'def': 'A transgene that is floxed. [SO:xp]'
  },
  'SO:0000364': {
    'name': 'transposable_element_flanking_region',
    'def': 'The region of sequence surrounding a transposable element. [SO:ke]'
  },
  'SO:0000365': {
    'name': 'integron',
    'def': 'A region encoding an integrase which acts at a site adjacent to it (attI_site) to insert DNA which must include but is not limited to an attC_site. [SO:as]'
  },
  'SO:0000366': {
    'name': 'insertion_site',
    'def': 'The junction where an insertion occurred. [SO:ke]'
  },
  'SO:0000367': {
    'name': 'attI_site',
    'def': 'A region within an integron, adjacent to an integrase, at which site specific recombination involving an attC_site takes place. [SO:as]'
  },
  'SO:0000368': {
    'name': 'transposable_element_insertion_site',
    'def': 'The junction in a genome where a transposable_element has inserted. [SO:ke]'
  },
  'SO:0000369': {
    'name': 'integrase_coding_region'
  },
  'SO:0000370': {
    'name': 'small_regulatory_ncRNA',
    'def': 'A non-coding RNA less than 200 nucleotides long, usually with a specific secondary structure, that acts to regulate gene expression. These include short ncRNAs such as piRNA, miRNA and siRNAs (among others). [PMID:28541282, PomBase:al, SO:ma]'
  },
  'SO:0000371': {
    'name': 'conjugative_transposon',
    'def': 'A transposon that encodes function required for conjugation. [http://www.sci.sdsu.edu/~smaloy/Glossary/C.html]'
  },
  'SO:0000372': {
    'name': 'enzymatic_RNA',
    'def': 'An RNA sequence that has catalytic activity with or without an associated ribonucleoprotein. [RSC:cb]'
  },
  'SO:0000373': {
    'name': 'recombinationally_inverted_gene',
    'def': 'A recombinationally rearranged gene by inversion. [SO:xp]'
  },
  'SO:0000374': {
    'name': 'ribozyme',
    'def': 'An RNA with catalytic activity. [SO:ma]'
  },
  'SO:0000375': {
    'name': 'cytosolic_5_8S_rRNA',
    'def': 'Cytosolic 5.8S rRNA is an RNA component of the large subunit of cytosolic ribosomes in eukaryotes. [https://rfam.xfam.org/family/RF00002]'
  },
  'SO:0000376': {
    'name': 'RNA_6S',
    'def': 'A small (184-nt in E. coli) RNA that forms a hairpin type structure. 6S RNA associates with RNA polymerase in a highly specific manner. 6S RNA represses expression from a sigma70-dependent promoter during stationary phase. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00013]'
  },
  'SO:0000377': {
    'name': 'CsrB_RsmB_RNA',
    'def': 'An enterobacterial RNA that binds the CsrA protein. The CsrB RNAs contain a conserved motif CAGGXXG that is found in up to 18 copies and has been suggested to bind CsrA. The Csr regulatory system has a strong negative regulatory effect on glycogen biosynthesis, glyconeogenesis and glycogen catabolism and a positive regulatory effect on glycolysis. In other bacteria such as Erwinia caratovara the RsmA protein has been shown to regulate the production of virulence determinants, such extracellular enzymes. RsmA binds to RsmB regulatory RNA which is also a member of this family. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00018]'
  },
  'SO:0000378': {
    'name': 'DsrA_RNA',
    'def': 'DsrA RNA regulates both transcription, by overcoming transcriptional silencing by the nucleoid-associated H-NS protein, and translation, by promoting efficient translation of the stress sigma factor, RpoS. These two activities of DsrA can be separated by mutation: the first of three stem-loops of the 85 nucleotide RNA is necessary for RpoS translation but not for anti-H-NS action, while the second stem-loop is essential for antisilencing and less critical for RpoS translation. The third stem-loop, which behaves as a transcription terminator, can be substituted by the trp transcription terminator without loss of either DsrA function. The sequence of the first stem-loop of DsrA is complementary with the upstream leader portion of RpoS messenger RNA, suggesting that pairing of DsrA with the RpoS message might be important for translational regulation. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00014]'
  },
  'SO:0000379': {
    'name': 'GcvB_RNA',
    'def': 'A small untranslated RNA involved in expression of the dipeptide and oligopeptide transport systems in Escherichia coli. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00022]'
  },
  'SO:0000380': {
    'name': 'hammerhead_ribozyme',
    'def': "A small catalytic RNA motif that catalyzes self-cleavage reaction. Its name comes from its secondary structure which resembles a carpenter's hammer. The hammerhead ribozyme is involved in the replication of some viroid and some satellite RNAs. [PMID:2436805]"
  },
  'SO:0000381': {
    'name': 'group_IIA_intron',
    'def': 'A group II intron that recognizes IBS1/EBS1 and IBS2/EBS2 for the 5-prime exon and gamma/gamma-prime for the 3-prime exon. [PMID:20463000]'
  },
  'SO:0000382': {
    'name': 'group_IIB_intron',
    'def': 'A group II intron that recognizes IBS1/EBS1 and IBS2/EBS2 for the 5-prime exon and IBS3/EBS3 for the 3-prime exon. [PMID:20463000]'
  },
  'SO:0000383': {
    'name': 'MicF_RNA',
    'def': 'A non-translated 93 nt antisense RNA that binds its target ompF mRNA and regulates ompF expression by inhibiting translation and inducing degradation of the message. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00033]'
  },
  'SO:0000384': {
    'name': 'OxyS_RNA',
    'def': 'A small untranslated RNA which is induced in response to oxidative stress in Escherichia coli. Acts as a global regulator to activate or repress the expression of as many as 40 genes, including the fhlA-encoded transcriptional activator and the rpoS-encoded sigma(s) subunit of RNA polymerase. OxyS is bound by the Hfq protein, that increases the OxyS RNA interaction with its target messages. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00035]'
  },
  'SO:0000385': {
    'name': 'RNase_MRP_RNA',
    'def': 'The RNA molecule essential for the catalytic activity of RNase MRP, an enzymatically active ribonucleoprotein with two distinct roles in eukaryotes. In mitochondria it plays a direct role in the initiation of mitochondrial DNA replication. In the nucleus it is involved in precursor rRNA processing, where it cleaves the internal transcribed spacer 1 between 18S and 5.8S rRNAs. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00030]'
  },
  'SO:0000386': {
    'name': 'RNase_P_RNA',
    'def': 'The RNA component of Ribonuclease P (RNase P), a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterized activity is the generation of mature 5 prime ends of tRNAs by cleaving the 5 prime leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzymes have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous. Helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00010]'
  },
  'SO:0000387': {
    'name': 'RprA_RNA',
    'def': 'Translational regulation of the stationary phase sigma factor RpoS is mediated by the formation of a double-stranded RNA stem-loop structure in the upstream region of the rpoS messenger RNA, occluding the translation initiation site. Clones carrying rprA (RpoS regulator RNA) increased the translation of RpoS. The rprA gene encodes a 106 nucleotide regulatory RNA. As with DsrA Rfam:RF00014, RprA is predicted to form three stem-loops. Thus, at least two small RNAs, DsrA and RprA, participate in the positive regulation of RpoS translation. Unlike DsrA, RprA does not have an extensive region of complementarity to the RpoS leader, leaving its mechanism of action unclear. RprA is non-essential. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00034]'
  },
  'SO:0000388': {
    'name': 'RRE_RNA',
    'def': 'The Rev response element (RRE) is encoded within the HIV-env gene. Rev is an essential regulatory protein of HIV that binds an internal loop of the RRE leading, encouraging further Rev-RRE binding. This RNP complex is critical for mRNA export and hence for expression of the HIV structural proteins. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00036]'
  },
  'SO:0000389': {
    'name': 'spot_42_RNA',
    'def': 'A 109-nucleotide RNA of E. coli that seems to have a regulatory role on the galactose operon. Changes in Spot 42 levels are implicated in affecting DNA polymerase I levels. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00021]'
  },
  'SO:0000390': {
    'name': 'telomerase_RNA',
    'def': 'The RNA component of telomerase, a reverse transcriptase that synthesizes telomeric DNA. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00025]'
  },
  'SO:0000391': {
    'name': 'U1_snRNA',
    'def': "U1 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Its 5' end forms complementary base pairs with the 5' splice junction, thus defining the 5' donor site of an intron. There are significant differences in sequence and secondary structure between metazoan and yeast U1 snRNAs, the latter being much longer (568 nucleotides as compared to 164 nucleotides in human). Nevertheless, secondary structure predictions suggest that all U1 snRNAs share a 'common core' consisting of helices I, II, the proximal region of III, and IV. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00003]"
  },
  'SO:0000392': {
    'name': 'U2_snRNA',
    'def': "U2 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Complementary binding between U2 snRNA (in an area lying towards the 5' end but 3' to hairpin I) and the branchpoint sequence (BPS) of the intron results in the bulging out of an unpaired adenine, on the BPS, which initiates a nucleophilic attack at the intronic 5' splice site, thus starting the first of two transesterification reactions that mediate splicing. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00004]"
  },
  'SO:0000393': {
    'name': 'U4_snRNA',
    'def': 'U4 small nuclear RNA (U4 snRNA) is a component of the major U2-dependent spliceosome. It forms a duplex with U6, and with each splicing round, it is displaced from U6 (and the spliceosome) in an ATP-dependent manner, allowing U6 to refold and create the active site for splicing catalysis. A recycling process involving protein Prp24 re-anneals U4 and U6. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00015]'
  },
  'SO:0000394': {
    'name': 'U4atac_snRNA',
    'def': 'An snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U6atac_snRNA (SO:0000397). [PMID:12409455]'
  },
  'SO:0000395': {
    'name': 'U5_snRNA',
    'def': "U5 RNA is a component of both types of known spliceosome. The precise function of this molecule is unknown, though it is known that the 5' loop is required for splice site selection and p220 binding, and that both the 3' stem-loop and the Sm site are important for Sm protein binding and cap methylation. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00020]"
  },
  'SO:0000396': {
    'name': 'U6_snRNA',
    'def': "U6 snRNA is a component of the spliceosome which is involved in splicing pre-mRNA. The putative secondary structure consensus base pairing is confined to a short 5' stem loop, but U6 snRNA is thought to form extensive base-pair interactions with U4 snRNA. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00015]"
  },
  'SO:0000397': {
    'name': 'U6atac_snRNA',
    'def': 'U6atac_snRNA is an snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U4atac_snRNA (SO:0000394). [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&db=pubmed&list_uids=12409455&dopt=Abstract]'
  },
  'SO:0000398': {
    'name': 'U11_snRNA',
    'def': "U11 snRNA plays a role in splicing of the minor U12-dependent class of eukaryotic nuclear introns, similar to U1 snRNA in the major class spliceosome it base pairs to the conserved 5' splice site sequence. [PMID:9622129]"
  },
  'SO:0000399': {
    'name': 'U12_snRNA',
    'def': 'The U12 small nuclear (snRNA), together with U4atac/U6atac, U5, and U11 snRNAs and associated proteins, forms a spliceosome that cleaves a divergent class of low-abundance pre-mRNA introns. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00007]'
  },
  'SO:0000400': {
    'name': 'sequence_attribute',
    'def': 'An attribute describes a quality of sequence. [SO:ke]'
  },
  'SO:0000401': {
    'name': 'gene_attribute',
    'def': 'An attribute describing a gene. []'
  },
  'SO:0000402': {
    'name': 'enhancer_attribute'
  },
  'SO:0000403': {
    'name': 'U14_snoRNA',
    'def': 'U14 small nucleolar RNA (U14 snoRNA) is required for early cleavages of eukaryotic precursor rRNAs. In yeasts, this molecule possess a stem-loop region (known as the Y-domain) which is essential for function. A similar structure, but with a different consensus sequence, is found in plants, but is absent in vertebrates. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00016, PMID:2551119]'
  },
  'SO:0000404': {
    'name': 'vault_RNA',
    'def': 'A family of RNAs are found as part of the enigmatic vault ribonucleoprotein complex. The complex consists of a major vault protein (MVP), two minor vault proteins (VPARP and TEP1), and several small untranslated RNA molecules. It has been suggested that the vault complex is involved in drug resistance. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00006]'
  },
  'SO:0000405': {
    'name': 'Y_RNA',
    'def': 'Y RNAs are components of the Ro ribonucleoprotein particle (Ro RNP), in association with Ro60 and La proteins. The Y RNAs and Ro60 and La proteins are well conserved, but the function of the Ro RNP is not known. In humans the RNA component can be one of four small RNAs: hY1, hY3, hY4 and hY5. These small RNAs are predicted to fold into a conserved secondary structure containing three stem structures. The largest of the four, hY1, contains an additional hairpin. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00019]'
  },
  'SO:0000406': {
    'name': 'twintron',
    'def': 'An intron within an intron. Twintrons are group II or III introns, into which another group II or III intron has been transposed. [PMID:1899376, PMID:7823908]'
  },
  'SO:0000407': {
    'name': 'cytosolic_18S_rRNA',
    'def': 'Cytosolic 18S rRNA is an RNA component of the small subunit of cytosolic ribosomes in eukaryotes. [SO:ke]'
  },
  'SO:0000408': {
    'name': 'site',
    'def': 'The interbase position where something (eg an aberration) occurred. [SO:ke]'
  },
  'SO:0000409': {
    'name': 'binding_site',
    'def': 'A biological_region of sequence that, in the molecule, interacts selectively and non-covalently with other molecules. A region on the surface of a molecule that may interact with another molecule. When applied to polypeptides: Amino acids involved in binding or interactions. It can also apply to an amino acid bond which is represented by the positions of the two flanking amino acids. [EBIBS:GAR, SO:ke]'
  },
  'SO:0000410': {
    'name': 'protein_binding_site',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with polypeptide molecules. [SO:ke]'
  },
  'SO:0000411': {
    'name': 'rescue_region',
    'def': 'A region that rescues. [SO:xp]'
  },
  'SO:0000412': {
    'name': 'restriction_fragment',
    'def': 'A region of polynucleotide sequence produced by digestion with a restriction endonuclease. [SO:ke]'
  },
  'SO:0000413': {
    'name': 'sequence_difference',
    'def': 'A region where the sequence differs from that of a specified sequence. [SO:ke]'
  },
  'SO:0000414': {
    'name': 'invalidated_by_genomic_contamination',
    'def': 'An attribute to describe a feature that is invalidated due to genomic contamination. [SO:ke]'
  },
  'SO:0000415': {
    'name': 'invalidated_by_genomic_polyA_primed_cDNA',
    'def': 'An attribute to describe a feature that is invalidated due to polyA priming. [SO:ke]'
  },
  'SO:0000416': {
    'name': 'invalidated_by_partial_processing',
    'def': 'An attribute to describe a feature that is invalidated due to partial processing. [SO:ke]'
  },
  'SO:0000417': {
    'name': 'polypeptide_domain',
    'def': 'A structurally or functionally defined protein region. In proteins with multiple domains, the combination of the domains determines the function of the protein. A region which has been shown to recur throughout evolution. [EBIBS:GAR]'
  },
  'SO:0000418': {
    'name': 'signal_peptide',
    'def': 'The signal_peptide is a short region of the peptide located at the N-terminus that directs the protein to be secreted or part of membrane components. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000419': {
    'name': 'mature_protein_region',
    'def': 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide. [EBIBS:GAR, http://www.insdc.org/files/feature_table.html, SO:cb]'
  },
  'SO:0000420': {
    'name': 'five_prime_terminal_inverted_repeat',
    'def': 'An inverted repeat (SO:0000294) occurring at the 5-prime termini of a DNA transposon. []'
  },
  'SO:0000421': {
    'name': 'three_prime_terminal_inverted_repeat',
    'def': 'An inverted repeat (SO:0000294) occurring at the 3-prime termini of a DNA transposon. []'
  },
  'SO:0000422': {
    'name': 'U5_LTR_region',
    'def': 'The U5 segment of the long terminal repeats. []'
  },
  'SO:0000423': {
    'name': 'R_LTR_region',
    'def': 'The R segment of the long terminal repeats. []'
  },
  'SO:0000424': {
    'name': 'U3_LTR_region',
    'def': 'The U3 segment of the long terminal repeats. []'
  },
  'SO:0000425': {
    'name': 'five_prime_LTR',
    'def': 'The long terminal repeat found at the five-prime end of the sequence to be inserted into the host genome. []'
  },
  'SO:0000426': {
    'name': 'three_prime_LTR',
    'def': 'The long terminal repeat found at the three-prime end of the sequence to be inserted into the host genome. []'
  },
  'SO:0000427': {
    'name': 'R_five_prime_LTR_region',
    'def': 'The R segment of the three-prime long terminal repeat. []'
  },
  'SO:0000428': {
    'name': 'U5_five_prime_LTR_region',
    'def': 'The U5 segment of the three-prime long terminal repeat. []'
  },
  'SO:0000429': {
    'name': 'U3_five_prime_LTR_region',
    'def': 'The U3 segment of the three-prime long terminal repeat. []'
  },
  'SO:0000430': {
    'name': 'R_three_prime_LTR_region',
    'def': 'The R segment of the three-prime long terminal repeat. []'
  },
  'SO:0000431': {
    'name': 'U3_three_prime_LTR_region',
    'def': 'The U3 segment of the three-prime long terminal repeat. []'
  },
  'SO:0000432': {
    'name': 'U5_three_prime_LTR_region',
    'def': 'The U5 segment of the three-prime long terminal repeat. []'
  },
  'SO:0000433': {
    'name': 'non_LTR_retrotransposon_polymeric_tract',
    'def': 'A polymeric tract, such as poly(dA), within a non_LTR_retrotransposon. [SO:ke]'
  },
  'SO:0000434': {
    'name': 'target_site_duplication',
    'def': 'A sequence of the target DNA that is duplicated when a transposable element or phage inserts; usually found at each end the insertion. [http://www.koko.gov.my/CocoaBioTech/Glossaryt.html]'
  },
  'SO:0000435': {
    'name': 'RR_tract',
    'def': 'A polypurine tract within an LTR_retrotransposon. [SO:ke]'
  },
  'SO:0000436': {
    'name': 'ARS',
    'def': 'A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host. [SO:ma]'
  },
  'SO:0000437': {
    'name': 'assortment_derived_duplication'
  },
  'SO:0000438': {
    'name': 'gene_not_polyadenylated'
  },
  'SO:0000439': {
    'name': 'inverted_ring_chromosome',
    'def': 'A ring chromosome is a chromosome whose arms have fused together to form a ring in an inverted fashion, often with the loss of the ends of the chromosome. []'
  },
  'SO:0000440': {
    'name': 'vector_replicon',
    'def': 'A replicon that has been modified to act as a vector for foreign sequence. [SO:ma]'
  },
  'SO:0000441': {
    'name': 'ss_oligo',
    'def': 'A single stranded oligonucleotide. [SO:ke]'
  },
  'SO:0000442': {
    'name': 'ds_oligo',
    'def': 'A double stranded oligonucleotide. [SO:ke]'
  },
  'SO:0000443': {
    'name': 'polymer_attribute',
    'def': 'An attribute to describe the kind of biological sequence. [SO:ke]'
  },
  'SO:0000444': {
    'name': 'three_prime_noncoding_exon',
    'def': "Non-coding exon in the 3' UTR. [SO:ke]"
  },
  'SO:0000445': {
    'name': 'five_prime_noncoding_exon',
    'def': "Non-coding exon in the 5' UTR. [SO:ke]"
  },
  'SO:0000446': {
    'name': 'UTR_intron',
    'def': 'Intron located in the untranslated region. [SO:ke]'
  },
  'SO:0000447': {
    'name': 'five_prime_UTR_intron',
    'def': "An intron located in the 5' UTR. [SO:ke]"
  },
  'SO:0000448': {
    'name': 'three_prime_UTR_intron',
    'def': "An intron located in the 3' UTR. [SO:ke]"
  },
  'SO:0000449': {
    'name': 'random_sequence',
    'def': 'A sequence of nucleotides or amino acids which, by design, has a "random" order of components, given a predetermined input frequency of these components. [SO:ma]'
  },
  'SO:0000450': {
    'name': 'interband',
    'def': 'A light region between two darkly staining bands in a polytene chromosome. [SO:ma]'
  },
  'SO:0000451': {
    'name': 'gene_with_polyadenylated_mRNA',
    'def': 'A gene that encodes a polyadenylated mRNA. [SO:xp]'
  },
  'SO:0000452': {
    'name': 'transgene_attribute'
  },
  'SO:0000453': {
    'name': 'chromosomal_transposition',
    'def': 'A chromosome structure variant whereby a region of a chromosome has been transferred to another position. Among interchromosomal rearrangements, the term transposition is reserved for that class in which the telomeres of the chromosomes involved are coupled (that is to say, form the two ends of a single DNA molecule) as in wild-type. [FB:reference_manual, SO:ke]'
  },
  'SO:0000454': {
    'name': 'rasiRNA',
    'def': 'A 17-28-nt, small interfering RNA derived from transcripts of repetitive elements. [http://www.developmentalcell.com/content/article/abstract?uid=PIIS1534580703002284, PMID:18032451]'
  },
  'SO:0000455': {
    'name': 'gene_with_mRNA_with_frameshift',
    'def': 'A gene that encodes an mRNA with a frameshift. [SO:xp]'
  },
  'SO:0000456': {
    'name': 'recombinationally_rearranged_gene',
    'def': 'A gene that is recombinationally rearranged. [SO:ke]'
  },
  'SO:0000457': {
    'name': 'interchromosomal_duplication',
    'def': 'A chromosome duplication involving an insertion from another chromosome. [SO:ke]'
  },
  'SO:0000458': {
    'name': 'D_gene_segment',
    'def': "Germline genomic DNA including D-region with 5' UTR and 3' UTR, also designated as D-segment. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000459': {
    'name': 'gene_with_trans_spliced_transcript',
    'def': 'A gene with a transcript that is trans-spliced. [SO:xp]'
  },
  'SO:0000460': {
    'name': 'vertebrate_immunoglobulin_T_cell_receptor_segment',
    'def': 'Germline genomic DNA with the sequence for a V, D, C, or J portion of an immunoglobulin/T-cell receptor. []'
  },
  'SO:0000461': {
    'name': 'inversion_derived_bipartite_deficiency',
    'def': 'A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at each end of the inversion. [FB:km]'
  },
  'SO:0000462': {
    'name': 'pseudogenic_region',
    'def': 'A non-functional descendant of a functional entity. [SO:cjm]'
  },
  'SO:0000463': {
    'name': 'encodes_alternately_spliced_transcripts',
    'def': 'A gene that encodes more than one transcript. [SO:ke]'
  },
  'SO:0000464': {
    'name': 'decayed_exon',
    'def': 'A non-functional descendant of an exon. [SO:ke]'
  },
  'SO:0000465': {
    'name': 'inversion_derived_deficiency_plus_duplication',
    'def': 'A chromosome deletion whereby a chromosome is generated by recombination between two inversions; there is a deficiency at one end of the inversion and a duplication at the other end of the inversion. [FB:km]'
  },
  'SO:0000466': {
    'name': 'V_gene_segment',
    'def': "Germline genomic DNA including L-part1, V-intron and V-exon, with the 5' UTR and 3' UTR. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000467': {
    'name': 'post_translationally_regulated_by_protein_stability',
    'def': 'An attribute describing a gene sequence where the resulting protein is regulated by the stability of the resulting protein. [SO:ke]'
  },
  'SO:0000468': {
    'name': 'golden_path_fragment',
    'def': 'One of the pieces of sequence that make up a golden path. [SO:rd]'
  },
  'SO:0000469': {
    'name': 'post_translationally_regulated_by_protein_modification',
    'def': 'An attribute describing a gene sequence where the resulting protein is modified to regulate it. [SO:ke]'
  },
  'SO:0000470': {
    'name': 'J_gene_segment',
    'def': "Germline genomic DNA of an immunoglobulin/T-cell receptor gene including J-region with 5' UTR (SO:0000204) and 3' UTR (SO:0000205), also designated as J-segment. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000471': {
    'name': 'autoregulated',
    'def': 'The gene product is involved in its own transcriptional regulation. [SO:ke]'
  },
  'SO:0000472': {
    'name': 'tiling_path',
    'def': 'A set of regions which overlap with minimal polymorphism to form a linear sequence. [SO:cjm]'
  },
  'SO:0000473': {
    'name': 'negatively_autoregulated',
    'def': 'The gene product is involved in its own transcriptional regulation where it decreases transcription. [SO:ke]'
  },
  'SO:0000474': {
    'name': 'tiling_path_fragment',
    'def': 'A piece of sequence that makes up a tiling_path (SO:0000472). [SO:ke]'
  },
  'SO:0000475': {
    'name': 'positively_autoregulated',
    'def': 'The gene product is involved in its own transcriptional regulation, where it increases transcription. [SO:ke]'
  },
  'SO:0000476': {
    'name': 'contig_read',
    'def': 'A DNA sequencer read which is part of a contig. [SO:ke]'
  },
  'SO:0000477': {
    'name': 'polycistronic_gene',
    'def': 'A gene that is polycistronic. [SO:ke]'
  },
  'SO:0000478': {
    'name': 'C_gene_segment',
    'def': "Genomic DNA of immunoglobulin/T-cell receptor gene including C-region (and introns if present) with 5' UTR (SO:0000204) and 3' UTR (SO:0000205). [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000479': {
    'name': 'trans_spliced_transcript',
    'def': 'A transcript that is trans-spliced. [SO:xp]'
  },
  'SO:0000480': {
    'name': 'tiling_path_clone',
    'def': 'A clone which is part of a tiling path. A tiling path is a set of sequencing substrates, typically clones, which have been selected in order to efficiently cover a region of the genome in preparation for sequencing and assembly. [SO:ke]'
  },
  'SO:0000481': {
    'name': 'terminal_inverted_repeat',
    'def': 'An inverted repeat (SO:0000294) occurring at the termini of a DNA transposon. [SO:ke]'
  },
  'SO:0000482': {
    'name': 'vertebrate_immunoglobulin_T_cell_receptor_gene_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration. []'
  },
  'SO:0000483': {
    'name': 'nc_primary_transcript',
    'def': 'A primary transcript that is never translated into a protein. [SO:ke]'
  },
  'SO:0000484': {
    'name': 'three_prime_coding_exon_noncoding_region',
    'def': "The sequence of the 3' exon that is not coding. [SO:ke]"
  },
  'SO:0000485': {
    'name': 'DJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene, and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000486': {
    'name': 'five_prime_coding_exon_noncoding_region',
    'def': "The sequence of the 5' exon preceding the start codon. [SO:ke]"
  },
  'SO:0000487': {
    'name': 'VDJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000488': {
    'name': 'VDJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000489': {
    'name': 'VJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000490': {
    'name': 'VJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000491': {
    'name': 'VJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000492': {
    'name': 'D_gene_recombination_feature',
    'def': "Recombination signal including D-heptamer, D-spacer and D-nonamer in 5' of D-region of a D-gene or D-sequence. []"
  },
  'SO:0000493': {
    'name': 'three_prime_D_heptamer',
    'def': "7 nucleotide recombination site like CACAGTG, part of a 3' D-recombination signal sequence of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000494': {
    'name': 'three_prime_D_nonamer',
    'def': "A 9 nucleotide recombination site (e.g. ACAAAAACC), part of a 3' D-recombination signal sequence of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000495': {
    'name': 'three_prime_D_spacer',
    'def': "A 12 or 23 nucleotide spacer between the 3'D-HEPTAMER and 3'D-NONAMER of a 3'D-RS. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000496': {
    'name': 'five_prime_D_heptamer',
    'def': "7 nucleotide recombination site (e.g. CACTGTG), part of a 5' D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000497': {
    'name': 'five_prime_D_nonamer',
    'def': '9 nucleotide recombination site (e.g. GGTTTTTGT), part of a five_prime_D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000498': {
    'name': 'five_prime_D_spacer',
    'def': "12 or 23 nucleotide spacer between the 5' D-heptamer (SO:0000496) and 5' D-nonamer (SO:0000497) of a 5' D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000499': {
    'name': 'virtual_sequence',
    'def': "A continuous piece of sequence similar to the 'virtual contig' concept of the Ensembl database. [SO:ke]"
  },
  'SO:0000500': {
    'name': 'Hoogsteen_base_pair',
    'def': 'A type of non-canonical base-pairing. This is less energetically favourable than watson crick base pairing. Hoogsteen GC base pairs only have two hydrogen bonds. [PMID:12177293]'
  },
  'SO:0000501': {
    'name': 'reverse_Hoogsteen_base_pair',
    'def': 'A type of non-canonical base-pairing. [SO:ke]'
  },
  'SO:0000502': {
    'name': 'transcribed_region',
    'def': 'A region of sequence that is transcribed. This region may cover the transcript of a gene, it may emcompas the sequence covered by all of the transcripts of a alternately spliced gene, or it may cover the region transcribed by a polycistronic transcript. A gene may have 1 or more transcribed regions and a transcribed_region may belong to one or more genes. [SO:ke]'
  },
  'SO:0000503': {
    'name': 'alternately_spliced_gene_encodeing_one_transcript'
  },
  'SO:0000504': {
    'name': 'D_DJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000505': {
    'name': 'D_DJ_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene and one DJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000506': {
    'name': 'D_DJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000507': {
    'name': 'pseudogenic_exon',
    'def': 'A non functional descendant of an exon, part of a pseudogene. [SO:ke]'
  },
  'SO:0000508': {
    'name': 'D_DJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000509': {
    'name': 'D_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000510': {
    'name': 'VD_gene_segment',
    'def': "Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including L-part1, V-intron and V-D-exon, with the 5' UTR (SO:0000204) and 3' UTR (SO:0000205). [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000511': {
    'name': 'J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000512': {
    'name': 'inversion_derived_deficiency_plus_aneuploid',
    'def': 'A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at one end and presumed to have a deficiency or duplication at the other end of the inversion. [FB:km]'
  },
  'SO:0000513': {
    'name': 'J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000514': {
    'name': 'J_nonamer',
    'def': '9 nucleotide recombination site (e.g. GGTTTTTGT), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000515': {
    'name': 'J_heptamer',
    'def': '7 nucleotide recombination site (e.g. CACAGTG), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000516': {
    'name': 'pseudogenic_transcript',
    'def': 'A non functional descendant of a transcript, part of a pseudogene. [SO:ke]'
  },
  'SO:0000517': {
    'name': 'J_spacer',
    'def': '12 or 23 nucleotide spacer between the J-nonamer and the J-heptamer of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000518': {
    'name': 'V_DJ_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one DJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000519': {
    'name': 'V_DJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000520': {
    'name': 'V_VDJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000521': {
    'name': 'V_VDJ_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VDJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000522': {
    'name': 'V_VDJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000523': {
    'name': 'V_VJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000524': {
    'name': 'V_VJ_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000525': {
    'name': 'V_VJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000526': {
    'name': 'V_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one V-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000527': {
    'name': 'V_D_DJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000528': {
    'name': 'V_D_DJ_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000529': {
    'name': 'V_D_DJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000530': {
    'name': 'V_D_DJ_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000531': {
    'name': 'V_D_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000532': {
    'name': 'V_D_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000533': {
    'name': 'V_heptamer',
    'def': '7 nucleotide recombination site (e.g. CACAGTG), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000534': {
    'name': 'V_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000535': {
    'name': 'V_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000536': {
    'name': 'V_nonamer',
    'def': '9 nucleotide recombination site (e.g. ACAAAAACC), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000537': {
    'name': 'V_spacer',
    'def': '12 or 23 nucleotide spacer between the V-heptamer and the V-nonamer of a V-gene recombination feature of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000538': {
    'name': 'V_gene_recombination_feature',
    'def': "Recombination signal including V-heptamer, V-spacer and V-nonamer in 3' of V-region of a V-gene or V-sequence of an immunoglobulin/T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000539': {
    'name': 'DJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000540': {
    'name': 'DJ_J_C_cluster',
    'def': 'Genomic DNA in rearranged configuration including at least one D-J-GENE, one J-GENE and one C-GENE. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000541': {
    'name': 'VDJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000542': {
    'name': 'V_DJ_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000543': {
    'name': 'alternately_spliced_gene_encoding_greater_than_one_transcript'
  },
  'SO:0000544': {
    'name': 'helitron',
    'def': "A rolling circle transposon. Autonomous helitrons encode a 5'-to-3' DNA helicase and nuclease/ligase similar to those encoded by known rolling-circle replicons. [http://www.pnas.org/cgi/content/full/100/11/6569]"
  },
  'SO:0000545': {
    'name': 'recoding_pseudoknot',
    'def': 'The pseudoknots involved in recoding are unique in that, as they play their role as a structure, they are immediately unfolded and their now linear sequence serves as a template for decoding. [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=33937]'
  },
  'SO:0000546': {
    'name': 'designed_sequence',
    'def': 'An oligonucleotide sequence that was designed by an experimenter that may or may not correspond with any natural sequence. []'
  },
  'SO:0000547': {
    'name': 'inversion_derived_bipartite_duplication',
    'def': 'A chromosome generated by recombination between two inversions; there is a duplication at each end of the inversion. [FB:km]'
  },
  'SO:0000548': {
    'name': 'gene_with_edited_transcript',
    'def': 'A gene that encodes a transcript that is edited. [SO:xp]'
  },
  'SO:0000549': {
    'name': 'inversion_derived_duplication_plus_aneuploid',
    'def': 'A chromosome generated by recombination between two inversions; has a duplication at one end and presumed to have a deficiency or duplication at the other end of the inversion. [FB:km]'
  },
  'SO:0000550': {
    'name': 'aneuploid_chromosome',
    'def': 'A chromosome structural variation whereby either a chromosome exists in addition to the normal chromosome complement or is lacking. [SO:ke]'
  },
  'SO:0000551': {
    'name': 'polyA_signal_sequence',
    'def': 'The recognition sequence necessary for endonuclease cleavage of an RNA transcript that is followed by polyadenylation; consensus=AATAAA. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000552': {
    'name': 'Shine_Dalgarno_sequence',
    'def': "A region in the 5' UTR that pairs with the 16S rRNA during formation of the preinitiation complex. [SO:jh]"
  },
  'SO:0000553': {
    'name': 'polyA_site',
    'def': 'The site on an RNA transcript to which will be added adenine residues by post-transcriptional polyadenylation. The boundary between the UTR and the polyA sequence. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000554': {
    'name': 'assortment_derived_deficiency_plus_duplication'
  },
  'SO:0000555': {
    'name': 'five_prime_clip',
    'def': "5' most region of a precursor transcript that is clipped off during processing. [http://www.insdc.org/files/feature_table.html]"
  },
  'SO:0000556': {
    'name': 'five_prime_D_recombination_signal_sequence',
    'def': "Recombination signal of an immunoglobulin/T-cell receptor gene, including the 5' D-nonamer (SO:0000497), 5' D-spacer (SO:0000498), and 5' D-heptamer (SO:0000396) in 5' of the D-region of a D-gene, or in 5' of the D-region of DJ-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000557': {
    'name': 'three_prime_clip',
    'def': "3'-most region of a precursor transcript that is clipped off during processing. [http://www.insdc.org/files/feature_table.html]"
  },
  'SO:0000558': {
    'name': 'C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene including more than one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000559': {
    'name': 'D_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one D-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000560': {
    'name': 'D_J_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene and one J-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000561': {
    'name': 'heptamer_of_recombination_feature_of_vertebrate_immune_system_gene',
    'def': 'Seven nucleotide recombination site (e.g. CACAGTG), part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000562': {
    'name': 'nonamer_of_recombination_feature_of_vertebrate_immune_system_gene',
    'def': 'Nine nucleotide recombination site, part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene. []'
  },
  'SO:0000563': {
    'name': 'vertebrate_immune_system_gene_recombination_spacer',
    'def': 'A 12 or 23 nucleotide spacer between two regions of an immunoglobulin/T-cell receptor gene that may be rearranged by recombinase. []'
  },
  'SO:0000564': {
    'name': 'V_DJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000565': {
    'name': 'V_VDJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000566': {
    'name': 'V_VJ_J_C_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene, one J-gene and one C-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]'
  },
  'SO:0000567': {
    'name': 'inversion_derived_aneuploid_chromosome',
    'def': 'A chromosome may be generated by recombination between two inversions; presumed to have a deficiency or duplication at each end of the inversion. [FB:km]'
  },
  'SO:0000568': {
    'name': 'bidirectional_promoter',
    'def': 'A promoter that can allow for transcription in both directions. [PMID:21601935, SO:ke]'
  },
  'SO:0000569': {
    'name': 'retrotransposed',
    'def': 'An attribute of a feature that occurred as the product of a reverse transcriptase mediated event. [SO:ke]'
  },
  'SO:0000570': {
    'name': 'three_prime_D_recombination_signal_sequence',
    'def': "Recombination signal of an immunoglobulin/T-cell receptor gene, including the 3' D-heptamer (SO:0000493), 3' D-spacer, and 3' D-nonamer (SO:0000494) in 3' of the D-region of a D-gene. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000571': {
    'name': 'miRNA_encoding',
    'def': 'A region that can be transcribed into a microRNA (miRNA). []'
  },
  'SO:0000572': {
    'name': 'DJ_gene_segment',
    'def': "Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including D-J-region with 5' UTR and 3' UTR, also designated as D-J-segment. [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000573': {
    'name': 'rRNA_encoding',
    'def': 'A region that can be transcribed into a ribosomal RNA (rRNA). []'
  },
  'SO:0000574': {
    'name': 'VDJ_gene_segment',
    'def': "Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-D-J-exon, with the 5'UTR (SO:0000204) and 3'UTR (SO:0000205). [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000575': {
    'name': 'scRNA_encoding',
    'def': 'A region that can be transcribed into a small cytoplasmic RNA (scRNA). []'
  },
  'SO:0000576': {
    'name': 'VJ_gene_segment',
    'def': "Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-J-exon, with the 5'UTR (SO:0000204) and 3'UTR (SO:0000205). [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]"
  },
  'SO:0000577': {
    'name': 'centromere',
    'def': 'A region of chromosome where the spindle fibers attach during mitosis and meiosis. [SO:ke]'
  },
  'SO:0000578': {
    'name': 'snoRNA_encoding',
    'def': 'A region that can be transcribed into a small nucleolar RNA (snoRNA). []'
  },
  'SO:0000579': {
    'name': 'edited_transcript_feature',
    'def': 'A locatable feature on a transcript that is edited. [SO:ma]'
  },
  'SO:0000580': {
    'name': 'methylation_guide_snoRNA_primary_transcript',
    'def': 'A primary transcript encoding a methylation guide small nucleolar RNA. [SO:ke]'
  },
  'SO:0000581': {
    'name': 'cap',
    'def': "A structure consisting of a 7-methylguanosine in 5'-5' triphosphate linkage with the first nucleotide of an mRNA. It is added post-transcriptionally, and is not encoded in the DNA. [http://seqcore.brcf.med.umich.edu/doc/educ/dnapr/mbglossary/mbgloss.html]"
  },
  'SO:0000582': {
    'name': 'rRNA_cleavage_snoRNA_primary_transcript',
    'def': 'A primary transcript encoding an rRNA cleavage snoRNA. [SO:ke]'
  },
  'SO:0000583': {
    'name': 'pre_edited_region',
    'def': 'The region of a transcript that will be edited. [http://dna.kdna.ucla.edu/rna/index.aspx]'
  },
  'SO:0000584': {
    'name': 'tmRNA',
    'def': 'A tmRNA liberates a mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNAs genes have been found encoded in two parts. TmRNAs have been identified in eubacteria and some chloroplasts but are absent from archeal and Eukaryote nuclear genomes. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00023]'
  },
  'SO:0000585': {
    'name': 'C_D_box_snoRNA_encoding',
    'def': 'snoRNA that is associated with guiding methylation of nucleotides. It contains two short conserved sequence motifs: C (RUGAUGA) near the 5-prime end and D (CUGA) near the 3-prime end. []'
  },
  'SO:0000586': {
    'name': 'tmRNA_primary_transcript',
    'def': 'A primary transcript encoding a tmRNA (SO:0000584). [SO:ke]'
  },
  'SO:0000587': {
    'name': 'group_I_intron',
    'def': 'Group I catalytic introns are large self-splicing ribozymes. They catalyze their own excision from mRNA, tRNA and rRNA precursors in a wide range of organisms. The core secondary structure consists of 9 paired regions (P1-P9). These fold to essentially two domains, the P4-P6 domain (formed from the stacking of P5, P4, P6 and P6a helices) and the P3-P9 domain (formed from the P8, P3, P7 and P9 helices). Group I catalytic introns often have long ORFs inserted in loop regions. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00028]'
  },
  'SO:0000588': {
    'name': 'autocatalytically_spliced_intron',
    'def': 'A self spliced intron. [SO:ke]'
  },
  'SO:0000589': {
    'name': 'SRP_RNA_primary_transcript',
    'def': 'A primary transcript encoding a signal recognition particle RNA. [SO:ke]'
  },
  'SO:0000590': {
    'name': 'SRP_RNA',
    'def': 'The signal recognition particle (SRP) is a universally conserved ribonucleoprotein. It is involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most eubacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Eukaryotic and archaeal 7S RNAs have very similar secondary structures, with eight helical elements. These fold into the Alu and S domains, separated by a long linker region. Eubacterial SRP is generally a simpler structure, with the M domain of Ffh bound to a region of the 4.5S RNA that corresponds to helix 8 of the eukaryotic and archaeal SRP S domain. Some Gram-positive bacteria (e.g. Bacillus subtilis), however, have a larger SRP RNA that also has an Alu domain. The Alu domain is thought to mediate the peptide chain elongation retardation function of the SRP. The universally conserved helix which interacts with the SRP54/Ffh M domain mediates signal sequence recognition. In eukaryotes and archaea, the SRP19-helix 6 complex is thought to be involved in SRP assembly and stabilizes helix 8 for SRP54 binding. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00017]'
  },
  'SO:0000591': {
    'name': 'pseudoknot',
    'def': 'A tertiary structure in RNA where nucleotides in a loop form base pairs with a region of RNA downstream of the loop. [RSC:cb]'
  },
  'SO:0000592': {
    'name': 'H_pseudoknot',
    'def': 'A pseudoknot which contains two stems and at least two loops. [http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=10334330&dopt=Abstract]'
  },
  'SO:0000593': {
    'name': 'C_D_box_snoRNA',
    'def': "Most box C/D snoRNAs also contain long (>10 nt) sequences complementary to rRNA. Boxes C and D, as well as boxes C' and D', are usually located in close proximity, and form a structure known as the box C/D motif. This motif is important for snoRNA stability, processing, nucleolar targeting and function. A small number of box C/D snoRNAs are involved in rRNA processing; most, however, are known or predicted to serve as guide RNAs in ribose methylation of rRNA. Targeting involves direct base pairing of the snoRNA at the rRNA site to be modified and selection of a rRNA nucleotide a fixed distance from box D or D'. [http://www.bio.umass.edu/biochem/rna-sequence/Yeast_snoRNA_Database/snoRNA_DataBase.html]"
  },
  'SO:0000594': {
    'name': 'H_ACA_box_snoRNA',
    'def': "Members of the box H/ACA family contain an ACA triplet, exactly 3 nt upstream from the 3' end and an H-box in a hinge region that links two structurally similar functional domains of the molecule. Both boxes are important for snoRNA biosynthesis and function. A few box H/ACA snoRNAs are involved in rRNA processing; most others are known or predicted to participate in selection of uridine nucleosides in rRNA to be converted to pseudouridines. Site selection is mediated by direct base pairing of the snoRNA with rRNA through one or both targeting domains. [http://www.bio.umass.edu/biochem/rna-sequence/Yeast_snoRNA_Database/snoRNA_DataBase.html]"
  },
  'SO:0000595': {
    'name': 'C_D_box_snoRNA_primary_transcript',
    'def': 'A primary transcript encoding a small nucleolar RNA of the box C/D family. [SO:ke]'
  },
  'SO:0000596': {
    'name': 'H_ACA_box_snoRNA_primary_transcript',
    'def': 'A primary transcript encoding a small nucleolar RNA of the box H/ACA family. [SO:ke]'
  },
  'SO:0000597': {
    'name': 'transcript_edited_by_U_insertion/deletion',
    'def': 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa. [http://www.rna.ucla.edu/index.html]'
  },
  'SO:0000598': {
    'name': 'edited_by_C_insertion_and_dinucleotide_insertion'
  },
  'SO:0000599': {
    'name': 'edited_by_C_to_U_substitution'
  },
  'SO:0000600': {
    'name': 'edited_by_A_to_I_substitution'
  },
  'SO:0000601': {
    'name': 'edited_by_G_addition'
  },
  'SO:0000602': {
    'name': 'guide_RNA',
    'def': "A short 3'-uridylated RNA that can form a duplex (except for its post-transcriptionally added oligo_U tail (SO:0000609)) with a stretch of mature edited mRNA. [http://www.rna.ucla.edu/index.html]"
  },
  'SO:0000603': {
    'name': 'group_II_intron',
    'def': "Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI). A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron to intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny. [http://www.sanger.ac.uk/Software/Rfam/browse/index.shtml]"
  },
  'SO:0000604': {
    'name': 'editing_block',
    'def': 'Edited mRNA sequence mediated by a single guide RNA (SO:0000602). [http://dna.kdna.ucla.edu/rna/index.aspx]'
  },
  'SO:0000605': {
    'name': 'intergenic_region',
    'def': 'A region containing or overlapping no genes that is bounded on either side by a gene, or bounded by a gene and the end of the chromosome. [SO:cjm]'
  },
  'SO:0000606': {
    'name': 'editing_domain',
    'def': 'Edited mRNA sequence mediated by two or more overlapping guide RNAs (SO:0000602). [http://dna.kdna.ucla.edu/rna/index.aspx]'
  },
  'SO:0000607': {
    'name': 'unedited_region',
    'def': 'The region of an edited transcript that will not be edited. [http://dna.kdna.ucla.edu/rna/index.aspx]'
  },
  'SO:0000608': {
    'name': 'H_ACA_box_snoRNA_encoding',
    'def': 'snoRNA that is associated with guiding polyuridylation. It contains two short conserved sequence motifs: H box (ANANNA) and ACA (ACA). []'
  },
  'SO:0000609': {
    'name': 'oligo_U_tail',
    'def': "The string of non-encoded U's at the 3' end of a guide RNA (SO:0000602). [http://www.rna.ucla.edu/]"
  },
  'SO:0000610': {
    'name': 'polyA_sequence',
    'def': "Sequence of about 100 nucleotides of A added to the 3' end of most eukaryotic mRNAs. [SO:ke]"
  },
  'SO:0000611': {
    'name': 'branch_site',
    'def': "A pyrimidine rich sequence near the 3' end of an intron to which the 5'end becomes covalently bound during nuclear splicing. The resulting structure resembles a lariat. [SO:ke]"
  },
  'SO:0000612': {
    'name': 'polypyrimidine_tract',
    'def': 'The polypyrimidine tract is one of the cis-acting sequence elements directing intron removal in pre-mRNA splicing. [http://nar.oupjournals.org/cgi/content/full/25/4/888]'
  },
  'SO:0000613': {
    'name': 'bacterial_RNApol_promoter',
    'def': 'A DNA sequence to which bacterial RNA polymerase binds, to begin transcription. [SO:ke]'
  },
  'SO:0000614': {
    'name': 'bacterial_terminator',
    'def': 'A terminator signal for bacterial transcription. [SO:ke]'
  },
  'SO:0000615': {
    'name': 'terminator_of_type_2_RNApol_III_promoter',
    'def': 'A terminator signal for RNA polymerase III transcription. [SO:ke]'
  },
  'SO:0000616': {
    'name': 'transcription_end_site',
    'def': 'The base where transcription ends. [SO:ke]'
  },
  'SO:0000617': {
    'name': 'RNApol_III_promoter_type_1',
    'def': 'This type of promoter recruits RNA pol III. This promoter is intragenic and includes an A box, an intermediate element, and a C box. This is well conserved in the 5s rRNA promoters across species. [PMID:12381659]'
  },
  'SO:0000618': {
    'name': 'RNApol_III_promoter_type_2',
    'def': 'This type of promoter recruits RNA pol III to transcribe genes mainly for t-RNA. This promoter is intragenic and includes an A box and a B box. [PMID:12381659]'
  },
  'SO:0000619': {
    'name': 'A_box',
    'def': 'A variably distant linear promoter region recognized by TFIIIC, with consensus sequence TGGCnnAGTGG. [SO:ke]'
  },
  'SO:0000620': {
    'name': 'B_box',
    'def': 'A variably distant linear promoter region recognized by TFIIIC, with consensus sequence AGGTTCCAnnCC. [SO:ke]'
  },
  'SO:0000621': {
    'name': 'RNApol_III_promoter_type_3',
    'def': 'This type of promoter recruits RNA pol III to transcribe predominantly noncoding RNAs. This promoter contains a proximal sequence element (PSE) and a TATA box upstream of the gene that it regulates. Transcription can also be activated by a distal sequence element (DSE), which is located further upstream. [PMID:12381659]'
  },
  'SO:0000622': {
    'name': 'C_box',
    'def': 'An RNA polymerase III type 1 promoter with consensus sequence CAnnCCn. [SO:ke]'
  },
  'SO:0000623': {
    'name': 'snRNA_encoding',
    'def': 'A region that can be transcribed into a small nuclear RNA (snRNA). []'
  },
  'SO:0000624': {
    'name': 'telomere',
    'def': 'A specific structure at the end of a linear chromosome, required for the integrity and maintenance of the end. [SO:ma]'
  },
  'SO:0000625': {
    'name': 'silencer',
    'def': 'A regulatory region which upon binding of transcription factors, suppress the transcription of the gene or genes they control. [SO:ke]'
  },
  'SO:0000626': {
    'name': 'chromosomal_regulatory_element',
    'def': 'Regions of the chromosome that are important for regulating binding of chromosomes to the nuclear matrix. []'
  },
  'SO:0000627': {
    'name': 'insulator',
    'def': "A regulatory region that 1) when located between a CRM and a gene's promoter prevents the CRM from modulating that genes expression and 2) acts as a chromatin boundary element or barrier that can block the encroachment of condensed chromatin from an adjacent region. [NCBI:cf, PMID:12154228, SO:regcreative]"
  },
  'SO:0000628': {
    'name': 'chromosomal_structural_element',
    'def': 'Regions of the chromosome that are important for structural elements. []'
  },
  'SO:0000629': {
    'name': 'five_prime_open_reading_frame',
    'def': "An open reading frame found within the 5' UTR that can be translated and stall the translation of the downstream open reading frame. [PMID:12890013]"
  },
  'SO:0000630': {
    'name': 'upstream_AUG_codon',
    'def': 'A start codon upstream of the ORF. [SO:ke]'
  },
  'SO:0000631': {
    'name': 'polycistronic_primary_transcript',
    'def': 'A primary transcript encoding for more than one gene product. [SO:ke]'
  },
  'SO:0000632': {
    'name': 'monocistronic_primary_transcript',
    'def': 'A primary transcript encoding for one gene product. [SO:ke]'
  },
  'SO:0000633': {
    'name': 'monocistronic_mRNA',
    'def': 'An mRNA with either a single protein product, or for which the regions encoding all its protein products overlap. [SO:rd]'
  },
  'SO:0000634': {
    'name': 'polycistronic_mRNA',
    'def': 'An mRNA that encodes multiple proteins from at least two non-overlapping regions. [SO:rd]'
  },
  'SO:0000635': {
    'name': 'mini_exon_donor_RNA',
    'def': 'A primary transcript that donates the spliced leader to other mRNA. [SO:ke]'
  },
  'SO:0000636': {
    'name': 'spliced_leader_RNA',
    'def': "Snall nuclear RNAs that are incorporated into the pre-mRNAs to replace the 5' end in some eukaryotes. [PMID:24130571]"
  },
  'SO:0000637': {
    'name': 'engineered_plasmid',
    'def': 'A plasmid that is engineered. [SO:xp]'
  },
  'SO:0000638': {
    'name': 'transcribed_spacer_region',
    'def': 'Part of an rRNA transcription unit that is transcribed but discarded during maturation, not giving rise to any part of rRNA. [http://oregonstate.edu/instruction/bb492/general/glossary.html]'
  },
  'SO:0000639': {
    'name': 'internal_transcribed_spacer_region',
    'def': 'Non-coding regions of DNA sequence that separate genes coding for the 28S, 5.8S, and 18S ribosomal RNAs. [SO:ke]'
  },
  'SO:0000640': {
    'name': 'external_transcribed_spacer_region',
    'def': 'Non-coding regions of DNA that precede the sequence that codes for the ribosomal RNA. [SO:ke]'
  },
  'SO:0000641': {
    'name': 'tetranucleotide_repeat_microsatellite_feature',
    'def': 'A region of a repeating tetranucleotide sequence (four bases). []'
  },
  'SO:0000642': {
    'name': 'SRP_RNA_encoding',
    'def': 'A region that can be transcribed into a signal recognition particle RNA (SRP RNA). []'
  },
  'SO:0000643': {
    'name': 'minisatellite',
    'def': 'A repeat region containing tandemly repeated sequences having a unit length of 10 to 40 bp. [http://www.informatics.jax.org/silver/glossary.shtml]'
  },
  'SO:0000644': {
    'name': 'antisense_RNA',
    'def': 'Antisense RNA is RNA that is transcribed from the coding, rather than the template, strand of DNA. It is therefore complementary to mRNA. [SO:ke]'
  },
  'SO:0000645': {
    'name': 'antisense_primary_transcript',
    'def': 'The reverse complement of the primary transcript. [SO:ke]'
  },
  'SO:0000646': {
    'name': 'siRNA',
    'def': 'A small RNA molecule that is the product of a longer exogenous or endogenous dsRNA, which is either a bimolecular duplex or very long hairpin, processed (via the Dicer pathway) such that numerous siRNAs accumulate from both strands of the dsRNA. siRNAs trigger the cleavage of their target molecules. [PMID:12592000]'
  },
  'SO:0000647': {
    'name': 'miRNA_primary_transcript',
    'def': 'A primary transcript encoding a micro RNA. [SO:ke]'
  },
  'SO:0000650': {
    'name': 'cytosolic_SSU_rRNA',
    'def': 'Cytosolic SSU rRNA is an RNA component of the small subunit of cytosolic ribosomes. [SO:ke]'
  },
  'SO:0000651': {
    'name': 'cytosolic_LSU_rRNA',
    'def': 'Cytosolic LSU rRNA is an RNA component of the large subunit of cytosolic ribosomes. [SO:ke]'
  },
  'SO:0000652': {
    'name': 'cytosolic_5S_rRNA',
    'def': 'Cytosolic 5S rRNA is an RNA component of the large subunit of cytosolic ribosomes in both prokaryotes and eukaryotes. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00001]'
  },
  'SO:0000653': {
    'name': 'cytosolic_28S_rRNA',
    'def': 'Cytosolic 28S rRNA is an RNA component of the large subunit of cytosolic ribosomes in metazoan eukaryotes. [SO:ke]'
  },
  'SO:0000654': {
    'name': 'maxicircle_gene',
    'def': 'A mitochondrial gene located in a maxicircle. [SO:xp]'
  },
  'SO:0000655': {
    'name': 'ncRNA',
    'def': 'An RNA transcript that does not encode for a protein rather the RNA molecule is the gene product. [SO:ke]'
  },
  'SO:0000656': {
    'name': 'stRNA_encoding',
    'def': 'A region that can be transcribed into a small temporal RNA (stRNA). Found in roundworm development. []'
  },
  'SO:0000657': {
    'name': 'repeat_region',
    'def': 'A region of sequence containing one or more repeat units. [SO:ke]'
  },
  'SO:0000658': {
    'name': 'dispersed_repeat',
    'def': 'A repeat that is located at dispersed sites in the genome. [SO:ke]'
  },
  'SO:0000659': {
    'name': 'tmRNA_encoding',
    'def': 'A region that can be transcribed into a transfer-messenger RNA (tmRNA). []'
  },
  'SO:0000660': {
    'name': 'DNA_invertase_target_sequence'
  },
  'SO:0000661': {
    'name': 'intron_attribute'
  },
  'SO:0000662': {
    'name': 'spliceosomal_intron',
    'def': 'An intron which is spliced by the spliceosome. [SO:ke]'
  },
  'SO:0000663': {
    'name': 'tRNA_encoding',
    'def': 'A region that can be transcribed into a transfer RNA (tRNA). []'
  },
  'SO:0000664': {
    'name': 'introgressed_chromosome_region',
    'def': 'A region of a chromosome that has been introduced by backcrossing with a separate species. [PMID:11454782]'
  },
  'SO:0000665': {
    'name': 'monocistronic_transcript',
    'def': 'A transcript that is monocistronic. [SO:xp]'
  },
  'SO:0000666': {
    'name': 'mobile_intron',
    'def': 'An intron (mitochondrial, chloroplast, nuclear or prokaryotic) that encodes a double strand sequence specific endonuclease allowing for mobility. [SO:ke]'
  },
  'SO:0000667': {
    'name': 'insertion',
    'def': 'The sequence of one or more nucleotides added between two adjacent nucleotides in the sequence. [SO:ke]'
  },
  'SO:0000668': {
    'name': 'EST_match',
    'def': 'A match against an EST sequence. [SO:ke]'
  },
  'SO:0000669': {
    'name': 'sequence_rearrangement_feature',
    'def': 'A feature where a segment of DNA has been rearranged from what it was in the parent cell. []'
  },
  'SO:0000670': {
    'name': 'chromosome_breakage_sequence',
    'def': 'A sequence within the micronuclear DNA of ciliates at which chromosome breakage and telomere addition occurs during nuclear differentiation. [SO:ma]'
  },
  'SO:0000671': {
    'name': 'internal_eliminated_sequence',
    'def': 'A sequence eliminated from the genome of ciliates during nuclear differentiation. [SO:ma]'
  },
  'SO:0000672': {
    'name': 'macronucleus_destined_segment',
    'def': 'A sequence that is conserved, although rearranged relative to the micronucleus, in the macronucleus of a ciliate genome. [SO:ma]'
  },
  'SO:0000673': {
    'name': 'transcript',
    'def': 'An RNA synthesized on a DNA or RNA template by an RNA polymerase. [SO:ma]'
  },
  'SO:0000674': {
    'name': 'non_canonical_splice_site',
    'def': 'A splice site where the donor and acceptor sites differ from the canonical form. [SO:ke]'
  },
  'SO:0000675': {
    'name': 'canonical_splice_site',
    'def': 'The major class of splice site with dinucleotides GT and AG for donor and acceptor sites, respectively. [SO:ke]'
  },
  'SO:0000676': {
    'name': 'canonical_three_prime_splice_site',
    'def': "The canonical 3' splice site has the sequence \"AG\". [SO:ke]"
  },
  'SO:0000677': {
    'name': 'canonical_five_prime_splice_site',
    'def': "The canonical 5' splice site has the sequence \"GT\". [SO:ke]"
  },
  'SO:0000678': {
    'name': 'non_canonical_three_prime_splice_site',
    'def': "A 3' splice site that does not have the sequence \"AG\". [SO:ke]"
  },
  'SO:0000679': {
    'name': 'non_canonical_five_prime_splice_site',
    'def': "A 5' splice site which does not have the sequence \"GT\". [SO:ke]"
  },
  'SO:0000680': {
    'name': 'non_canonical_start_codon',
    'def': 'A start codon that is not the usual AUG sequence. [SO:ke]'
  },
  'SO:0000681': {
    'name': 'aberrant_processed_transcript',
    'def': 'A transcript that has been processed "incorrectly", for example by the failure of splicing of one or more exons. [SO:ke]'
  },
  'SO:0000682': {
    'name': 'splicing_feature'
  },
  'SO:0000683': {
    'name': 'exonic_splice_enhancer',
    'def': 'Exonic splicing enhancers (ESEs) facilitate exon definition by assisting in the recruitment of splicing factors to the adjacent intron. [http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12403462&dopt=Abstract]'
  },
  'SO:0000684': {
    'name': 'nuclease_sensitive_site',
    'def': 'A region of nucleotide sequence targeted by a nuclease enzyme. [SO:ma]'
  },
  'SO:0000685': {
    'name': 'DNaseI_hypersensitive_site',
    'def': 'DNA region representing open chromatin structure that is hypersensitive to digestion by DNase I. []'
  },
  'SO:0000686': {
    'name': 'translocation_element',
    'def': 'A chromosomal translocation whereby the chromosomes carrying non-homologous centromeres may be recovered independently. These chromosomes are described as translocation elements. This occurs for some translocations, particularly but not exclusively, reciprocal translocations. [SO:ma]'
  },
  'SO:0000687': {
    'name': 'deletion_junction',
    'def': 'The space between two bases in a sequence which marks the position where a deletion has occurred. [SO:ke]'
  },
  'SO:0000688': {
    'name': 'golden_path',
    'def': 'A set of subregions selected from sequence contigs which when concatenated form a nonredundant linear sequence. [SO:ls]'
  },
  'SO:0000689': {
    'name': 'cDNA_match',
    'def': 'A match against cDNA sequence. [SO:ke]'
  },
  'SO:0000690': {
    'name': 'gene_with_polycistronic_transcript',
    'def': 'A gene that encodes a polycistronic transcript. [SO:xp]'
  },
  'SO:0000691': {
    'name': 'cleaved_initiator_methionine',
    'def': 'The initiator methionine that has been cleaved from a mature polypeptide sequence. [EBIBS:GAR]'
  },
  'SO:0000692': {
    'name': 'gene_with_dicistronic_transcript',
    'def': 'A gene that encodes a dicistronic transcript. [SO:xp]'
  },
  'SO:0000693': {
    'name': 'gene_with_recoded_mRNA',
    'def': 'A gene that encodes an mRNA that is recoded. [SO:xp]'
  },
  'SO:0000694': {
    'name': 'SNP',
    'def': 'SNPs are single base pair positions in genomic DNA at which different sequence alternatives exist in normal individuals in some population(s), wherein the least frequent variant has an abundance of 1% or greater. [SO:cb]'
  },
  'SO:0000695': {
    'name': 'reagent',
    'def': 'A sequence used in experiment. [SO:ke]'
  },
  'SO:0000696': {
    'name': 'oligo',
    'def': "A short oligonucleotide sequence, of length on the order of 10's of bases; either single or double stranded. [SO:ma]"
  },
  'SO:0000697': {
    'name': 'gene_with_stop_codon_read_through',
    'def': 'A gene that encodes a transcript with stop codon readthrough. [SO:xp]'
  },
  'SO:0000698': {
    'name': 'gene_with_stop_codon_redefined_as_pyrrolysine',
    'def': 'A gene encoding an mRNA that has the stop codon redefined as pyrrolysine. [SO:xp]'
  },
  'SO:0000699': {
    'name': 'junction',
    'def': 'A sequence_feature with an extent of zero. [SO:ke]'
  },
  'SO:0000700': {
    'name': 'remark',
    'def': 'A comment about the sequence. [SO:ke]'
  },
  'SO:0000701': {
    'name': 'possible_base_call_error',
    'def': 'A region of sequence where the validity of the base calling is questionable. [SO:ke]'
  },
  'SO:0000702': {
    'name': 'possible_assembly_error',
    'def': 'A region of sequence where there may have been an error in the assembly. [SO:ke]'
  },
  'SO:0000703': {
    'name': 'experimental_result_region',
    'def': 'A region of sequence implicated in an experimental result. [SO:ke]'
  },
  'SO:0000704': {
    'name': 'gene',
    'def': 'A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene may include regulatory regions, transcribed regions and/or other functional sequence regions. [SO:immuno_workshop]'
  },
  'SO:0000705': {
    'name': 'tandem_repeat',
    'def': 'Two or more adjacent copies of a region (of length greater than 1). [SO:ke]'
  },
  'SO:0000706': {
    'name': 'trans_splice_acceptor_site',
    'def': "The 3' splice site of the acceptor primary transcript. [SO:ke]"
  },
  'SO:0000707': {
    'name': 'trans_splice_donor_site',
    'def': "The 5' five prime splice site region of the donor RNA. [SO:ke]"
  },
  'SO:0000708': {
    'name': 'SL1_acceptor_site',
    'def': "A trans_splicing_acceptor_site which appends the 22nt SL1 RNA leader sequence to the 5' end of most mRNAs. [SO:nlw]"
  },
  'SO:0000709': {
    'name': 'SL2_acceptor_site',
    'def': "A trans_splicing_acceptor_site which appends the 22nt SL2 RNA leader sequence to the 5' end of mRNAs. SL2 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0000710': {
    'name': 'gene_with_stop_codon_redefined_as_selenocysteine',
    'def': 'A gene encoding an mRNA that has the stop codon redefined as selenocysteine. [SO:xp]'
  },
  'SO:0000711': {
    'name': 'gene_with_mRNA_recoded_by_translational_bypass',
    'def': 'A gene with mRNA recoded by translational bypass. [SO:xp]'
  },
  'SO:0000712': {
    'name': 'gene_with_transcript_with_translational_frameshift',
    'def': 'A gene encoding a transcript that has a translational frameshift. [SO:xp]'
  },
  'SO:0000713': {
    'name': 'DNA_motif',
    'def': 'A motif that is active in the DNA form of the sequence. [SO:ke]'
  },
  'SO:0000714': {
    'name': 'nucleotide_motif',
    'def': 'A region of nucleotide sequence corresponding to a known motif. [SO:ke]'
  },
  'SO:0000715': {
    'name': 'RNA_motif',
    'def': 'A motif that is active in RNA sequence. [SO:ke]'
  },
  'SO:0000716': {
    'name': 'dicistronic_mRNA',
    'def': 'An mRNA that has the quality dicistronic. [SO:ke]'
  },
  'SO:0000717': {
    'name': 'reading_frame',
    'def': 'A nucleic acid sequence that when read as sequential triplets, has the potential of encoding a sequential string of amino acids. It need not contain the start or stop codon. [SGD:rb]'
  },
  'SO:0000718': {
    'name': 'blocked_reading_frame',
    'def': 'A reading_frame that is interrupted by one or more stop codons; usually identified through inter-genomic sequence comparisons. [SGD:rb]'
  },
  'SO:0000719': {
    'name': 'ultracontig',
    'def': 'An ordered and oriented set of scaffolds based on somewhat weaker sets of inferential evidence such as one set of mate pair reads together with supporting evidence from ESTs or location of markers from SNP or microsatellite maps, or cytogenetic localization of contained markers. [FB:WG]'
  },
  'SO:0000720': {
    'name': 'foreign_transposable_element',
    'def': 'A transposable element that is foreign. [SO:ke]'
  },
  'SO:0000721': {
    'name': 'gene_with_dicistronic_primary_transcript',
    'def': 'A gene that encodes a dicistronic primary transcript. [SO:xp]'
  },
  'SO:0000722': {
    'name': 'gene_with_dicistronic_mRNA',
    'def': 'A gene that encodes a polycistronic mRNA. [SO:xp]'
  },
  'SO:0000723': {
    'name': 'iDNA',
    'def': 'Genomic sequence removed from the genome, as a normal event, by a process of recombination. [SO:ma]'
  },
  'SO:0000724': {
    'name': 'oriT',
    'def': 'A region of a DNA molecule where transfer is initiated during the process of conjugation or mobilization. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000725': {
    'name': 'transit_peptide',
    'def': 'The transit_peptide is a short region at the N-terminus of the peptide that directs the protein to an organelle (chloroplast, mitochondrion, microbody or cyanelle). [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0000726': {
    'name': 'repeat_unit',
    'def': 'The simplest repeated component of a repeat region. A single repeat. [SO:ke]'
  },
  'SO:0000727': {
    'name': 'cis_regulatory_module',
    'def': 'A regulatory region where transcription factor binding sites are clustered to regulate various aspects of transcription activities. (CRMs can be located a few kb to hundreds of kb upstream of the core promoter, in the coding sequence, within introns, or in the untranslated regions (UTR) sequences, and even on a different chromosome). A single gene can be regulated by multiple CRMs to give precise control of its spatial and temporal expression. CRMs function as nodes in large, intertwined regulatory network. CRM DNA accessibility is subject to regulation by dbTFs and transcription co-TFs. [PMID:19660565, SO:SG]'
  },
  'SO:0000728': {
    'name': 'intein',
    'def': 'A region of a peptide that is able to excise itself and rejoin the remaining portions with a peptide bond. [SO:ke]'
  },
  'SO:0000729': {
    'name': 'intein_containing',
    'def': 'An attribute of protein-coding genes where the initial protein product contains an intein. [SO:ke]'
  },
  'SO:0000730': {
    'name': 'gap',
    'def': "A gap in the sequence of known length. The unknown bases are filled in with N's. [SO:ke]"
  },
  'SO:0000731': {
    'name': 'fragmentary',
    'def': 'An attribute to describe a feature that is incomplete. [SO:ke]'
  },
  'SO:0000732': {
    'name': 'predicted',
    'def': 'An attribute describing an unverified region. [SO:ke]'
  },
  'SO:0000733': {
    'name': 'feature_attribute',
    'def': 'An attribute describing a located_sequence_feature. [SO:ke]'
  },
  'SO:0000734': {
    'name': 'exemplar_mRNA',
    'def': 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group. [http://mged.sourceforge.net/ontologies/MGEDontology.php]'
  },
  'SO:0000735': {
    'name': 'sequence_location',
    'def': 'The location of a sequence. []'
  },
  'SO:0000736': {
    'name': 'organelle_sequence',
    'def': 'A sequence of DNA that originates from a an organelle. []'
  },
  'SO:0000737': {
    'name': 'mitochondrial_sequence',
    'def': 'DNA belonging to the genome of a mitochondria. []'
  },
  'SO:0000738': {
    'name': 'nuclear_sequence',
    'def': 'DNA belonging to the nuclear genome of cell. []'
  },
  'SO:0000739': {
    'name': 'nucleomorphic_sequence',
    'def': 'DNA belonging to the genome of a plastid such as a chloroplast. The nucleomorph is the nuclei of the plastic. []'
  },
  'SO:0000740': {
    'name': 'plastid_sequence',
    'def': 'DNA belonging to the genome of a plastid such as a chloroplast. []'
  },
  'SO:0000741': {
    'name': 'kinetoplast',
    'def': 'A kinetoplast is an interlocked network of thousands of minicircles and tens of maxicircles, located near the base of the flagellum of some protozoan species. [PMID:8395055]'
  },
  'SO:0000742': {
    'name': 'maxicircle',
    'def': 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method. [PMID:8395055]'
  },
  'SO:0000743': {
    'name': 'apicoplast_sequence',
    'def': 'DNA belonging to the genome of an apicoplast, a non-photosynthetic plastid. []'
  },
  'SO:0000744': {
    'name': 'chromoplast_sequence',
    'def': 'DNA belonging to the genome of a chromoplast, a colored plastid for synthesis and storage of pigments. []'
  },
  'SO:0000745': {
    'name': 'chloroplast_sequence',
    'def': 'DNA belonging to the genome of a chloroplast, a green plastid for photosynthesis. []'
  },
  'SO:0000746': {
    'name': 'cyanelle_sequence',
    'def': 'DNA belonging to the genome of a cyanelle, a photosynthetic plastid found in algae. []'
  },
  'SO:0000747': {
    'name': 'leucoplast_sequence',
    'def': 'DNA belonging to the genome of a leucoplast, a colorless plastid generally containing starch or oil. []'
  },
  'SO:0000748': {
    'name': 'proplastid_sequence',
    'def': 'DNA belonging to the genome of a proplastid such as an immature chloroplast. []'
  },
  'SO:0000749': {
    'name': 'plasmid_location',
    'def': 'The location of DNA that has come from a plasmid sequence. []'
  },
  'SO:0000750': {
    'name': 'amplification_origin',
    'def': 'An origin_of_replication that is used for the amplification of a chromosomal nucleic acid sequence. [SO:ma]'
  },
  'SO:0000751': {
    'name': 'proviral_location',
    'def': 'The location of DNA that has come from a viral origin. []'
  },
  'SO:0000752': {
    'name': 'gene_group_regulatory_region',
    'def': 'A region that is involved in the regulation of transcription of a group of regulated genes. []'
  },
  'SO:0000753': {
    'name': 'clone_insert',
    'def': 'The region of sequence that has been inserted and is being propagated by the clone. [SO:ke]'
  },
  'SO:0000754': {
    'name': 'lambda_vector',
    'def': 'The lambda bacteriophage is the vector for the linear lambda clone. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome. [ISBN:0-1767-2380-8]'
  },
  'SO:0000755': {
    'name': 'plasmid_vector',
    'def': 'A plasmid that has been generated to act as a vector for foreign sequence. []'
  },
  'SO:0000756': {
    'name': 'cDNA',
    'def': 'DNA synthesized by reverse transcriptase using RNA as a template. [SO:ma]'
  },
  'SO:0000757': {
    'name': 'single_stranded_cDNA',
    'def': 'DNA synthesized from RNA by reverse transcriptase, single stranded. []'
  },
  'SO:0000758': {
    'name': 'double_stranded_cDNA',
    'def': 'DNA synthesized from RNA by reverse transcriptase that has been copied by PCR to make it double stranded. []'
  },
  'SO:0000759': {
    'name': 'plasmid_clone'
  },
  'SO:0000760': {
    'name': 'YAC_clone'
  },
  'SO:0000761': {
    'name': 'phagemid_clone'
  },
  'SO:0000762': {
    'name': 'PAC_clone'
  },
  'SO:0000763': {
    'name': 'fosmid_clone'
  },
  'SO:0000764': {
    'name': 'BAC_clone'
  },
  'SO:0000765': {
    'name': 'cosmid_clone'
  },
  'SO:0000766': {
    'name': 'pyrrolysyl_tRNA',
    'def': "A tRNA sequence that has a pyrrolysine anticodon, and a 3' pyrrolysine binding region. [SO:ke]"
  },
  'SO:0000767': {
    'name': 'clone_insert_start'
  },
  'SO:0000768': {
    'name': 'episome',
    'def': 'A plasmid that may integrate with a chromosome. [SO:ma]'
  },
  'SO:0000769': {
    'name': 'tmRNA_coding_piece',
    'def': 'The region of a two-piece tmRNA that bears the reading frame encoding the proteolysis tag. The tmRNA gene undergoes circular permutation in some groups of bacteria. Processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together. [doi:10.1093/nar/gkh795, Indiana:kw, issn:1362-4962]'
  },
  'SO:0000770': {
    'name': 'tmRNA_acceptor_piece',
    'def': "The acceptor region of a two-piece tmRNA that when mature is charged at its 3' end with alanine. The tmRNA gene undergoes circular permutation in some groups of bacteria; processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together. [doi:10.1093/nar/gkh795, Indiana:kw]"
  },
  'SO:0000771': {
    'name': 'QTL',
    'def': 'A quantitative trait locus (QTL) is a polymorphic locus which contains alleles that differentially affect the expression of a continuously distributed phenotypic trait. Usually it is a marker described by statistical association to quantitative variation in the particular phenotypic trait that is thought to be controlled by the cumulative action of alleles at multiple loci. [http://rgd.mcw.edu/tu/qtls/]'
  },
  'SO:0000772': {
    'name': 'genomic_island',
    'def': 'A genomic island is an integrated mobile genetic element, characterized by size (over 10 Kb). It that has features that suggest a foreign origin. These can include nucleotide distribution (oligonucleotides signature, CG content etc.) that differs from the bulk of the chromosome and/or genes suggesting DNA mobility. [Phigo:at, SO:ke]'
  },
  'SO:0000773': {
    'name': 'pathogenic_island',
    'def': 'Mobile genetic elements that contribute to rapid changes in virulence potential. They are present on the genomes of pathogenic strains but absent from the genomes of non pathogenic members of the same or related species. [SO:ke]'
  },
  'SO:0000774': {
    'name': 'metabolic_island',
    'def': 'A transmissible element containing genes involved in metabolism, analogous to the pathogenicity islands of gram negative bacteria. [SO:ke]'
  },
  'SO:0000775': {
    'name': 'adaptive_island',
    'def': 'An adaptive island is a genomic island that provides an adaptive advantage to the host. [SO:ke]'
  },
  'SO:0000776': {
    'name': 'symbiosis_island',
    'def': 'A transmissible element containing genes involved in symbiosis, analogous to the pathogenicity islands of gram negative bacteria. [SO:ke]'
  },
  'SO:0000777': {
    'name': 'pseudogenic_rRNA',
    'def': 'A non functional descendant of an rRNA. [SO:ke]'
  },
  'SO:0000778': {
    'name': 'pseudogenic_tRNA',
    'def': 'A non functional descendent of a tRNA. [SO:ke]'
  },
  'SO:0000779': {
    'name': 'engineered_episome',
    'def': 'An episome that is engineered. [SO:xp]'
  },
  'SO:0000780': {
    'name': 'transposable_element_attribute'
  },
  'SO:0000781': {
    'name': 'transgenic',
    'def': 'Attribute describing sequence that has been integrated with foreign sequence. [SO:ke]'
  },
  'SO:0000782': {
    'name': 'natural',
    'def': 'An attribute describing a feature that occurs in nature. [SO:ke]'
  },
  'SO:0000783': {
    'name': 'engineered',
    'def': 'An attribute to describe a region that was modified in vitro. [SO:ke]'
  },
  'SO:0000784': {
    'name': 'foreign',
    'def': 'An attribute to describe a region from another species. [SO:ke]'
  },
  'SO:0000785': {
    'name': 'cloned_region',
    'def': 'The region of sequence that has been inserted and is being propagated by the clone. []'
  },
  'SO:0000786': {
    'name': 'reagent_attribute'
  },
  'SO:0000787': {
    'name': 'clone_attribute'
  },
  'SO:0000788': {
    'name': 'cloned'
  },
  'SO:0000789': {
    'name': 'validated',
    'def': 'An attribute to describe a feature that has been proven. [SO:ke]'
  },
  'SO:0000790': {
    'name': 'invalidated',
    'def': 'An attribute describing a feature that is invalidated. [SO:ke]'
  },
  'SO:0000791': {
    'name': 'cloned_genomic'
  },
  'SO:0000792': {
    'name': 'cloned_cDNA'
  },
  'SO:0000793': {
    'name': 'engineered_DNA'
  },
  'SO:0000794': {
    'name': 'engineered_rescue_region',
    'def': 'A rescue region that is engineered. [SO:xp]'
  },
  'SO:0000795': {
    'name': 'rescue_mini_gene',
    'def': 'A mini_gene that rescues. [SO:xp]'
  },
  'SO:0000796': {
    'name': 'transgenic_transposable_element',
    'def': 'TE that has been modified in vitro, including insertion of DNA derived from a source other than the originating TE. [FB:mc]'
  },
  'SO:0000797': {
    'name': 'natural_transposable_element',
    'def': 'TE that exists (or existed) in nature. [FB:mc]'
  },
  'SO:0000798': {
    'name': 'engineered_transposable_element',
    'def': 'TE that has been modified by manipulations in vitro. [FB:mc]'
  },
  'SO:0000799': {
    'name': 'engineered_foreign_transposable_element',
    'def': 'A transposable_element that is engineered and foreign. [FB:mc]'
  },
  'SO:0000800': {
    'name': 'assortment_derived_duplication',
    'def': 'A multi-chromosome duplication aberration generated by reassortment of other aberration components. [FB:gm]'
  },
  'SO:0000801': {
    'name': 'assortment_derived_deficiency_plus_duplication',
    'def': 'A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency and a duplication. [FB:gm]'
  },
  'SO:0000802': {
    'name': 'assortment_derived_deficiency',
    'def': 'A multi-chromosome deficiency aberration generated by reassortment of other aberration components. [FB:gm]'
  },
  'SO:0000803': {
    'name': 'assortment_derived_aneuploid',
    'def': 'A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency or a duplication. [FB:gm]'
  },
  'SO:0000804': {
    'name': 'engineered_region',
    'def': 'A region that is engineered. [SO:xp]'
  },
  'SO:0000805': {
    'name': 'engineered_foreign_region',
    'def': 'A region that is engineered and foreign. [SO:xp]'
  },
  'SO:0000806': {
    'name': 'fusion',
    'def': 'When two regions of DNA are joined together that are not normally together. []'
  },
  'SO:0000807': {
    'name': 'engineered_tag',
    'def': 'A tag that is engineered. [SO:xp]'
  },
  'SO:0000808': {
    'name': 'validated_cDNA_clone',
    'def': 'A cDNA clone that has been validated. [SO:xp]'
  },
  'SO:0000809': {
    'name': 'invalidated_cDNA_clone',
    'def': 'A cDNA clone that is invalid. [SO:xp]'
  },
  'SO:0000810': {
    'name': 'chimeric_cDNA_clone',
    'def': 'A cDNA clone invalidated because it is chimeric. [SO:xp]'
  },
  'SO:0000811': {
    'name': 'genomically_contaminated_cDNA_clone',
    'def': 'A cDNA clone invalidated by genomic contamination. [SO:xp]'
  },
  'SO:0000812': {
    'name': 'polyA_primed_cDNA_clone',
    'def': 'A cDNA clone invalidated by polyA priming. [SO:xp]'
  },
  'SO:0000813': {
    'name': 'partially_processed_cDNA_clone',
    'def': 'A cDNA invalidated clone by partial processing. [SO:xp]'
  },
  'SO:0000814': {
    'name': 'rescue',
    'def': "An attribute describing a region's ability, when introduced to a mutant organism, to re-establish (rescue) a phenotype. [SO:ke]"
  },
  'SO:0000815': {
    'name': 'mini_gene',
    'def': "By definition, minigenes are short open-reading frames (ORF), usually encoding approximately 9 to 20 amino acids, which are expressed in vivo (as distinct from being synthesized as peptide or protein ex vivo and subsequently injected). The in vivo synthesis confers a distinct advantage: the expressed sequences can enter both antigen presentation pathways, MHC I (inducing CD8+ T- cells, which are usually cytotoxic T-lymphocytes (CTL)) and MHC II (inducing CD4+ T-cells, usually 'T-helpers' (Th)); and can encounter B-cells, inducing antibody responses. Three main vector approaches have been used to deliver minigenes: viral vectors, bacterial vectors and plasmid DNA. [PMID:15992143]"
  },
  'SO:0000816': {
    'name': 'rescue_gene',
    'def': 'A gene that rescues. [SO:xp]'
  },
  'SO:0000817': {
    'name': 'wild_type',
    'def': 'An attribute describing sequence with the genotype found in nature and/or standard laboratory stock. [SO:ke]'
  },
  'SO:0000818': {
    'name': 'wild_type_rescue_gene',
    'def': 'A gene that rescues. [SO:xp]'
  },
  'SO:0000819': {
    'name': 'mitochondrial_chromosome',
    'def': 'A chromosome originating in a mitochondria. [SO:xp]'
  },
  'SO:0000820': {
    'name': 'chloroplast_chromosome',
    'def': 'A chromosome originating in a chloroplast. [SO:xp]'
  },
  'SO:0000821': {
    'name': 'chromoplast_chromosome',
    'def': 'A chromosome originating in a chromoplast. [SO:xp]'
  },
  'SO:0000822': {
    'name': 'cyanelle_chromosome',
    'def': 'A chromosome originating in a cyanelle. [SO:xp]'
  },
  'SO:0000823': {
    'name': 'leucoplast_chromosome',
    'def': 'A chromosome with origin in a leucoplast. [SO:xp]'
  },
  'SO:0000824': {
    'name': 'macronuclear_chromosome',
    'def': 'A chromosome originating in a macronucleus. [SO:xp]'
  },
  'SO:0000825': {
    'name': 'micronuclear_chromosome',
    'def': 'A chromosome originating in a micronucleus. [SO:xp]'
  },
  'SO:0000828': {
    'name': 'nuclear_chromosome',
    'def': 'A chromosome originating in a nucleus. [SO:xp]'
  },
  'SO:0000829': {
    'name': 'nucleomorphic_chromosome',
    'def': 'A chromosome originating in a nucleomorph. [SO:xp]'
  },
  'SO:0000830': {
    'name': 'chromosome_part',
    'def': 'A region of a chromosome. [SO:ke]'
  },
  'SO:0000831': {
    'name': 'gene_member_region',
    'def': 'A region of a gene. [SO:ke]'
  },
  'SO:0000832': {
    'name': 'promoter_region',
    'def': 'A region of sequence which is part of a promoter. [SO:ke]'
  },
  'SO:0000833': {
    'name': 'transcript_region',
    'def': 'A region of a transcript. [SO:ke]'
  },
  'SO:0000834': {
    'name': 'mature_transcript_region',
    'def': 'A region of a mature transcript. [SO:ke]'
  },
  'SO:0000835': {
    'name': 'primary_transcript_region',
    'def': 'A part of a primary transcript. [SO:ke]'
  },
  'SO:0000836': {
    'name': 'mRNA_region',
    'def': 'A region of an mRNA. [SO:cb]'
  },
  'SO:0000837': {
    'name': 'UTR_region',
    'def': 'A region of UTR. [SO:ke]'
  },
  'SO:0000838': {
    'name': 'rRNA_primary_transcript_region',
    'def': 'A region of an rRNA primary transcript. [SO:ke]'
  },
  'SO:0000839': {
    'name': 'polypeptide_region',
    'def': 'Biological sequence region that can be assigned to a specific subsequence of a polypeptide. [SO:GAR, SO:ke]'
  },
  'SO:0000840': {
    'name': 'repeat_component',
    'def': 'A region of a repeated sequence. [SO:ke]'
  },
  'SO:0000841': {
    'name': 'spliceosomal_intron_region',
    'def': 'A region within an intron. [SO:ke]'
  },
  'SO:0000842': {
    'name': 'gene_component_region',
    'def': 'A region of a gene that has a specific function. []'
  },
  'SO:0000843': {
    'name': 'bacterial_RNApol_promoter_region',
    'def': 'A region which is part of a bacterial RNA polymerase promoter. [SO:ke]'
  },
  'SO:0000844': {
    'name': 'RNApol_II_promoter_region',
    'def': 'A region of sequence which is a promoter for RNA polymerase II. [SO:ke]'
  },
  'SO:0000845': {
    'name': 'RNApol_III_promoter_type_1_region',
    'def': 'A region of sequence which is a promoter for RNA polymerase III type 1. [SO:ke]'
  },
  'SO:0000846': {
    'name': 'RNApol_III_promoter_type_2_region',
    'def': 'A region of sequence which is a promoter for RNA polymerase III type 2. [SO:ke]'
  },
  'SO:0000847': {
    'name': 'tmRNA_region',
    'def': 'A region of a tmRNA. [SO:cb]'
  },
  'SO:0000848': {
    'name': 'LTR_component',
    'def': 'The long terminal repeat found at the ends of the sequence to be inserted into the host genome. []'
  },
  'SO:0000849': {
    'name': 'three_prime_LTR_component',
    'def': 'A component of the three-prime long terminal repeat. [PMID:8649407]'
  },
  'SO:0000850': {
    'name': 'five_prime_LTR_component',
    'def': 'A component of the five-prime long terminal repeat. [PMID:8649407]'
  },
  'SO:0000851': {
    'name': 'CDS_region',
    'def': 'A region of a CDS. [SO:cb]'
  },
  'SO:0000852': {
    'name': 'exon_region',
    'def': 'A region of an exon. [RSC:cb]'
  },
  'SO:0000853': {
    'name': 'homologous_region',
    'def': 'A region that is homologous to another region. [SO:ke]'
  },
  'SO:0000854': {
    'name': 'paralogous_region',
    'def': 'A homologous_region that is paralogous to another region. [SO:ke]'
  },
  'SO:0000855': {
    'name': 'orthologous_region',
    'def': 'A homologous_region that is orthologous to another region. [SO:ke]'
  },
  'SO:0000856': {
    'name': 'conserved',
    'def': 'A region that is similar or identical across more than one species. []'
  },
  'SO:0000857': {
    'name': 'homologous',
    'def': 'Similarity due to common ancestry. [SO:ke]'
  },
  'SO:0000858': {
    'name': 'orthologous',
    'def': 'An attribute describing a kind of homology where divergence occurred after a speciation event. [SO:ke]'
  },
  'SO:0000859': {
    'name': 'paralogous',
    'def': 'An attribute describing a kind of homology where divergence occurred after a duplication event. [SO:ke]'
  },
  'SO:0000860': {
    'name': 'syntenic',
    'def': 'Attribute describing sequence regions occurring in same order on chromosome of different species. [SO:ke]'
  },
  'SO:0000861': {
    'name': 'capped_primary_transcript',
    'def': 'A primary transcript that is capped. [SO:xp]'
  },
  'SO:0000862': {
    'name': 'capped_mRNA',
    'def': 'An mRNA that is capped. [SO:xp]'
  },
  'SO:0000863': {
    'name': 'mRNA_attribute',
    'def': 'An attribute describing an mRNA feature. [SO:ke]'
  },
  'SO:0000864': {
    'name': 'exemplar',
    'def': 'An attribute describing a sequence is representative of a class of similar sequences. [SO:ke]'
  },
  'SO:0000865': {
    'name': 'frameshift',
    'def': 'An attribute describing a sequence that contains a mutation involving the deletion or insertion of one or more bases, where this number is not divisible by 3. [SO:ke]'
  },
  'SO:0000866': {
    'name': 'minus_1_frameshift',
    'def': 'A frameshift caused by deleting one base. [SO:ke]'
  },
  'SO:0000867': {
    'name': 'minus_2_frameshift',
    'def': 'A frameshift caused by deleting two bases. [SO:ke]'
  },
  'SO:0000868': {
    'name': 'plus_1_frameshift',
    'def': 'A frameshift caused by inserting one base. [SO:ke]'
  },
  'SO:0000869': {
    'name': 'plus_2_framshift',
    'def': 'A frameshift caused by inserting two bases. [SO:ke]'
  },
  'SO:0000870': {
    'name': 'trans_spliced',
    'def': 'An attribute describing transcript sequence that is created by splicing exons from diferent genes. [SO:ke]'
  },
  'SO:0000871': {
    'name': 'polyadenylated_mRNA',
    'def': 'An mRNA that is polyadenylated. [SO:xp]'
  },
  'SO:0000872': {
    'name': 'trans_spliced_mRNA',
    'def': 'An mRNA that is trans-spliced. [SO:xp]'
  },
  'SO:0000873': {
    'name': 'edited_transcript',
    'def': 'A transcript that is edited. [SO:ke]'
  },
  'SO:0000874': {
    'name': 'edited_transcript_by_A_to_I_substitution',
    'def': 'A transcript that has been edited by A to I substitution. [SO:ke]'
  },
  'SO:0000875': {
    'name': 'bound_by_protein',
    'def': 'An attribute describing a sequence that is bound by a protein. [SO:ke]'
  },
  'SO:0000876': {
    'name': 'bound_by_nucleic_acid',
    'def': 'An attribute describing a sequence that is bound by a nucleic acid. [SO:ke]'
  },
  'SO:0000877': {
    'name': 'alternatively_spliced',
    'def': 'An attribute describing a situation where a gene may encode for more than 1 transcript. [SO:ke]'
  },
  'SO:0000878': {
    'name': 'monocistronic',
    'def': 'An attribute describing a sequence that contains the code for one gene product. [SO:ke]'
  },
  'SO:0000879': {
    'name': 'dicistronic',
    'def': 'An attribute describing a sequence that contains the code for two gene products. [SO:ke]'
  },
  'SO:0000880': {
    'name': 'polycistronic',
    'def': 'An attribute describing a sequence that contains the code for more than one gene product. [SO:ke]'
  },
  'SO:0000881': {
    'name': 'recoded',
    'def': 'An attribute describing an mRNA sequence that has been reprogrammed at translation, causing localized alterations. [SO:ke]'
  },
  'SO:0000882': {
    'name': 'codon_redefined',
    'def': 'An attribute describing the alteration of codon meaning. [SO:ke]'
  },
  'SO:0000883': {
    'name': 'stop_codon_read_through',
    'def': 'A stop codon redefined to be a new amino acid. [SO:ke]'
  },
  'SO:0000884': {
    'name': 'stop_codon_redefined_as_pyrrolysine',
    'def': 'A stop codon redefined to be the new amino acid, pyrrolysine. [SO:ke]'
  },
  'SO:0000885': {
    'name': 'stop_codon_redefined_as_selenocysteine',
    'def': 'A stop codon redefined to be the new amino acid, selenocysteine. [SO:ke]'
  },
  'SO:0000886': {
    'name': 'recoded_by_translational_bypass',
    'def': 'Recoded mRNA where a block of nucleotides is not translated. [SO:ke]'
  },
  'SO:0000887': {
    'name': 'translationally_frameshifted',
    'def': 'Recoding by frameshifting a particular site. [SO:ke]'
  },
  'SO:0000888': {
    'name': 'maternally_imprinted_gene',
    'def': 'A gene that is maternally_imprinted. [SO:xp]'
  },
  'SO:0000889': {
    'name': 'paternally_imprinted_gene',
    'def': 'A gene that is paternally imprinted. [SO:xp]'
  },
  'SO:0000890': {
    'name': 'post_translationally_regulated_gene',
    'def': 'A gene that is post translationally regulated. [SO:xp]'
  },
  'SO:0000891': {
    'name': 'negatively_autoregulated_gene',
    'def': 'A gene that is negatively autoreguated. [SO:xp]'
  },
  'SO:0000892': {
    'name': 'positively_autoregulated_gene',
    'def': 'A gene that is positively autoregulated. [SO:xp]'
  },
  'SO:0000893': {
    'name': 'silenced',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated at transcriptional or translational level. [SO:ke]'
  },
  'SO:0000894': {
    'name': 'silenced_by_DNA_modification',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by DNA modifications, resulting in repression of transcription. [SO:ke]'
  },
  'SO:0000895': {
    'name': 'silenced_by_DNA_methylation',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by DNA methylation, resulting in repression of transcription. [SO:ke]'
  },
  'SO:0000896': {
    'name': 'translationally_regulated_gene',
    'def': 'A gene that is translationally regulated. [SO:xp]'
  },
  'SO:0000897': {
    'name': 'allelically_excluded_gene',
    'def': 'A gene that is allelically_excluded. [SO:xp]'
  },
  'SO:0000898': {
    'name': 'epigenetically_modified_gene',
    'def': 'A gene that is epigenetically modified. [SO:ke]'
  },
  'SO:0000899': {
    'name': 'nuclear_mitochondrial',
    'def': 'An attribute describing a nuclear pseudogene of a mitochndrial gene. [SO:ke]'
  },
  'SO:0000900': {
    'name': 'processed',
    'def': 'An attribute describing a pseudogene where by an mRNA was retrotransposed. The mRNA sequence is transcribed back into the genome, lacking introns and promotors, but often including a polyA tail. [SO:ke]'
  },
  'SO:0000901': {
    'name': 'unequally_crossed_over',
    'def': 'An attribute describing a pseudogene that was created by tandem duplication and unequal crossing over during recombination. [SO:ke]'
  },
  'SO:0000902': {
    'name': 'transgene',
    'def': 'A transgene is a gene that has been transferred naturally or by any of a number of genetic engineering techniques from one organism to another. [SO:xp]'
  },
  'SO:0000903': {
    'name': 'endogenous_retroviral_sequence',
    'def': 'Endogenous DNA sequence that are likely to have arisen from retroviruses. []'
  },
  'SO:0000904': {
    'name': 'rearranged_at_DNA_level',
    'def': 'An attribute to describe the sequence of a feature, where the DNA is rearranged. [SO:ke]'
  },
  'SO:0000905': {
    'name': 'status',
    'def': 'An attribute describing the status of a feature, based on the available evidence. [SO:ke]'
  },
  'SO:0000906': {
    'name': 'independently_known',
    'def': 'Attribute to describe a feature that is independently known - not predicted. [SO:ke]'
  },
  'SO:0000907': {
    'name': 'supported_by_sequence_similarity',
    'def': 'An attribute to describe a feature that has been predicted using sequence similarity techniques. [SO:ke]'
  },
  'SO:0000908': {
    'name': 'supported_by_domain_match',
    'def': 'An attribute to describe a feature that has been predicted using sequence similarity of a known domain. [SO:ke]'
  },
  'SO:0000909': {
    'name': 'supported_by_EST_or_cDNA',
    'def': 'An attribute to describe a feature that has been predicted using sequence similarity to EST or cDNA data. [SO:ke]'
  },
  'SO:0000910': {
    'name': 'orphan',
    'def': 'A gene whose predicted amino acid sequence is unsupported by any experimental evidence or by any match with any other known sequence. []'
  },
  'SO:0000911': {
    'name': 'predicted_by_ab_initio_computation',
    'def': 'An attribute describing a feature that is predicted by a computer program that did not rely on sequence similarity. [SO:ke]'
  },
  'SO:0000912': {
    'name': 'asx_turn',
    'def': 'A motif of three consecutive residues and one H-bond in which: residue(i) is Aspartate or Asparagine (Asx), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2). [http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0000913': {
    'name': 'cloned_cDNA_insert',
    'def': 'A clone insert made from cDNA. [SO:xp]'
  },
  'SO:0000914': {
    'name': 'cloned_genomic_insert',
    'def': 'A clone insert made from genomic DNA. [SO:xp]'
  },
  'SO:0000915': {
    'name': 'engineered_insert',
    'def': 'A clone insert that is engineered. [SO:xp]'
  },
  'SO:0000916': {
    'name': 'edit_operation'
  },
  'SO:0000917': {
    'name': 'insert_U',
    'def': 'An edit to insert a U. [SO:ke]'
  },
  'SO:0000918': {
    'name': 'delete_U',
    'def': 'An edit to delete a uridine. [SO:ke]'
  },
  'SO:0000919': {
    'name': 'substitute_A_to_I',
    'def': 'An edit to substitute an I for an A. [SO:ke]'
  },
  'SO:0000920': {
    'name': 'insert_C',
    'def': 'An edit to insert a cytidine. [SO:ke]'
  },
  'SO:0000921': {
    'name': 'insert_dinucleotide',
    'def': 'An edit to insert a dinucleotide. [SO:ke]'
  },
  'SO:0000922': {
    'name': 'substitute_C_to_U',
    'def': 'An edit to substitute an U for a C. [SO:ke]'
  },
  'SO:0000923': {
    'name': 'insert_G',
    'def': 'An edit to insert a G. [SO:ke]'
  },
  'SO:0000924': {
    'name': 'insert_GC',
    'def': 'An edit to insert a GC dinucleotide. [SO:ke]'
  },
  'SO:0000925': {
    'name': 'insert_GU',
    'def': 'An edit to insert a GU dinucleotide. [SO:ke]'
  },
  'SO:0000926': {
    'name': 'insert_CU',
    'def': 'An edit to insert a CU dinucleotide. [SO:ke]'
  },
  'SO:0000927': {
    'name': 'insert_AU',
    'def': 'An edit to insert a AU dinucleotide. [SO:ke]'
  },
  'SO:0000928': {
    'name': 'insert_AA',
    'def': 'An edit to insert a AA dinucleotide. [SO:ke]'
  },
  'SO:0000929': {
    'name': 'edited_mRNA',
    'def': 'An mRNA that is edited. [SO:xp]'
  },
  'SO:0000930': {
    'name': 'guide_RNA_region',
    'def': 'A region of guide RNA. [SO:ma]'
  },
  'SO:0000931': {
    'name': 'anchor_region',
    'def': 'A region of a guide_RNA that base-pairs to a target mRNA. [SO:jk]'
  },
  'SO:0000932': {
    'name': 'pre_edited_mRNA',
    'def': 'A primary transcript that, at least in part, encodes one or more proteins that has not been edited. []'
  },
  'SO:0000933': {
    'name': 'intermediate',
    'def': 'An attribute to describe a feature between stages of processing. [SO:ke]'
  },
  'SO:0000934': {
    'name': 'miRNA_target_site',
    'def': 'A miRNA target site is a binding site where the molecule is a micro RNA. [FB:cds]'
  },
  'SO:0000935': {
    'name': 'edited_CDS',
    'def': 'A CDS that is edited. [SO:xp]'
  },
  'SO:0000936': {
    'name': 'vertebrate_immunoglobulin_T_cell_receptor_rearranged_segment',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA. []'
  },
  'SO:0000937': {
    'name': 'vertebrate_immune_system_feature'
  },
  'SO:0000938': {
    'name': 'vertebrate_immunoglobulin_T_cell_receptor_rearranged_gene_cluster',
    'def': 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration. []'
  },
  'SO:0000939': {
    'name': 'vertebrate_immune_system_gene_recombination_signal_feature',
    'def': 'Feature used for the recombination of genomic material for the purpose of generating diversity of the immune system. []'
  },
  'SO:0000940': {
    'name': 'recombinationally_rearranged',
    'def': 'A gene that is recombinationally rearranged. []'
  },
  'SO:0000941': {
    'name': 'recombinationally_rearranged_vertebrate_immune_system_gene',
    'def': 'A recombinationally rearranged gene of the vertebrate immune system. [SO:xp]'
  },
  'SO:0000942': {
    'name': 'attP_site',
    'def': 'An integration/excision site of a phage chromosome at which a recombinase acts to insert the phage DNA at a cognate integration/excision site on a bacterial chromosome. [SO:as]'
  },
  'SO:0000943': {
    'name': 'attB_site',
    'def': 'An integration/excision site of a bacterial chromosome at which a recombinase acts to insert foreign DNA containing a cognate integration/excision site. [SO:as]'
  },
  'SO:0000944': {
    'name': 'attL_site',
    'def': "A region that results from recombination between attP_site and attB_site, composed of the 5' portion of attB_site and the 3' portion of attP_site. [SO:as]"
  },
  'SO:0000945': {
    'name': 'attR_site',
    'def': "A region that results from recombination between attP_site and attB_site, composed of the 5' portion of attP_site and the 3' portion of attB_site. [SO:as]"
  },
  'SO:0000946': {
    'name': 'integration_excision_site',
    'def': 'A region specifically recognised by a recombinase, which inserts or removes another region marked by a distinct cognate integration/excision site. [SO:as]'
  },
  'SO:0000947': {
    'name': 'resolution_site',
    'def': 'A region specifically recognized by a recombinase, which separates a physically contiguous circle of DNA into two physically separate circles. [SO:as]'
  },
  'SO:0000948': {
    'name': 'inversion_site',
    'def': 'A region specifically recognised by a recombinase, which inverts the region flanked by a pair of sites. [SO:ma]'
  },
  'SO:0000949': {
    'name': 'dif_site',
    'def': 'A site at which replicated bacterial circular chromosomes are decatenated by site specific resolvase. [SO:as]'
  },
  'SO:0000950': {
    'name': 'attC_site',
    'def': 'An attC site is a sequence required for the integration of a DNA of an integron. [SO:as]'
  },
  'SO:0000951': {
    'name': 'eukaryotic_terminator',
    'def': 'A signal for RNA polymerase to terminate transcription. []'
  },
  'SO:0000952': {
    'name': 'oriV',
    'def': 'An origin of vegetative replication in plasmids and phages. [SO:as]'
  },
  'SO:0000953': {
    'name': 'oriC',
    'def': 'An origin of bacterial chromosome replication. [SO:as]'
  },
  'SO:0000954': {
    'name': 'DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, DNA molecule. [SO:ma]'
  },
  'SO:0000955': {
    'name': 'double_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded DNA molecule. [SO:ma]'
  },
  'SO:0000956': {
    'name': 'single_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded DNA molecule. [SO:ma]'
  },
  'SO:0000957': {
    'name': 'linear_double_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded, linear DNA molecule. [SO:ma]'
  },
  'SO:0000958': {
    'name': 'circular_double_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded, circular DNA molecule. [SO:ma]'
  },
  'SO:0000959': {
    'name': 'linear_single_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded, linear DNA molecule. [SO:ma]'
  },
  'SO:0000960': {
    'name': 'circular_single_stranded_DNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded, circular DNA molecule. [SO:ma]'
  },
  'SO:0000961': {
    'name': 'RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, RNA molecule. [SO:ma]'
  },
  'SO:0000962': {
    'name': 'single_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded RNA molecule. [SO:ma]'
  },
  'SO:0000963': {
    'name': 'linear_single_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded, linear RNA molecule. [SO:ma]'
  },
  'SO:0000964': {
    'name': 'linear_double_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded, linear RNA molecule. [SO:ma]'
  },
  'SO:0000965': {
    'name': 'double_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded RNA molecule. [SO:ma]'
  },
  'SO:0000966': {
    'name': 'circular_single_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, single-stranded, circular DNA molecule. [SO:ma]'
  },
  'SO:0000967': {
    'name': 'circular_double_stranded_RNA_chromosome',
    'def': 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule. [SO:ma]'
  },
  'SO:0000968': {
    'name': 'sequence_replication_mode'
  },
  'SO:0000969': {
    'name': 'rolling_circle'
  },
  'SO:0000970': {
    'name': 'theta_replication'
  },
  'SO:0000971': {
    'name': 'DNA_replication_mode'
  },
  'SO:0000972': {
    'name': 'RNA_replication_mode'
  },
  'SO:0000973': {
    'name': 'insertion_sequence',
    'def': 'A terminal_inverted_repeat_element that is bacterial and only encodes the functions required for its transposition between these inverted repeats. [SO:as]'
  },
  'SO:0000975': {
    'name': 'minicircle_gene',
    'def': 'A gene found within a minicircle. []'
  },
  'SO:0000976': {
    'name': 'cryptic',
    'def': 'A feature_attribute describing a feature that is not manifest under normal conditions. [SO:ke]'
  },
  'SO:0000977': {
    'name': 'anchor_binding_site'
  },
  'SO:0000978': {
    'name': 'template_region',
    'def': 'A region of a guide_RNA that specifies the insertions and deletions of bases in the editing of a target mRNA. [SO:jk]'
  },
  'SO:0000979': {
    'name': 'gRNA_encoding',
    'def': 'A non-protein_coding gene that encodes a guide_RNA. [SO:ma]'
  },
  'SO:0000980': {
    'name': 'minicircle',
    'def': 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs. [PMID:8395055]'
  },
  'SO:0000981': {
    'name': 'rho_dependent_bacterial_terminator',
    'def': 'A transcription terminator that is dependent upon Rho. []'
  },
  'SO:0000982': {
    'name': 'rho_independent_bacterial_terminator',
    'def': 'A transcription terminator that is not dependent upon Rho. Rather, the mRNA contains a sequence that allows it to base-pair with itself and make a stem-loop structure. []'
  },
  'SO:0000983': {
    'name': 'strand_attribute',
    'def': 'The attribute of how many strands are present in a nucleotide polymer. []'
  },
  'SO:0000984': {
    'name': 'single',
    'def': 'When a nucleotide polymer has only one strand. []'
  },
  'SO:0000985': {
    'name': 'double',
    'def': 'When a nucleotide polymer has two strands that are reverse-complement to one another and pair together. []'
  },
  'SO:0000986': {
    'name': 'topology_attribute',
    'def': 'The attribute of whether a nucleotide polymer is linear or circular. []'
  },
  'SO:0000987': {
    'name': 'linear',
    'def': "A quality of a nucleotide polymer that has a 3'-terminal residue and a 5'-terminal residue. [SO:cb]"
  },
  'SO:0000988': {
    'name': 'circular',
    'def': 'A quality of a nucleotide polymer that has no terminal nucleotide residues. [SO:cb]'
  },
  'SO:0000989': {
    'name': 'class_II_RNA',
    'def': "Small non-coding RNA (59-60 nt long) containing 5' and 3' ends that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm. [PMID:15333696]"
  },
  'SO:0000990': {
    'name': 'class_I_RNA',
    'def': "Small non-coding RNA (55-65 nt long) containing highly conserved 5' and 3' ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm. [PMID:15333696]"
  },
  'SO:0000991': {
    'name': 'genomic_DNA',
    'def': 'DNA located in the genome and able to be transmitted to the offspring. [BCS:etrwz]'
  },
  'SO:0000992': {
    'name': 'BAC_cloned_genomic_insert',
    'def': 'A region of DNA that has been inserted into the bacterial genome using a bacterial artificial chromosome. []'
  },
  'SO:0000993': {
    'name': 'consensus',
    'def': 'A sequence produced from an aligment algorithm that uses multiple sequences as input. []'
  },
  'SO:0000994': {
    'name': 'consensus_region',
    'def': 'A region that has a known consensus sequence. []'
  },
  'SO:0000995': {
    'name': 'consensus_mRNA',
    'def': 'An mRNA sequence produced from an aligment algorithm that uses multiple sequences as input. []'
  },
  'SO:0000996': {
    'name': 'predicted_gene',
    'def': 'A region of the genome that has been predicted to be a gene but has not been confirmed by laboratory experiments. []'
  },
  'SO:0000997': {
    'name': 'gene_fragment',
    'def': 'A portion of a gene that is not the complete gene. []'
  },
  'SO:0000998': {
    'name': 'recursive_splice_site',
    'def': 'A recursive splice site is a splice site which subdivides a large intron. Recursive splicing is a mechanism that splices large introns by sub dividing the intron at non exonic elements and alternate exons. [http://www.genetics.org/cgi/content/full/170/2/661]'
  },
  'SO:0000999': {
    'name': 'BAC_end',
    'def': 'A region of sequence from the end of a BAC clone that may provide a highly specific marker. [SO:ke]'
  },
  'SO:0001000': {
    'name': 'cytosolic_16S_rRNA',
    'def': 'Cytosolic 16S rRNA is an RNA component of the small subunit of cytosolic ribosomes in prokaryotes. [SO:ke]'
  },
  'SO:0001001': {
    'name': 'cytosolic_23S_rRNA',
    'def': 'Cytosolic 23S rRNA is an RNA component of the large subunit of cytosolic ribosomes in prokaryotes. [SO:ke]'
  },
  'SO:0001002': {
    'name': 'cytosolic_25S_rRNA',
    'def': 'Cytosolic 25S rRNA is an RNA component of the large subunit of cytosolic ribosomes most eukaryotes. [PMID:15493135, PMID:2100998, RSC:cb]'
  },
  'SO:0001003': {
    'name': 'solo_LTR',
    'def': 'A recombination product between the 2 LTR of the same element. [SO:ke]'
  },
  'SO:0001004': {
    'name': 'low_complexity',
    'def': 'When a sequence does not contain an equal distribution of all four possible nucleotide bases or does not contain all nucleotide bases. []'
  },
  'SO:0001005': {
    'name': 'low_complexity_region',
    'def': 'A region where the DNA does not contain an equal distrubution of all four possible nucleotides or does not contain all four nucleotides. []'
  },
  'SO:0001006': {
    'name': 'prophage',
    'def': 'A phage genome after it has established in the host genome in a latent/immune state either as a plasmid or as an integrated "island". [GOC:jl]'
  },
  'SO:0001007': {
    'name': 'cryptic_prophage',
    'def': 'A remnant of an integrated prophage in the host genome or an "island" in the host genome that includes phage like-genes. [GOC:jl]'
  },
  'SO:0001008': {
    'name': 'tetraloop',
    'def': 'A base-paired stem with loop of 4 non-hydrogen bonded nucleotides. [SO:ke]'
  },
  'SO:0001009': {
    'name': 'DNA_constraint_sequence',
    'def': 'A double-stranded DNA used to control macromolecular structure and function." [http:/www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=pubmed&term=SILVERMAN+S [au]&dispmax=50]'
  },
  'SO:0001010': {
    'name': 'i_motif',
    'def': 'A cytosine rich domain whereby strands associate both inter- and intramolecularly at moderately acidic pH. [PMID:9753739]'
  },
  'SO:0001011': {
    'name': 'PNA_oligo',
    'def': 'Peptide nucleic acid, is a chemical not known to occur naturally but is artificially synthesized and used in some biological research and medical treatments. The PNA backbone is composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds. [SO:ke]'
  },
  'SO:0001012': {
    'name': 'DNAzyme',
    'def': 'A DNA sequence with catalytic activity. [SO:cb]'
  },
  'SO:0001013': {
    'name': 'MNP',
    'def': 'A multiple nucleotide polymorphism with alleles of common length > 1, for example AAA/TTT. [http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=rs2067431]'
  },
  'SO:0001014': {
    'name': 'intron_domain',
    'def': 'An intronic region that has an attribute. []'
  },
  'SO:0001015': {
    'name': 'wobble_base_pair',
    'def': 'A type of non-canonical base pairing, most commonly between G and U, which is important for the secondary structure of RNAs. It has similar thermodynamic stability to the Watson-Crick pairing. Wobble base pairs only have two hydrogen bonds. Other wobble base pair possibilities are I-A, I-U and I-C. [PMID:11256617]'
  },
  'SO:0001016': {
    'name': 'internal_guide_sequence',
    'def': 'A purine-rich sequence in the group I introns which determines the locations of the splice sites in group I intron splicing and has catalytic activity. [SO:cb]'
  },
  'SO:0001017': {
    'name': 'silent_mutation',
    'def': 'A sequence variant that does not affect protein function. Silent mutations may occur in genic ( CDS, UTR, intron etc) and intergenic regions. Silent mutations may have affects on processes such as splicing and regulation. [SO:ke]'
  },
  'SO:0001018': {
    'name': 'epitope',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with antibodies, B cells or T cells. [http://en.wikipedia.org/wiki/Epitope, SO:cb]'
  },
  'SO:0001019': {
    'name': 'copy_number_variation',
    'def': 'A variation that increases or decreases the copy number of a given region. [SO:ke]'
  },
  'SO:0001020': {
    'name': 'sequence_variant_affecting_copy_number'
  },
  'SO:0001021': {
    'name': 'chromosome_breakpoint',
    'def': 'A chromosomal region that may sustain a double-strand break, resulting in a recombination event. []'
  },
  'SO:0001022': {
    'name': 'inversion_breakpoint',
    'def': 'The point within a chromosome where an inversion begins or ends. [SO:cb]'
  },
  'SO:0001023': {
    'name': 'allele',
    'def': 'An allele is one of a set of coexisting sequence variants of a gene. [SO:immuno_workshop]'
  },
  'SO:0001024': {
    'name': 'haplotype',
    'def': 'A haplotype is one of a set of coexisting sequence variants of a haplotype block. [SO:immuno_workshop]'
  },
  'SO:0001025': {
    'name': 'polymorphic_sequence_variant',
    'def': 'A sequence variant that is segregating in one or more natural populations of a species. [SO:immuno_workshop]'
  },
  'SO:0001026': {
    'name': 'genome',
    'def': 'A genome is the sum of genetic material within a cell or virion. [SO:immuno_workshop]'
  },
  'SO:0001027': {
    'name': 'genotype',
    'def': 'A genotype is a variant genome, complete or incomplete. [SO:immuno_workshop]'
  },
  'SO:0001028': {
    'name': 'diplotype',
    'def': 'A diplotype is a pair of haplotypes from a given individual. It is a genotype where the phase is known. [SO:immuno_workshop]'
  },
  'SO:0001029': {
    'name': 'direction_attribute',
    'def': 'The attribute of whether the sequence is the same direction as a feature (forward) or the opposite direction as a feature (reverse). []'
  },
  'SO:0001030': {
    'name': 'forward',
    'def': "Forward is an attribute of the feature, where the feature is in the 5' to 3' direction. [SO:ke]"
  },
  'SO:0001031': {
    'name': 'reverse',
    'def': "Reverse is an attribute of the feature, where the feature is in the 3' to 5' direction. Again could be applied to primer. [SO:ke]"
  },
  'SO:0001032': {
    'name': 'mitochondrial_DNA',
    'def': 'DNA belonging to the genome of a mitochondria. []'
  },
  'SO:0001033': {
    'name': 'chloroplast_DNA',
    'def': 'DNA belonging to the genome of a chloroplast, a photosynthetic plastid. []'
  },
  'SO:0001034': {
    'name': 'miRtron',
    'def': 'A de-branched intron which mimics the structure of pre-miRNA and enters the miRNA processing pathway without Drosha mediated cleavage. [PMID:17589500, SO:ma]'
  },
  'SO:0001035': {
    'name': 'piRNA',
    'def': 'A small non coding RNA, part of a silencing system that prevents the spreading of selfish genetic elements. [SO:ke]'
  },
  'SO:0001036': {
    'name': 'arginyl_tRNA',
    'def': "A tRNA sequence that has an arginine anticodon, and a 3' arginine binding region. [SO:ke]"
  },
  'SO:0001037': {
    'name': 'mobile_genetic_element',
    'def': 'A nucleotide region with either intra-genome or intracellular mobility, of varying length, which often carry the information necessary for transfer and recombination with the host genome. [PMID:14681355]'
  },
  'SO:0001038': {
    'name': 'extrachromosomal_mobile_genetic_element',
    'def': 'An MGE that is not integrated into the host chromosome. [SO:ke]'
  },
  'SO:0001039': {
    'name': 'integrated_mobile_genetic_element',
    'def': 'An MGE that is integrated into the host chromosome. [SO:ke]'
  },
  'SO:0001040': {
    'name': 'integrated_plasmid',
    'def': 'A plasmid sequence that is integrated within the host chromosome. [SO:ke]'
  },
  'SO:0001041': {
    'name': 'viral_sequence',
    'def': 'The region of nucleotide sequence of a virus, a submicroscopic particle that replicates by infecting a host cell. [SO:ke]'
  },
  'SO:0001042': {
    'name': 'phage_sequence',
    'def': 'The nucleotide sequence of a virus that infects bacteria. [SO:ke]'
  },
  'SO:0001043': {
    'name': 'attCtn_site',
    'def': 'An attachment site located on a conjugative transposon and used for site-specific integration of a conjugative transposon. [Phigo:at]'
  },
  'SO:0001044': {
    'name': 'nuclear_mt_pseudogene',
    'def': 'A nuclear pseudogene of either coding or non-coding mitochondria derived sequence. [SO:xp]'
  },
  'SO:0001045': {
    'name': 'cointegrated_plasmid',
    'def': 'A MGE region consisting of two fused plasmids resulting from a replicative transposition event. [phigo:at]'
  },
  'SO:0001046': {
    'name': 'IRLinv_site',
    'def': 'Component of the inversion site located at the left of a region susceptible to site-specific inversion. [Phigo:at]'
  },
  'SO:0001047': {
    'name': 'IRRinv_site',
    'def': 'Component of the inversion site located at the right of a region susceptible to site-specific inversion. [Phigo:at]'
  },
  'SO:0001048': {
    'name': 'inversion_site_part',
    'def': 'A region located within an inversion site. [SO:ke]'
  },
  'SO:0001049': {
    'name': 'defective_conjugative_transposon',
    'def': 'An island that contains genes for integration/excision and the gene and site for the initiation of intercellular transfer by conjugation. It can be complemented for transfer by a conjugative transposon. [Phigo:ariane]'
  },
  'SO:0001050': {
    'name': 'repeat_fragment',
    'def': 'A portion of a repeat, interrupted by the insertion of another element. [SO:ke]'
  },
  'SO:0001051': {
    'name': 'nested_region'
  },
  'SO:0001052': {
    'name': 'nested_repeat'
  },
  'SO:0001053': {
    'name': 'nested_transposon'
  },
  'SO:0001054': {
    'name': 'transposon_fragment',
    'def': 'A portion of a transposon, interrupted by the insertion of another element. [SO:ke]'
  },
  'SO:0001055': {
    'name': 'transcriptional_cis_regulatory_region',
    'def': 'A regulatory_region that modulates the transcription of a gene or genes. [PMID:9679020, SO:regcreative]'
  },
  'SO:0001056': {
    'name': 'splicing_regulatory_region',
    'def': 'A regulatory_region that modulates splicing. [SO:ke]'
  },
  'SO:0001057': {
    'name': 'enhanceosome'
  },
  'SO:0001058': {
    'name': 'promoter_targeting_sequence',
    'def': 'A transcriptional_cis_regulatory_region that restricts the activity of a CRM to a single promoter and which functions only when both itself and an insulator are located between the CRM and the promoter. [SO:regcreative]'
  },
  'SO:0001059': {
    'name': 'sequence_alteration',
    'def': 'A sequence_alteration is a sequence_feature whose extent is the deviation from another sequence. [SO:ke]'
  },
  'SO:0001060': {
    'name': 'sequence_variant',
    'def': 'A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration. [SO:ke]'
  },
  'SO:0001061': {
    'name': 'propeptide_cleavage_site',
    'def': 'The propeptide_cleavage_site is the arginine/lysine boundary on a propeptide where cleavage occurs. [EBIBS:GAR]'
  },
  'SO:0001062': {
    'name': 'propeptide',
    'def': 'Part of a peptide chain which is cleaved off during the formation of the mature protein. [EBIBS:GAR]'
  },
  'SO:0001063': {
    'name': 'immature_peptide_region',
    'def': 'An immature_peptide_region is the extent of the peptide after it has been translated and before any processing occurs. [EBIBS:GAR]'
  },
  'SO:0001064': {
    'name': 'active_peptide',
    'def': 'Active peptides are proteins which are biologically active, released from a precursor molecule. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001066': {
    'name': 'compositionally_biased_region_of_peptide',
    'def': 'Polypeptide region that is rich in a particular amino acid or homopolymeric and greater than three residues in length. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001067': {
    'name': 'polypeptide_motif',
    'def': 'A sequence motif is a short (up to 20 amino acids) region of biological interest. Such motifs, although they are too short to constitute functional domains, share sequence similarities and are conserved in different proteins. They display a common function (protein-binding, subcellular location etc.). [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001068': {
    'name': 'polypeptide_repeat',
    'def': 'A polypeptide_repeat is a single copy of an internal sequence repetition. [EBIBS:GAR]'
  },
  'SO:0001070': {
    'name': 'polypeptide_structural_region',
    'def': 'Region of polypeptide with a given structural property. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001071': {
    'name': 'membrane_structure',
    'def': 'Arrangement of the polypeptide with respect to the lipid bilayer. [EBIBS:GAR]'
  },
  'SO:0001072': {
    'name': 'extramembrane_polypeptide_region',
    'def': 'Polypeptide region that is localized outside of a lipid bilayer. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001073': {
    'name': 'cytoplasmic_polypeptide_region',
    'def': 'Polypeptide region that is localized inside the cytoplasm. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001074': {
    'name': 'non_cytoplasmic_polypeptide_region',
    'def': 'Polypeptide region that is localized outside of a lipid bilayer and outside of the cytoplasm. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001075': {
    'name': 'intramembrane_polypeptide_region',
    'def': 'Polypeptide region present in the lipid bilayer. [EBIBS:GAR]'
  },
  'SO:0001076': {
    'name': 'membrane_peptide_loop',
    'def': 'Polypeptide region localized within the lipid bilayer where both ends traverse the same membrane. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001077': {
    'name': 'transmembrane_polypeptide_region',
    'def': 'Polypeptide region traversing the lipid bilayer. [EBIBS:GAR, UniProt:curator_manual]'
  },
  'SO:0001078': {
    'name': 'polypeptide_secondary_structure',
    'def': 'A region of peptide with secondary structure has hydrogen bonding along the peptide chain that causes a defined conformation of the chain. [EBIBS:GAR]'
  },
  'SO:0001079': {
    'name': 'polypeptide_structural_motif',
    'def': 'Motif is a three-dimensional structural element within the chain, which appears also in a variety of other molecules. Unlike a domain, a motif does not need to form a stable globular unit. [EBIBS:GAR]'
  },
  'SO:0001080': {
    'name': 'coiled_coil',
    'def': 'A coiled coil is a structural motif in proteins, in which alpha-helices are coiled together like the strands of a rope. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001081': {
    'name': 'helix_turn_helix',
    'def': 'A motif comprising two helices separated by a turn. [EBIBS:GAR]'
  },
  'SO:0001082': {
    'name': 'polypeptide_sequencing_information',
    'def': 'Incompatibility in the sequence due to some experimental problem. [EBIBS:GAR]'
  },
  'SO:0001083': {
    'name': 'non_adjacent_residues',
    'def': 'Indicates that two consecutive residues in a fragment sequence are not consecutive in the full-length protein and that there are a number of unsequenced residues between them. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001084': {
    'name': 'non_terminal_residue',
    'def': 'The residue at an extremity of the sequence is not the terminal residue. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001085': {
    'name': 'sequence_conflict',
    'def': 'Different sources report differing sequences. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001086': {
    'name': 'sequence_uncertainty',
    'def': 'Describes the positions in a sequence where the authors are unsure about the sequence assignment. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001087': {
    'name': 'cross_link',
    'def': 'Posttranslationally formed amino acid bonds. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001088': {
    'name': 'disulfide_bond',
    'def': 'The covalent bond between sulfur atoms that binds two peptide chains or different parts of one peptide chain and is a structural determinant in many protein molecules. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001089': {
    'name': 'post_translationally_modified_region',
    'def': 'A region where a transformation occurs in a protein after it has been synthesized. This which may regulate, stabilize, crosslink or introduce new chemical functionalities in the protein. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001090': {
    'name': 'covalent_binding_site',
    'def': 'Binding involving a covalent bond. [EBIBS:GAR]'
  },
  'SO:0001091': {
    'name': 'non_covalent_binding_site',
    'def': 'Binding site for any chemical group (co-enzyme, prosthetic group, etc.). [EBIBS:GAR]'
  },
  'SO:0001092': {
    'name': 'polypeptide_metal_contact',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with metal ions. [EBIBS:GAR, SO:cb, UniProt:curation_manual]'
  },
  'SO:0001093': {
    'name': 'protein_protein_contact',
    'def': 'A binding site that, in the protein molecule, interacts selectively and non-covalently with polypeptide residues. [EBIBS:GAR, UniProt:Curation_manual]'
  },
  'SO:0001094': {
    'name': 'polypeptide_calcium_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with calcium ions. [EBIBS:GAR]'
  },
  'SO:0001095': {
    'name': 'polypeptide_cobalt_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with cobalt ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001096': {
    'name': 'polypeptide_copper_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with copper ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001097': {
    'name': 'polypeptide_iron_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with iron ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001098': {
    'name': 'polypeptide_magnesium_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with magnesium ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001099': {
    'name': 'polypeptide_manganese_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with manganese ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001100': {
    'name': 'polypeptide_molybdenum_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with molybdenum ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001101': {
    'name': 'polypeptide_nickel_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with nickel ions. [EBIBS:GAR]'
  },
  'SO:0001102': {
    'name': 'polypeptide_tungsten_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with tungsten ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001103': {
    'name': 'polypeptide_zinc_ion_contact_site',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with zinc ions. [EBIBS:GAR, SO:cb]'
  },
  'SO:0001104': {
    'name': 'catalytic_residue',
    'def': 'Amino acid involved in the activity of an enzyme. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001105': {
    'name': 'polypeptide_ligand_contact',
    'def': 'Residues which interact with a ligand. [EBIBS:GAR]'
  },
  'SO:0001106': {
    'name': 'asx_motif',
    'def': 'A motif of five consecutive residues and two H-bonds in which: Residue(i) is Aspartate or Asparagine (Asx), side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3), main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4). [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001107': {
    'name': 'beta_bulge',
    'def': 'A motif of three residues within a beta-sheet in which the main chains of two consecutive residues are H-bonded to that of the third, and in which the dihedral angles are as follows: Residue(i): -140 degrees < phi(l) -20 degrees , -90 degrees < psi(l) < 40 degrees. Residue (i+1): -180 degrees < phi < -25 degrees or +120 degrees < phi < +180 degrees, +40 degrees < psi < +180 degrees or -180 degrees < psi < -120 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001108': {
    'name': 'beta_bulge_loop',
    'def': 'A motif of three residues within a beta-sheet consisting of two H-bonds. Beta bulge loops often occur at the loop ends of beta-hairpins. [EBIBS:GAR, Http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001109': {
    'name': 'beta_bulge_loop_five',
    'def': 'A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+4), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+3), these loops have an RL nest at residues i+2 and i+3. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001110': {
    'name': 'beta_bulge_loop_six',
    'def': 'A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+5), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+4), these loops have an RL nest at residues i+3 and i+4. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001111': {
    'name': 'beta_strand',
    'def': 'A beta strand describes a single length of polypeptide chain that forms part of a beta sheet. A single continuous stretch of amino acids adopting an extended conformation of hydrogen bonds between the N-O and the C=O of another part of the peptide. This forms a secondary protein structure in which two or more extended polypeptide regions are hydrogen-bonded to one another in a planar array. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001112': {
    'name': 'antiparallel_beta_strand',
    'def': "A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (one running N-terminal to C-terminal and one running C-terminal to N-terminal). Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i) and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they form two mutual backbone hydrogen bonds to each other's flanking peptide groups; this is known as a close pair of hydrogen bonds. The peptide backbone dihedral angles (phi, psi) are about (-140 degrees, 135 degrees) in antiparallel sheets. [EBIBS:GAR, UniProt:curation_manual]"
  },
  'SO:0001113': {
    'name': 'parallel_beta_strand',
    'def': 'A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (both running N-terminal to C-terminal). This orientation is slightly less stable because it introduces nonplanarity in the inter-strand hydrogen bonding pattern. Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i)and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they do not hydrogen bond to each other; rather, one residue forms hydrogen bonds to the residues that flank the other (but not vice versa). For example, residue i may form hydrogen bonds to residues j - 1 and j + 1; this is known as a wide pair of hydrogen bonds. By contrast, residue j may hydrogen-bond to different residues altogether, or to none at all. The dihedral angles (phi, psi) are about (-120 degrees, 115 degrees) in parallel sheets. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001114': {
    'name': 'peptide_helix',
    'def': 'A helix is a secondary_structure conformation where the peptide backbone forms a coil. [EBIBS:GAR]'
  },
  'SO:0001115': {
    'name': 'left_handed_peptide_helix',
    'def': 'A left handed helix is a region of peptide where the coiled conformation turns in an anticlockwise, left handed screw. [EBIBS:GAR]'
  },
  'SO:0001116': {
    'name': 'right_handed_peptide_helix',
    'def': 'A right handed helix is a region of peptide where the coiled conformation turns in a clockwise, right handed screw. [EBIBS:GAR]'
  },
  'SO:0001117': {
    'name': 'alpha_helix',
    'def': 'The helix has 3.6 residues per turn which corresponds to a translation of 1.5 angstroms (= 0.15 nm) along the helical axis. Every backbone N-H group donates a hydrogen bond to the backbone C=O group of the amino acid four residues earlier. [EBIBS:GAR]'
  },
  'SO:0001118': {
    'name': 'pi_helix',
    'def': 'The pi helix has 4.1 residues per turn and a translation of 1.15 (=0.115 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid five residues earlier. [EBIBS:GAR]'
  },
  'SO:0001119': {
    'name': 'three_ten_helix',
    'def': 'The 3-10 helix has 3 residues per turn with a translation of 2.0 angstroms (=0.2 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid three residues earlier. [EBIBS:GAR]'
  },
  'SO:0001120': {
    'name': 'polypeptide_nest_motif',
    'def': 'A motif of two consecutive residues with dihedral angles. Nest should not have Proline as any residue. Nests frequently occur as parts of other motifs such as Schellman loops. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001121': {
    'name': 'polypeptide_nest_left_right_motif',
    'def': 'A motif of two consecutive residues with dihedral angles: Residue(i): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001122': {
    'name': 'polypeptide_nest_right_left_motif',
    'def': 'A motif of two consecutive residues with dihedral angles: Residue(i): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001123': {
    'name': 'schellmann_loop',
    'def': 'A motif of six or seven consecutive residues that contains two H-bonds. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001124': {
    'name': 'schellmann_loop_seven',
    'def': 'Wild type: A motif of seven consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+6), the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+5). [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001125': {
    'name': 'schellmann_loop_six',
    'def': 'Common Type: A motif of six consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+5) the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+4). [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001126': {
    'name': 'serine_threonine_motif',
    'def': 'A motif of five consecutive residues and two hydrogen bonds in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3) , the main-chain CO group of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4). [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001127': {
    'name': 'serine_threonine_staple_motif',
    'def': 'A motif of four or five consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain OH of residue(i) is H-bonded to the main-chain CO of residue(i3) or (i4), Phi angles of residues(i1), (i2) and (i3) are negative. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001128': {
    'name': 'polypeptide_turn_motif',
    'def': 'A reversal in the direction of the backbone of a protein that is stabilized by hydrogen bond between backbone NH and CO groups, involving no more than 4 amino acid residues. [EBIBS:GAR, uniprot:feature_type]'
  },
  'SO:0001129': {
    'name': 'asx_turn_left_handed_type_one',
    'def': 'Left handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001130': {
    'name': 'asx_turn_left_handed_type_two',
    'def': 'Left handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001131': {
    'name': 'asx_turn_right_handed_type_two',
    'def': 'Right handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001132': {
    'name': 'asx_turn_right_handed_type_one',
    'def': 'Right handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001133': {
    'name': 'beta_turn',
    'def': 'A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles of the second and third residues, which are the basis for sub-categorization. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001134': {
    'name': 'beta_turn_left_handed_type_one',
    'def': 'Left handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles:- Residue(i+1): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees. Residue(i+2): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001135': {
    'name': 'beta_turn_left_handed_type_two',
    'def': 'Left handed type II: A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees > phi > -20 degrees, +80 degrees > psi > +180 degrees. Residue(i+2): +20 degrees > phi > +140 degrees, -40 degrees > psi > +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001136': {
    'name': 'beta_turn_right_handed_type_one',
    'def': 'Right handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+2): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001137': {
    'name': 'beta_turn_right_handed_type_two',
    'def': 'Right handed type II:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, +80 degrees < psi < +180 degrees. Residue(i+2): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001138': {
    'name': 'gamma_turn',
    'def': 'Gamma turns, defined for 3 residues i,( i+1),( i+2) if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001139': {
    'name': 'gamma_turn_classic',
    'def': 'Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=75.0 - psi(i+1)=-64.0. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001140': {
    'name': 'gamma_turn_inverse',
    'def': 'Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=-79.0 - psi(i+1)=69.0. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001141': {
    'name': 'serine_threonine_turn',
    'def': 'A motif of three consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2). [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001142': {
    'name': 'st_turn_left_handed_type_one',
    'def': 'The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001143': {
    'name': 'st_turn_left_handed_type_two',
    'def': 'The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001144': {
    'name': 'st_turn_right_handed_type_one',
    'def': 'The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001145': {
    'name': 'st_turn_right_handed_type_two',
    'def': 'The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0001146': {
    'name': 'polypeptide_variation_site',
    'def': 'A site of sequence variation (alteration). Alternative sequence due to naturally occurring events such as polymorphisms and alternative splicing or experimental methods such as site directed mutagenesis. [EBIBS:GAR, SO:ke]'
  },
  'SO:0001147': {
    'name': 'natural_variant_site',
    'def': 'Describes the natural sequence variants due to polymorphisms, disease-associated mutations, RNA editing and variations between strains, isolates or cultivars. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001148': {
    'name': 'mutated_variant_site',
    'def': 'Site which has been experimentally altered. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001149': {
    'name': 'alternate_sequence_site',
    'def': 'Description of sequence variants produced by alternative splicing, alternative promoter usage, alternative initiation and ribosomal frameshifting. [EBIBS:GAR, UniProt:curation_manual]'
  },
  'SO:0001150': {
    'name': 'beta_turn_type_six',
    'def': 'A motif of four consecutive peptide resides of type VIa or type VIb and where the i+2 residue is cis-proline. [SO:cb]'
  },
  'SO:0001151': {
    'name': 'beta_turn_type_six_a',
    'def': 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -90 degrees, psi ~ 0 degrees. [PMID:2371257, SO:cb]'
  },
  'SO:0001152': {
    'name': 'beta_turn_type_six_a_one',
    'def': 'A type VIa beta turn with the following phi and psi sngles on amino acid residues 2 and 3: phi-2 = -60 degrees, psi-2 = 120 degrees, phi-3 = -90 degrees, psi-3 = 0 degrees. [PMID:27428516]'
  },
  'SO:0001153': {
    'name': 'beta_turn_type_six_a_two',
    'def': 'A type VIa beta turn with the following phi and psi sngles on amino acid residues 2 and 3: phi-2 = -120 degrees, psi-2 = 120 degrees, phi-3 = -60 degrees, psi-3 = 0 degrees. [PMID:27428516]'
  },
  'SO:0001154': {
    'name': 'beta_turn_type_six_b',
    'def': 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -120 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -60 degrees, psi ~ 0 degrees. [PMID:2371257, SO:cb]'
  },
  'SO:0001155': {
    'name': 'beta_turn_type_eight',
    'def': 'A motif of four consecutive peptide residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ -30 degrees. Residue(i+2): phi ~ -120 degrees, psi ~ 120 degrees. [PMID:2371257, SO:cb]'
  },
  'SO:0001156': {
    'name': 'DRE_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -10 and -60 relative to the TSS. Consensus sequence is WATCGATW. [PMID:12537576]'
  },
  'SO:0001157': {
    'name': 'DMv4_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements with respect to the TSS (+1). Consensus sequence is YGGTCACACTR. Marked spatial preference within core promoter; tend to occur near the TSS, although not as tightly as INR (SO:0000014). [PMID:16827941:12537576]'
  },
  'SO:0001158': {
    'name': 'E_box_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and +1 relative to the TSS. Consensus sequence is AWCAGCTGWT. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015). [PMID:12537576:16827941]'
  },
  'SO:0001159': {
    'name': 'DMv5_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -50 and -10 relative to the TSS. Consensus sequence is KTYRGTATWTTT. Tends to co-occur with DMv4 (SO:0001157) . Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162). [PMID:12537576:16827941]'
  },
  'SO:0001160': {
    'name': 'DMv3_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -30 and +15 relative to the TSS. Consensus sequence is KNNCAKCNCTRNY. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015) or MTE (0001162). [PMID:12537576:16827941]'
  },
  'SO:0001161': {
    'name': 'DMv2_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and -45 relative to the TSS. Consensus sequence is MKSYGGCARCGSYSS. Tends to co-occur with DMv3 (SO:0001160). Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162). [PMID:12537576:16827941]'
  },
  'SO:0001162': {
    'name': 'MTE',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters, usually located between +20 and +30 relative to the TSS. Consensus sequence is CSARCSSAACGS. Tends to co-occur with INR motif (SO:0000014). Tends to not occur with DPE motif (SO:0000015) or DMv5 (SO:0001159). [PMID:12537576:15231738, PMID:16858867]'
  },
  'SO:0001163': {
    'name': 'INR1_motif',
    'def': 'A promoter motif with consensus sequence TCATTCG. [PMID:16827941]'
  },
  'SO:0001164': {
    'name': 'DPE1_motif',
    'def': 'A promoter motif with consensus sequence CGGACGT. [PMID:16827941]'
  },
  'SO:0001165': {
    'name': 'DMv1_motif',
    'def': 'A promoter motif with consensus sequence CARCCCT. [PMID:16827941]'
  },
  'SO:0001166': {
    'name': 'GAGA_motif',
    'def': 'A non directional promoter motif with consensus sequence GAGAGCG. [PMID:16827941]'
  },
  'SO:0001167': {
    'name': 'NDM2_motif',
    'def': 'A non directional promoter motif with consensus CGMYGYCR. [PMID:16827941]'
  },
  'SO:0001168': {
    'name': 'NDM3_motif',
    'def': 'A non directional promoter motif with consensus sequence GAAAGCT. [PMID:16827941]'
  },
  'SO:0001169': {
    'name': 'ds_RNA_viral_sequence',
    'def': 'A ds_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded RNA. [SO:ke]'
  },
  'SO:0001170': {
    'name': 'polinton',
    'def': "A kind of DNA transposon that populates the genomes of protists, fungi, and animals, characterized by a unique set of proteins necessary for their transposition, including a protein-primed DNA polymerase B, retroviral integrase, cysteine protease, and ATPase. Polintons are characterized by 6-bp target site duplications, terminal-inverted repeats that are several hundred nucleotides long, and 5'-AG and TC-3' termini. Polintons exist as autonomous and nonautonomous elements. [PMID:16537396]"
  },
  'SO:0001171': {
    'name': 'rRNA_21S',
    'def': 'A component of the large ribosomal subunit in mitochondrial rRNA. [RSC:cb]'
  },
  'SO:0001172': {
    'name': 'tRNA_region',
    'def': 'A region of a tRNA. [RSC:cb]'
  },
  'SO:0001173': {
    'name': 'anticodon_loop',
    'def': "A sequence of seven nucleotide bases in tRNA which contains the anticodon. It has the sequence 5'-pyrimidine-purine-anticodon-modified purine-any base-3. [ISBN:0716719207]"
  },
  'SO:0001174': {
    'name': 'anticodon',
    'def': 'A sequence of three nucleotide bases in tRNA which recognizes a codon in mRNA. [RSC:cb]'
  },
  'SO:0001175': {
    'name': 'CCA_tail',
    'def': "Base sequence at the 3' end of a tRNA. The 3'-hydroxyl group on the terminal adenosine is the attachment point for the amino acid. [ISBN:0716719207]"
  },
  'SO:0001176': {
    'name': 'DHU_loop',
    'def': 'Non-base-paired sequence of nucleotide bases in tRNA. It contains several dihydrouracil residues. [ISBN:071671920]'
  },
  'SO:0001177': {
    'name': 'T_loop',
    'def': 'Non-base-paired sequence of three nucleotide bases in tRNA. It has sequence T-Psi-C. [ISBN:0716719207]'
  },
  'SO:0001178': {
    'name': 'pyrrolysine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding pyrrolysyl tRNA (SO:0000766). [RSC:cb]'
  },
  'SO:0001179': {
    'name': 'U3_snoRNA',
    'def': "U3 snoRNA is a member of the box C/D class of small nucleolar RNAs. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localization, stability and hypermethylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00012]"
  },
  'SO:0001180': {
    'name': 'AU_rich_element',
    'def': "A cis-acting element found in the 3' UTR of some mRNA which is rich in AUUUA pentamers. Messenger RNAs bearing multiple AU-rich elements are often unstable. [PMID:7892223]"
  },
  'SO:0001181': {
    'name': 'Bruno_response_element',
    'def': "A cis-acting element found in the 3' UTR of some mRNA which is bound by the Drosophila Bruno protein and its homologs. [PMID:10893231]"
  },
  'SO:0001182': {
    'name': 'iron_responsive_element',
    'def': "A regulatory sequence found in the 5' and 3' UTRs of many mRNAs which encode iron-binding proteins. It has a hairpin structure and is recognized by trans-acting proteins known as iron-regulatory proteins. [PMID:3198610, PMID:8710843]"
  },
  'SO:0001183': {
    'name': 'morpholino_backbone',
    'def': 'An attribute describing a sequence composed of nucleobases bound to a morpholino backbone. A morpholino backbone consists of morpholine (CHEBI:34856) rings connected by phosphorodiamidate linkages. [RSC:cb]'
  },
  'SO:0001184': {
    'name': 'PNA',
    'def': 'An attribute describing a sequence composed of peptide nucleic acid (CHEBI:48021), a chemical consisting of nucleobases bound to a backbone composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds. [RSC:cb]'
  },
  'SO:0001185': {
    'name': 'enzymatic',
    'def': 'An attribute describing the sequence of a transcript that has catalytic activity with or without an associated ribonucleoprotein. [RSC:cb]'
  },
  'SO:0001186': {
    'name': 'ribozymic',
    'def': 'An attribute describing the sequence of a transcript that has catalytic activity even without an associated ribonucleoprotein. [RSC:cb]'
  },
  'SO:0001187': {
    'name': 'pseudouridylation_guide_snoRNA',
    'def': 'A snoRNA that specifies the site of pseudouridylation in an RNA molecule by base pairing with a short sequence around the target residue. [GOC:mah, PMID:12457565]'
  },
  'SO:0001188': {
    'name': 'LNA',
    'def': "An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of 'locked' deoxyribose rings connected to a phosphate backbone. The deoxyribose unit's conformation is 'locked' by a 2'-C,4'-C-oxymethylene link. [CHEBI:48010]"
  },
  'SO:0001189': {
    'name': 'LNA_oligo',
    'def': 'An oligo composed of LNA residues. [RSC:cb]'
  },
  'SO:0001190': {
    'name': 'TNA',
    'def': 'An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of threose rings connected to a phosphate backbone. [CHEBI:48019]'
  },
  'SO:0001191': {
    'name': 'TNA_oligo',
    'def': 'An oligo composed of TNA residues. [RSC:cb]'
  },
  'SO:0001192': {
    'name': 'GNA',
    'def': 'An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of an acyclic three-carbon propylene glycol connected to a phosphate backbone. It has two enantiomeric forms, (R)-GNA and (S)-GNA. [CHEBI:48015]'
  },
  'SO:0001193': {
    'name': 'GNA_oligo',
    'def': 'An oligo composed of GNA residues. [RSC:cb]'
  },
  'SO:0001194': {
    'name': 'R_GNA',
    'def': 'An attribute describing a GNA sequence in the (R)-GNA enantiomer. [CHEBI:48016]'
  },
  'SO:0001195': {
    'name': 'R_GNA_oligo',
    'def': 'An oligo composed of (R)-GNA residues. [RSC:cb]'
  },
  'SO:0001196': {
    'name': 'S_GNA',
    'def': 'An attribute describing a GNA sequence in the (S)-GNA enantiomer. [CHEBI:48017]'
  },
  'SO:0001197': {
    'name': 'S_GNA_oligo',
    'def': 'An oligo composed of (S)-GNA residues. [RSC:cb]'
  },
  'SO:0001198': {
    'name': 'ds_DNA_viral_sequence',
    'def': 'A ds_DNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded DNA. [SO:ke]'
  },
  'SO:0001199': {
    'name': 'ss_RNA_viral_sequence',
    'def': 'A ss_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as single stranded RNA. [SO:ke]'
  },
  'SO:0001200': {
    'name': 'negative_sense_ssRNA_viral_sequence',
    'def': 'A negative_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that is complementary to mRNA and must be converted to positive sense RNA by RNA polymerase before translation. [SO:ke]'
  },
  'SO:0001201': {
    'name': 'positive_sense_ssRNA_viral_sequence',
    'def': 'A positive_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that can be immediately translated by the host. [SO:ke]'
  },
  'SO:0001202': {
    'name': 'ambisense_ssRNA_viral_sequence',
    'def': 'A ambisense_RNA_virus is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus with both messenger and anti messenger polarity. [SO:ke]'
  },
  'SO:0001203': {
    'name': 'RNA_polymerase_promoter',
    'def': 'A region (DNA) to which RNA polymerase binds, to begin transcription. [xenbase:jb]'
  },
  'SO:0001204': {
    'name': 'Phage_RNA_Polymerase_Promoter',
    'def': 'A region (DNA) to which Bacteriophage RNA polymerase binds, to begin transcription. [xenbase:jb]'
  },
  'SO:0001205': {
    'name': 'SP6_RNA_Polymerase_Promoter',
    'def': 'A region (DNA) to which the SP6 RNA polymerase binds, to begin transcription. [xenbase:jb]'
  },
  'SO:0001206': {
    'name': 'T3_RNA_Polymerase_Promoter',
    'def': 'A DNA sequence to which the T3 RNA polymerase binds, to begin transcription. [xenbase:jb]'
  },
  'SO:0001207': {
    'name': 'T7_RNA_Polymerase_Promoter',
    'def': 'A region (DNA) to which the T7 RNA polymerase binds, to begin transcription. [xenbase:jb]'
  },
  'SO:0001208': {
    'name': 'five_prime_EST',
    'def': "An EST read from the 5' end of a transcript that usually codes for a protein. These regions tend to be conserved across species and do not change much within a gene family. [http://www.ncbi.nlm.nih.gov/About/primer/est.html]"
  },
  'SO:0001209': {
    'name': 'three_prime_EST',
    'def': "An EST read from the 3' end of a transcript. They are more likely to fall within non-coding, or untranslated regions(UTRs). [http://www.ncbi.nlm.nih.gov/About/primer/est.html]"
  },
  'SO:0001210': {
    'name': 'translational_frameshift',
    'def': 'The region of mRNA (not divisible by 3 bases) that is skipped or added during the process of translational frameshifting (GO:0006452), causing the reading frame to be different. [http://www.insdc.org/files/feature_table.html, SO:ke]'
  },
  'SO:0001211': {
    'name': 'plus_1_translational_frameshift',
    'def': 'The region of mRNA 1 base long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different. [SO:ke]'
  },
  'SO:0001212': {
    'name': 'plus_2_translational_frameshift',
    'def': 'The region of mRNA 2 bases long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different. [SO:ke]'
  },
  'SO:0001213': {
    'name': 'group_III_intron',
    'def': 'Group III introns are introns found in the mRNA of the plastids of euglenoid protists. They are spliced by a two step transesterification with bulged adenosine as initiating nucleophile. [PMID:11377794]'
  },
  'SO:0001214': {
    'name': 'noncoding_region_of_exon',
    'def': 'The maximal intersection of exon and UTR. [SO:ke]'
  },
  'SO:0001215': {
    'name': 'coding_region_of_exon',
    'def': 'The region of an exon that encodes for protein sequence. [SO:ke]'
  },
  'SO:0001216': {
    'name': 'endonuclease_spliced_intron',
    'def': 'An intron that spliced via endonucleolytic cleavage and ligation rather than transesterification. [SO:ke]'
  },
  'SO:0001217': {
    'name': 'protein_coding_gene',
    'def': 'A gene that codes for an RNA that can be translated into a protein. []'
  },
  'SO:0001218': {
    'name': 'transgenic_insertion',
    'def': 'An insertion that derives from another organism, via the use of recombinant DNA technology. [SO:bm]'
  },
  'SO:0001219': {
    'name': 'retrogene',
    'def': 'A gene that has been produced as the product of a reverse transcriptase mediated event. []'
  },
  'SO:0001220': {
    'name': 'silenced_by_RNA_interference',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by RNA interference. [RSC:cb]'
  },
  'SO:0001221': {
    'name': 'silenced_by_histone_modification',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by histone modification. [RSC:cb]'
  },
  'SO:0001222': {
    'name': 'silenced_by_histone_methylation',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by histone methylation. [RSC:cb]'
  },
  'SO:0001223': {
    'name': 'silenced_by_histone_deacetylation',
    'def': 'An attribute describing an epigenetic process where a gene is inactivated by histone deacetylation. [RSC:cb]'
  },
  'SO:0001224': {
    'name': 'gene_silenced_by_RNA_interference',
    'def': 'A gene that is silenced by RNA interference. [SO:xp]'
  },
  'SO:0001225': {
    'name': 'gene_silenced_by_histone_modification',
    'def': 'A gene that is silenced by histone modification. [SO:xp]'
  },
  'SO:0001226': {
    'name': 'gene_silenced_by_histone_methylation',
    'def': 'A gene that is silenced by histone methylation. [SO:xp]'
  },
  'SO:0001227': {
    'name': 'gene_silenced_by_histone_deacetylation',
    'def': 'A gene that is silenced by histone deacetylation. [SO:xp]'
  },
  'SO:0001228': {
    'name': 'dihydrouridine',
    'def': 'A modified RNA base in which the 5,6-dihydrouracil is bound to the ribose ring. [RSC:cb]'
  },
  'SO:0001229': {
    'name': 'pseudouridine',
    'def': 'A modified RNA base in which the 5- position of the uracil is bound to the ribose ring instead of the 4- position. [RSC:cb]'
  },
  'SO:0001230': {
    'name': 'inosine',
    'def': 'A modified RNA base in which hypoxanthine is bound to the ribose ring. [http://library.med.utah.edu/RNAmods/, RSC:cb]'
  },
  'SO:0001231': {
    'name': 'seven_methylguanine',
    'def': 'A modified RNA base in which guanine is methylated at the 7- position. [RSC:cb]'
  },
  'SO:0001232': {
    'name': 'ribothymidine',
    'def': 'A modified RNA base in which thymine is bound to the ribose ring. [RSC:cb]'
  },
  'SO:0001233': {
    'name': 'methylinosine',
    'def': 'A modified RNA base in which methylhypoxanthine is bound to the ribose ring. [RSC:cb]'
  },
  'SO:0001234': {
    'name': 'mobile',
    'def': 'An attribute describing a feature that has either intra-genome or intracellular mobility. [RSC:cb]'
  },
  'SO:0001235': {
    'name': 'replicon',
    'def': 'A region containing at least one unique origin of replication and a unique termination site. [ISBN:0716719207]'
  },
  'SO:0001236': {
    'name': 'base',
    'def': 'A base is a sequence feature that corresponds to a single unit of a nucleotide polymer. [SO:ke]'
  },
  'SO:0001237': {
    'name': 'amino_acid',
    'def': 'A sequence feature that corresponds to a single amino acid residue in a polypeptide. [RSC:cb]'
  },
  'SO:0001238': {
    'name': 'major_TSS',
    'def': 'The tanscription start site that is most frequently used for transcription of a gene. []'
  },
  'SO:0001239': {
    'name': 'minor_TSS',
    'def': 'A tanscription start site that is not the most frequently used for transcription of a gene. []'
  },
  'SO:0001240': {
    'name': 'TSS_region',
    'def': "The region of a gene from the 5' most TSS to the 3' TSS. [BBOP:nw]"
  },
  'SO:0001241': {
    'name': 'encodes_alternate_transcription_start_sites',
    'def': 'A gene that has multiple possible transcription start sites. []'
  },
  'SO:0001243': {
    'name': 'miRNA_primary_transcript_region',
    'def': 'A part of an miRNA primary_transcript. [SO:ke]'
  },
  'SO:0001244': {
    'name': 'pre_miRNA',
    'def': 'The 60-70 nucleotide region remain after Drosha processing of the primary transcript, that folds back upon itself to form a hairpin structure. [SO:ke]'
  },
  'SO:0001245': {
    'name': 'miRNA_stem',
    'def': 'The stem of the hairpin loop formed by folding of the pre-miRNA. [SO:ke]'
  },
  'SO:0001246': {
    'name': 'miRNA_loop',
    'def': 'The loop of the hairpin loop formed by folding of the pre-miRNA. [SO:ke]'
  },
  'SO:0001247': {
    'name': 'synthetic_oligo',
    'def': 'An oligo composed of synthetic nucleotides. [SO:ke]'
  },
  'SO:0001248': {
    'name': 'assembly',
    'def': 'A region of the genome of known length that is composed by ordering and aligning two or more different regions. [SO:ke]'
  },
  'SO:0001249': {
    'name': 'fragment_assembly',
    'def': 'A fragment assembly is a genome assembly that orders overlapping fragments of the genome based on landmark sequences. The base pair distance between the landmarks is known allowing additivity of lengths. [SO:ke]'
  },
  'SO:0001250': {
    'name': 'fingerprint_map',
    'def': 'A fingerprint_map is a physical map composed of restriction fragments. [SO:ke]'
  },
  'SO:0001251': {
    'name': 'STS_map',
    'def': 'An STS map is a physical map organized by the unique STS landmarks. [SO:ke]'
  },
  'SO:0001252': {
    'name': 'RH_map',
    'def': 'A radiation hybrid map is a physical map. [SO:ke]'
  },
  'SO:0001253': {
    'name': 'sonicate_fragment',
    'def': 'A DNA fragment generated by sonication. Sonication is a technique used to sheer DNA into smaller fragments. [SO:ke]'
  },
  'SO:0001254': {
    'name': 'polyploid',
    'def': 'A kind of chromosome variation where the chromosome complement is an exact multiple of the haploid number and is greater than the diploid number. [SO:ke]'
  },
  'SO:0001255': {
    'name': 'autopolyploid',
    'def': 'A polyploid where the multiple chromosome set was derived from the same organism. [SO:ke]'
  },
  'SO:0001256': {
    'name': 'allopolyploid',
    'def': 'A polyploid where the multiple chromosome set was derived from a different organism. [SO:ke]'
  },
  'SO:0001257': {
    'name': 'homing_endonuclease_binding_site',
    'def': 'The binding site (recognition site) of a homing endonuclease. The binding site is typically large. [SO:ke]'
  },
  'SO:0001258': {
    'name': 'octamer_motif',
    'def': 'A sequence element characteristic of some RNA polymerase II promoters with sequence ATTGCAT that binds Pou-domain transcription factors. [GOC:dh, PMID:3095662]'
  },
  'SO:0001259': {
    'name': 'apicoplast_chromosome',
    'def': 'A chromosome originating in an apicoplast. [SO:xp]'
  },
  'SO:0001260': {
    'name': 'sequence_collection',
    'def': 'A collection of discontinuous sequences. [SO:ke]'
  },
  'SO:0001261': {
    'name': 'overlapping_feature_set',
    'def': 'A continuous region of sequence composed of the overlapping of multiple sequence_features, which ultimately provides evidence for another sequence_feature. [SO:ke]'
  },
  'SO:0001262': {
    'name': 'overlapping_EST_set',
    'def': "A continous experimental result region extending the length of multiple overlapping EST's. [SO:ke]"
  },
  'SO:0001263': {
    'name': 'ncRNA_gene',
    'def': 'A gene that encodes a non-coding RNA. []'
  },
  'SO:0001264': {
    'name': 'gRNA_gene',
    'def': 'A noncoding RNA that guides the insertion or deletion of uridine residues in mitochondrial mRNAs. This may also refer to synthetic RNAs used to guide DNA editing using the CRIPSR/Cas9 system. []'
  },
  'SO:0001265': {
    'name': 'miRNA_gene',
    'def': 'A small noncoding RNA of approximately 22 nucleotides in length which may be involved in regulation of gene expression. []'
  },
  'SO:0001266': {
    'name': 'scRNA_gene',
    'def': 'A gene encoding a small noncoding RNA that is generally found only in the cytoplasm. []'
  },
  'SO:0001267': {
    'name': 'snoRNA_gene',
    'def': 'A gene encoding a small noncoding RNA that participates in the processing or chemical modifications of many RNAs, including ribosomal RNAs and spliceosomal RNAs. []'
  },
  'SO:0001268': {
    'name': 'snRNA_gene',
    'def': 'A gene that encodes a small nuclear RNA. [http://en.wikipedia.org/wiki/Small_nuclear_RNA]'
  },
  'SO:0001269': {
    'name': 'SRP_RNA_gene',
    'def': 'A gene that encodes a signal recognition particle (SRP) RNA. []'
  },
  'SO:0001271': {
    'name': 'tmRNA_gene',
    'def': 'A bacterial RNA with both tRNA and mRNA like properties. []'
  },
  'SO:0001272': {
    'name': 'tRNA_gene',
    'def': 'A noncoding RNA that binds to a specific amino acid to allow that amino acid to be used by the ribosome during translation of RNA. []'
  },
  'SO:0001273': {
    'name': 'modified_adenosine',
    'def': 'A modified adenine is an adenine base feature that has been altered. [SO:ke]'
  },
  'SO:0001274': {
    'name': 'modified_inosine',
    'def': 'A modified inosine is an inosine base feature that has been altered. [SO:ke]'
  },
  'SO:0001275': {
    'name': 'modified_cytidine',
    'def': 'A modified cytidine is a cytidine base feature which has been altered. [SO:ke]'
  },
  'SO:0001276': {
    'name': 'modified_guanosine',
    'def': 'A guanosine base that has been modified. []'
  },
  'SO:0001277': {
    'name': 'modified_uridine',
    'def': 'A uridine base that has been modified. []'
  },
  'SO:0001278': {
    'name': 'one_methylinosine',
    'def': '1-methylinosine is a modified inosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001279': {
    'name': 'one_two_prime_O_dimethylinosine',
    'def': "1,2'-O-dimethylinosine is a modified inosine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001280': {
    'name': 'two_prime_O_methylinosine',
    'def': "2'-O-methylinosine is a modified inosine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001281': {
    'name': 'three_methylcytidine',
    'def': '3-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001282': {
    'name': 'five_methylcytidine',
    'def': '5-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001283': {
    'name': 'two_prime_O_methylcytidine',
    'def': "2'-O-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001284': {
    'name': 'two_thiocytidine',
    'def': '2-thiocytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001285': {
    'name': 'N4_acetylcytidine',
    'def': 'N4-acetylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001286': {
    'name': 'five_formylcytidine',
    'def': '5-formylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001287': {
    'name': 'five_two_prime_O_dimethylcytidine',
    'def': "5,2'-O-dimethylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001288': {
    'name': 'N4_acetyl_2_prime_O_methylcytidine',
    'def': "N4-acetyl-2'-O-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001289': {
    'name': 'lysidine',
    'def': 'Lysidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001290': {
    'name': 'N4_methylcytidine',
    'def': 'N4-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001291': {
    'name': 'N4_2_prime_O_dimethylcytidine',
    'def': "N4,2'-O-dimethylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001292': {
    'name': 'five_hydroxymethylcytidine',
    'def': '5-hydroxymethylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001293': {
    'name': 'five_formyl_two_prime_O_methylcytidine',
    'def': "5-formyl-2'-O-methylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001294': {
    'name': 'N4_N4_2_prime_O_trimethylcytidine',
    'def': 'N4_N4_2_prime_O_trimethylcytidine is a modified cytidine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001295': {
    'name': 'one_methyladenosine',
    'def': '1_methyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001296': {
    'name': 'two_methyladenosine',
    'def': '2_methyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001297': {
    'name': 'N6_methyladenosine',
    'def': 'N6_methyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001298': {
    'name': 'two_prime_O_methyladenosine',
    'def': '2prime_O_methyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001299': {
    'name': 'two_methylthio_N6_methyladenosine',
    'def': '2_methylthio_N6_methyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001300': {
    'name': 'N6_isopentenyladenosine',
    'def': 'N6_isopentenyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001301': {
    'name': 'two_methylthio_N6_isopentenyladenosine',
    'def': '2_methylthio_N6_isopentenyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001302': {
    'name': 'N6_cis_hydroxyisopentenyl_adenosine',
    'def': 'N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001303': {
    'name': 'two_methylthio_N6_cis_hydroxyisopentenyl_adenosine',
    'def': '2_methylthio_N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001304': {
    'name': 'N6_glycinylcarbamoyladenosine',
    'def': 'N6_glycinylcarbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001305': {
    'name': 'N6_threonylcarbamoyladenosine',
    'def': 'N6_threonylcarbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001306': {
    'name': 'two_methylthio_N6_threonyl_carbamoyladenosine',
    'def': '2_methylthio_N6_threonyl_carbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001307': {
    'name': 'N6_methyl_N6_threonylcarbamoyladenosine',
    'def': 'N6_methyl_N6_threonylcarbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001308': {
    'name': 'N6_hydroxynorvalylcarbamoyladenosine',
    'def': 'N6_hydroxynorvalylcarbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001309': {
    'name': 'two_methylthio_N6_hydroxynorvalyl_carbamoyladenosine',
    'def': '2_methylthio_N6_hydroxynorvalyl_carbamoyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001310': {
    'name': 'two_prime_O_ribosyladenosine_phosphate',
    'def': '2prime_O_ribosyladenosine_phosphate is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001311': {
    'name': 'N6_N6_dimethyladenosine',
    'def': 'N6_N6_dimethyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001312': {
    'name': 'N6_2_prime_O_dimethyladenosine',
    'def': 'N6_2prime_O_dimethyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001313': {
    'name': 'N6_N6_2_prime_O_trimethyladenosine',
    'def': 'N6_N6_2prime_O_trimethyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001314': {
    'name': 'one_two_prime_O_dimethyladenosine',
    'def': "1,2'-O-dimethyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]"
  },
  'SO:0001315': {
    'name': 'N6_acetyladenosine',
    'def': 'N6_acetyladenosine is a modified adenosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001316': {
    'name': 'seven_deazaguanosine',
    'def': '7-deazaguanosine is a modified guanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001317': {
    'name': 'queuosine',
    'def': 'Queuosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001318': {
    'name': 'epoxyqueuosine',
    'def': 'Epoxyqueuosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001319': {
    'name': 'galactosyl_queuosine',
    'def': 'Galactosyl_queuosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001320': {
    'name': 'mannosyl_queuosine',
    'def': 'Mannosyl_queuosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001321': {
    'name': 'seven_cyano_seven_deazaguanosine',
    'def': '7_cyano_7_deazaguanosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001322': {
    'name': 'seven_aminomethyl_seven_deazaguanosine',
    'def': '7_aminomethyl_7_deazaguanosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001323': {
    'name': 'archaeosine',
    'def': 'Archaeosine is a modified 7-deazoguanosine. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001324': {
    'name': 'one_methylguanosine',
    'def': '1_methylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001325': {
    'name': 'N2_methylguanosine',
    'def': 'N2_methylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001326': {
    'name': 'seven_methylguanosine',
    'def': '7_methylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001327': {
    'name': 'two_prime_O_methylguanosine',
    'def': '2prime_O_methylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001328': {
    'name': 'N2_N2_dimethylguanosine',
    'def': 'N2_N2_dimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001329': {
    'name': 'N2_2_prime_O_dimethylguanosine',
    'def': 'N2_2prime_O_dimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001330': {
    'name': 'N2_N2_2_prime_O_trimethylguanosine',
    'def': 'N2_N2_2prime_O_trimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001331': {
    'name': 'two_prime_O_ribosylguanosine_phosphate',
    'def': '2prime_O_ribosylguanosine_phosphate is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001332': {
    'name': 'wybutosine',
    'def': 'Wybutosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001333': {
    'name': 'peroxywybutosine',
    'def': 'Peroxywybutosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001334': {
    'name': 'hydroxywybutosine',
    'def': 'Hydroxywybutosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001335': {
    'name': 'undermodified_hydroxywybutosine',
    'def': 'Undermodified_hydroxywybutosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001336': {
    'name': 'wyosine',
    'def': 'Wyosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001337': {
    'name': 'methylwyosine',
    'def': 'Methylwyosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001338': {
    'name': 'N2_7_dimethylguanosine',
    'def': 'N2_7_dimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001339': {
    'name': 'N2_N2_7_trimethylguanosine',
    'def': 'N2_N2_7_trimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001340': {
    'name': 'one_two_prime_O_dimethylguanosine',
    'def': '1_2prime_O_dimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001341': {
    'name': 'four_demethylwyosine',
    'def': '4_demethylwyosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001342': {
    'name': 'isowyosine',
    'def': 'Isowyosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001343': {
    'name': 'N2_7_2prirme_O_trimethylguanosine',
    'def': 'N2_7_2prirme_O_trimethylguanosine is a modified guanosine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001344': {
    'name': 'five_methyluridine',
    'def': '5_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001345': {
    'name': 'two_prime_O_methyluridine',
    'def': '2prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001346': {
    'name': 'five_two_prime_O_dimethyluridine',
    'def': '5_2_prime_O_dimethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001347': {
    'name': 'one_methylpseudouridine',
    'def': '1_methylpseudouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001348': {
    'name': 'two_prime_O_methylpseudouridine',
    'def': '2prime_O_methylpseudouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001349': {
    'name': 'two_thiouridine',
    'def': '2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001350': {
    'name': 'four_thiouridine',
    'def': '4_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001351': {
    'name': 'five_methyl_2_thiouridine',
    'def': '5_methyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001352': {
    'name': 'two_thio_two_prime_O_methyluridine',
    'def': '2_thio_2prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001353': {
    'name': 'three_three_amino_three_carboxypropyl_uridine',
    'def': '3_3_amino_3_carboxypropyl_uridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001354': {
    'name': 'five_hydroxyuridine',
    'def': '5_hydroxyuridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001355': {
    'name': 'five_methoxyuridine',
    'def': '5_methoxyuridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001356': {
    'name': 'uridine_five_oxyacetic_acid',
    'def': 'Uridine_5_oxyacetic_acid is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001357': {
    'name': 'uridine_five_oxyacetic_acid_methyl_ester',
    'def': 'Uridine_5_oxyacetic_acid_methyl_ester is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001358': {
    'name': 'five_carboxyhydroxymethyl_uridine',
    'def': '5_carboxyhydroxymethyl_uridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001359': {
    'name': 'five_carboxyhydroxymethyl_uridine_methyl_ester',
    'def': '5_carboxyhydroxymethyl_uridine_methyl_ester is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001360': {
    'name': 'five_methoxycarbonylmethyluridine',
    'def': 'Five_methoxycarbonylmethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001361': {
    'name': 'five_methoxycarbonylmethyl_two_prime_O_methyluridine',
    'def': 'Five_methoxycarbonylmethyl_2_prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001362': {
    'name': 'five_methoxycarbonylmethyl_two_thiouridine',
    'def': '5_methoxycarbonylmethyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001363': {
    'name': 'five_aminomethyl_two_thiouridine',
    'def': '5_aminomethyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001364': {
    'name': 'five_methylaminomethyluridine',
    'def': '5_methylaminomethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001365': {
    'name': 'five_methylaminomethyl_two_thiouridine',
    'def': '5_methylaminomethyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001366': {
    'name': 'five_methylaminomethyl_two_selenouridine',
    'def': '5_methylaminomethyl_2_selenouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001367': {
    'name': 'five_carbamoylmethyluridine',
    'def': '5_carbamoylmethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001368': {
    'name': 'five_carbamoylmethyl_two_prime_O_methyluridine',
    'def': '5_carbamoylmethyl_2_prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001369': {
    'name': 'five_carboxymethylaminomethyluridine',
    'def': '5_carboxymethylaminomethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001370': {
    'name': 'five_carboxymethylaminomethyl_two_prime_O_methyluridine',
    'def': '5_carboxymethylaminomethyl_2_prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001371': {
    'name': 'five_carboxymethylaminomethyl_two_thiouridine',
    'def': '5_carboxymethylaminomethyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001372': {
    'name': 'three_methyluridine',
    'def': '3_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001373': {
    'name': 'one_methyl_three_three_amino_three_carboxypropyl_pseudouridine',
    'def': '1_methyl_3_3_amino_3_carboxypropyl_pseudouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001374': {
    'name': 'five_carboxymethyluridine',
    'def': '5_carboxymethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001375': {
    'name': 'three_two_prime_O_dimethyluridine',
    'def': '3_2prime_O_dimethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001376': {
    'name': 'five_methyldihydrouridine',
    'def': '5_methyldihydrouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001377': {
    'name': 'three_methylpseudouridine',
    'def': '3_methylpseudouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001378': {
    'name': 'five_taurinomethyluridine',
    'def': '5_taurinomethyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001379': {
    'name': 'five_taurinomethyl_two_thiouridine',
    'def': '5_taurinomethyl_2_thiouridineis a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001380': {
    'name': 'five_isopentenylaminomethyl_uridine',
    'def': '5_isopentenylaminomethyl_uridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001381': {
    'name': 'five_isopentenylaminomethyl_two_thiouridine',
    'def': '5_isopentenylaminomethyl_2_thiouridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001382': {
    'name': 'five_isopentenylaminomethyl_two_prime_O_methyluridine',
    'def': '5_isopentenylaminomethyl_2prime_O_methyluridine is a modified uridine base feature. [http://library.med.utah.edu/RNAmods/]'
  },
  'SO:0001383': {
    'name': 'histone_binding_site',
    'def': 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues of a histone. [SO:ke]'
  },
  'SO:0001384': {
    'name': 'CDS_fragment',
    'def': 'A portion of a CDS that is not the complete CDS. []'
  },
  'SO:0001385': {
    'name': 'modified_amino_acid_feature',
    'def': 'A post translationally modified amino acid feature. [SO:ke]'
  },
  'SO:0001386': {
    'name': 'modified_glycine',
    'def': 'A post translationally modified glycine amino acid feature. [SO:ke]'
  },
  'SO:0001387': {
    'name': 'modified_L_alanine',
    'def': 'A post translationally modified alanine amino acid feature. [SO:ke]'
  },
  'SO:0001388': {
    'name': 'modified_L_asparagine',
    'def': 'A post translationally modified asparagine amino acid feature. [SO:ke]'
  },
  'SO:0001389': {
    'name': 'modified_L_aspartic_acid',
    'def': 'A post translationally modified aspartic acid amino acid feature. [SO:ke]'
  },
  'SO:0001390': {
    'name': 'modified_L_cysteine',
    'def': 'A post translationally modified cysteine amino acid feature. [SO:ke]'
  },
  'SO:0001391': {
    'name': 'modified_L_glutamic_acid',
    'def': 'A post translationally modified glutamic acid. []'
  },
  'SO:0001392': {
    'name': 'modified_L_threonine',
    'def': 'A post translationally modified threonine amino acid feature. [SO:ke]'
  },
  'SO:0001393': {
    'name': 'modified_L_tryptophan',
    'def': 'A post translationally modified tryptophan amino acid feature. [SO:ke]'
  },
  'SO:0001394': {
    'name': 'modified_L_glutamine',
    'def': 'A post translationally modified glutamine amino acid feature. [SO:ke]'
  },
  'SO:0001395': {
    'name': 'modified_L_methionine',
    'def': 'A post translationally modified methionine amino acid feature. [SO:ke]'
  },
  'SO:0001396': {
    'name': 'modified_L_isoleucine',
    'def': 'A post translationally modified isoleucine amino acid feature. [SO:ke]'
  },
  'SO:0001397': {
    'name': 'modified_L_phenylalanine',
    'def': 'A post translationally modified phenylalanine amino acid feature. [SO:ke]'
  },
  'SO:0001398': {
    'name': 'modified_L_histidine',
    'def': 'A post translationally modified histidine amino acid feature. [SO:ke]'
  },
  'SO:0001399': {
    'name': 'modified_L_serine',
    'def': 'A post translationally modified serine amino acid feature. [SO:ke]'
  },
  'SO:0001400': {
    'name': 'modified_L_lysine',
    'def': 'A post translationally modified lysine amino acid feature. [SO:ke]'
  },
  'SO:0001401': {
    'name': 'modified_L_leucine',
    'def': 'A post translationally modified leucine amino acid feature. [SO:ke]'
  },
  'SO:0001402': {
    'name': 'modified_L_selenocysteine',
    'def': 'A post translationally modified selenocysteine amino acid feature. [SO:ke]'
  },
  'SO:0001403': {
    'name': 'modified_L_valine',
    'def': 'A post translationally modified valine amino acid feature. [SO:ke]'
  },
  'SO:0001404': {
    'name': 'modified_L_proline',
    'def': 'A post translationally modified proline amino acid feature. [SO:ke]'
  },
  'SO:0001405': {
    'name': 'modified_L_tyrosine',
    'def': 'A post translationally modified tyrosine amino acid feature. [SO:ke]'
  },
  'SO:0001406': {
    'name': 'modified_L_arginine',
    'def': 'A post translationally modified arginine amino acid feature. [SO:ke]'
  },
  'SO:0001407': {
    'name': 'peptidyl',
    'def': 'An attribute describing the nature of a proteinaceous polymer, where by the amino acid units are joined by peptide bonds. [SO:ke]'
  },
  'SO:0001408': {
    'name': 'cleaved_for_gpi_anchor_region',
    'def': 'The C-terminal residues of a polypeptide which are exchanged for a GPI-anchor. [EBI:rh]'
  },
  'SO:0001409': {
    'name': 'biomaterial_region',
    'def': 'A region which is intended for use in an experiment. [SO:cb]'
  },
  'SO:0001410': {
    'name': 'experimental_feature',
    'def': 'A region which is the result of some arbitrary experimental procedure. The procedure may be carried out with biological material or inside a computer. [SO:cb]'
  },
  'SO:0001411': {
    'name': 'biological_region',
    'def': 'A region defined by its disposition to be involved in a biological process. [SO:cb]'
  },
  'SO:0001412': {
    'name': 'topologically_defined_region',
    'def': 'A DNA region within which self-interaction occurs more often than expected by chance because of DNA-looping. [PMID:32782014, SO:cb]'
  },
  'SO:0001413': {
    'name': 'translocation_breakpoint',
    'def': 'The point within a chromosome where a translocation begins or ends. [SO:cb]'
  },
  'SO:0001414': {
    'name': 'insertion_breakpoint',
    'def': 'The point within a chromosome where a insertion begins or ends. [SO:cb]'
  },
  'SO:0001415': {
    'name': 'deletion_breakpoint',
    'def': 'The point within a chromosome where a deletion begins or ends. [SO:cb]'
  },
  'SO:0001416': {
    'name': 'five_prime_flanking_region',
    'def': 'A flanking region located five prime of a specific region. [SO:chado]'
  },
  'SO:0001417': {
    'name': 'three_prime_flanking_region',
    'def': 'A flanking region located three prime of a specific region. [SO:chado]'
  },
  'SO:0001418': {
    'name': 'transcribed_fragment',
    'def': 'An experimental region, defined by a tiling array experiment to be transcribed at some level. [SO:ke]'
  },
  'SO:0001419': {
    'name': 'cis_splice_site',
    'def': 'Intronic 2 bp region bordering exon. A splice_site that adjacent_to exon and overlaps intron. [SO:cjm, SO:ke]'
  },
  'SO:0001420': {
    'name': 'trans_splice_site',
    'def': 'Primary transcript region bordering trans-splice junction. [SO:ke]'
  },
  'SO:0001421': {
    'name': 'splice_junction',
    'def': 'The boundary between an intron and an exon. [SO:ke]'
  },
  'SO:0001422': {
    'name': 'conformational_switch',
    'def': 'A region of a polypeptide, involved in the transition from one conformational state to another. [SO:ke]'
  },
  'SO:0001423': {
    'name': 'dye_terminator_read',
    'def': 'A read produced by the dye terminator method of sequencing. [SO:ke]'
  },
  'SO:0001424': {
    'name': 'pyrosequenced_read',
    'def': 'A read produced by pyrosequencing technology. [SO:ke]'
  },
  'SO:0001425': {
    'name': 'ligation_based_read',
    'def': 'A read produced by ligation based sequencing technologies. [SO:ke]'
  },
  'SO:0001426': {
    'name': 'polymerase_synthesis_read',
    'def': 'A read produced by the polymerase based sequence by synthesis method. [SO:ke]'
  },
  'SO:0001427': {
    'name': 'cis_regulatory_frameshift_element',
    'def': 'A structural region in an RNA molecule which promotes ribosomal frameshifting of cis coding sequence. [RFAM:jd]'
  },
  'SO:0001428': {
    'name': 'expressed_sequence_assembly',
    'def': 'A sequence assembly derived from expressed sequences. [SO:ke]'
  },
  'SO:0001429': {
    'name': 'DNA_binding_site',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with DNA. [SO:ke]'
  },
  'SO:0001431': {
    'name': 'cryptic_gene',
    'def': 'A gene that is not transcribed under normal conditions and is not critical to normal cellular functioning. [SO:ke]'
  },
  'SO:0001432': {
    'name': 'sequence_variant_affecting_polyadenylation'
  },
  'SO:0001433': {
    'name': 'three_prime_RACE_clone',
    'def': "A three prime RACE (Rapid Amplification of cDNA Ends) clone is a cDNA clone copied from the 3' end of an mRNA (using a poly-dT primer to capture the polyA tail and a gene-specific or randomly primed 5' primer), and spliced into a vector for propagation in a suitable host. [modENCODE:nlw]"
  },
  'SO:0001434': {
    'name': 'cassette_pseudogene',
    'def': 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy. [SO:ke]'
  },
  'SO:0001435': {
    'name': 'alanine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons GCN (GCT, GCC, GCA and GCG). []'
  },
  'SO:0001436': {
    'name': 'valine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons GTN (GTT, GTC, GTA and GTG). []'
  },
  'SO:0001437': {
    'name': 'leucine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons CTN (CTT, CTC, CTA and CTG), TTA and TTG. []'
  },
  'SO:0001438': {
    'name': 'isoleucine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons ATH (ATT, ATC and ATA). []'
  },
  'SO:0001439': {
    'name': 'proline',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons CCN (CCT, CCC, CCA and CCG). []'
  },
  'SO:0001440': {
    'name': 'tryptophan',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codon TGG. []'
  },
  'SO:0001441': {
    'name': 'phenylalanine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codons TTT and TTC. []'
  },
  'SO:0001442': {
    'name': 'methionine',
    'def': 'A non-polar, hydorophobic amino acid encoded by the codon ATG. []'
  },
  'SO:0001443': {
    'name': 'glycine',
    'def': 'A non-polar, hydorophilic amino acid encoded by the codons GGN (GGT, GGC, GGA and GGG). []'
  },
  'SO:0001444': {
    'name': 'serine',
    'def': 'A polar, hydorophilic amino acid encoded by the codons TCN (TCT, TCC, TCA, TCG), AGT and AGC. []'
  },
  'SO:0001445': {
    'name': 'threonine',
    'def': 'A polar, hydorophilic amino acid encoded by the codons ACN (ACT, ACC, ACA and ACG). []'
  },
  'SO:0001446': {
    'name': 'tyrosine',
    'def': 'A polar, hydorophilic amino acid encoded by the codons TAT and TAC. []'
  },
  'SO:0001447': {
    'name': 'cysteine',
    'def': 'A polar amino acid encoded by the codons TGT and TGC. []'
  },
  'SO:0001448': {
    'name': 'glutamine',
    'def': 'A polar, hydorophilic amino acid encoded by the codons CAA and CAG. []'
  },
  'SO:0001449': {
    'name': 'asparagine',
    'def': 'A polar, hydorophilic amino acid encoded by the codons AAT and AAC. []'
  },
  'SO:0001450': {
    'name': 'lysine',
    'def': 'A positively charged, hydorophilic amino acid encoded by the codons AAA and AAG. []'
  },
  'SO:0001451': {
    'name': 'arginine',
    'def': 'A positively charged, hydorophilic amino acid encoded by the codons CGN (CGT, CGC, CGA and CGG), AGA and AGG. []'
  },
  'SO:0001452': {
    'name': 'histidine',
    'def': 'A positively charged, hydorophilic amino acid encoded by the codons CAT and CAC. []'
  },
  'SO:0001453': {
    'name': 'aspartic_acid',
    'def': 'A negatively charged, hydorophilic amino acid encoded by the codons GAT and GAC. []'
  },
  'SO:0001454': {
    'name': 'glutamic_acid',
    'def': 'A negatively charged, hydorophilic amino acid encoded by the codons GAA and GAG. []'
  },
  'SO:0001455': {
    'name': 'selenocysteine',
    'def': 'A relatively rare amino acid encoded by the codon UGA in some contexts, whereas UGA is a termination codon in other contexts. [PMID:23275319]'
  },
  'SO:0001456': {
    'name': 'pyrrolysine',
    'def': 'A relatively rare amino acid encoded by the codon UAG in some contexts, whereas UAG is a termination codon in other contexts. [PMID:15788401]'
  },
  'SO:0001457': {
    'name': 'transcribed_cluster',
    'def': 'A region defined by a set of transcribed sequences from the same gene or expressed pseudogene. [SO:ke]'
  },
  'SO:0001458': {
    'name': 'unigene_cluster',
    'def': 'A kind of transcribed_cluster defined by a set of transcribed sequences from the a unique gene. [SO:ke]'
  },
  'SO:0001459': {
    'name': 'CRISPR',
    'def': 'Clustered Palindromic Repeats interspersed with bacteriophage derived spacer sequences. [RFAM:jd]'
  },
  'SO:0001460': {
    'name': 'insulator_binding_site',
    'def': 'A binding site that, in an insulator region of a nucleotide molecule, interacts selectively and non-covalently with polypeptide residues. [SO:ke]'
  },
  'SO:0001461': {
    'name': 'enhancer_binding_site',
    'def': 'A binding site that, in the enhancer region of a nucleotide molecule, interacts selectively and non-covalently with polypeptide residues. [SO:ke]'
  },
  'SO:0001462': {
    'name': 'contig_collection',
    'def': 'A collection of contigs. [SO:ke]'
  },
  'SO:0001463': {
    'name': 'lincRNA',
    'def': 'Long, intervening non-coding RNA. A transcript that does not overlap within the start or end genomic coordinates of a coding gene or pseudogene on either strand. [http://www.gencodegenes.org/gencode_biotypes.html, PMID:19182780, PMID:23463798, SO:ke]'
  },
  'SO:0001464': {
    'name': 'UST',
    'def': 'An EST spanning part or all of the untranslated regions of a protein-coding transcript. [SO:nlw]'
  },
  'SO:0001465': {
    'name': 'three_prime_UST',
    'def': "A UST located in the 3'UTR of a protein-coding transcript. [SO:nlw]"
  },
  'SO:0001466': {
    'name': 'five_prime_UST',
    'def': "An UST located in the 5'UTR of a protein-coding transcript. [SO:nlw]"
  },
  'SO:0001467': {
    'name': 'RST',
    'def': 'A tag produced from a single sequencing read from a RACE product; typically a few hundred base pairs long. [SO:nlw]'
  },
  'SO:0001468': {
    'name': 'three_prime_RST',
    'def': "A tag produced from a single sequencing read from a 3'-RACE product; typically a few hundred base pairs long. [SO:nlw]"
  },
  'SO:0001469': {
    'name': 'five_prime_RST',
    'def': "A tag produced from a single sequencing read from a 5'-RACE product; typically a few hundred base pairs long. [SO:nlw]"
  },
  'SO:0001470': {
    'name': 'UST_match',
    'def': 'A match against an UST sequence. [SO:nlw]'
  },
  'SO:0001471': {
    'name': 'RST_match',
    'def': 'A match against an RST sequence. [SO:nlw]'
  },
  'SO:0001472': {
    'name': 'primer_match',
    'def': 'A nucleotide match to a primer sequence. [SO:nlw]'
  },
  'SO:0001473': {
    'name': 'miRNA_antiguide',
    'def': 'A region of the pri miRNA that base pairs with the guide to form the hairpin. [SO:ke]'
  },
  'SO:0001474': {
    'name': 'trans_splice_junction',
    'def': 'The boundary between the spliced leader and the first exon of the mRNA. [SO:ke]'
  },
  'SO:0001475': {
    'name': 'outron',
    'def': 'A region of a primary transcript, that is removed via trans splicing. [PMID:16401417, SO:ke]'
  },
  'SO:0001476': {
    'name': 'natural_plasmid',
    'def': 'A plasmid that occurs naturally. [SO:xp]'
  },
  'SO:0001477': {
    'name': 'gene_trap_construct',
    'def': 'A gene trap construct is a type of engineered plasmid which is designed to integrate into a genome and produce a fusion transcript between exons of the gene into which it inserts and a reporter element in the construct. Gene traps contain a splice acceptor, do not contain promoter elements for the reporter, and are mutagenic. Gene traps may be bicistronic with the second cassette containing a promoter driving an a selectable marker. [ZFIN:dh]'
  },
  'SO:0001478': {
    'name': 'promoter_trap_construct',
    'def': 'A promoter trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when inserted in close proximity to a promoter element. Promoter traps typically do not contain promoter elements and are mutagenic. [ZFIN:dh]'
  },
  'SO:0001479': {
    'name': 'enhancer_trap_construct',
    'def': 'An enhancer trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when the expression from a basic minimal promoter is enhanced by genomic enhancer elements. Enhancer traps contain promoter elements and are not usually mutagenic. [ZFIN:dh]'
  },
  'SO:0001480': {
    'name': 'PAC_end',
    'def': 'A region of sequence from the end of a PAC clone that may provide a highly specific marker. [ZFIN:mh]'
  },
  'SO:0001481': {
    'name': 'RAPD',
    'def': "RAPD is a 'PCR product' where a sequence variant is identified through the use of PCR with random primers. [ZFIN:mh]"
  },
  'SO:0001482': {
    'name': 'shadow_enhancer',
    'def': 'An enhancer that drives the pattern of transcription and binds to the same TF as the primary enhancer, but is located in the intron of or on the far side of a neighboring gene. [PMID:22083793]'
  },
  'SO:0001483': {
    'name': 'SNV',
    'def': 'SNVs are single nucleotide positions in genomic DNA at which different sequence alternatives exist. [SO:bm]'
  },
  'SO:0001484': {
    'name': 'X_element_combinatorial_repeat',
    'def': "An X element combinatorial repeat is a repeat region located between the X element and the telomere or adjacent Y' element. [http://www.yeastgenome.org/help/glossary.html]"
  },
  'SO:0001485': {
    'name': 'Y_prime_element',
    'def': "A Y' element is a repeat region (SO:0000657) located adjacent to telomeric repeats or X element combinatorial repeats, either as a single copy or tandem repeat of two to four copies. [http:http://www.yeastgenome.org/help/glossary.html]"
  },
  'SO:0001486': {
    'name': 'standard_draft',
    'def': 'The status of a whole genome sequence, where the data is minimally filtered or un-filtered, from any number of sequencing platforms, and is assembled into contigs. Genome sequence of this quality may harbour regions of poor quality and can be relatively incomplete. [DOI:10.1126]'
  },
  'SO:0001487': {
    'name': 'high_quality_draft',
    'def': 'The status of a whole genome sequence, where overall coverage represents at least 90 percent of the genome. [DOI:10.1126]'
  },
  'SO:0001488': {
    'name': 'improved_high_quality_draft',
    'def': 'The status of a whole genome sequence, where additional work has been performed, using either manual or automated methods, such as gap resolution. [DOI:10.1126]'
  },
  'SO:0001489': {
    'name': 'annotation_directed_improved_draft',
    'def': 'The status of a whole genome sequence,where annotation, and verification of coding regions has occurred. [DOI:10.1126]'
  },
  'SO:0001490': {
    'name': 'noncontiguous_finished',
    'def': 'The status of a whole genome sequence, where the assembly is high quality, closure approaches have been successful for most gaps, misassemblies and low quality regions. [DOI:10.1126]'
  },
  'SO:0001491': {
    'name': 'finished_genome',
    'def': 'The status of a whole genome sequence, with less than 1 error per 100,000 base pairs. [DOI:10.1126]'
  },
  'SO:0001492': {
    'name': 'intronic_regulatory_region',
    'def': 'A regulatory region that is part of an intron. [SO:ke]'
  },
  'SO:0001493': {
    'name': 'centromere_DNA_Element_I',
    'def': 'A centromere DNA Element I (CDEI) is a conserved region, part of the centromere, consisting of a consensus region composed of 8-11bp which enables binding by the centromere binding factor 1(Cbf1p). [PMID:11222754]'
  },
  'SO:0001494': {
    'name': 'centromere_DNA_Element_II',
    'def': 'A centromere DNA Element II (CDEII) is part a conserved region of the centromere, consisting of a consensus region that is AT-rich and ~ 75-100 bp in length. [PMID:11222754]'
  },
  'SO:0001495': {
    'name': 'centromere_DNA_Element_III',
    'def': 'A centromere DNA Element I (CDEI) is a conserved region, part of the centromere, consisting of a consensus region that consists of a 25-bp which enables binding by the centromere DNA binding factor 3 (CBF3) complex. [PMID:11222754]'
  },
  'SO:0001496': {
    'name': 'telomeric_repeat',
    'def': 'The telomeric repeat is a repeat region, part of the chromosome, which in yeast, is a G-rich terminal sequence of the form (TG(1-3))n or more precisely ((TG)(1-6)TG(2-3))n. [PMID:8720065]'
  },
  'SO:0001497': {
    'name': 'X_element',
    'def': 'The X element is a conserved region, of the telomere, of ~475 bp that contains an ARS sequence and in most cases an Abf1p binding site. [http://www.yeastgenome.org/help/glossary.html#xelemcoresequence, PMID:7785338, PMID:8005434]'
  },
  'SO:0001498': {
    'name': 'YAC_end',
    'def': 'A region of sequence from the end of a YAC clone that may provide a highly specific marker. [SO:ke]'
  },
  'SO:0001499': {
    'name': 'whole_genome_sequence_status',
    'def': 'The status of whole genome sequence. [DOI:10.1126]'
  },
  'SO:0001500': {
    'name': 'heritable_phenotypic_marker',
    'def': 'A biological_region characterized as a single heritable trait in a phenotype screen. The heritable phenotype may be mapped to a chromosome but generally has not been characterized to a specific gene locus. [JAX:hdene]'
  },
  'SO:0001501': {
    'name': 'peptide_collection',
    'def': 'A collection of peptide sequences. [BBOP:nlw]'
  },
  'SO:0001502': {
    'name': 'high_identity_region',
    'def': 'An experimental feature with high sequence identity to another sequence. [SO:ke]'
  },
  'SO:0001503': {
    'name': 'processed_transcript',
    'def': 'A transcript for which no open reading frame has been identified and for which no other function has been determined. [MGI:hdeen]'
  },
  'SO:0001504': {
    'name': 'assortment_derived_variation',
    'def': 'A chromosome variation derived from an event during meiosis. [SO:ke]'
  },
  'SO:0001505': {
    'name': 'reference_genome',
    'def': 'A collection of sequences (often chromosomes) taken as the standard for a given organism and genome assembly. [SO:ke]'
  },
  'SO:0001506': {
    'name': 'variant_genome',
    'def': 'A collection of sequences (often chromosomes) of an individual. [SO:ke]'
  },
  'SO:0001507': {
    'name': 'variant_collection',
    'def': 'A collection of one or more sequences of an individual. [SO:ke]'
  },
  'SO:0001508': {
    'name': 'alteration_attribute',
    'def': 'An attribute of alteration of one or more chromosomes. []'
  },
  'SO:0001509': {
    'name': 'chromosomal_variation_attribute',
    'def': 'An attribute of a change in the structure or number of a chromosomes. []'
  },
  'SO:0001510': {
    'name': 'intrachromosomal',
    'def': 'A change in chromosomes that occurs between two separate chromosomes. []'
  },
  'SO:0001511': {
    'name': 'interchromosomal',
    'def': 'A change in chromosomes that occurs between two sections of the same chromosome or between homologous chromosomes. []'
  },
  'SO:0001512': {
    'name': 'insertion_attribute',
    'def': 'A quality of a chromosomal insertion,. [SO:ke]'
  },
  'SO:0001513': {
    'name': 'tandem',
    'def': 'An insertion of extension of a tandem repeat. []'
  },
  'SO:0001514': {
    'name': 'direct',
    'def': 'A quality of an insertion where the insert is not in a cytologically inverted orientation. [SO:ke]'
  },
  'SO:0001515': {
    'name': 'inverted',
    'def': 'A quality of an insertion where the insert is in a cytologically inverted orientation. [SO:ke]'
  },
  'SO:0001516': {
    'name': 'free',
    'def': 'The quality of a duplication where the new region exists independently of the original. [SO:ke]'
  },
  'SO:0001517': {
    'name': 'inversion_attribute',
    'def': 'When a region of a chromosome is changed to the reverse order without duplication or deletion. []'
  },
  'SO:0001518': {
    'name': 'pericentric',
    'def': 'An inversion event that includes the centromere. []'
  },
  'SO:0001519': {
    'name': 'paracentric',
    'def': 'An inversion event that does not include the centromere. []'
  },
  'SO:0001520': {
    'name': 'translocaton_attribute',
    'def': 'An attribute of a translocation, which is then a region of nucleotide sequence that has translocated to a new position. The observed adjacency of two previously separated regions. []'
  },
  'SO:0001521': {
    'name': 'reciprocal',
    'def': 'When translocation occurs between nonhomologous chromosomes and involved an equal exchange of genetic materials. []'
  },
  'SO:0001522': {
    'name': 'insertional',
    'def': 'When a translocation is simply moving genetic material from one chromosome to another. []'
  },
  'SO:0001523': {
    'name': 'duplication_attribute',
    'def': 'An attribute of a duplication, which is an insertion which derives from, or is identical in sequence to, nucleotides present at a known location in the genome. []'
  },
  'SO:0001524': {
    'name': 'chromosomally_aberrant_genome',
    'def': 'When a genome contains an abnormal amount of chromosomes. []'
  },
  'SO:0001525': {
    'name': 'assembly_error_correction',
    'def': 'A region of sequence where the final nucleotide assignment differs from the original assembly due to an improvement that replaces a mistake. [SO:ke]'
  },
  'SO:0001526': {
    'name': 'base_call_error_correction',
    'def': 'A region of sequence where the final nucleotide assignment is different from that given by the base caller due to an improvement that replaces a mistake. [SO:ke]'
  },
  'SO:0001527': {
    'name': 'peptide_localization_signal',
    'def': 'A region of peptide sequence used to target the polypeptide molecule to a specific organelle. [SO:ke]'
  },
  'SO:0001528': {
    'name': 'nuclear_localization_signal',
    'def': 'A polypeptide region that targets a polypeptide to the nucleus. [SO:ke]'
  },
  'SO:0001529': {
    'name': 'endosomal_localization_signal',
    'def': 'A polypeptide region that targets a polypeptide to the endosome. [SO:ke]'
  },
  'SO:0001530': {
    'name': 'lysosomal_localization_signal',
    'def': 'A polypeptide region that targets a polypeptide to the lysosome. [SO:ke]'
  },
  'SO:0001531': {
    'name': 'nuclear_export_signal',
    'def': 'A polypeptide region that targets a polypeptide to he cytoplasm. [SO:ke]'
  },
  'SO:0001532': {
    'name': 'recombination_signal_sequence',
    'def': 'A region recognized by a recombinase. [SO:ke]'
  },
  'SO:0001533': {
    'name': 'cryptic_splice_site',
    'def': 'A splice site that is in part of the transcript not normally spliced. They occur via mutation or transcriptional error. [SO:ke]'
  },
  'SO:0001534': {
    'name': 'nuclear_rim_localization_signal',
    'def': 'A polypeptide region that targets a polypeptide to the nuclear rim. [SO:ke]'
  },
  'SO:0001535': {
    'name': 'P_TIR_transposon',
    'def': 'A P-element is a DNA transposon responsible for hybrid dysgenesis. P elements in this terminal inverted repeat (TIR) transposon superfamily have 31 bp perfect TIR and upon insertion duplicate an 8 bp sequence. It contains transposase that may lack the DDE domain. [PMID:6309410, SO:ke]'
  },
  'SO:0001536': {
    'name': 'functional_effect_variant',
    'def': 'A variant whereby the effect is evaluated with respect to a reference. [SO:ke]'
  },
  'SO:0001537': {
    'name': 'structural_variant',
    'def': 'A sequence variant that changes one or more structural features. [SO:ke]'
  },
  'SO:0001538': {
    'name': 'transcript_function_variant',
    'def': 'A sequence variant which alters the functioning of a transcript with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001539': {
    'name': 'translational_product_function_variant',
    'def': 'A sequence variant that affects the functioning of a translational product with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001540': {
    'name': 'level_of_transcript_variant',
    'def': 'A sequence variant which alters the level of a transcript. [SO:ke]'
  },
  'SO:0001541': {
    'name': 'decreased_transcript_level_variant',
    'def': 'A sequence variant that decreases the level of mature, spliced and processed RNA with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001542': {
    'name': 'increased_transcript_level_variant',
    'def': 'A sequence variant that increases the level of mature, spliced and processed RNA with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001543': {
    'name': 'transcript_processing_variant',
    'def': 'A sequence variant that affects the post transcriptional processing of a transcript with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001544': {
    'name': 'editing_variant',
    'def': 'A transcript processing variant whereby the process of editing is disrupted with respect to the reference. [SO:ke]'
  },
  'SO:0001545': {
    'name': 'polyadenylation_variant',
    'def': 'A sequence variant that changes polyadenylation with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001546': {
    'name': 'transcript_stability_variant',
    'def': 'A variant that changes the stability of a transcript with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001547': {
    'name': 'decreased_transcript_stability_variant',
    'def': 'A sequence variant that decreases transcript stability with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001548': {
    'name': 'increased_transcript_stability_variant',
    'def': 'A sequence variant that increases transcript stability with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001549': {
    'name': 'transcription_variant',
    'def': 'A variant that changes alters the transcription of a transcript with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001550': {
    'name': 'rate_of_transcription_variant',
    'def': 'A sequence variant that changes the rate of transcription with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001551': {
    'name': 'increased_transcription_rate_variant',
    'def': 'A sequence variant that increases the rate of transcription with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001552': {
    'name': 'decreased_transcription_rate_variant',
    'def': 'A sequence variant that decreases the rate of transcription with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001553': {
    'name': 'translational_product_level_variant',
    'def': 'A functional variant that changes the translational product level with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001554': {
    'name': 'polypeptide_function_variant',
    'def': 'A sequence variant which changes polypeptide functioning with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001555': {
    'name': 'decreased_translational_product_level',
    'def': 'A sequence variant which decreases the translational product level with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001556': {
    'name': 'increased_translational_product_level',
    'def': 'A sequence variant which increases the translational product level with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001557': {
    'name': 'polypeptide_gain_of_function_variant',
    'def': 'A sequence variant which causes gain of polypeptide function with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001558': {
    'name': 'polypeptide_localization_variant',
    'def': 'A sequence variant which changes the localization of a polypeptide with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001559': {
    'name': 'polypeptide_loss_of_function_variant',
    'def': 'A sequence variant that causes the loss of a polypeptide function with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001560': {
    'name': 'inactive_ligand_binding_site',
    'def': 'A sequence variant that causes the inactivation of a ligand binding site with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001561': {
    'name': 'polypeptide_partial_loss_of_function',
    'def': 'A sequence variant that causes some but not all loss of polypeptide function with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001562': {
    'name': 'polypeptide_post_translational_processing_variant',
    'def': 'A sequence variant that causes a change in post translational processing of the peptide with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001563': {
    'name': 'copy_number_change',
    'def': 'A sequence variant where copies of a feature (CNV) are either increased or decreased. [SO:ke]'
  },
  'SO:0001564': {
    'name': 'gene_variant',
    'def': 'A sequence variant where the structure of the gene is changed. [SO:ke]'
  },
  'SO:0001565': {
    'name': 'gene_fusion',
    'def': 'A sequence variant whereby a two genes have become joined. [SO:ke]'
  },
  'SO:0001566': {
    'name': 'regulatory_region_variant',
    'def': 'A sequence variant located within a regulatory region. [SO:ke]'
  },
  'SO:0001567': {
    'name': 'stop_retained_variant',
    'def': 'A sequence variant where at least one base in the terminator codon is changed, but the terminator remains. [SO:ke]'
  },
  'SO:0001568': {
    'name': 'splicing_variant',
    'def': 'A sequence variant that changes the process of splicing. [SO:ke]'
  },
  'SO:0001569': {
    'name': 'cryptic_splice_site_variant',
    'def': 'A sequence variant causing a new (functional) splice site. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:0001570': {
    'name': 'cryptic_splice_acceptor',
    'def': 'A sequence variant whereby a new splice site is created due to the activation of a new acceptor. [SO:ke]'
  },
  'SO:0001571': {
    'name': 'cryptic_splice_donor',
    'def': 'A sequence variant whereby a new splice site is created due to the activation of a new donor. [SO:ke]'
  },
  'SO:0001572': {
    'name': 'exon_loss_variant',
    'def': 'A sequence variant whereby an exon is lost from the transcript. [SO:ke]'
  },
  'SO:0001573': {
    'name': 'intron_gain_variant',
    'def': 'A sequence variant whereby an intron is gained by the processed transcript; usually a result of an alteration of the donor or acceptor. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:0001574': {
    'name': 'splice_acceptor_variant',
    'def': "A splice variant that changes the 2 base region at the 3' end of an intron. [SO:ke]"
  },
  'SO:0001575': {
    'name': 'splice_donor_variant',
    'def': "A splice variant that changes the 2 base pair region at the 5' end of an intron. [SO:ke]"
  },
  'SO:0001576': {
    'name': 'transcript_variant',
    'def': 'A sequence variant that changes the structure of the transcript. [SO:ke]'
  },
  'SO:0001577': {
    'name': 'complex_transcript_variant',
    'def': 'A transcript variant with a complex INDEL- Insertion or deletion that spans an exon/intron border or a coding sequence/UTR border. [http://ensembl.org/info/docs/variation/index.html]'
  },
  'SO:0001578': {
    'name': 'stop_lost',
    'def': 'A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript. [SO:ke]'
  },
  'SO:0001579': {
    'name': 'transcript_sequence_variant'
  },
  'SO:0001580': {
    'name': 'coding_sequence_variant',
    'def': 'A sequence variant that changes the coding sequence. [SO:ke]'
  },
  'SO:0001582': {
    'name': 'initiator_codon_variant',
    'def': 'A codon variant that changes at least one base of the first codon of a transcript. [SO:ke]'
  },
  'SO:0001583': {
    'name': 'missense_variant',
    'def': 'A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved. [EBI:fc, EBI:gr, SO:ke]'
  },
  'SO:0001585': {
    'name': 'conservative_missense_variant',
    'def': 'A sequence variant whereby at least one base of a codon is changed resulting in a codon that encodes for a different but similar amino acid. These variants may or may not be deleterious. [SO:ke]'
  },
  'SO:0001586': {
    'name': 'non_conservative_missense_variant',
    'def': 'A sequence variant whereby at least one base of a codon is changed resulting in a codon that encodes for an amino acid with different biochemical properties. [SO:ke]'
  },
  'SO:0001587': {
    'name': 'stop_gained',
    'def': 'A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened polypeptide. [SO:ke]'
  },
  'SO:0001589': {
    'name': 'frameshift_variant',
    'def': 'A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three. [SO:ke]'
  },
  'SO:0001590': {
    'name': 'terminator_codon_variant',
    'def': 'A sequence variant whereby at least one of the bases in the terminator codon is changed. [SO:ke]'
  },
  'SO:0001591': {
    'name': 'frame_restoring_variant',
    'def': 'A sequence variant that reverts the sequence of a previous frameshift mutation back to the initial frame. [SO:ke]'
  },
  'SO:0001592': {
    'name': 'minus_1_frameshift_variant',
    'def': 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base ahead. [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]'
  },
  'SO:0001593': {
    'name': 'minus_2_frameshift_variant',
    'def': 'A sequence variant which causes a disruption of the translational reading frame, by shifting two bases forward. []'
  },
  'SO:0001594': {
    'name': 'plus_1_frameshift_variant',
    'def': 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base backward. [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]'
  },
  'SO:0001595': {
    'name': 'plus_2_frameshift_variant',
    'def': 'A sequence variant which causes a disruption of the translational reading frame, by shifting two bases backward. []'
  },
  'SO:0001596': {
    'name': 'transcript_secondary_structure_variant',
    'def': 'A sequence variant within a transcript that changes the secondary structure of the RNA product. [SO:ke]'
  },
  'SO:0001597': {
    'name': 'compensatory_transcript_secondary_structure_variant',
    'def': 'A secondary structure variant that compensate for the change made by a previous variant. [SO:ke]'
  },
  'SO:0001598': {
    'name': 'translational_product_structure_variant',
    'def': 'A sequence variant within the transcript that changes the structure of the translational product. [SO:ke]'
  },
  'SO:0001599': {
    'name': '3D_polypeptide_structure_variant',
    'def': 'A sequence variant that changes the resulting polypeptide structure. [SO:ke]'
  },
  'SO:0001600': {
    'name': 'complex_3D_structural_variant',
    'def': 'A sequence variant that changes the resulting polypeptide structure. [SO:ke]'
  },
  'SO:0001601': {
    'name': 'conformational_change_variant',
    'def': 'A sequence variant in the CDS region that causes a conformational change in the resulting polypeptide sequence. [SO:ke]'
  },
  'SO:0001602': {
    'name': 'complex_change_of_translational_product_variant',
    'def': 'A variant that changes the translational product with respect to the reference. []'
  },
  'SO:0001603': {
    'name': 'polypeptide_sequence_variant',
    'def': 'A sequence variant with in the CDS that causes a change in the resulting polypeptide sequence. [SO:ke]'
  },
  'SO:0001604': {
    'name': 'amino_acid_deletion',
    'def': 'A sequence variant within a CDS resulting in the loss of an amino acid from the resulting polypeptide. [SO:ke]'
  },
  'SO:0001605': {
    'name': 'amino_acid_insertion',
    'def': 'A sequence variant within a CDS resulting in the gain of an amino acid to the resulting polypeptide. [SO:ke]'
  },
  'SO:0001606': {
    'name': 'amino_acid_substitution',
    'def': 'A sequence variant of a codon resulting in the substitution of one amino acid for another in the resulting polypeptide. [SO:ke]'
  },
  'SO:0001607': {
    'name': 'conservative_amino_acid_substitution',
    'def': 'A sequence variant of a codon causing the substitution of a similar amino acid for another in the resulting polypeptide. [SO:ke]'
  },
  'SO:0001608': {
    'name': 'non_conservative_amino_acid_substitution',
    'def': 'A sequence variant of a codon causing the substitution of a non conservative amino acid for another in the resulting polypeptide. [SO:ke]'
  },
  'SO:0001609': {
    'name': 'elongated_polypeptide',
    'def': 'An elongation of a polypeptide sequence deriving from a sequence variant extending the CDS. [SO:ke]'
  },
  'SO:0001610': {
    'name': 'elongated_polypeptide_C_terminal',
    'def': 'An elongation of a polypeptide sequence at the C terminus deriving from a sequence variant extending the CDS. [SO:ke]'
  },
  'SO:0001611': {
    'name': 'elongated_polypeptide_N_terminal',
    'def': 'An elongation of a polypeptide sequence at the N terminus deriving from a sequence variant extending the CDS. [SO:ke]'
  },
  'SO:0001612': {
    'name': 'elongated_in_frame_polypeptide_C_terminal',
    'def': 'A sequence variant with in the CDS that causes in frame elongation of the resulting polypeptide sequence at the C terminus. [SO:ke]'
  },
  'SO:0001613': {
    'name': 'elongated_out_of_frame_polypeptide_C_terminal',
    'def': 'A sequence variant with in the CDS that causes out of frame elongation of the resulting polypeptide sequence at the C terminus. [SO:ke]'
  },
  'SO:0001614': {
    'name': 'elongated_in_frame_polypeptide_N_terminal_elongation',
    'def': 'A sequence variant with in the CDS that causes in frame elongation of the resulting polypeptide sequence at the N terminus. [SO:ke]'
  },
  'SO:0001615': {
    'name': 'elongated_out_of_frame_polypeptide_N_terminal',
    'def': 'A sequence variant with in the CDS that causes out of frame elongation of the resulting polypeptide sequence at the N terminus. [SO:ke]'
  },
  'SO:0001616': {
    'name': 'polypeptide_fusion',
    'def': 'A sequence variant that causes a fusion of two polypeptide sequences. [SO:ke]'
  },
  'SO:0001617': {
    'name': 'polypeptide_truncation',
    'def': 'A sequence variant of the CD that causes a truncation of the resulting polypeptide. [SO:ke]'
  },
  'SO:0001618': {
    'name': 'inactive_catalytic_site',
    'def': 'A sequence variant that causes the inactivation of a catalytic site with respect to a reference sequence. [SO:ke]'
  },
  'SO:0001619': {
    'name': 'non_coding_transcript_variant',
    'def': 'A transcript variant of a non coding RNA gene. [SO:ke]'
  },
  'SO:0001620': {
    'name': 'mature_miRNA_variant',
    'def': 'A transcript variant located with the sequence of the mature miRNA. [SO:ke]'
  },
  'SO:0001621': {
    'name': 'NMD_transcript_variant',
    'def': 'A variant in a transcript that is the target of nonsense-mediated mRNA decay. [SO:ke]'
  },
  'SO:0001622': {
    'name': 'UTR_variant',
    'def': 'A transcript variant that is located within the UTR. [SO:ke]'
  },
  'SO:0001623': {
    'name': '5_prime_UTR_variant',
    'def': "A UTR variant of the 5' UTR. [SO:ke]"
  },
  'SO:0001624': {
    'name': '3_prime_UTR_variant',
    'def': "A UTR variant of the 3' UTR. [SO:ke]"
  },
  'SO:0001626': {
    'name': 'incomplete_terminal_codon_variant',
    'def': 'A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed. [SO:ke]'
  },
  'SO:0001627': {
    'name': 'intron_variant',
    'def': 'A transcript variant occurring within an intron. [SO:ke]'
  },
  'SO:0001628': {
    'name': 'intergenic_variant',
    'def': 'A sequence variant located in the intergenic region, between genes. [SO:ke]'
  },
  'SO:0001629': {
    'name': 'splice_site_variant',
    'def': 'A sequence variant that changes the first two or last two bases of an intron, or the 5th base from the start of the intron in the orientation of the transcript. [http://ensembl.org/info/docs/variation/index.html]'
  },
  'SO:0001630': {
    'name': 'splice_region_variant',
    'def': 'A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron. [http://ensembl.org/info/docs/variation/index.html]'
  },
  'SO:0001631': {
    'name': 'upstream_gene_variant',
    'def': "A sequence variant located 5' of a gene. [SO:ke]"
  },
  'SO:0001632': {
    'name': 'downstream_gene_variant',
    'def': "A sequence variant located 3' of a gene. [SO:ke]"
  },
  'SO:0001633': {
    'name': '5KB_downstream_variant',
    'def': 'A sequence variant located within 5 KB of the end of a gene. [SO:ke]'
  },
  'SO:0001634': {
    'name': '500B_downstream_variant',
    'def': 'A sequence variant located within a half KB of the end of a gene. [SO:ke]'
  },
  'SO:0001635': {
    'name': '5KB_upstream_variant',
    'def': "A sequence variant located within 5KB 5' of a gene. [SO:ke]"
  },
  'SO:0001636': {
    'name': '2KB_upstream_variant',
    'def': "A sequence variant located within 2KB 5' of a gene. [SO:ke]"
  },
  'SO:0001637': {
    'name': 'rRNA_gene',
    'def': 'A gene that encodes for ribosomal RNA. [SO:ke]'
  },
  'SO:0001638': {
    'name': 'piRNA_gene',
    'def': 'A gene that encodes for an piwi associated RNA. [SO:ke]'
  },
  'SO:0001639': {
    'name': 'RNase_P_RNA_gene',
    'def': 'A gene that encodes an RNase P RNA. [SO:ke]'
  },
  'SO:0001640': {
    'name': 'RNase_MRP_RNA_gene',
    'def': 'A gene that encodes a RNase_MRP_RNA. [SO:ke]'
  },
  'SO:0001641': {
    'name': 'lincRNA_gene',
    'def': 'A gene that encodes a long, intervening non-coding RNA. [http://www.gencodegenes.org/gencode_biotypes.html, PMID:23463798, SO:ke]'
  },
  'SO:0001642': {
    'name': 'mathematically_defined_repeat',
    'def': 'A mathematically defined repeat (MDR) is a experimental feature that is determined by querying overlapping oligomers of length k against a database of shotgun sequence data and identifying regions in the query sequence that exceed a statistically determined threshold of repetitiveness. [SO:jestill]'
  },
  'SO:0001643': {
    'name': 'telomerase_RNA_gene',
    'def': 'A telomerase RNA gene is a non coding RNA gene the RNA product of which is a component of telomerase. [SO:ke]'
  },
  'SO:0001644': {
    'name': 'targeting_vector',
    'def': 'An engineered vector that is able to take part in homologous recombination in a host with the intent of introducing site specific genomic modifications. [MGD:tm, PMID:10354467]'
  },
  'SO:0001645': {
    'name': 'genetic_marker',
    'def': 'A measurable sequence feature that varies within a population. [SO:db]'
  },
  'SO:0001646': {
    'name': 'DArT_marker',
    'def': 'A genetic marker, discovered using Diversity Arrays Technology (DArT) technology. [SO:ke]'
  },
  'SO:0001647': {
    'name': 'kozak_sequence',
    'def': "A kind of ribosome entry site, specific to Eukaryotic organisms that overlaps part of both 5' UTR and CDS sequence. [SO:ke]"
  },
  'SO:0001648': {
    'name': 'nested_transposon',
    'def': 'A transposon that is disrupted by the insertion of another element. [SO:ke]'
  },
  'SO:0001649': {
    'name': 'nested_repeat',
    'def': 'A repeat that is disrupted by the insertion of another element. [SO:ke]'
  },
  'SO:0001650': {
    'name': 'inframe_variant',
    'def': 'A sequence variant which does not cause a disruption of the translational reading frame. [SO:ke]'
  },
  'SO:0001653': {
    'name': 'retinoic_acid_responsive_element',
    'def': 'A transcription factor binding site of variable direct repeats of the sequence PuGGTCA spaced by five nucleotides (DR5) found in the promoters of retinoic acid-responsive genes, to which retinoic acid receptors bind. [PMID:11327309, PMID:19917671]'
  },
  'SO:0001654': {
    'name': 'nucleotide_to_protein_binding_site',
    'def': 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues. [SO:ke]'
  },
  'SO:0001655': {
    'name': 'nucleotide_binding_site',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with nucleotide residues. [SO:cb]'
  },
  'SO:0001656': {
    'name': 'metal_binding_site',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with metal ions. [SO:cb]'
  },
  'SO:0001657': {
    'name': 'ligand_binding_site',
    'def': 'A binding site that, in the molecule, interacts selectively and non-covalently with a small molecule such as a drug, or hormone. [SO:ke]'
  },
  'SO:0001658': {
    'name': 'nested_tandem_repeat',
    'def': 'An NTR is a nested repeat of two distinct tandem motifs interspersed with each other. [SO:AF]'
  },
  'SO:0001659': {
    'name': 'promoter_element',
    'def': 'An element that can exist within the promoter region of a gene. []'
  },
  'SO:0001660': {
    'name': 'core_eukaryotic_promoter_element',
    'def': 'An element that only exists within the promoter region of a eukaryotic gene. [GREEKC:cl]'
  },
  'SO:0001661': {
    'name': 'RNA_polymerase_II_TATA_box',
    'def': 'A TATA box core promoter of a gene transcribed by RNA polymerase II. [PMID:16858867]'
  },
  'SO:0001662': {
    'name': 'RNA_polymerase_III_TATA_box',
    'def': 'A TATA box core promoter of a gene transcribed by RNA polymerase III. [SO:ke]'
  },
  'SO:0001663': {
    'name': 'BREd_motif',
    'def': 'A core RNA polymerase II promoter element with consensus (G/A)T(T/G/A)(T/A)(G/T)(T/G)(T/G). [PMID:16858867]'
  },
  'SO:0001664': {
    'name': 'DCE',
    'def': 'A discontinuous core element of RNA polymerase II transcribed genes, situated downstream of the TSS. It is composed of three sub elements: SI, SII and SIII. [PMID:16858867]'
  },
  'SO:0001665': {
    'name': 'DCE_SI',
    'def': 'A sub element of the DCE core promoter element, with consensus sequence CTTC. [PMID:16858867, SO:ke]'
  },
  'SO:0001666': {
    'name': 'DCE_SII',
    'def': 'A sub element of the DCE core promoter element with consensus sequence CTGT. [PMID:16858867, SO:ke]'
  },
  'SO:0001667': {
    'name': 'DCE_SIII',
    'def': 'A sub element of the DCE core promoter element with consensus sequence AGC. [PMID:16858867, SO:ke]'
  },
  'SO:0001668': {
    'name': 'proximal_promoter_element',
    'def': 'DNA segment that ranges from about -250 to -40 relative to +1 of RNA transcription start site, where sequence specific DNA-binding transcription factors binds, such as Sp1, CTF (CCAAT-binding transcription factor), and CBF (CCAAT-box binding factor). [PMID:12515390, PMID:9679020, SO:ml]'
  },
  'SO:0001669': {
    'name': 'RNApol_II_core_promoter',
    'def': 'The minimal portion of the promoter required to properly initiate transcription in RNA polymerase II transcribed genes. [PMID:16858867]'
  },
  'SO:0001670': {
    'name': 'distal_promoter_element',
    'def': 'A regulatory promoter element that is distal from the TSS. []'
  },
  'SO:0001671': {
    'name': 'bacterial_RNApol_promoter_sigma_70_element',
    'def': 'A DNA sequence to which bacterial RNA polymerase sigma 70 binds, to begin transcription. []'
  },
  'SO:0001672': {
    'name': 'bacterial_RNApol_promoter_sigma54_element',
    'def': 'A DNA sequence to which bacterial RNA polymerase sigma 54 binds, to begin transcription. []'
  },
  'SO:0001673': {
    'name': 'minus_12_signal',
    'def': 'A conserved region about 12-bp upstream of the start point of bacterial transcription units, involved with sigma factor 54. [PMID:18331472]'
  },
  'SO:0001674': {
    'name': 'minus_24_signal',
    'def': 'A conserved region about 24-bp upstream of the start point of bacterial transcription units, involved with sigma factor 54. [PMID:18331472]'
  },
  'SO:0001675': {
    'name': 'A_box_type_1',
    'def': 'An A box within an RNA polymerase III type 1 promoter. [SO:ke]'
  },
  'SO:0001676': {
    'name': 'A_box_type_2',
    'def': 'An A box within an RNA polymerase III type 2 promoter. [SO:ke]'
  },
  'SO:0001677': {
    'name': 'intermediate_element',
    'def': 'A core promoter region of RNA polymerase III type 1 promoters. [PMID:12381659]'
  },
  'SO:0001678': {
    'name': 'regulatory_promoter_element',
    'def': 'A promoter element that is not part of the core promoter, but provides the promoter with a specific regulatory region. [PMID:12381659]'
  },
  'SO:0001679': {
    'name': 'transcription_regulatory_region',
    'def': 'A regulatory region that is involved in the control of the process of transcription. [SO:ke]'
  },
  'SO:0001680': {
    'name': 'translation_regulatory_region',
    'def': 'A regulatory region that is involved in the control of the process of translation. [SO:ke]'
  },
  'SO:0001681': {
    'name': 'recombination_regulatory_region',
    'def': 'A regulatory region that is involved in the control of the process of recombination. [SO:ke]'
  },
  'SO:0001682': {
    'name': 'replication_regulatory_region',
    'def': 'A regulatory region that is involved in the control of the process of nucleotide replication. [SO:ke]'
  },
  'SO:0001683': {
    'name': 'sequence_motif',
    'def': 'A sequence motif is a nucleotide or amino-acid sequence pattern that may have biological significance. [http://en.wikipedia.org/wiki/Sequence_motif]'
  },
  'SO:0001684': {
    'name': 'experimental_feature_attribute',
    'def': 'An attribute of an experimentally derived feature. [SO:ke]'
  },
  'SO:0001685': {
    'name': 'score',
    'def': 'The score of an experimentally derived feature such as a p-value. [SO:ke]'
  },
  'SO:0001686': {
    'name': 'quality_value',
    'def': 'An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score. [SO:ke]'
  },
  'SO:0001687': {
    'name': 'restriction_enzyme_recognition_site',
    'def': 'The nucleotide region (usually a palindrome) that is recognized by a restriction enzyme. This may or may not be equal to the restriction enzyme binding site. [SO:ke]'
  },
  'SO:0001688': {
    'name': 'restriction_enzyme_cleavage_junction',
    'def': 'The boundary at which a restriction enzyme breaks the nucleotide sequence. [SO:ke]'
  },
  'SO:0001689': {
    'name': 'five_prime_restriction_enzyme_junction',
    'def': "The restriction enzyme cleavage junction on the 5' strand of the nucleotide sequence. [SO:ke]"
  },
  'SO:0001690': {
    'name': 'three_prime_restriction_enzyme_junction',
    'def': "The restriction enzyme cleavage junction on the 3' strand of the nucleotide sequence. []"
  },
  'SO:0001691': {
    'name': 'blunt_end_restriction_enzyme_cleavage_site',
    'def': 'A restriction enzyme recognition site that, when cleaved, results in no overhangs. [SBOL:jgquinn, SO:ke]'
  },
  'SO:0001692': {
    'name': 'sticky_end_restriction_enzyme_cleavage_site',
    'def': "A site where restriction enzymes can cleave that will produce an overhang or 'sticky end'. []"
  },
  'SO:0001693': {
    'name': 'blunt_end_restriction_enzyme_cleavage_junction',
    'def': 'A restriction enzyme cleavage site where both strands are cut at the same position. [SO:ke]'
  },
  'SO:0001694': {
    'name': 'single_strand_restriction_enzyme_cleavage_site',
    'def': 'A restriction enzyme cleavage site whereby only one strand is cut. [SO:ke]'
  },
  'SO:0001695': {
    'name': 'restriction_enzyme_single_strand_overhang',
    'def': 'A terminal region of DNA sequence where the end of the region is not blunt ended. [SO:ke]'
  },
  'SO:0001696': {
    'name': 'experimentally_defined_binding_region',
    'def': 'A region that has been implicated in binding although the exact coordinates of binding may be unknown. [SO:ke]'
  },
  'SO:0001697': {
    'name': 'ChIP_seq_region',
    'def': 'A region of sequence identified by CHiP seq technology to contain a protein binding site. [SO:ke]'
  },
  'SO:0001698': {
    'name': 'ASPE_primer',
    'def': "\"A primer containing an SNV at the 3' end for accurate genotyping. [http://www.ncbi.nlm.nih.gov/pubmed/11252801]"
  },
  'SO:0001699': {
    'name': 'dCAPS_primer',
    'def': 'A primer with one or more mismatches to the DNA template corresponding to a position within a restriction enzyme recognition site. [http://www.ncbi.nlm.nih.gov/pubmed/9628033]'
  },
  'SO:0001700': {
    'name': 'histone_modification',
    'def': 'Histone modification is a post translationally modified region whereby residues of the histone protein are modified by methylation, acetylation, phosphorylation, ubiquitination, sumoylation, citrullination, or ADP-ribosylation. [http:en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001701': {
    'name': 'histone_methylation_site',
    'def': 'A histone modification site where the modification is the methylation of the residue. [SO:ke]'
  },
  'SO:0001702': {
    'name': 'histone_acetylation_site',
    'def': 'A histone modification where the modification is the acylation of the residue. [SO:ke]'
  },
  'SO:0001703': {
    'name': 'H3K9_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is acetylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001704': {
    'name': 'H3K14_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 14th residue (a lysine), from the start of the H3 histone protein is acetylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001705': {
    'name': 'H3K4_monomethylation_site',
    'def': 'A kind of histone modification, whereby the 4th residue (a lysine), from the start of the H3 protein is mono-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001706': {
    'name': 'H3K4_trimethylation',
    'def': 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 protein is tri-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001707': {
    'name': 'H3K9_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is tri-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001708': {
    'name': 'H3K27_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is mono-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001709': {
    'name': 'H3K27_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is tri-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001710': {
    'name': 'H3K79_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is mono- methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001711': {
    'name': 'H3K79_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is di-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001712': {
    'name': 'H3K79_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is tri-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001713': {
    'name': 'H4K20_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H4histone protein is mono-methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001714': {
    'name': 'H2BK5_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2B protein is methylated. [http://en.wikipedia.org/wiki/Histone]'
  },
  'SO:0001715': {
    'name': 'ISRE',
    'def': 'An ISRE is a transcriptional cis regulatory region, containing the consensus region: YAGTTTC(A/T)YTTTYCC, responsible for increased transcription via interferon binding. [http://genesdev.cshlp.org/content/2/4/383.abstrac]'
  },
  'SO:0001716': {
    'name': 'histone_ubiqitination_site',
    'def': 'A histone modification site where ubiquitin may be added. [SO:ke]'
  },
  'SO:0001717': {
    'name': 'H2B_ubiquitination_site',
    'def': 'A histone modification site on H2B where ubiquitin may be added. [SO:ke]'
  },
  'SO:0001718': {
    'name': 'H3K18_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 18th residue (a lysine), from the start of the H3 histone protein is acetylated. [SO:ke]'
  },
  'SO:0001719': {
    'name': 'H3K23_acetylation_site',
    'def': 'A kind of histone modification, whereby the 23rd residue (a lysine), from the start of the H3 histone protein is acetylated. [SO:ke]'
  },
  'SO:0001720': {
    'name': 'epigenetically_modified_region',
    'def': 'A biological DNA region implicated in epigenomic changes caused by mechanisms other than changes in the underlying DNA sequence. This includes, nucleosomal histone post-translational modifications, nucleosome depletion to render DNA accessible and post-replicational base modifications such as cytosine modification. [http://en.wikipedia.org/wiki/Epigenetics, SO:ke]'
  },
  'SO:0001721': {
    'name': 'H3K27_acylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is acylated. [SO:ke]'
  },
  'SO:0001722': {
    'name': 'H3K36_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is mono-methylated. [SO:ke]'
  },
  'SO:0001723': {
    'name': 'H3K36_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is dimethylated. [SO:ke]'
  },
  'SO:0001724': {
    'name': 'H3K36_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is tri-methylated. [SO:ke]'
  },
  'SO:0001725': {
    'name': 'H3K4_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 histone protein is di-methylated. [SO:ke]'
  },
  'SO:0001726': {
    'name': 'H3K27_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is di-methylated. [SO:ke]'
  },
  'SO:0001727': {
    'name': 'H3K9_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is mono-methylated. [SO:ke]'
  },
  'SO:0001728': {
    'name': 'H3K9_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein may be dimethylated. [SO:ke]'
  },
  'SO:0001729': {
    'name': 'H4K16_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 16th residue (a lysine), from the start of the H4 histone protein is acetylated. [SO:ke]'
  },
  'SO:0001730': {
    'name': 'H4K5_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H4 histone protein is acetylated. [SO:ke]'
  },
  'SO:0001731': {
    'name': 'H4K8_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 8th residue (a lysine), from the start of the H4 histone protein is acetylated. [SO:KE]'
  },
  'SO:0001732': {
    'name': 'H3K27_methylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is methylated. [SO:ke]'
  },
  'SO:0001733': {
    'name': 'H3K36_methylation_site',
    'def': 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is methylated. [SO:ke]'
  },
  'SO:0001734': {
    'name': 'H3K4_methylation_site',
    'def': 'A kind of histone modification, whereby the 4th residue (a lysine), from the start of the H3 protein is methylated. [SO:ke]'
  },
  'SO:0001735': {
    'name': 'H3K79_methylation_site',
    'def': 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is methylated. [SO:ke]'
  },
  'SO:0001736': {
    'name': 'H3K9_methylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is methylated. [SO:ke]'
  },
  'SO:0001737': {
    'name': 'histone_acylation_region',
    'def': 'A histone modification, whereby the histone protein is acylated at multiple sites in a region. [SO:ke]'
  },
  'SO:0001738': {
    'name': 'H4K_acylation_region',
    'def': 'A region of the H4 histone whereby multiple lysines are acylated. [SO:ke]'
  },
  'SO:0001739': {
    'name': 'gene_with_non_canonical_start_codon',
    'def': 'A gene with a start codon other than AUG. [SO:xp]'
  },
  'SO:0001740': {
    'name': 'gene_with_start_codon_CUG',
    'def': 'A gene with a translational start codon of CUG. [SO:mc]'
  },
  'SO:0001741': {
    'name': 'pseudogenic_gene_segment',
    'def': 'A gene segment which when incorporated by somatic recombination in the final gene transcript results in a nonfunctional product. [SO:hd]'
  },
  'SO:0001742': {
    'name': 'copy_number_gain',
    'def': 'A sequence alteration whereby the copy number of a given regions is greater than the reference sequence. [SO:ke]'
  },
  'SO:0001743': {
    'name': 'copy_number_loss',
    'def': 'A sequence alteration whereby the copy number of a given region is less than the reference sequence. [SO:ke]'
  },
  'SO:0001744': {
    'name': 'UPD',
    'def': 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from one parent and no copies of the same chromosome or region from the other parent. [SO:BM]'
  },
  'SO:0001745': {
    'name': 'maternal_uniparental_disomy',
    'def': 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from the mother and no copies of the same chromosome or region from the father. [SO:bm]'
  },
  'SO:0001746': {
    'name': 'paternal_uniparental_disomy',
    'def': 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from the father and no copies of the same chromosome or region from the mother. [SO:bm]'
  },
  'SO:0001747': {
    'name': 'open_chromatin_region',
    'def': 'A DNA sequence that in the normal state of the chromosome corresponds to an unfolded, un-complexed stretch of double-stranded DNA. [SO:cb]'
  },
  'SO:0001748': {
    'name': 'SL3_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL3 RNA leader sequence to the 5' end of an mRNA. SL3 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001749': {
    'name': 'SL4_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL4 RNA leader sequence to the 5' end of an mRNA. SL4 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001750': {
    'name': 'SL5_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL5 RNA leader sequence to the 5' end of an mRNA. SL5 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001751': {
    'name': 'SL6_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL6 RNA leader sequence to the 5' end of an mRNA. SL6 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001752': {
    'name': 'SL7_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL7 RNA leader sequence to the 5' end of an mRNA. SL7 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001753': {
    'name': 'SL8_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL8 RNA leader sequence to the 5' end of an mRNA. SL8 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001754': {
    'name': 'SL9_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL9 RNA leader sequence to the 5' end of an mRNA. SL9 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001755': {
    'name': 'SL10_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL10 RNA leader sequence to the 5' end of an mRNA. SL10 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001756': {
    'name': 'SL11_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL11 RNA leader sequence to the 5' end of an mRNA. SL11 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001757': {
    'name': 'SL12_acceptor_site',
    'def': "A SL2_acceptor_site which appends the SL12 RNA leader sequence to the 5' end of an mRNA. SL12 acceptor sites occur in genes in internal segments of polycistronic transcripts. [SO:nlw]"
  },
  'SO:0001758': {
    'name': 'duplicated_pseudogene',
    'def': 'A pseudogene that arose via gene duplication. Generally duplicated pseudogenes have the same structure as the original gene, including intron-exon structure and some regulatory sequence. [http://en.wikipedia.org/wiki/Pseudogene]'
  },
  'SO:0001759': {
    'name': 'unitary_pseudogene',
    'def': 'A pseudogene, deactivated from original state by mutation, fixed in a population,where the ortholog in a reference species such as mouse remains functional. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, http://en.wikipedia.org/wiki/Pseudogene, SO:ke]'
  },
  'SO:0001760': {
    'name': 'non_processed_pseudogene',
    'def': 'A pseudogene that arose from a means other than retrotransposition. A pseudogene created via genomic duplication of a functional protein-coding parent gene followed by accumulation of deleterious mutations. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, SO:ke]'
  },
  'SO:0001761': {
    'name': 'variant_quality',
    'def': 'A dependent entity that inheres in a bearer, a sequence variant. [PMID:17597783, SO:ke]'
  },
  'SO:0001762': {
    'name': 'variant_origin',
    'def': 'A quality inhering in a variant by virtue of its origin. [PMID:17597783, SO:ke]'
  },
  'SO:0001763': {
    'name': 'variant_frequency',
    'def': 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population. [PMID:17597783, SO:ke]'
  },
  'SO:0001764': {
    'name': 'unique_variant',
    'def': 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population. [SO:ke]'
  },
  'SO:0001765': {
    'name': 'rare_variant',
    'def': "When a variant from the genomic sequence is rarely found in the general population. The threshold for 'rare' varies between studies. []"
  },
  'SO:0001766': {
    'name': 'polymorphic_variant',
    'def': 'A variant that affects one of several possible alleles at that location, such as the major histocompatibility complex (MHC) genes. []'
  },
  'SO:0001767': {
    'name': 'common_variant',
    'def': 'When a variant from the genomic sequence is commonly found in the general population. []'
  },
  'SO:0001768': {
    'name': 'fixed_variant',
    'def': 'When a variant has become fixed in the population so that it is now the only variant. []'
  },
  'SO:0001769': {
    'name': 'variant_phenotype',
    'def': 'A quality inhering in a variant by virtue of its phenotype. [PMID:17597783, SO:ke]'
  },
  'SO:0001770': {
    'name': 'benign_variant',
    'def': 'A variant that does not affect the function of the gene or cause disease. []'
  },
  'SO:0001771': {
    'name': 'disease_associated_variant',
    'def': 'A variant that has been found to be associated with disease. []'
  },
  'SO:0001772': {
    'name': 'disease_causing_variant',
    'def': 'A variant that has been found to cause disease. []'
  },
  'SO:0001773': {
    'name': 'lethal_variant',
    'def': 'A sequence variant where the mutated gene product does not allow for one or more basic functions necessary for survival. []'
  },
  'SO:0001774': {
    'name': 'quantitative_variant',
    'def': 'A variant within a gene that contributes to a quantitative trait such as height or weight. []'
  },
  'SO:0001775': {
    'name': 'maternal_variant',
    'def': 'A variant in the genetic material inherited from the mother. []'
  },
  'SO:0001776': {
    'name': 'paternal_variant',
    'def': 'A variant in the genetic material inherited from the father. []'
  },
  'SO:0001777': {
    'name': 'somatic_variant',
    'def': 'A variant that has arisen after splitting of the embryo, resulting in the variant being found in only some of the tissues or cells of the body. []'
  },
  'SO:0001778': {
    'name': 'germline_variant',
    'def': 'A variant present in the embryo that is carried by every cell in the body. []'
  },
  'SO:0001779': {
    'name': 'pedigree_specific_variant',
    'def': 'A variant that is found only by individuals that belong to the same pedigree. []'
  },
  'SO:0001780': {
    'name': 'population_specific_variant',
    'def': 'A variant found within only speficic populations. []'
  },
  'SO:0001781': {
    'name': 'de_novo_variant',
    'def': 'A variant arising in the offspring that is not found in either of the parents. []'
  },
  'SO:0001782': {
    'name': 'TF_binding_site_variant',
    'def': 'A sequence variant located within a transcription factor binding site. [EBI:fc]'
  },
  'SO:0001784': {
    'name': 'complex_structural_alteration',
    'def': 'A structural sequence alteration or rearrangement encompassing one or more genome fragments, with 4 or more breakpoints. [FB:reference_manual, NCBI:th, SO:ke]'
  },
  'SO:0001785': {
    'name': 'structural_alteration',
    'def': 'An alteration of the genome that leads to a change in the structure of one or more chromosomes. []'
  },
  'SO:0001786': {
    'name': 'loss_of_heterozygosity',
    'def': 'A functional variant whereby the sequence alteration causes a loss of function of one allele of a gene. [SO:ke]'
  },
  'SO:0001787': {
    'name': 'splice_donor_5th_base_variant',
    'def': 'A sequence variant that causes a change at the 5th base pair after the start of the intron in the orientation of the transcript. [EBI:gr]'
  },
  'SO:0001788': {
    'name': 'U_box',
    'def': 'An U-box is a conserved T-rich region upstream of a retroviral polypurine tract that is involved in PPT primer creation during reverse transcription. [PMID:10556309, PMID:11577982, PMID:9649446]'
  },
  'SO:0001789': {
    'name': 'mating_type_region',
    'def': 'A specialized region in the genomes of some yeast and fungi, the genes of which regulate mating type. [SO:ke]'
  },
  'SO:0001790': {
    'name': 'paired_end_fragment',
    'def': 'An assembly region that has been sequenced from both ends resulting in a read_pair (mate_pair). [SO:ke]'
  },
  'SO:0001791': {
    'name': 'exon_variant',
    'def': 'A sequence variant that changes exon sequence. [SO:ke]'
  },
  'SO:0001792': {
    'name': 'non_coding_transcript_exon_variant',
    'def': 'A sequence variant that changes non-coding exon sequence in a non-coding transcript. [EBI:fc, SO:ke]'
  },
  'SO:0001793': {
    'name': 'clone_end',
    'def': 'A read from an end of the clone sequence. [SO:ke]'
  },
  'SO:0001794': {
    'name': 'point_centromere',
    'def': 'A point centromere is a relatively small centromere (about 125 bp DNA) in discrete sequence, found in some yeast including S. cerevisiae. [PMID:7502067, SO:vw]'
  },
  'SO:0001795': {
    'name': 'regional_centromere',
    'def': 'A regional centromere is a large modular centromere found in fission yeast and higher eukaryotes. It consist of a central core region flanked by inverted inner and outer repeat regions. [PMID:7502067, SO:vw]'
  },
  'SO:0001796': {
    'name': 'regional_centromere_central_core',
    'def': 'A conserved region within the central region of a modular centromere, where the kinetochore is formed. [SO:vw]'
  },
  'SO:0001797': {
    'name': 'centromeric_repeat',
    'def': 'A repeat region found within the modular centromere. [SO:ke]'
  },
  'SO:0001798': {
    'name': 'regional_centromere_inner_repeat_region',
    'def': 'The inner inverted repeat region of a modular centromere and part of the central core surrounding a non-conserved central region. This region is adjacent to the central core, on each chromosome arm. [SO:vw]'
  },
  'SO:0001799': {
    'name': 'regional_centromere_outer_repeat_region',
    'def': 'The heterochromatic outer repeat region of a modular centromere. These repeats exist in tandem arrays on both chromosome arms. [SO:vw]'
  },
  'SO:0001800': {
    'name': 'tasiRNA',
    'def': 'The sequence of a 21 nucleotide double stranded, polyadenylated non coding RNA, transcribed from the TAS gene. [PMID:16145017]'
  },
  'SO:0001801': {
    'name': 'tasiRNA_primary_transcript',
    'def': 'A primary transcript encoding a tasiRNA. [PMID:16145017]'
  },
  'SO:0001802': {
    'name': 'increased_polyadenylation_variant',
    'def': 'A transcript processing variant whereby polyadenylation of the encoded transcript is increased with respect to the reference. [SO:ke]'
  },
  'SO:0001803': {
    'name': 'decreased_polyadenylation_variant',
    'def': 'A transcript processing variant whereby polyadenylation of the encoded transcript is decreased with respect to the reference. [SO:ke]'
  },
  'SO:0001804': {
    'name': 'DDB_box',
    'def': 'A conserved polypeptide motif that mediates protein-protein interaction and defines adaptor proteins for DDB1/cullin 4 ubiquitin ligases. [PMID:18794354, PMID:19818632]'
  },
  'SO:0001805': {
    'name': 'destruction_box',
    'def': 'A conserved polypeptide motif that can be recognized by both Fizzy/Cdc20- and FZR/Cdh1-activated anaphase-promoting complex/cyclosome (APC/C) and targets a protein for ubiquitination and subsequent degradation by the APC/C. The consensus sequence is RXXLXXXXN. [PMID:12208841, PMID:1842691]'
  },
  'SO:0001806': {
    'name': 'ER_retention_signal',
    'def': 'A C-terminal tetrapeptide motif that mediates retention of a protein in (or retrieval to) the endoplasmic reticulum. In mammals the sequence is KDEL, and in fungi HDEL or DDEL. [doi:10.1093/jxb/50.331.157, PMID:2077689]'
  },
  'SO:0001807': {
    'name': 'KEN_box',
    'def': 'A conserved polypeptide motif that can be recognized by FZR/Cdh1-activated anaphase-promoting complex/cyclosome (APC/C) and targets a protein for ubiquitination and subsequent degradation by the APC/C. The consensus sequence is KENXXXN. [PMID:10733526, PMID:1220884, PMID:18426916]'
  },
  'SO:0001808': {
    'name': 'mitochondrial_targeting_signal',
    'def': 'A polypeptide region that targets a polypeptide to the mitochondrion. [PomBase:mah]'
  },
  'SO:0001809': {
    'name': 'signal_anchor',
    'def': 'A signal sequence that is not cleaved from the polypeptide. Anchors a Type II membrane protein to the membrane. [http://www.cbs.dtu.dk/services/SignalP/background/biobackground.php]'
  },
  'SO:0001810': {
    'name': 'PIP_box',
    'def': 'A polypeptide region that mediates binding to PCNA. The consensus sequence is QXX(hh)XX(aa), where (h) denotes residues with moderately hydrophobic side chains and (a) denotes residues with highly hydrophobic aromatic side chains. [PMID:9631646]'
  },
  'SO:0001811': {
    'name': 'phosphorylation_site',
    'def': 'A post-translationally modified region in which residues of the protein are modified by phosphorylation. [PomBase:mah]'
  },
  'SO:0001812': {
    'name': 'transmembrane_helix',
    'def': 'A region that traverses the lipid bilayer and adopts a helical secondary structure. [PomBase:mah]'
  },
  'SO:0001813': {
    'name': 'vacuolar_sorting_signal',
    'def': 'A polypeptide region that targets a polypeptide to the vacuole. [PomBase:mah]'
  },
  'SO:0001814': {
    'name': 'coding_variant_quality',
    'def': 'An attribute of a coding genomic variant. []'
  },
  'SO:0001815': {
    'name': 'synonymous',
    'def': 'A variant that does not lead to any change in the amino acid sequence. []'
  },
  'SO:0001816': {
    'name': 'non_synonymous',
    'def': 'A variant that leads to the change of an amino acid within the protein. []'
  },
  'SO:0001817': {
    'name': 'inframe',
    'def': 'An attribute describing a sequence that contains a mutation involving the deletion or insertion of one or more bases, where this number is divisible by 3. [SO:ke]'
  },
  'SO:0001818': {
    'name': 'protein_altering_variant',
    'def': 'A sequence_variant which is predicted to change the protein encoded in the coding sequence. [EBI:gr]'
  },
  'SO:0001819': {
    'name': 'synonymous_variant',
    'def': 'A sequence variant where there is no resulting change to the encoded amino acid. [SO:ke]'
  },
  'SO:0001820': {
    'name': 'inframe_indel',
    'def': 'A coding sequence variant where the change does not alter the frame of the transcript. [SO:ke]'
  },
  'SO:0001821': {
    'name': 'inframe_insertion',
    'def': 'An inframe non synonymous variant that inserts bases into in the coding sequence. [EBI:gr]'
  },
  'SO:0001822': {
    'name': 'inframe_deletion',
    'def': 'An inframe non synonymous variant that deletes bases from the coding sequence. [EBI:gr]'
  },
  'SO:0001823': {
    'name': 'conservative_inframe_insertion',
    'def': 'An inframe increase in cds length that inserts one or more codons into the coding sequence between existing codons. [EBI:gr]'
  },
  'SO:0001824': {
    'name': 'disruptive_inframe_insertion',
    'def': 'An inframe increase in cds length that inserts one or more codons into the coding sequence within an existing codon. [EBI:gr]'
  },
  'SO:0001825': {
    'name': 'conservative_inframe_deletion',
    'def': 'An inframe decrease in cds length that deletes one or more entire codons from the coding sequence but does not change any remaining codons. [EBI:gr]'
  },
  'SO:0001826': {
    'name': 'disruptive_inframe_deletion',
    'def': 'An inframe decrease in cds length that deletes bases from the coding sequence starting within an existing codon. [EBI:gr]'
  },
  'SO:0001827': {
    'name': 'mRNA_read',
    'def': 'A sequencer read of an mRNA substrate. [SO:ke]'
  },
  'SO:0001828': {
    'name': 'genomic_DNA_read',
    'def': 'A sequencer read of a genomic DNA substrate. [SO:ke]'
  },
  'SO:0001829': {
    'name': 'mRNA_contig',
    'def': 'A contig composed of mRNA_reads. [SO:ke]'
  },
  'SO:0001830': {
    'name': 'AFLP_fragment',
    'def': 'A PCR product obtained by applying the AFLP technique, based on a restriction enzyme digestion of genomic DNA and an amplification of the resulting fragments. [GMOD:ea]'
  },
  'SO:0001831': {
    'name': 'protein_hmm_match',
    'def': 'A match to a protein HMM such as pfam. [SO:ke]'
  },
  'SO:0001832': {
    'name': 'immunoglobulin_region',
    'def': 'A region of immunoglobulin sequence, either constant or variable. [SO:ke]'
  },
  'SO:0001833': {
    'name': 'V_region',
    'def': 'The variable region of an immunoglobulin polypeptide sequence. [SO:ke]'
  },
  'SO:0001834': {
    'name': 'C_region',
    'def': 'The constant region of an immunoglobulin polypeptide sequence. [SO:ke]'
  },
  'SO:0001835': {
    'name': 'N_region',
    'def': 'Extra nucleotides inserted between rearranged immunoglobulin segments. [SO:ke]'
  },
  'SO:0001836': {
    'name': 'S_region',
    'def': 'The switch region of immunoglobulin heavy chains; it is involved in the rearrangement of heavy chain DNA leading to the expression of a different immunoglobulin classes from the same B-cell. [SO:ke]'
  },
  'SO:0001837': {
    'name': 'mobile_element_insertion',
    'def': 'A kind of insertion where the inserted sequence is a mobile element. [EBI:dvga]'
  },
  'SO:0001838': {
    'name': 'novel_sequence_insertion',
    'def': 'An insertion the sequence of which cannot be mapped to the reference genome. [NCBI:th]'
  },
  'SO:0001839': {
    'name': 'CSL_response_element',
    'def': 'A promoter element with consensus sequence GTGRGAA, bound by CSL (CBF1/RBP-JK/Suppressor of Hairless/LAG-1) transcription factors. [PMID:19101542]'
  },
  'SO:0001840': {
    'name': 'GATA_box',
    'def': 'A GATA transcription factor element containing the consensus sequence WGATAR (in which W indicates A/T and R indicates A/G). [PMID:8321208]'
  },
  'SO:0001841': {
    'name': 'polymorphic_pseudogene',
    'def': 'A pseudogene in the reference genome, though known to be intact in the genomes of other individuals of the same species. The annotation process has confirmed that the pseudogenisation event is not a genomic sequencing error. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, JAX:hd]'
  },
  'SO:0001842': {
    'name': 'AP_1_binding_site',
    'def': 'A promoter element with consensus sequence TGACTCA, bound by AP-1 and related transcription factors. [PMID:1899230, PMID:3034432, PMID:3125983]'
  },
  'SO:0001843': {
    'name': 'CRE',
    'def': 'MERGED DEFINITION:\nTARGET DEFINITION: A promoter element with consensus sequence TGACGTCA; bound by the ATF/CREB family of transcription factors.\n--------------------\nSOURCE DEFINITION: A promoter element that contains a core sequence TGACGT, bound by a protein complex that regulates transcription of genes encoding PKA pathway components. [PMID:11483355, PMID:11483993, PMID:15448137]'
  },
  'SO:0001844': {
    'name': 'CuRE',
    'def': 'A promoter element bound by copper ion-sensing transcription factors such as S. cerevisiae Mac1p or S. pombe Cuf1; the consensus sequence is HTHNNGCTGD (more specifically TTTGCKCR in budding yeast). [PMID:10593913, PMID:9188496, PMID:9211922]'
  },
  'SO:0001845': {
    'name': 'DRE',
    'def': 'A promoter element with consensus sequence CGWGGWNGMM, bound by transcription factors related to RecA and found in promoters of genes expressed following several types of DNA damage or inhibition of DNA synthesis. [PMID:11073995, PMID:8668127]'
  },
  'SO:0001846': {
    'name': 'FLEX_element',
    'def': 'A promoter element that has consensus sequence GTAAACAAACAAAM and contains a heptameric core GTAAACA, bound by transcription factors with a forkhead DNA-binding domain. [PMID:10747048, PMID:14871934]'
  },
  'SO:0001847': {
    'name': 'forkhead_motif',
    'def': 'A promoter element with consensus sequence TTTRTTTACA, bound by transcription factors with a forkhead DNA-binding domain. [PMID:15195092]'
  },
  'SO:0001848': {
    'name': 'homol_D_box',
    'def': 'A core promoter element that has the consensus sequence CAGTCACA (or its inverted form TGTGACTG), and plays the role of a TATA box in promoters that do not contain a canonical TATA sequence. [PMID:21673110, PMID:7501449, PMID:8458332]'
  },
  'SO:0001849': {
    'name': 'homol_E_box',
    'def': 'A core promoter element that has the consensus sequence ACCCTACCCT (or its inverted form AGGGTAGGGT), and is found near the homol D box in some promoters that use a homol D box instead of a canonical TATA sequence. [PMID:7501449]'
  },
  'SO:0001850': {
    'name': 'HSE',
    'def': 'A promoter element that consists of at least three copies of the pentanucleotide NGAAN, bound by the heat shock transcription factor HSF. [PMID:17347150, PMID:8689565]'
  },
  'SO:0001851': {
    'name': 'iron_repressed_GATA_element',
    'def': 'A GATA promoter element with consensus sequence WGATAA, found in promoters of genes repressed in the presence of iron. [PMID:11956219, PMID:17211681]'
  },
  'SO:0001852': {
    'name': 'mating_type_M_box',
    'def': 'A promoter element with consensus sequence ACAAT, found in promoters of mating type M-specific genes in fission yeast and bound by the transcription factor Mat1-Mc. [PMID:9233811]'
  },
  'SO:0001853': {
    'name': 'androgen_response_element',
    'def': 'A non-palindromic sequence found in the promoters of genes whose expression is regulated in response to androgen. [PMID:21796522]'
  },
  'SO:0001854': {
    'name': 'smFISH_probe',
    'def': 'A smFISH is a probe that binds RNA in a single molecule in situ hybridization experiment. [PMID:18806792]'
  },
  'SO:0001855': {
    'name': 'MCB',
    'def': 'A promoter element with consensus sequence ACGCGT, bound by the transcription factor complex MBF (MCB-binding factor) and found in promoters of genes expressed during the G1/S transition of the cell cycle. [PMID:16285853]'
  },
  'SO:0001856': {
    'name': 'CCAAT_motif',
    'def': 'A promoter element with consensus sequence CCAAT, bound by a protein complex that represses transcription in response to low iron levels. [PMID:16963626]'
  },
  'SO:0001857': {
    'name': 'Ace2_UAS',
    'def': 'A promoter element with consensus sequence CCAGCC, bound by the fungal transcription factor Ace2. [PMID:16678171]'
  },
  'SO:0001858': {
    'name': 'TR_box',
    'def': 'A promoter element with consensus sequence TTCTTTGTTY, bound an HMG-box transcription factor such as S. pombe Ste11, and found in promoters of genes up-regulated early in meiosis. [PMID:1657709]'
  },
  'SO:0001859': {
    'name': 'STREP_motif',
    'def': 'A promoter element with consensus sequence CCCCTC, bound by the PKA-responsive zinc finger transcription factor Rst2. [PMID:11739717]'
  },
  'SO:0001860': {
    'name': 'rDNA_intergenic_spacer_element',
    'def': 'A DNA motif that contains a core consensus sequence AGGTAAGGGTAATGCAC, is found in the intergenic regions of rDNA repeats, and is bound by an RNA polymerase I transcription termination factor (e.g. S. pombe Reb1). The S. pombe telomeric repeat consensus is TTAC(0-1)A(0-1)G(1-8). [ISBN:978-0199638901, PMID:9016645]'
  },
  'SO:0001861': {
    'name': 'sterol_regulatory_element',
    'def': 'A 10-bp promoter element bound by sterol regulatory element binding proteins (SREBPs), found in promoters of genes involved in sterol metabolism. Many variants of the sequence ATCACCCCAC function as SREs. [GO:mah, PMID:11111080, PMID:16537923]'
  },
  'SO:0001862': {
    'name': 'GT_dinucleotide_repeat',
    'def': 'A dinucleotide repeat region composed of GT repeating elements. [SO:ke]'
  },
  'SO:0001863': {
    'name': 'GTT_trinucleotide_repeat',
    'def': 'A trinucleotide repeat region composed of GTT repeating elements. [SO:ke]'
  },
  'SO:0001864': {
    'name': 'Sap1_recognition_motif',
    'def': "A DNA motif to which the S. pombe Sap1 protein binds. The consensus sequence is 5'-TARGCAGNTNYAACGMG-3'; it is found at the mating type locus, where it is important for mating type switching, and at replication fork barriers in rDNA repeats. [PMID:16166653, PMID:7651412]"
  },
  'SO:0001865': {
    'name': 'CDRE_motif',
    'def': 'An RNA polymerase II promoter element found in the promoters of genes regulated by calcineurin. The consensus sequence is GNGGCKCA. [PMID:16928959]'
  },
  'SO:0001866': {
    'name': 'BAC_read_contig',
    'def': 'A contig of BAC reads. [GMOD:ea]'
  },
  'SO:0001867': {
    'name': 'candidate_gene',
    'def': 'A gene suspected of being involved in the expression of a trait. [GMOD:ea]'
  },
  'SO:0001868': {
    'name': 'positional_candidate_gene',
    'def': "A candidate gene whose association with a trait is based on the gene's location on a chromosome. [GMOD:ea]"
  },
  'SO:0001869': {
    'name': 'functional_candidate_gene',
    'def': 'A candidate gene whose function has something in common biologically with the trait under investigation. [GMOD:ea]'
  },
  'SO:0001870': {
    'name': 'enhancerRNA',
    'def': 'A short ncRNA that is transcribed from an enhancer. May have a regulatory function. [doi:10.1038/465173a, SO:cjm]'
  },
  'SO:0001871': {
    'name': 'PCB',
    'def': 'A promoter element with consensus sequence GNAACR, bound by the transcription factor complex PBF (PCB-binding factor) and found in promoters of genes expressed during the M/G1 transition of the cell cycle. [GO:mah, PMID:12411492]'
  },
  'SO:0001872': {
    'name': 'rearrangement_region',
    'def': 'A region of a chromosome, where the chromosome has undergone a large structural rearrangement that altered the genome organization. There is no longer synteny to the reference genome. [NCBI:th, PMID:18564416]'
  },
  'SO:0001873': {
    'name': 'interchromosomal_breakpoint',
    'def': 'A rearrangement breakpoint between two different chromosomes. [NCBI:th]'
  },
  'SO:0001874': {
    'name': 'intrachromosomal_breakpoint',
    'def': 'A rearrangement breakpoint within the same chromosome. [NCBI:th]'
  },
  'SO:0001875': {
    'name': 'unassigned_supercontig',
    'def': 'A supercontig that is not been assigned to any ultracontig during a genome assembly project. [GMOD:ea]'
  },
  'SO:0001876': {
    'name': 'partial_genomic_sequence_assembly',
    'def': "A partial DNA sequence assembly of a chromosome or full genome, which contains gaps that are filled with N's. [GMOD:ea]"
  },
  'SO:0001877': {
    'name': 'lncRNA',
    'def': "A non-coding RNA generally longer than 200 nucleotides that cannot be classified as any other ncRNA subtype. Similar to mRNAs, lncRNAs are mainly transcribed by RNA polymerase II, are often capped by 7-methyl guanosine at their 5' ends, polyadenylated at their 3' ends and may be spliced. [HGNC:mw] {comment='PMID:33353982'}"
  },
  'SO:0001878': {
    'name': 'feature_variant',
    'def': 'A sequence variant that falls entirely or partially within a genomic feature. [EBI:fc, SO:ke]'
  },
  'SO:0001879': {
    'name': 'feature_ablation',
    'def': 'A sequence variant, caused by an alteration of the genomic sequence, where the deletion, is greater than the extent of the underlying genomic features. [SO:ke]'
  },
  'SO:0001880': {
    'name': 'feature_amplification',
    'def': 'A sequence variant, caused by an alteration of the genomic sequence, where the structural change, an amplification of sequence, is greater than the extent of the underlying genomic features. [SO:ke]'
  },
  'SO:0001881': {
    'name': 'feature_translocation',
    'def': 'A sequence variant, caused by an alteration of the genomic sequence, where the structural change, a translocation, is greater than the extent of the underlying genomic features. [SO:ke]'
  },
  'SO:0001882': {
    'name': 'feature_fusion',
    'def': 'A sequence variant, caused by an alteration of the genomic sequence, where a deletion fuses genomic features. [SO:ke]'
  },
  'SO:0001883': {
    'name': 'transcript_translocation',
    'def': 'A feature translocation where the region contains a transcript. [SO:ke]'
  },
  'SO:0001884': {
    'name': 'regulatory_region_translocation',
    'def': 'A feature translocation where the region contains a regulatory region. [SO:ke]'
  },
  'SO:0001885': {
    'name': 'TFBS_translocation',
    'def': 'A feature translocation where the region contains a transcription factor binding site. [SO:ke]'
  },
  'SO:0001886': {
    'name': 'transcript_fusion',
    'def': 'A feature fusion where the deletion brings together transcript regions. [SO:ke]'
  },
  'SO:0001887': {
    'name': 'regulatory_region_fusion',
    'def': 'A feature fusion where the deletion brings together regulatory regions. [SO:ke]'
  },
  'SO:0001888': {
    'name': 'TFBS_fusion',
    'def': 'A fusion where the deletion brings together transcription factor binding sites. [SO:ke]'
  },
  'SO:0001889': {
    'name': 'transcript_amplification',
    'def': 'A feature amplification of a region containing a transcript. [SO:ke]'
  },
  'SO:0001890': {
    'name': 'transcript_regulatory_region_fusion',
    'def': 'A feature fusion where the deletion brings together a regulatory region and a transcript region. [SO:ke]'
  },
  'SO:0001891': {
    'name': 'regulatory_region_amplification',
    'def': 'A feature amplification of a region containing a regulatory region. [SO:ke]'
  },
  'SO:0001892': {
    'name': 'TFBS_amplification',
    'def': 'A feature amplification of a region containing a transcription factor binding site. [SO:ke]'
  },
  'SO:0001893': {
    'name': 'transcript_ablation',
    'def': 'A feature ablation whereby the deleted region includes a transcript feature. [SO:ke]'
  },
  'SO:0001894': {
    'name': 'regulatory_region_ablation',
    'def': 'A feature ablation whereby the deleted region includes a regulatory region. [SO:ke]'
  },
  'SO:0001895': {
    'name': 'TFBS_ablation',
    'def': 'A feature ablation whereby the deleted region includes a transcription factor binding site. [SO:ke]'
  },
  'SO:0001896': {
    'name': 'transposable_element_CDS',
    'def': 'A CDS that is part of a transposable element. [SO:ke]'
  },
  'SO:0001897': {
    'name': 'transposable_element_pseudogene',
    'def': 'A pseudogene contained within a transposable element. [SO:ke]'
  },
  'SO:0001898': {
    'name': 'dg_repeat',
    'def': 'A repeat region which is part of the regional centromere outer repeat region. [PMID:16407326, SO:vw]'
  },
  'SO:0001899': {
    'name': 'dh_repeat',
    'def': 'A repeat region which is part of the regional centromere outer repeat region. [PMID:16407326, SO:vw]'
  },
  'SO:0001901': {
    'name': 'AACCCT_box',
    'def': "A conserved 17-bp sequence (5'-ATCA(C/A)AACCCTAACCCT-3') commonly present upstream of the start site of histone transcription units functioning as a transcription factor binding site. [PMID:17452352, PMID:4092687]"
  },
  'SO:0001902': {
    'name': 'splice_region',
    'def': 'A region surrounding a cis_splice site, either within 1-3 bases of the exon or 3-8 bases of the intron. [SO:bm]'
  },
  'SO:0001904': {
    'name': 'antisense_lncRNA',
    'def': 'Non-coding RNA transcribed from the opposite DNA strand compared with other transcripts and overlap in part with sense RNA. [PMID:19638999]'
  },
  'SO:0001905': {
    'name': 'regional_centromere_outer_repeat_transcript',
    'def': 'A transcript that is transcribed from the outer repeat region of a regional centromere. [PomBase:mah]'
  },
  'SO:0001906': {
    'name': 'feature_truncation',
    'def': 'A sequence variant that causes the reduction of a genomic feature, with regard to the reference sequence. [SO:ke]'
  },
  'SO:0001907': {
    'name': 'feature_elongation',
    'def': 'A sequence variant that causes the extension of a genomic feature, with regard to the reference sequence. [SO:ke]'
  },
  'SO:0001908': {
    'name': 'internal_feature_elongation',
    'def': 'A sequence variant that causes the extension of a genomic feature from within the feature rather than from the terminus of the feature, with regard to the reference sequence. [SO:ke]'
  },
  'SO:0001909': {
    'name': 'frameshift_elongation',
    'def': 'A frameshift variant that causes the translational reading frame to be extended relative to the reference feature. [SO:ke]'
  },
  'SO:0001910': {
    'name': 'frameshift_truncation',
    'def': 'A frameshift variant that causes the translational reading frame to be shortened relative to the reference feature. [SO:ke]'
  },
  'SO:0001911': {
    'name': 'copy_number_increase',
    'def': 'A sequence variant where copies of a feature are increased relative to the reference. [SO:ke]'
  },
  'SO:0001912': {
    'name': 'copy_number_decrease',
    'def': 'A sequence variant where copies of a feature are decreased relative to the reference. [SO:ke]'
  },
  'SO:0001913': {
    'name': 'bacterial_RNApol_promoter_sigma_ecf_element',
    'def': 'A bacterial promoter with sigma ecf factor binding dependency. This is a type of bacterial promoters that requires a sigma ECF factor to bind to identified -10 and -35 sequence regions in order to mediate binding of the RNA polymerase to the promoter region as part of transcription initiation. [Invitrogen:kc]'
  },
  'SO:0001914': {
    'name': 'rDNA_replication_fork_barrier',
    'def': 'A DNA motif that is found in eukaryotic rDNA repeats, and is a site of replication fork pausing. [PMID:14645529]'
  },
  'SO:0001915': {
    'name': 'transcription_start_cluster',
    'def': 'A region defined by a cluster of experimentally determined transcription starting sites. [PMID:19624849, PMID:21372179, SO:andrewgibson]'
  },
  'SO:0001916': {
    'name': 'CAGE_tag',
    'def': "A CAGE tag is a sequence tag hat corresponds to 5' ends of mRNA at cap sites, produced by cap analysis gene expression and used to identify transcriptional start sites. [SO:andrewgibson]"
  },
  'SO:0001917': {
    'name': 'CAGE_cluster',
    'def': 'A kind of transcription_initiation_cluster defined by the clustering of CAGE tags on a sequence region. [PMID:16645617, SO:andrewgibson]'
  },
  'SO:0001918': {
    'name': '5_methylcytosine',
    'def': 'A cytosine methylated at the 5 carbon. [SO:rtapella]'
  },
  'SO:0001919': {
    'name': '4_methylcytosine',
    'def': 'A cytosine methylated at the 4 nitrogen. [SO:rtapella]'
  },
  'SO:0001920': {
    'name': 'N6_methyladenine',
    'def': 'An adenine methylated at the 6 nitrogen. [SO:rtapella]'
  },
  'SO:0001921': {
    'name': 'mitochondrial_contig',
    'def': 'A contig of mitochondria derived sequences. [GMOD:ea]'
  },
  'SO:0001922': {
    'name': 'mitochondrial_supercontig',
    'def': 'A scaffold composed of mitochondrial contigs. [GMOD:ea]'
  },
  'SO:0001923': {
    'name': 'TERRA',
    'def': 'A non-coding RNA transcript, derived from the transcription of the telomere. These transcripts contain G rich telomeric RNA repeats and RNA tracts corresponding to adjacent subtelomeric sequences. They are 100-9000 bases long. [PMID:22139915]'
  },
  'SO:0001924': {
    'name': 'ARRET',
    'def': 'A non coding RNA transcript, complementary to subtelomeric tract of TERRA transcript but devoid of the repeats. [PMID:2139915]'
  },
  'SO:0001925': {
    'name': 'ARIA',
    'def': 'A non-coding RNA transcript, derived from the transcription of the telomere. These transcripts consist of C rich repeats. [PMID:22139915]'
  },
  'SO:0001926': {
    'name': 'anti_ARRET',
    'def': 'A non-coding RNA transcript, derived from the transcription of the telomere. These transcripts are antisense of ARRET transcripts. [PMID:22139915]'
  },
  'SO:0001927': {
    'name': 'telomeric_transcript',
    'def': 'A non-coding transcript derived from the transcript of the telomere. [PMID:22139915]'
  },
  'SO:0001928': {
    'name': 'distal_duplication',
    'def': 'A duplication of the distal region of a chromosome. [SO:bm]'
  },
  'SO:0001929': {
    'name': 'mitochondrial_DNA_read',
    'def': 'A sequencer read of a mitochondrial DNA sample. [GMOD:ea]'
  },
  'SO:0001930': {
    'name': 'chloroplast_DNA_read',
    'def': 'A sequencer read of a chloroplast DNA sample. [GMOD:ea]'
  },
  'SO:0001931': {
    'name': 'consensus_gDNA',
    'def': 'Genomic DNA sequence produced from some base calling or alignment algorithm which uses aligned or assembled multiple gDNA sequences as input. [GMOD:ea]'
  },
  'SO:0001932': {
    'name': 'restriction_enzyme_five_prime_single_strand_overhang',
    'def': "A terminal region of DNA sequence where the end of the region is not blunt ended and the exposed single strand terminates at the 5' end. [SO:ke]"
  },
  'SO:0001933': {
    'name': 'restriction_enzyme_three_prime_single_strand_overhang',
    'def': "A terminal region of DNA sequence where the end of the region is not blunt ended and the exposed single strand terminates at the 3' end. [SO:ke]"
  },
  'SO:0001934': {
    'name': 'monomeric_repeat',
    'def': 'A repeat_region containing repeat_units of 1 bp that is repeated multiple times in tandem. [SO:ke]'
  },
  'SO:0001935': {
    'name': 'H3K20_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H3 protein is tri-methylated. [EBI:nj]'
  },
  'SO:0001936': {
    'name': 'H3K36_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001937': {
    'name': 'H2BK12_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 12th residue (a lysine), from the start of the H2B protein is acetylated. [EBI:nj]'
  },
  'SO:0001938': {
    'name': 'H2AK5_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2A histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001939': {
    'name': 'H4K12_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 12th residue (a lysine), from the start of the H4 histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001940': {
    'name': 'H2BK120_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 120th residue (a lysine), from the start of the H2B histone protein is acetylated. [EBI:nj, http://dx.doi.org/10.4161/epi.6.5.15623]'
  },
  'SO:0001941': {
    'name': 'H4K91_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 91st residue (a lysine), from the start of the H4 histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001942': {
    'name': 'H2BK20_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H2B histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001943': {
    'name': 'H3K4_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001944': {
    'name': 'H2AK9_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H2A histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001945': {
    'name': 'H3K56_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 56th residue (a lysine), from the start of the H3 histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001946': {
    'name': 'H2BK15_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 15th residue (a lysine), from the start of the H2B histone protein is acetylated. [EBI:nj]'
  },
  'SO:0001947': {
    'name': 'H3R2_monomethylation_site',
    'def': 'A kind of histone modification site, whereby the 2nd residue (an arginine), from the start of the H3 protein is mono-methylated. [EBI:nj]'
  },
  'SO:0001948': {
    'name': 'H3R2_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 2nd residue (an arginine), from the start of the H3 protein is di-methylated. [EBI:nj]'
  },
  'SO:0001949': {
    'name': 'H4R3_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 3nd residue (an arginine), from the start of the H4 protein is di-methylated. [EBI:nj]'
  },
  'SO:0001950': {
    'name': 'H4K4_trimethylation_site',
    'def': 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H4 protein is tri-methylated. [EBI:nj]'
  },
  'SO:0001951': {
    'name': 'H3K23_dimethylation_site',
    'def': 'A kind of histone modification site, whereby the 23rd residue (a lysine), from the start of the H3 protein is di-methylated. [EBI:nj]'
  },
  'SO:0001952': {
    'name': 'promoter_flanking_region',
    'def': 'A region immediately adjacent to a promoter which may or may not contain transcription factor binding sites. [EBI:nj]'
  },
  'SO:0001953': {
    'name': 'restriction_enzyme_assembly_scar',
    'def': 'A region of DNA sequence formed from the ligation of two sticky ends where the palindrome is broken and no longer comprises the recognition site and thus cannot be re-cut by the restriction enzymes used to create the sticky ends. [SO:ke]'
  },
  'SO:0001954': {
    'name': 'restriction_enzyme_region',
    'def': 'A region related to restriction enzyme function. [SO:ke]'
  },
  'SO:0001955': {
    'name': 'protein_stability_element',
    'def': 'A polypeptide region that proves structure in a protein that affects the stability of the protein. [SO:ke]'
  },
  'SO:0001956': {
    'name': 'protease_site',
    'def': 'A polypeptide_region that codes for a protease cleavage site. [SO:ke]'
  },
  'SO:0001957': {
    'name': 'RNA_stability_element',
    'def': 'RNA secondary structure that affects the stability of an RNA molecule. [SO:ke]'
  },
  'SO:0001958': {
    'name': 'lariat_intron',
    'def': 'A kind of intron whereby the excision is driven by lariat formation. [SO:ke]'
  },
  'SO:0001959': {
    'name': 'TCT_motif',
    'def': 'A cis-regulatory element, conserved sequence YYC+1TTTYY, and spans -2 to +6 relative to +1 TSS. It is present in most ribosomal protein genes in Drosophila and mammals but not in the yeast Saccharomyces cerevisiae. Resembles the initiator (TCAKTY in Drosophila) but functionally distinct from initiator. [PMID:20801935, SO:myl]'
  },
  'SO:0001960': {
    'name': '5_hydroxymethylcytosine',
    'def': 'A modified DNA cytosine base feature, modified by a hydroxymethyl group at the 5 carbon. [SO:ke]'
  },
  'SO:0001961': {
    'name': '5_formylcytosine',
    'def': 'A modified DNA cytosine base feature, modified by a formyl group at the 5 carbon. [SO:ke]'
  },
  'SO:0001962': {
    'name': 'modified_adenine',
    'def': 'A modified adenine DNA base feature. [SO:ke]'
  },
  'SO:0001963': {
    'name': 'modified_cytosine',
    'def': 'A modified cytosine DNA base feature. [SO:ke]'
  },
  'SO:0001964': {
    'name': 'modified_guanine',
    'def': 'A modified guanine DNA base feature. [SO:ke]'
  },
  'SO:0001965': {
    'name': '8_oxoguanine',
    'def': 'A modified DNA guanine base,at the 8 carbon, often the product of DNA damage. [SO:ke]'
  },
  'SO:0001966': {
    'name': '5_carboxylcytosine',
    'def': 'A modified DNA cytosine base feature, modified by a carboxy group at the 5 carbon. [SO:ke]'
  },
  'SO:0001967': {
    'name': '8_oxoadenine',
    'def': 'A modified DNA adenine base,at the 8 carbon, often the product of DNA damage. [SO:ke]'
  },
  'SO:0001968': {
    'name': 'coding_transcript_variant',
    'def': 'A transcript variant of a protein coding gene. [SO:ke]'
  },
  'SO:0001969': {
    'name': 'coding_transcript_intron_variant',
    'def': 'A transcript variant occurring within an intron of a coding transcript. [SO:ke]'
  },
  'SO:0001970': {
    'name': 'non_coding_transcript_intron_variant',
    'def': 'A transcript variant occurring within an intron of a non coding transcript. [SO:ke]'
  },
  'SO:0001971': {
    'name': 'zinc_finger_binding_site',
    'def': 'A binding site to which a polypeptide will bind with a zinc finger motif, which is characterized by requiring one or more Zinc 2+ ions for stabilized folding. []'
  },
  'SO:0001972': {
    'name': 'histone_4_acetylation_site',
    'def': 'A histone 4 modification where the modification is the acetylation of the residue. [EBI:nj, ISBN:0815341059, SO:ke]'
  },
  'SO:0001973': {
    'name': 'histone_3_acetylation_site',
    'def': 'A histone 3 modification where the modification is the acetylation of the residue. [EBI:nj, ISBN:0815341059, SO:ke]'
  },
  'SO:0001974': {
    'name': 'CTCF_binding_site',
    'def': 'A transcription factor binding site with consensus sequence CCGCGNGGNGGCAG, bound by CCCTF-binding factor. [EBI:nj]'
  },
  'SO:0001975': {
    'name': 'five_prime_sticky_end_restriction_enzyme_cleavage_site',
    'def': 'A restriction enzyme recognition site that, when cleaved, results in 5 prime overhangs. [SO:ke]'
  },
  'SO:0001976': {
    'name': 'three_prime_sticky_end_restriction_enzyme_cleavage_site',
    'def': 'A restriction enzyme recognition site that, when cleaved, results in 3 prime overhangs. [SO:ke]'
  },
  'SO:0001977': {
    'name': 'ribonuclease_site',
    'def': 'A region of a transcript encoding the cleavage site for a ribonuclease enzyme. [SO:ke]'
  },
  'SO:0001978': {
    'name': 'signature',
    'def': 'A region of sequence where developer information is encoded. [SO:ke]'
  },
  'SO:0001979': {
    'name': 'RNA_stability_element',
    'def': 'A motif that affects the stability of RNA. [PMID:22495308, SO:ke]'
  },
  'SO:0001980': {
    'name': 'G_box',
    'def': 'A regulatory promoter element identified in mutation experiments, with consensus sequence: CACGTG. Present in promoters, intergenic regions, coding regions, and introns. They are involved in gene expression responses to light and interact with G-box binding factor and I-box binding factor 1a. [PMID:19249238, PMID:8571452, SO:ml]'
  },
  'SO:0001981': {
    'name': 'L_box',
    'def': 'An orientation dependent regulatory promoter element, with consensus sequence of TTGCACAN4TTGCACA, found in plants. [PMID:17381552, PMID:2902624, SO:ml]'
  },
  'SO:0001982': {
    'name': 'I-box',
    'def': 'A plant regulatory promoter motif, composed of a highly conserved hexamer GATAAG (I-box core). [PMID:2347304, PMID:2902624, SO:ml]'
  },
  'SO:0001983': {
    'name': '5_prime_UTR_premature_start_codon_variant',
    'def': "A 5' UTR variant where a premature start codon is introduced, moved or lost. [SANGER:am]"
  },
  'SO:0001984': {
    'name': 'silent_mating_type_cassette_array',
    'def': 'A gene cassette array that corresponds to a silenced version of a mating type region. [PomBase:mah]'
  },
  'SO:0001985': {
    'name': 'Okazaki_fragment',
    'def': 'Any of the DNA segments produced by discontinuous synthesis of the lagging strand during DNA replication. [ISBN:0805350152]'
  },
  'SO:0001986': {
    'name': 'upstream_transcript_variant',
    'def': 'A feature variant, where the alteration occurs upstream of the transcript TSS. [EBI:gr]'
  },
  'SO:0001987': {
    'name': 'downstream_transcript_variant',
    'def': 'A feature variant, where the alteration occurs downstream of the transcript termination site. []'
  },
  'SO:0001988': {
    'name': '5_prime_UTR_premature_start_codon_gain_variant',
    'def': "A 5' UTR variant where a premature start codon is gained. [Sanger:am]"
  },
  'SO:0001989': {
    'name': '5_prime_UTR_premature_start_codon_loss_variant',
    'def': "A 5' UTR variant where a premature start codon is lost. [SANGER:am]"
  },
  'SO:0001990': {
    'name': 'five_prime_UTR_premature_start_codon_location_variant',
    'def': "A 5' UTR variant where a premature start codon is moved. [SANGER:am]"
  },
  'SO:0001991': {
    'name': 'consensus_AFLP_fragment',
    'def': 'A consensus AFLP fragment is an AFLP sequence produced from any alignment algorithm which uses assembled multiple AFLP sequences as input. [GMOD:ea]'
  },
  'SO:0001992': {
    'name': 'nonsynonymous_variant',
    'def': 'A non-synonymous variant is an inframe, protein altering variant, resulting in a codon change. [SO:ke]'
  },
  'SO:0001993': {
    'name': 'extended_cis_splice_site',
    'def': 'Intronic positions associated with cis-splicing. Contains the first and second positions immediately before the exon and the first, second and fifth positions immediately after. [SANGER:am]'
  },
  'SO:0001994': {
    'name': 'intron_base_5',
    'def': "Fifth intronic position after the intron exon boundary, close to the 5' edge of the intron. [SANGER:am]"
  },
  'SO:0001995': {
    'name': 'extended_intronic_splice_region_variant',
    'def': 'A sequence variant occurring in the intron, within 10 bases of exon. [sanger:am]'
  },
  'SO:0001996': {
    'name': 'extended_intronic_splice_region',
    'def': 'Region of intronic sequence within 10 bases of an exon. [SANGER:am]'
  },
  'SO:0001997': {
    'name': 'subtelomere',
    'def': 'A heterochromatic region of the chromosome, adjacent to the telomere (on the centromeric side) that contains repetitive DNA and sometimes genes and it is transcribed. [POMBE:al]'
  },
  'SO:0001998': {
    'name': 'sgRNA',
    'def': 'A small RNA oligo, typically about 20 bases, that guides the cas nuclease to a target DNA sequence in the CRISPR/cas mutagenesis method. [PMID:23934893]'
  },
  'SO:0001999': {
    'name': 'mating_type_region_motif',
    'def': 'DNA motif that is a component of a mating type region. [SO:ke]'
  },
  'SO:0002001': {
    'name': 'Y_region',
    'def': 'A segment of non-homology between a and alpha mating alleles, found at all three mating loci (HML, MAT, and HMR), has two forms (Ya and Yalpha). [SGD:jd]'
  },
  'SO:0002002': {
    'name': 'Z1_region',
    'def': 'A mating type region motif, one of two segments of homology found at all three mating loci (HML, MAT, and HMR). [SGD:jd]'
  },
  'SO:0002003': {
    'name': 'Z2_region',
    'def': 'A mating type region motif, the rightmost segment of homology in the HML and MAT mating loci (not present in HMR). [SGD:jd]'
  },
  'SO:0002004': {
    'name': 'ARS_consensus_sequence',
    'def': "The ACS is an 11-bp sequence of the form 5'-WTTTAYRTTTW-3' which is at the core of every yeast ARS, and is necessary but not sufficient for recognition and binding by the origin recognition complex (ORC). Functional ARSs require an ACS, as well as other cis elements in the 5' (C domain) and 3' (B domain) flanking sequences of the ACS. [SGD:jd]"
  },
  'SO:0002005': {
    'name': 'DSR_motif',
    'def': 'The determinant of selective removal (DSR) motif consists of repeats of U(U/C)AAAC. The motif targets meiotic transcripts for removal during mitosis via the exosome. [PMID:22645662]'
  },
  'SO:0002006': {
    'name': 'zinc_repressed_element',
    'def': 'A promoter element that has the consensus sequence GNMGATC, and is found in promoters of genes repressed in the presence of zinc. [PMID:24003116, POMBE:mh]'
  },
  'SO:0002007': {
    'name': 'MNV',
    'def': 'An MNV is a multiple nucleotide variant (substitution) in which the inserted sequence is the same length as the replaced sequence. [NCBI:th]'
  },
  'SO:0002008': {
    'name': 'rare_amino_acid_variant',
    'def': 'A sequence variant whereby at least one base of a codon encoding a rare amino acid is changed, resulting in a different encoded amino acid. [SO:ke]'
  },
  'SO:0002009': {
    'name': 'selenocysteine_loss',
    'def': 'A sequence variant whereby at least one base of a codon encoding selenocysteine is changed, resulting in a different encoded amino acid. [SO:ke]'
  },
  'SO:0002010': {
    'name': 'pyrrolysine_loss',
    'def': 'A sequence variant whereby at least one base of a codon encoding pyrrolysine is changed, resulting in a different encoded amino acid. [SO:ke]'
  },
  'SO:0002011': {
    'name': 'intragenic_variant',
    'def': 'A variant that occurs within a gene but falls outside of all transcript features. This occurs when alternate transcripts of a gene do not share overlapping sequence. [SO:ke]'
  },
  'SO:0002012': {
    'name': 'start_lost',
    'def': 'A codon variant that changes at least one base of the canonical start codon. [SO:ke]'
  },
  'SO:0002013': {
    'name': '5_prime_UTR_truncation',
    'def': "A sequence variant that causes the reduction of a the 5'UTR with regard to the reference sequence. [SO:ke]"
  },
  'SO:0002014': {
    'name': '5_prime_UTR_elongation',
    'def': "A sequence variant that causes the extension of 5' UTR, with regard to the reference sequence. [SO:ke]"
  },
  'SO:0002015': {
    'name': '3_prime_UTR_truncation',
    'def': "A sequence variant that causes the reduction of a the 3' UTR with regard to the reference sequence. [SO:ke]"
  },
  'SO:0002016': {
    'name': '3_prime_UTR_elongation',
    'def': "A sequence variant that causes the extension of 3' UTR, with regard to the reference sequence. [SO:ke]"
  },
  'SO:0002017': {
    'name': 'conserved_intergenic_variant',
    'def': 'A sequence variant located in a conserved intergenic region, between genes. [SO:ke]'
  },
  'SO:0002018': {
    'name': 'conserved_intron_variant',
    'def': 'A transcript variant occurring within a conserved region of an intron. [SO:ke]'
  },
  'SO:0002019': {
    'name': 'start_retained_variant',
    'def': 'A sequence variant where at least one base in the start codon is changed, but the start remains. [SO:ke]'
  },
  'SO:0002020': {
    'name': 'boundary_element',
    'def': 'Boundary elements are DNA motifs that prevent heterochromatin from spreading into neighboring euchromatic regions. [PMID:24013502]'
  },
  'SO:0002021': {
    'name': 'mating_type_region_replication_fork_barrier',
    'def': 'A DNA motif that is found in eukaryotic rDNA repeats, and is a site of replication fork pausing. [PMID:17614787]'
  },
  'SO:0002022': {
    'name': 'priRNA',
    'def': 'A small RNA molecule, 22-23 nt in size, that is the product of a longer RNA. The production of priRNAs is independent of dicer and involves binding of RNA by argonaute and trimming by triman. In fission yeast, priRNAs trigger the establishment of heterochromatin. PriRNAs are primarily generated from centromeric transcripts (dg and dh repeats), but may also be produced from degradation products of primary transcripts. [PMID:20178743, PMID:24095277, PomBase:al]'
  },
  'SO:0002023': {
    'name': 'multiplexing_sequence_identifier',
    'def': 'A nucleic tag which is used in a ligation step of library preparation process to allow pooling of samples while maintaining ability to identify individual source material and creation of a multiplexed library. [OBO:prs, PMID:22574170]'
  },
  'SO:0002024': {
    'name': 'W_region',
    'def': 'The leftmost segment of homology in the HML and MAT mating loci, but not present in HMR. [SGD:jd]'
  },
  'SO:0002025': {
    'name': 'cis_acting_homologous_chromosome_pairing_region',
    'def': 'A genome region where chromosome pairing occurs preferentially during homologous chromosome pairing during early meiotic prophase of Meiosis I. [PMID:22582262, PMID:23117617, PMID:24173580, PomBase:vw]'
  },
  'SO:0002026': {
    'name': 'intein_encoding_region',
    'def': 'The nucleotide sequence which encodes the intein portion of the precursor gene. [PMID:8165123]'
  },
  'SO:0002027': {
    'name': 'uORF',
    'def': "A short open reading frame that is found in the 5' untranslated region of an mRNA and plays a role in translational regulation. [PMID:12890013, PMID:16153175, POMBASE:mah]"
  },
  'SO:0002028': {
    'name': 'sORF',
    'def': 'An open reading frame that encodes a peptide of less than 100 amino acids. [PMID:23970561, PMID:24705786, POMBASE:mah]'
  },
  'SO:0002029': {
    'name': 'tnaORF',
    'def': 'A translated ORF encoded entirely within the antisense strand of a known protein coding gene. [POMBASE:vw]'
  },
  'SO:0002030': {
    'name': 'X_region',
    'def': 'One of two segments of homology found at all three mating loci (HML, MAT and HMR). [SGD:jd]'
  },
  'SO:0002031': {
    'name': 'shRNA',
    'def': 'A short hairpin RNA (shRNA) is an RNA transcript that makes a tight hairpin turn that can be used to silence target gene expression via RNA interference. [PMID:6699500, SO:ke]'
  },
  'SO:0002032': {
    'name': 'moR',
    'def': "A non-coding transcript encoded by sequences adjacent to the ends of the 5' and 3' miR-encoding sequences that abut the loop in precursor miRNA. [SO:ke]"
  },
  'SO:0002033': {
    'name': 'loR',
    'def': 'A short, non coding transcript of loop-derived sequences encoded in precursor miRNA. [SO:ke]'
  },
  'SO:0002034': {
    'name': 'miR_encoding_snoRNA_primary_transcript',
    'def': 'A snoRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002035': {
    'name': 'lncRNA_primary_transcript',
    'def': 'A primary transcript encoding a lncRNA. [SO:ke]'
  },
  'SO:0002036': {
    'name': 'miR_encoding_lncRNA_primary_transcript',
    'def': 'A lncRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002037': {
    'name': 'miR_encoding_tRNA_primary_transcript',
    'def': 'A tRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002038': {
    'name': 'shRNA_primary_transcript',
    'def': 'A primary transcript encoding an shRNA. [SO:ke]'
  },
  'SO:0002039': {
    'name': 'miR_encoding_shRNA_primary_transcript',
    'def': 'A shRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002040': {
    'name': 'vaultRNA_primary_transcript',
    'def': 'A primary transcript encoding a vaultRNA. [SO:ke]'
  },
  'SO:0002041': {
    'name': 'miR_encoding_vaultRNA_primary_transcript',
    'def': 'A vaultRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002042': {
    'name': 'Y_RNA_primary_transcript',
    'def': 'A primary transcript encoding a Y-RNA. [SO:ke]'
  },
  'SO:0002043': {
    'name': 'miR_encoding_Y_RNA_primary_transcript',
    'def': 'A Y-RNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA. [SO:ke]'
  },
  'SO:0002044': {
    'name': 'TCS_element',
    'def': 'A TCS element is a (yeast) transcription factor binding site, bound by the TEA DNA binding domain (DBD) of transcription factors. The consensus site is CATTCC or CATTCT. [PMID:1489142, PMID:20118212, SO:ke]'
  },
  'SO:0002045': {
    'name': 'pheromone_response_element',
    'def': 'A PRE is a (yeast) TFBS with consensus site [TGAAAC(A/G)]. [PMID:1489142, SO:ke]'
  },
  'SO:0002046': {
    'name': 'FRE',
    'def': 'A FRE is an enhancer element necessary and sufficient to confer filamentation associated expression in S. cerevisiae. [PMID:1489142, SO:ke]'
  },
  'SO:0002047': {
    'name': 'transcription_pause_site',
    'def': 'Transcription pause sites are regions of a gene where RNA polymerase may pause during transcription. The functional role of pausing may be to facilitate factor recruitment, RNA folding, and synchronization with translation. Consensus transcription pause site have been observed in E. coli. [PMID:24789973, SO:ke]'
  },
  'SO:0002048': {
    'name': 'disabled_reading_frame',
    'def': 'A reading frame that could encode a full-length protein but which contains obvious mid-sequence disablements (frameshifts or premature stop codons). [SGD:se]'
  },
  'SO:0002049': {
    'name': 'H3K27_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is acetylated. [SO:rs]'
  },
  'SO:0002050': {
    'name': 'constitutive_promoter',
    'def': 'A promoter that allows for continual transcription of gene. [SO:ke]'
  },
  'SO:0002051': {
    'name': 'inducible_promoter',
    'def': 'A promoter whereby activity is induced by the presence or absence of biotic or abiotic factors. [SO:ke]'
  },
  'SO:0002052': {
    'name': 'dominant_negative_variant',
    'def': 'A variant where the mutated gene product adversely affects the other (wild type) gene product. [SO:ke]'
  },
  'SO:0002053': {
    'name': 'gain_of_function_variant',
    'def': 'A sequence variant whereby new or enhanced function is conferred on the gene product. [SO:ke]'
  },
  'SO:0002054': {
    'name': 'loss_of_function_variant',
    'def': 'A sequence variant whereby the gene product has diminished or abolished function. [SO:ke]'
  },
  'SO:0002055': {
    'name': 'null_mutation',
    'def': 'A variant whereby the gene product is not functional or the gene product is not produced. [SO:ke]'
  },
  'SO:0002056': {
    'name': 'intronic_splicing_silencer',
    'def': 'An intronic splicing regulatory element that functions to recruit trans acting splicing factors suppress the transcription of the gene or genes they control. [PMID:23241926, SO:ke]'
  },
  'SO:0002057': {
    'name': 'intronic_splicing_enhancer'
  },
  'SO:0002058': {
    'name': 'exonic_splicing_silencer',
    'def': 'An exonic splicing regulatory element that functions to recruit trans acting splicing factors suppress the transcription of the gene or genes they control. [PMID:23241926, SO:ke]'
  },
  'SO:0002059': {
    'name': 'recombination_enhancer',
    'def': 'A regulatory_region that promotes or induces the process of recombination. [PMID:8861911, SGD:se]'
  },
  'SO:0002060': {
    'name': 'interchromosomal_translocation',
    'def': 'A translocation where the regions involved are from different chromosomes. [NCBI:th]'
  },
  'SO:0002061': {
    'name': 'intrachromosomal_translocation',
    'def': 'A translocation where the regions involved are from the same chromosome. [NCBI:th]'
  },
  'SO:0002062': {
    'name': 'complex_chromosomal_rearrangement',
    'def': 'A contiguous cluster of translocations, usually the result of a single catastrophic event such as chromothripsis or chromoanasynthesis. [NCBI:th]'
  },
  'SO:0002063': {
    'name': 'Alu_insertion',
    'def': 'An insertion of sequence from the Alu family of mobile elements. [NCBI:th]'
  },
  'SO:0002064': {
    'name': 'LINE1_insertion',
    'def': 'An insertion from the Line1 family of mobile elements. [NCBI:th]'
  },
  'SO:0002065': {
    'name': 'SVA_insertion',
    'def': 'An insertion of sequence from the SVA family of mobile elements. [NCBI:th]'
  },
  'SO:0002066': {
    'name': 'mobile_element_deletion',
    'def': 'A deletion of a mobile element when comparing a reference sequence (has mobile element) to a individual sequence (does not have mobile element). [NCBI:th]'
  },
  'SO:0002067': {
    'name': 'HERV_deletion',
    'def': 'A deletion of the HERV mobile element with respect to a reference. [NCBI:th]'
  },
  'SO:0002068': {
    'name': 'SVA_deletion',
    'def': 'A deletion of an SVA mobile element. [NCBI:th]'
  },
  'SO:0002069': {
    'name': 'LINE1_deletion',
    'def': 'A deletion of a LINE1 mobile element with respect to a reference. [NCBI:th]'
  },
  'SO:0002070': {
    'name': 'Alu_deletion',
    'def': 'A deletion of an Alu mobile element with respect to a reference. [NCBI:th]'
  },
  'SO:0002071': {
    'name': 'CDS_supported_by_peptide_spectrum_match',
    'def': 'A CDS that is supported by proteomics data. [SO:ke]'
  },
  'SO:0002072': {
    'name': 'sequence_comparison',
    'def': 'A position or feature where two sequences have been compared. []'
  },
  'SO:0002073': {
    'name': 'no_sequence_alteration',
    'def': 'A position or feature within a sequence that is identical to the comparable position or feature of a specified reference sequence. [SO:ke]'
  },
  'SO:0002074': {
    'name': 'intergenic_1kb_variant',
    'def': 'A variant that falls in an intergenic region that is 1 kb or less between 2 genes. [SO:ke]'
  },
  'SO:0002075': {
    'name': 'incomplete_transcript_variant',
    'def': 'A sequence variant that intersects an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002076': {
    'name': 'incomplete_transcript_3UTR_variant',
    'def': "A sequence variant that intersects the 3' UTR of an incompletely annotated transcript. [SO:ke]"
  },
  'SO:0002077': {
    'name': 'incomplete_transcript_5UTR_variant',
    'def': "A sequence variant that intersects the 5' UTR of an incompletely annotated transcript. [SO:ke]"
  },
  'SO:0002078': {
    'name': 'incomplete_transcript_intronic_variant',
    'def': 'A sequence variant that intersects the intron of an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002079': {
    'name': 'incomplete_transcript_splice_region_variant',
    'def': 'A sequence variant that intersects the splice region of an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002080': {
    'name': 'incomplete_transcript_exonic_variant',
    'def': 'A sequence variant that intersects the exon of an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002081': {
    'name': 'incomplete_transcript_CDS',
    'def': 'A sequence variant that intersects the coding regions of an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002082': {
    'name': 'incomplete_transcript_coding_splice_variant',
    'def': 'A sequence variant that intersects the coding sequence near a splice region of an incompletely annotated transcript. [SO:ke]'
  },
  'SO:0002083': {
    'name': '2KB_downstream_variant',
    'def': "A sequence variant located within 2KB 3' of a gene. [SO:ke]"
  },
  'SO:0002084': {
    'name': 'exonic_splice_region_variant',
    'def': 'A sequence variant in which a change has occurred within the exonic region of the splice site, 1-2 bases from boundary. [SO:ke]'
  },
  'SO:0002085': {
    'name': 'unidirectional_gene_fusion',
    'def': 'A sequence variant whereby two genes, on the same strand have become joined. [SO:ke]'
  },
  'SO:0002086': {
    'name': 'bidirectional_gene_fusion',
    'def': 'A sequence variant whereby two genes, on alternate strands have become joined. [SO:ke]'
  },
  'SO:0002087': {
    'name': 'pseudogenic_CDS',
    'def': 'A non functional descendant of the coding portion of a coding transcript, part of a pseudogene. [SO:ke]'
  },
  'SO:0002088': {
    'name': 'non_coding_transcript_splice_region_variant',
    'def': 'A transcript variant occurring within the splice region (1-3 bases of the exon or 3-8 bases of the intron) of a non coding transcript. [SO:ke]'
  },
  'SO:0002089': {
    'name': '3_prime_UTR_exon_variant',
    'def': "A UTR variant of exonic sequence of the 3' UTR. [SO:ke]"
  },
  'SO:0002090': {
    'name': '3_prime_UTR_intron_variant',
    'def': "A UTR variant of intronic sequence of the 3' UTR. [SO:ke]"
  },
  'SO:0002091': {
    'name': '5_prime_UTR_intron_variant',
    'def': "A UTR variant of intronic sequence of the 5' UTR. [SO:ke]"
  },
  'SO:0002092': {
    'name': '5_prime_UTR_exon_variant',
    'def': "A UTR variant of exonic sequence of the 5' UTR. [SO:ke]"
  },
  'SO:0002093': {
    'name': 'structural_interaction_variant',
    'def': 'A variant that impacts the internal interactions of the resulting polypeptide structure. [SO:ke]'
  },
  'SO:0002094': {
    'name': 'non_allelic_homologous_recombination_region',
    'def': 'A genomic region at a non-allelic position where exchange of genetic material happens as a result of homologous recombination. []'
  },
  'SO:0002095': {
    'name': 'scaRNA',
    'def': "A ncRNA, specific to the Cajal body, that has been demonstrated to function as a guide RNA in the site-specific synthesis of 2'-O-ribose-methylated nucleotides and pseudouridines in the RNA polymerase II-transcribed U1, U2, U4 and U5 spliceosomal small nuclear RNAs (snRNAs). [PMC:126017, PMID:27775477, PMID:28869095, SO:nrs]"
  },
  'SO:0002096': {
    'name': 'short_tandem_repeat_variation',
    'def': 'A variation that expands or contracts a tandem repeat with regard to a reference. [SO:ke]'
  },
  'SO:0002097': {
    'name': 'vertebrate_immune_system_pseudogene',
    'def': 'A pseudogene derived from a vertebrate immune system gene. [SO:ke]'
  },
  'SO:0002098': {
    'name': 'immunoglobulin_pseudogene',
    'def': 'A pseudogene derived from an immunoglobulin gene. [SO:ke]'
  },
  'SO:0002099': {
    'name': 'T_cell_receptor_pseudogene',
    'def': 'A pseudogene derived from a T-cell receptor gene. [SO:ke]'
  },
  'SO:0002100': {
    'name': 'IG_C_pseudogene',
    'def': 'A pseudogenic constant region of an immunoglobulin gene which closely resembles a known functional Imunoglobulin constant gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002101': {
    'name': 'IG_J_pseudogene',
    'def': 'A pseudogenic joining region which closely resembles a known functional imunoglobulin joining gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002102': {
    'name': 'IG_V_pseudogene',
    'def': 'A pseudogenic variable region which closely resembles a known functional imunoglobulin variable gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the variable region of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002103': {
    'name': 'TR_V_pseudogene',
    'def': 'A pseudogenic variable region which closely resembles a known functional T receptor variable gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the variable region of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002104': {
    'name': 'TR_J_pseudogene',
    'def': 'A pseudogenic joining region which closely resembles a known functional T receptor (TR) joining gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002105': {
    'name': 'translated_processed_pseudogene',
    'def': 'A processed pseudogene where there is evidence, (mass spec data) suggesting that it is also translated. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002106': {
    'name': 'translated_unprocessed_pseudogene',
    'def': 'A non-processed pseudogene where there is evidence, (mass spec data) suggesting that it is also translated. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002107': {
    'name': 'transcribed_unprocessed_pseudogene',
    'def': 'A unprocessed pseudogene supported by locus-specific evidence of transcription. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002108': {
    'name': 'transcribed_unitary_pseudogene',
    'def': 'A species specific unprocessed pseudogene without a parent gene, as it has an active orthologue in another species. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002109': {
    'name': 'transcribed_processed_pseudogene',
    'def': 'A processed_pseudogene overlapped by locus-specific evidence of transcription. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002110': {
    'name': 'polymorphic_pseudogene_with_retained_intron',
    'def': 'A polymorphic pseudogene in the reference genome, containing a retained intron, known to be intact in the genomes of other individuals of the same species. The annotation process has confirmed that the pseudogenisation event is not a genomic sequencing error. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002111': {
    'name': 'pseudogene_processed_transcript',
    'def': 'A processed_transcript supported by EST and/or mRNA evidence that aligns unambiguously to a pseudogene locus (i.e. alignment to the pseudogene locus clearly better than alignment to parent locus). [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002112': {
    'name': 'coding_transcript_with_retained_intron',
    'def': 'A protein coding transcript containing a retained intron. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002113': {
    'name': 'lncRNA_with_retained_intron',
    'def': 'A lncRNA transcript containing a retained intron. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002114': {
    'name': 'NMD_transcript',
    'def': 'A protein coding transcript that contains a CDS but has one or more splice junctions >50bp downstream of stop codon, making it susceptible to nonsense mediated decay. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002115': {
    'name': 'pseudogenic_transcript_with_retained_intron',
    'def': 'A transcript supported by EST and/or mRNA evidence that aligns unambiguously to the pseudogene locus; has retained intronic sequence compared to a reference transcript sequence. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002116': {
    'name': 'polymorphic_pseudogene_processed_transcript',
    'def': 'A processed transcript that does not contain a CDS that fullfills annotation criteria and not necessarily functionally non-coding. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002117': {
    'name': '<new term>'
  },
  'SO:0002118': {
    'name': 'NMD_polymorphic_pseudogene_transcript',
    'def': 'A polymorphic pseudogene transcript that contains a CDS but has one or more splice junctions >50bp downstream of stop codon. Premature stop codon is not introduced, directly or indirectly, as a result of the variation i.e. must be present in both protein_coding and pseudogenic alleles. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002119': {
    'name': 'allelic_frequency',
    'def': 'A physical quality which inheres to the allele by virtue of the number instances of the allele within a population. This is the relative frequency of the allele at a given locus in a population. [SO:ke]'
  },
  'SO:0002120': {
    'name': 'three_prime_overlapping_ncrna',
    'def': "Transcript where ditag (digital gene expression profiling)and/or published experimental data strongly supports the existence of short non-coding transcripts transcribed from the 3'UTR. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]"
  },
  'SO:0002121': {
    'name': 'vertebrate_immune_system_gene',
    'def': 'The configuration of the IG and TR variable (V), diversity (D) and joining (J) germline genes before DNA rearrangements (with or without constant (C) genes in undefined configuration. (germline, non rearranged regions of the IG DNA loci). [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002122': {
    'name': 'immunoglobulin_gene',
    'def': 'A germline immunoglobulin gene. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002123': {
    'name': 'IG_C_gene',
    'def': 'A constant (C) gene, a gene that codes the constant region of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002124': {
    'name': 'IG_D_gene',
    'def': 'A gene that rearranges at the DNA level and codes the diversity region of the variable domain of an immunoglobuin (IG) gene. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002125': {
    'name': 'IG_J_gene',
    'def': 'A joining gene that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002126': {
    'name': 'IG_V_gene',
    'def': 'A variable gene that rearranges at the DNA level and codes the variable region of the variable domain of an Immunoglobulin chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002127': {
    'name': 'lncRNA_gene',
    'def': 'A gene that encodes a long non-coding RNA. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002128': {
    'name': 'mt_rRNA',
    'def': 'Mitochondrial rRNA is an RNA component of the small or large subunits of mitochondrial ribosomes. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002129': {
    'name': 'mt_tRNA',
    'def': 'Mitochondrial transfer RNA. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002130': {
    'name': 'NSD_transcript',
    'def': 'A transcript that contains a CDS but has no stop codon before the polyA site is reached. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002131': {
    'name': 'sense_intronic_lncRNA',
    'def': 'A long non-coding transcript found within an intron of a coding or non-coding gene, with no overlap of exonic sequence. [GENECODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002132': {
    'name': 'sense_overlap_lncRNA',
    'def': 'A long non-coding transcript that contains a protein coding gene within its intronic sequence on the same strand, with no overlap of exonic sequence. [GENECODE:http://www.gencodegenes.org/gencode_biotypes.html]'
  },
  'SO:0002133': {
    'name': 'T_cell_receptor_gene',
    'def': 'A T-cell receptor germline gene. []'
  },
  'SO:0002134': {
    'name': 'TR_C_Gene',
    'def': 'A constant (C) gene, a gene that codes the constant region of a T-cell receptor chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002135': {
    'name': 'TR_D_Gene',
    'def': 'A gene that rearranges at the DNA level and codes the diversity region of the variable domain of aT-cell receptor gene. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002136': {
    'name': 'TR_J_Gene',
    'def': 'A joining gene that rearranges at the DNA level and codes the joining region of the variable domain of aT-cell receptor chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002137': {
    'name': 'TR_V_Gene',
    'def': 'A variable gene that rearranges at the DNA level and codes the variable region of the variable domain of aT-cell receptor chain. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html, IGMT:http://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]'
  },
  'SO:0002138': {
    'name': 'predicted_transcript',
    'def': 'A transcript feature that has been predicted but is not yet validated. [SO:ke]'
  },
  'SO:0002139': {
    'name': 'unconfirmed_transcript',
    'def': "This is used for non-spliced EST clusters that have polyA features. This category has been specifically created for the ENCODE project to highlight regions that could indicate the presence of protein coding genes that require experimental validation, either by 5' RACE or RT-PCR to extend the transcripts, or by confirming expression of the putatively-encoded peptide with specific antibodies. [GENCODE:http://www.gencodegenes.org/gencode_biotypes.html]"
  },
  'SO:0002140': {
    'name': 'early_origin_of_replication',
    'def': 'An origin of replication that initiates early in S phase. [PMID:23348837, PMID:9115207]'
  },
  'SO:0002141': {
    'name': 'late_origin_of_replication',
    'def': 'An origin of replication that initiates late in S phase. [PMID:23348837, PMID:9115207]'
  },
  'SO:0002142': {
    'name': 'histone_2A_acetylation_site',
    'def': 'A histone 2A modification where the modification is the acetylation of the residue. [ISBN:0815341059]'
  },
  'SO:0002143': {
    'name': 'histone_2B_acetylation_site',
    'def': 'A histone 2B modification where the modification is the acetylation of the residue. [ISBN:0815341059]'
  },
  'SO:0002144': {
    'name': 'histone_2AZ_acetylation_site',
    'def': 'A histone 2AZ modification where the modification is the acetylation of the residue. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002145': {
    'name': 'H2AZK4_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H2AZ histone protein is acetylated. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002146': {
    'name': 'H2AZK7_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 7th residue (a lysine), from the start of the H2AZ histone protein is acetylated. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002147': {
    'name': 'H2AZK11_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 11th residue (a lysine), from the start of the H2AZ histone protein is acetylated. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002148': {
    'name': 'H2AZK13_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 13th residue (a lysine), from the start of the H2AZ histone protein is acetylated. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002149': {
    'name': 'H2AZK15_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 15th residue (a lysine), from the start of the H2AZ histone protein is acetylated. [PMID:19385636, PMID:24316985, PMID:27087541]'
  },
  'SO:0002150': {
    'name': 'AUG_initiated_uORF',
    'def': 'A uORF beginning with the canonical start codon AUG. [PMID:26684391, PMID:27313038]'
  },
  'SO:0002151': {
    'name': 'non_AUG_initiated_uORF',
    'def': 'A uORF beginning with a codon other than AUG. [PMID:26684391, PMID:27313038]'
  },
  'SO:0002152': {
    'name': 'genic_downstream_transcript_variant',
    'def': 'A variant that falls downstream of a transcript, but within the genic region of the gene due to alternately transcribed isoforms. [NCBI:dm, SO:ke]'
  },
  'SO:0002153': {
    'name': 'genic_upstream_transcript_variant',
    'def': 'A variant that falls upstream of a transcript, but within the genic region of the gene due to alternately transcribed isoforms. [NCBI:dm, SO:ke]'
  },
  'SO:0002154': {
    'name': 'mitotic_recombination_region',
    'def': 'A genomic region where there is an exchange of genetic material with another genomic region, occurring in somatic cells. [NCBI:cf, SO:ke]'
  },
  'SO:0002155': {
    'name': 'meiotic_recombination_region',
    'def': 'A genomic region in which there is an exchange of genetic material as a result of the repair of meiosis-specific double strand breaks that occur during meiotic prophase. [NCBI:cf, SO:ke]'
  },
  'SO:0002156': {
    'name': 'CArG_box',
    'def': "A promoter element bound by the MADS family of transcription factors with consensus 5'-(C/T)TA(T/A)4TA(G/A)-3'. [PMID:1748287, PMID:7623803]"
  },
  'SO:0002157': {
    'name': 'Mat2P',
    'def': 'A gene cassette array containing H+ mating type specific information. [PMID:18354497]'
  },
  'SO:0002158': {
    'name': 'Mat3M',
    'def': 'A gene cassette array containing H- mating type specific information. [PMID:18354497]'
  },
  'SO:0002159': {
    'name': 'SHP_box',
    'def': 'A conserved Cdc48/p97 interaction motif with strict consensus sequence F[PI]GKG[TK][RK]LG[GT] and relaxed consensus sequence FXGKGX[RK]LG. [PMID:17083136, PMID:27655872]'
  },
  'SO:0002160': {
    'name': 'sequence_length_variant',
    'def': 'A sequence variant that changes the length of one or more sequence features. []'
  },
  'SO:0002161': {
    'name': 'short_tandem_repeat_change',
    'def': 'A sequence variant where the copies of a short tandem repeat (STR) feature are either contracted or expanded. []'
  },
  'SO:0002162': {
    'name': 'short_tandem_repeat_expansion',
    'def': 'A short tandem repeat variant containing more repeat units than the reference sequence. []'
  },
  'SO:0002163': {
    'name': 'short_tandem_repeat_contraction',
    'def': 'A short tandem repeat variant containing fewer repeat units than the reference sequence. []'
  },
  'SO:0002164': {
    'name': 'H2BK5_acetylation_site',
    'def': 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2B histone protein is acetylated. [http://www.actrec.gov.in/histome/ptm_sp.php?ptm_sp=H2BK5ac, PMID:18552846]'
  },
  'SO:0002165': {
    'name': 'trinucleotide_repeat_expansion',
    'def': 'A short tandem repeat expansion with an increase in a sequence of three nucleotide units repeated in tandem compared to a reference sequence. []'
  },
  'SO:0002166': {
    'name': 'ref_miRNA',
    'def': 'A ref_miRNA (RefSeq-miRNA) sequence is assigned at the creation of a new mature miRNA entry in a database. The ref_miRNA sequence designation remains unchanged even if a different isomiR is later shown to be expressed at a higher level. A ref_miRNA can be produced by one or multiple pre-miRNA. [PMID:26453491]'
  },
  'SO:0002167': {
    'name': 'isomiR',
    'def': 'IsomiRs are all the bona fide variants of a mature product. IsomiRs should be connected to the ref_miRNA it is most likely to be the variant of. Some isomiRs can be variations of one or multiple ref_miRNA. [PMID:26453491]'
  },
  'SO:0002168': {
    'name': 'RNA_thermometer',
    'def': "An RNA_thermometer is a cis element in the 5' end of an mRNA that can change its secondary structure in response to temperature and coordinate temperature-dependent gene expression. [PMID:22421878]"
  },
  'SO:0002169': {
    'name': 'splice_polypyrimidine_tract_variant',
    'def': "A sequence variant that falls in the polypyrimidine tract at 3' end of intron between 17 and 3 bases from the end (acceptor -3 to acceptor -17). []"
  },
  'SO:0002170': {
    'name': 'splice_donor_region_variant',
    'def': "A sequence variant that falls in the region between the 3rd and 6th base after splice junction (5' end of intron). []"
  },
  'SO:0002171': {
    'name': 'telomeric_D_loop',
    'def': "A telomeric D-loop is a three-stranded DNA displacement loop that forms at the site where the telomeric 3' single-stranded DNA overhang (formed of the repeat sequence TTAGGG in mammals) is tucked back inside the double-stranded component of telomeric DNA molecule, thus forming a t-loop or telomeric-loop and protecting the chromosome terminus. [PMID:10338204, PMID:15071557, PMID:24012755]"
  },
  'SO:0002172': {
    'name': 'sequence_alteration_artifact',
    'def': 'A sequence_alteration where the source of the alteration is due to an artifact in the base-calling or assembly process. []'
  },
  'SO:0002173': {
    'name': 'indel_artifact',
    'def': 'An indel that is the result of base-calling or assembly error. []'
  },
  'SO:0002174': {
    'name': 'deletion_artifact',
    'def': 'A deletion that is the result of base-calling or assembly error. []'
  },
  'SO:0002175': {
    'name': 'insertion_artifact',
    'def': 'An insertion that is the result of base-calling or assembly error. []'
  },
  'SO:0002176': {
    'name': 'substitution_artifact',
    'def': 'A substitution that is the result of base-calling or assembly error. []'
  },
  'SO:0002177': {
    'name': 'duplication_artifact',
    'def': 'A duplication that is the result of base-calling or assembly error. []'
  },
  'SO:0002178': {
    'name': 'SNV_artifact',
    'def': 'An SNV that is the result of base-calling or assembly error. []'
  },
  'SO:0002179': {
    'name': 'MNV_artifact',
    'def': 'An MNV that is the result of base-calling or assembly error. []'
  },
  'SO:0002180': {
    'name': 'enzymatic_RNA_gene',
    'def': 'A gene that encodes an enzymatic RNA. []'
  },
  'SO:0002181': {
    'name': 'ribozyme_gene',
    'def': 'A gene that encodes a ribozyme. []'
  },
  'SO:0002182': {
    'name': 'antisense_lncRNA_gene',
    'def': 'A gene that encodes an antisense long, non-coding RNA. []'
  },
  'SO:0002183': {
    'name': 'sense_overlap_lncRNA_gene',
    'def': 'A gene that encodes a sense overlap long non-coding RNA. []'
  },
  'SO:0002184': {
    'name': 'sense_intronic_lncRNA_gene',
    'def': 'A gene that encodes a sense intronic long non-coding RNA. []'
  },
  'SO:0002185': {
    'name': 'bidirectional_promoter_lncRNA_gene',
    'def': 'A non-coding locus that originates from within the promoter region of a protein-coding gene, with transcription proceeding in the opposite direction on the other strand. [https://www.gencodegenes.org/pages/biotypes.html]'
  },
  'SO:0002186': {
    'name': 'mutational_hotspot',
    'def': 'A region of genomic sequence known to undergo mutational events with greater frequency than expected by chance. []'
  },
  'SO:0002187': {
    'name': 'HERV_insertion',
    'def': 'An insertion of sequence from the HERV family of mobile elements with respect to a reference. [NCBI:th]'
  },
  'SO:0002188': {
    'name': 'functional_gene_region',
    'def': 'A gene_member_region that encodes sequence that directly contributes to the molecular function of its gene or gene product. [Clingen:mb]'
  },
  'SO:0002189': {
    'name': 'allelic_pseudogene',
    'def': 'A (unitary) pseudogene that is stable in the population but importantly it has a functional alternative allele also in the population. i.e., one strain may have the gene, another strain may have the pseudogene. MHC haplotypes have allelic pseudogenes. []'
  },
  'SO:0002190': {
    'name': 'enhancer_blocking_element',
    'def': "A transcriptional cis regulatory region that when located between an enhancer and a gene's promoter prevents the enhancer from modulating the expression of the gene. Sometimes referred to as an insulator but may not include the barrier function of an insulator. [NCBI:cf]"
  },
  'SO:0002191': {
    'name': 'imprinting_control_region',
    'def': 'A regulatory region that controls epigenetic imprinting and affects the expression of target genes in an allele- or parent-of-origin-specific manner. Associated regulatory elements may include differentially methylated regions and non-coding RNAs. []'
  },
  'SO:0002192': {
    'name': 'flanking_repeat',
    'def': 'A repeat lying outside the sequence for which it has functional significance (eg. transposon insertion target sites). []'
  },
  'SO:0002193': {
    'name': 'processed_pseudogenic_rRNA',
    'def': 'The pseudogene has arisen by reverse transcription of a mRNA into cDNA, followed by reintegration into the genome. Therefore, it has lost any intron/exon structure, and it might have a pseudo-polyA-tail. []'
  },
  'SO:0002194': {
    'name': 'unprocessed_pseudogenic_rRNA',
    'def': 'The pseudogene has arisen from a copy of the parent gene by duplication followed by accumulation of random mutation. The changes, compared to their functional homolog, include insertions, deletions, premature stop codons, frameshifts and a higher proportion of non-synonymous versus synonymous substitutions. []'
  },
  'SO:0002195': {
    'name': 'unitary_pseudogenic_rRNA',
    'def': 'The pseudogene has no parent. It is the original gene, which is functional in some species but disrupted in some way (indels, mutation, recombination) in another species or strain. []'
  },
  'SO:0002196': {
    'name': 'allelic_pseudogenic_rRNA',
    'def': 'A (unitary) pseudogene that is stable in the population but importantly it has a functional alternative allele also in the population. i.e., one strain may have the gene, another strain may have the pseudogene. MHC haplotypes have allelic pseudogenes. []'
  },
  'SO:0002197': {
    'name': 'processed_pseudogenic_tRNA',
    'def': 'The pseudogene has arisen by reverse transcription of a mRNA into cDNA, followed by reintegration into the genome. Therefore, it has lost any intron/exon structure, and it might have a pseudo-polyA-tail. []'
  },
  'SO:0002198': {
    'name': 'unprocessed_pseudogenic_tRNA',
    'def': 'The pseudogene has arisen from a copy of the parent gene by duplication followed by accumulation of random mutation. The changes, compared to their functional homolog, include insertions, deletions, premature stop codons, frameshifts and a higher proportion of non-synonymous versus synonymous substitutions. []'
  },
  'SO:0002199': {
    'name': 'unitary_pseudogenic_tRNA',
    'def': 'The pseudogene has no parent. It is the original gene, which is functional in some species but disrupted in some way (indels, mutation, recombination) in another species or strain. []'
  },
  'SO:0002200': {
    'name': 'allelic_pseudogenic_tRNA',
    'def': 'A (unitary) pseudogene that is stable in the population but importantly it has a functional alternative allele also in the population. i.e., one strain may have the gene, another strain may have the pseudogene. MHC haplotypes have allelic pseudogenes. []'
  },
  'SO:0002201': {
    'name': 'terminal_repeat',
    'def': 'A repeat at the ends of and within the sequence for which it has functional significance other than long terminal repeat. []'
  },
  'SO:0002202': {
    'name': 'repeat_instability_region',
    'def': 'A repeat region that is prone to expansions and/or contractions. []'
  },
  'SO:0002203': {
    'name': 'replication_start_site',
    'def': 'A nucleotide site from which replication initiates. [NCBI:cf]'
  },
  'SO:0002204': {
    'name': 'nucleotide_cleavage_site',
    'def': 'A point in nucleic acid where a cleavage event occurs. []'
  },
  'SO:0002205': {
    'name': 'response_element',
    'def': 'A regulatory element that acts in response to a stimulus, usually via transcription factor binding. []'
  },
  'SO:0002206': {
    'name': 'sequence_source',
    'def': 'Identifies the biological source of the specified span of the sequence [NCBI:tm]'
  },
  'SO:0002207': {
    'name': 'UNAAAC_motif',
    'def': 'A hexameric RNA motif consisting of nucleotides UNAAAC (where N can be any nucleotide) that targets the RNA for degradation. [PMID:22645662, PMID:28765164, PomBase:al]'
  },
  'SO:0002208': {
    'name': 'long_terminal_repeat_transcript',
    'def': 'An RNA that is transcribed from a long terminal repeat. [PMID:24256266, PomBase:mh]'
  },
  'SO:0002209': {
    'name': 'genomic_DNA_contig',
    'def': 'A contig composed of genomic DNA derived sequences. [BCS:etrwz]'
  },
  'SO:0002210': {
    'name': 'presence_absence_variation',
    'def': 'A variation qualifying the presence of a sequence in a genome which is entirely missing in another genome. [BCS:bbean, PMID:19956538, PMID:25881062]'
  },
  'SO:0002211': {
    'name': 'circular_plasmid',
    'def': 'A self replicating circular nucleic acid molecule that is distinct from a chromosome in the organism. [PMID:21719542, SBOL:jb]'
  },
  'SO:0002212': {
    'name': 'linear_plasmid',
    'def': "A self replicating linear nucleic acid molecule that is distinct from a chromosome in the organism. They are capped by terminal proteins covalently bound to the 5' ends of the DNA. [PMID:21719542, SBOL:jb]"
  },
  'SO:0002213': {
    'name': 'transcription_termination_signal',
    'def': 'Termination signal preferentially observed downstream of polyadenylation signal [PMID:28367989]'
  },
  'SO:0002214': {
    'name': 'redundant_inserted_stop_gained',
    'def': 'A sequence variant whereby at least one base of a codon is changed, resulting in a stop codon inserted next to an existing stop codon. This leads to a polypeptide of the same length. []'
  },
  'SO:0002215': {
    'name': 'Zas1_recognition_motif',
    'def': "A DNA motif to which the S. pombe Zas1 protein binds. The consensus sequence is 5'-(Y)CCCCAY-3'. [PMID:29735745, PomBase:vw]"
  },
  'SO:0002216': {
    'name': 'Pho7_binding_site',
    'def': "A promoter element with consensus sequence [5'-TCG(G/C)(A/T)xxTTxAA], bound by the transcription factor Pho7. [PMID:28811350]"
  },
  'SO:0002217': {
    'name': 'unspecified_indel',
    'def': 'A sequence alteration which includes an insertion or a deletion. This describes a sequence length change when the direction of the change is unspecified or when such changes are pooled into one category. [ZFIN:st]'
  },
  'SO:0002218': {
    'name': 'functionally_abnormal',
    'def': 'A sequence variant in which the function of a gene product is altered with respect to a reference. []'
  },
  'SO:0002219': {
    'name': 'functionally_normal',
    'def': 'A sequence variant in which the function of a gene product is retained with respect to a reference. []'
  },
  'SO:0002220': {
    'name': 'function_uncertain_variant',
    'def': 'A sequence variant in which the function of a gene product is unknown with respect to a reference. []'
  },
  'SO:0002221': {
    'name': 'eukaryotic_promoter',
    'def': 'A regulatory_region including the Transcription Start Site (TSS) of a gene and serving as a platform for Pre-Initiation Complex (PIC) assembly, enabling transcription of a gene under certain conditions. []'
  },
  'SO:0002222': {
    'name': 'prokaryotic_promoter',
    'def': 'A regulatory_region essential for the specific initiation of transcription at a defined location in a DNA molecule, although this location might not be one single base. It is recognized by a specific RNA polymerase(RNAP)-holoenzyme, and this recognition is not necessarily autonomous. [PMID:32665585]'
  },
  'SO:0002223': {
    'name': 'inert_DNA_spacer',
    'def': "Sequences that decrease interactions between biological regions, such as between a promoter, its 5' context and/or the translational unit(s) it regulates. Spacers can affect regulation of translation, transcription, and other biological processes. [PMID:20843779, PMID:24933158, PMID:27034378, PMID:28422998]"
  },
  'SO:0002224': {
    'name': '2A_self_cleaving_peptide_region',
    'def': 'A region that codes for a 2A self-cleaving polypeptide region, which is a region that can result in a break in the peptide sequence at its terminal G-P junction. [PMID:22301656, PMID:28526819]'
  },
  'SO:0002225': {
    'name': 'LOZ1_response_element',
    'def': "A conserved sequence (5'-CGNMGATCNTY-3') transcription repressor binding site required for gene repression in the presence of high zinc. []"
  },
  'SO:0002226': {
    'name': 'group_IIC_intron',
    'def': 'A group II intron that recognizes IBS1/EBS1 for the 5-prime exon and IBS3/EBS3 for the 3-prime exon and may also recognize a stem-loop in the RNA. [PMID:20463000]'
  },
  'SO:0002227': {
    'name': 'CDS_extension',
    'def': 'A sequence variant extending the CDS, that causes elongation of the resulting polypeptide sequence. [PMID:14732127, PMID:15864293, PMID:27984720, PMID:31216041, PMID:32020195]'
  },
  'SO:0002228': {
    'name': 'CDS_five_prime_extension',
    'def': "A sequence variant extending the CDS at the 5' end, that causes elongation of the resulting polypeptide sequence at the N terminus. [PMID:14732127, PMID:15864293, PMID:27984720, PMID:31216041, PMID:32020195]"
  },
  'SO:0002229': {
    'name': 'CDS_three_prime_extension',
    'def': "A sequence variant extending the CDS at the 3' end, that causes elongation of the resulting polypeptide sequence at the C terminus. [PMID:14732127, PMID:15864293, PMID:27984720, PMID:31216041, PMID:32020195]"
  },
  'SO:0002230': {
    'name': 'CAAX_box',
    'def': 'A C-terminus protein motif (CAAX) serving as a post-translational prenylation site modified by the attachment of either a farnesyl or a geranyl-geranyl group to a cysteine residue. Farnesyltransferase recognizes CaaX boxes where X = M, S, Q, A, or C, whereas Geranylgeranyltransferase I recognizes CaaX boxes with X = L or E. []'
  },
  'SO:0002231': {
    'name': 'self_cleaving_ribozyme',
    'def': 'An RNA that catalyzes its own cleavage. []'
  },
  'SO:0002232': {
    'name': 'selection_marker',
    'def': 'A genetic feature that encodes a trait used for artificial selection of a subpopulation. []'
  },
  'SO:0002233': {
    'name': 'homologous_chromosome_recognition_and_pairing_locus',
    'def': 'A chromosomal locus where complementary lncRNA and associated proteins accumulate at the corresponding lncRNA gene loci to tether homologous chromosome during chromosome pairing at meiosis I. [PMID:22582262, PMID:31811152]'
  },
  'SO:0002234': {
    'name': 'pumilio_response_element',
    'def': "A cis-acting element involved in RNA stability found in the 3' UTR of some RNA (consensus UGUAAAUA). [PMID:30601114]"
  },
  'SO:0002235': {
    'name': 'SUMO_interaction_motif',
    'def': 'A polypeptide region that mediates binding to SUMO. The motif contains a hydrophobic core sequence consisting of three or four Ile, Leu, or Val residues plus one acidic or polar residue at position 2 or 3. [PMID:15388847,PMID:16524884]'
  },
  'SO:0002236': {
    'name': 'cytosolic_rRNA_18S_gene',
    'def': 'A gene which codes for 18S_rRNA, which functions as the small subunit of the ribosome in eukaryotes. []'
  },
  'SO:0002237': {
    'name': 'cytosolic_rRNA_16S_gene',
    'def': 'A gene which codes for 16S_rRNA, which functions as the small subunit of the ribosome in prokaryotes. []'
  },
  'SO:0002238': {
    'name': 'cytosolic_rRNA_5S_gene',
    'def': 'A gene which codes for 5S_rRNA, which is a portion of the large subunit of the ribosome in both eukaryotes and prokaryotes. []'
  },
  'SO:0002239': {
    'name': 'cytosolic_rRNA_28S_gene',
    'def': 'A gene which codes for 28S_rRNA, which functions as a component of the large subunit of the ribosome in eukaryotes. []'
  },
  'SO:0002240': {
    'name': 'cytosolic_rRNA_5_8S_gene',
    'def': 'A gene which codes for 5_8S_rRNA (5.8S rRNA), which functions as a component of the large subunit of the ribosome in eukaryotes. []'
  },
  'SO:0002241': {
    'name': 'rRNA_21S_gene',
    'def': 'A gene which codes for 21S_rRNA, which functions as a component of the large subunit of the ribosome in mitochondria. []'
  },
  'SO:0002242': {
    'name': 'cytosolic_rRNA_25S_gene',
    'def': 'A gene which codes for 25S_rRNA, which functions as a component of the large subunit of the ribosome in some eukaryotes. []'
  },
  'SO:0002243': {
    'name': 'cytosolic_rRNA_23S_gene',
    'def': 'A gene which codes for 23S_rRNA, which functions as a component of the large subunit of the ribosome in prokaryotes. []'
  },
  'SO:0002244': {
    'name': 'partially_duplicated_transcript',
    'def': 'A transcript which is partially duplicated due to duplication of DNA, leading to a new transcript that is only partial and likely nonfunctional. []'
  },
  'SO:0002245': {
    'name': 'five_prime_duplicated_transcript',
    'def': "A partially_duplicated_transcript where the 5' end of the transcript is duplicated. []"
  },
  'SO:0002246': {
    'name': 'three_prime_duplicated_transcript',
    'def': "A partially_duplicated_transcript where the 3' end of the transcript is duplicated. []"
  },
  'SO:0002247': {
    'name': 'sncRNA',
    'def': 'A non-coding RNA less than 200 nucleotides in length. [PMID:30069443]'
  },
  'SO:0002248': {
    'name': 'spurious_protein',
    'def': 'A region of DNA that is predicted to be translated and transcribed into a protein by a protein detection algorithm that does not get transcribed in nature. [PMID:21771858]'
  },
  'SO:0002249': {
    'name': 'mature_protein_region_of_CDS',
    'def': 'A CDS region corresponding to a mature protein region of a polypeptide. []'
  },
  'SO:0002250': {
    'name': 'propeptide_region_of_CDS',
    'def': 'A CDS region corresponding to a propeptide of a polypeptide. []'
  },
  'SO:0002251': {
    'name': 'signal_peptide_region_of_CDS',
    'def': 'A CDS region corresponding to a signal peptide of a polypeptide. []'
  },
  'SO:0002252': {
    'name': 'transit_peptide_region_of_CDS',
    'def': 'CDS region corresponding to a transit peptide region of a polypeptide. []'
  },
  'SO:0002253': {
    'name': 'stem_loop_region',
    'def': 'A portion of a stem loop secondary structure in RNA. []'
  },
  'SO:0002254': {
    'name': 'loop',
    'def': 'The loop portion of a stem loop, which is not folded back upon itself. []'
  },
  'SO:0002255': {
    'name': 'stem',
    'def': 'The portion of a stem loop where the RNA is folded back upon itself. []'
  },
  'SO:0002256': {
    'name': 'non_complimentary_stem',
    'def': 'A region of a stem in a stem loop structure where the sequences are non-complimentary. []'
  },
  'SO:0002257': {
    'name': 'knob',
    'def': 'Cytologically observable heterochromatic regions of chromosomes away from centromeres that contain predominatly large tandem repeats and retrotransposons. [PMID:6439888]'
  },
  'SO:0002258': {
    'name': 'teb1_recognition_motif',
    'def': 'A binding motif with the consensus sequence TTAGGG to which Teb1 binds. [PMID:23314747, PMID:27901072]'
  },
  'SO:0002259': {
    'name': 'polyA_site_cluster',
    'def': 'A region defined by a cluster of experimentally determined polyadenylation sites, typically less than 25 bp in length and associated with a single polyadenylation signal. [PMID:17202160, PMID:24072873, PMID:25906188]'
  },
  'SO:0002260': {
    'name': 'LARD',
    'def': 'Large Retrotransposon Derivative elements are long-terminal repeats that contain reverse transcriptase priming sites and are conserved in sequence but contain no open reading frames encoding typical retrotransposon proteins . The LARDs identified in barley and other Triticeae have LTRs ~5.5 kb and an interal domain of ~3.5 kb. LARDs lack coding domains and thus do not encode proteins. [PMID:15082561]'
  },
  'SO:0002261': {
    'name': 'TRIM',
    'def': 'TRIM elements have terminal direct repeat sequences of 100-250 bp in length that flank an internal domain of 100â300 bp. TRIMs lack coding domains and thus do not encode proteins. [PMID:11717436]'
  },
  'SO:0002262': {
    'name': 'Watson_strand',
    'def': "An absolute reference to the strand. When a chromosome has p and q arms, the Watson strand is the strand whose 5'-end is on the short arm of the chromosome. Of note, the term 'plus strand' is typically based on a reference sequence where it's preferred for the plus strand to be the Watson strand, but might not be and 'plus strand' is therefore not an exact synonym. [PMID:21303550]"
  },
  'SO:0002263': {
    'name': 'Crick_strand',
    'def': "An absolute reference to the strand. When a chromosome has p and q arms, the Crick strand is the strand whose 5'-end is on the long arm of the chromosome. Of note, the term 'minus strand' is typically based on a reference sequence where it's preferred for the minus strand to be the Crick strand, but might not be and 'minus strand' is therefore not an exact synonym. [PMID:21303550]"
  },
  'SO:0002264': {
    'name': 'Copia_LTR_retrotransposon',
    'def': 'LTR retrotransposons in the Copia superfamily contain elements coding for specific proteins in this order: GAG, AP, INT, RT, RH. GAG is a structural protein for virus-like particles. AP is aspartic proteinase. INT is a DDE integrase. RT is a reverse transcriptase. RH is RNAse H. [PMID:17984973]'
  },
  'SO:0002265': {
    'name': 'Gypsy_LTR_retrotransposon',
    'def': 'LTR retrotransposons in the Gypsy superfamily contain elements coding for specific proteins in this order: GAG, AP, RT, RH, INT. GAG is a structural protein for virus-like particles. AP is aspartic proteinase. INT is a DDE integrase. RT is a reverse transcriptase. RH is RNAse H. [PMID:17984973]'
  },
  'SO:0002266': {
    'name': 'Bel_Pao_LTR_retrotransposon',
    'def': 'LTR retrotransposons in the Bel-Pao superfamily are similar to LTRs in the Gypsy and Retrovirus superfamilies. Mainly described in metazoan genomes, this superfamily contain elements coding for specific proteins in this order: GAG, AP, RT, RH and INT. GAG is a structural protein for virus-like particles. AP is aspartic proteinase. INT is a DDE integrase. RT is a reverse transcriptase. RH is RNAse H. [PMID:17984973]'
  },
  'SO:0002267': {
    'name': 'Retrovirus_LTR_retrotransposon',
    'def': 'LTR retrotransposons in the retrovirus superfamily are similar to LTR retrotransposons in the Gypsy and Bel-Pao superfamilies. Mainly described in vertebrate animals, this superfamily contain elements coding for specific proteins in this order: GAG, AP, RT, RH, INT, and ENV. GAG is a structural protein for virus-like particles. AP is aspartic proteinase. INT is a DDE integrase. RT is a reverse transcriptase. RH is RNAse H. ENV is envelop protein. [PMID:17984973]'
  },
  'SO:0002268': {
    'name': 'Endogenous_Retrovirus_LTR_retrotransposon',
    'def': 'Endogenous retrovirus (ERV) retrotransposons are abundant in the genomes of jawed vertebrates. Human ERVs (HERVs) are classified based on their homologies to animal retroviruses. Class I families are similar in sequence to mammalian Gammaretroviruses (type C) and Epsilonretroviruses (Type E). Class II families show homology to mammalian Betaretroviruses (Type B) and Deltaretroviruses (Type D). F-Class III families are similar to foamy viruses. [PMID:17984973]'
  },
  'SO:0002269': {
    'name': 'R2_LINE_retrotransposon',
    'def': "R2 retrotransposons are LINE elements (SO:0000194) that insert site-specifically into the host organism's 28S ribosomal RNA (rRNA) genes. [PMID:21734471]"
  },
  'SO:0002270': {
    'name': 'RTE_LINE_retrotransposon',
    'def': 'RTE retrotransposons are LINE elements (SO:0000194) that contain a domain with homology to the apurinic-apyrimidic (AP) endonucleases in addition to the previously identified reverse transcriptase domain. [PMID:9729877]'
  },
  'SO:0002271': {
    'name': 'Jockey_LINE_retrotransposon',
    'def': 'Jockey retrotransposons are LINE elements (SO:0000194) found only in arthropods. The full-length element is ~â5âkb and contains two open reading frames (SO:0000236), ORF1 (568 aa) and ORF2 (916 aa), the second of which encodes an apurinic endonuclease (APE) and a reverse transcriptase (RT). [PMID:31709017]'
  },
  'SO:0002272': {
    'name': 'L1_LINE_retrotransposon',
    'def': 'Long interspersed element-1 (LINE-1) elements are found in the human genome, which contains ORF1 (open reading frame1, including CC, coiled coil; RRM, RNA recognition motif; CTD, carboxyl-terminal domain) and ORF2 (including EN, endonuclease; RT, reverse transcriptase; C, cysteine-rich domain). The L1-encoded proteins (ORF1p and ORF2p) can mobilize nonautonomous retrotransposons, other noncoding RNAs, and messenger RNAs. [PMID:31709017]'
  },
  'SO:0002273': {
    'name': 'I_LINE_retrotransposon',
    'def': 'Elements of the LINE I superfamily are similar to the Jockey and L1 superfamily. They contains two ORFs, the.second of which includesÂ  Apurinic endonuclease (APE) andÂ  reverse transcriptase (RT). The I superfamily encodes an RH (RNase H) domain downstream of the RT domain. []'
  },
  'SO:0002274': {
    'name': 'tRNA_SINE_retrotransposon',
    'def': 'Short interspersed elements that originated from tRNAs. [PMID:21673742]'
  },
  'SO:0002275': {
    'name': '7SL_SINE_retrotransposon',
    'def': 'Short interspersed elements that originated from 7SL RNAs. [PMID:21673742]'
  },
  'SO:0002276': {
    'name': '5S_SINE_retrotransposon',
    'def': 'Short interspersed elements that originated from 5S rRNAs. [PMID:21673742]'
  },
  'SO:0002277': {
    'name': 'Crypton_YR_transposon',
    'def': 'Crypton is a superfamily of DNA transposons that use tyrosine recombinase (YR) to cut and rejoin the recombining DNA molecules. [PMID:22011512]'
  },
  'SO:0002278': {
    'name': 'Tc1_Mariner_TIR_transposon',
    'def': 'Elements of the Tc1-Mariner terminal inverted repeat transposon superfamily (also called mariner transposons) are named after the Transponon of C. elegans number 1 transposasse. Their activity creates a 2-bp (TA) target-site duplication (TSD). Stowaway is the non-autonomous element in this superfamily usually shorter than 600 bp. [PMID:17984973, PMID:8556864]'
  },
  'SO:0002279': {
    'name': 'hAT_TIR_transposon',
    'def': 'The hAT terminal inverted repeat transposon superfamily elements were first found in maize (the Ac/Ds elements). Members of the hAT superfamily have TSDs of 8 bp, relatively short TIRs of 5â27 bp and overall lengths of less than 4 kb. [PMID:11454746]'
  },
  'SO:0002280': {
    'name': 'Mutator_TIR_transposon',
    'def': 'Members of the Mutator family of terminal inverted repeat (TIR) transposon are usually long but are also highly divergent, either sharing only terminal Gâ¦C nucleotides, or with the Gâ¦C nucleotides absent. The length of the TSD (7-11 bp, usually 9 bp) remains probably the most useful criterion for identification. [PMID:17984973]'
  },
  'SO:0002281': {
    'name': 'Merlin_TIR_transposon',
    'def': 'Terminal inverted repeat transposon superfamily Merlin elements create 8-9 bp target-site duplications (TSD). [PMID:17984973]'
  },
  'SO:0002282': {
    'name': 'Transib_TIR_transposon',
    'def': 'Terminal inverted repeat (TIR) transposons of the superfamily Transib contain the DDE motif, which is related to the RAG1 protein involved in V(D)J recombination. [PMID:17984973]'
  },
  'SO:0002283': {
    'name': 'piggyBac_TIR_transposon',
    'def': 'Primarily found in animals, the terminal inverted repeat (TIR) transposon superfamily piggyBac elements favour insertion adjacent to TTAA. [PMID:17984973]'
  },
  'SO:0002284': {
    'name': 'PIF_Harbinger_TIR_transposon',
    'def': "Terminal inverted repeat transposons in the PIF/Harbinger/tourist superfamily create 3-bp target site duplication that are mainly 'TAA' or 'TTA'. The autonomous PIF-Harbinger elements are relatively small in size, usually a few kb in length. Non-autonomous elements in this superfamily usually shorter than 600 bp are referrred to as Tourist elements. The terminal sequences for PIF/Harbinger/Tourist elements are 'GGG/CCCâ¦GGC/GCC' or 'GA/GGCAâ¦TGCC/TC'. [PMID:26709091]"
  },
  'SO:0002285': {
    'name': 'CACTA_TIR_transposon',
    'def': "This terminal inverted repeat of the CACTA family generate 3-bp target site duplication (TSD) upon insertion. CACTA elements do not have a significant preference for genic region insertions. This terminal inverted repeat (TIR) transposon superfamily is named CACTA because their terminal sequences are 'CACTA/Gâ¦C/TAGTG'. [PMID:26709091]"
  },
  'SO:0002286': {
    'name': 'YR_retrotransposon',
    'def': 'Tyrosine Kinase (YR) retrotransposons are a subclass of non-LTR retrotransposons. These YR-encoding elements consist of central gag, pol and tyrosine recombinase (YR) open reading frames (ORFs) flanked with terminal repeat. The pol ORF includes a reverse transcriptase (RT), a RNase H (RH) and, in case of DIRS, a domain similar to bacterial and phage DNA N-6-adenine-methyltransferase (MT). Compared to the retroviral pol (LTR retrotransposons, non-LTR retrotransposons and Penelope elements), both aspartic protease and DDE integrase are absent from YR retrotransposons. YR retrotransposons have inverted terminal repeats (ITRs). [PMID:24086727]'
  },
  'SO:0002287': {
    'name': 'DIRS_YR_retrotransposon',
    'def': 'Dictyostelium intermediate repeat sequence (DIRS) retrotransposons are members of the YR_retrotransposon (SO:0002286) superfamily with the following protein domains: RT, RH, YR, and MT. RT is a reverse transcriptase. RH is RNAse H. YR is tyrosine recombinase. MT is DNA N-6-adenine-methyltransferase. [PMID:24086727]'
  },
  'SO:0002288': {
    'name': 'Ngaro_YR_retrotransposon',
    'def': 'Ngaro retrotransposons are members of the YR_retrotransposon (SO:0002286) superfamily with the following protein domains: RT, RH, YR. RT is a reverse transcriptase. RH is RNAse H. YR is Tyrosine recombinase. Inverted terminal repeats (ITRs) in Ngaro are arranged in A-pol-B-A-B order where A and B represent ITRs. [PMID:24086727]'
  },
  'SO:0002289': {
    'name': 'Viper_YR_retrotransposon',
    'def': 'VIPER retrotransposons are members of the YR_retrotransposon (SO:0002286 superfamily with protein domains: RT, RH, YR. RT is a reverse transcriptase. RH is RNAse H. YR is Tyrosine recombinase. Inverted terminal repeats (ITRs) in VIPER are arranged in A-pol-B-A-B order where A and B represent ITRs. VIPER is only found in kinetoplastida genomes. [PMID:16297462]'
  },
  'SO:0002290': {
    'name': 'Penelope_retrotransposon',
    'def': 'Penelope is a subclass of non_LTR_retrotransposons (SO:0000189). Penelope retrotransposons contains structural features of TR, RT, EN, TR, terminal repeats which can be in tandem or inverse orientation in different Penelope copies. RT is reverse transcriptase. EN is endonuclease. [PMID:23914310]'
  },
  'SO:0002291': {
    'name': 'circular_ncRNA',
    'def': 'A non-coding RNA that is generated by backsplicing of exons or introns, resulting in a covalently closed loop without a 5â cap or 3â polyA tail. [PMID:29086764, PMID:29182528, PMID:29230098, PMID:29576969, PMID:29626935]'
  },
  'SO:0002292': {
    'name': 'circular_mRNA',
    'def': 'An mRNA that is generated by backsplicing of exons or introns, resulting in a covalently closed loop without a 5â cap or 3â polyA tail. [PMID:29086764, PMID:29182528, PMID:29576969]'
  },
  'SO:0002293': {
    'name': 'mitochondrial_control_region',
    'def': 'The non-coding region of the mitochondrial genome that controls RNA and DNA synthesis. [PMID: 19407924, PMID:10968878]'
  },
  'SO:0002294': {
    'name': 'mitochondrial_D_loop',
    'def': 'Mitochondrial displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0002295': {
    'name': 'transcription_factor_regulatory_site',
    'def': 'A TF_binding_site that is involved in regulation of expression. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002296': {
    'name': 'TFRS_module',
    'def': 'The possible discontinuous stretch of DNA that is the combination of one or several TFRSs whose bound TFs work jointly in the regulation of a promoter. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002297': {
    'name': 'TFRS_collection',
    'def': 'The possible discontinous stretch of DNA that encompass all the TFRSs that regulate a promoter. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002298': {
    'name': 'simple_operon',
    'def': 'An operon whose transcription is coordinated on a single transcription unit. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002299': {
    'name': 'complex_operon',
    'def': 'An operon whose transcription is coordinated on several mutually overlapping transcription units transcribed in the same direction and sharing at least one gene. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002300': {
    'name': 'unit_of_gene_expression',
    'def': 'Transcription units or transcribed coding sequences. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002301': {
    'name': 'transcription_unit',
    'def': 'DNA regions delimited by different nonspurious TSS-TTS pairs. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002302': {
    'name': 'simple_regulon',
    'def': 'A regulon defined by considering one regulatory gene product. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002303': {
    'name': 'complex_regulon',
    'def': 'A regulon defined by considering the units of expression regulated by a specified set of regulatory gene products. [Bacterial_regulation_working_group:CMA, PMID:32665585]'
  },
  'SO:0002304': {
    'name': 'topologically_associated_domain',
    'def': 'An instance of a self-interacting DNA region flanked by left and right TAD boundaries. [GREEKC:cl, PMID:32782014]'
  },
  'SO:0002305': {
    'name': 'topologically_associated_domain_boundary',
    'def': 'A DNA region enriched in DNA loop anchors and across which DNA loops occur less often than expected by chance. [GREEKC:cl, PMID:32782014]'
  },
  'SO:0002306': {
    'name': 'chromatin_regulatory_region',
    'def': 'A region of a chromosome where regulatory events occur, including epigenetic modifications. These epigenetic modifications can include nucleosome modifications and post-replicational DNA modifications. [GREEKC:cl, PMID:32782014]'
  },
  'SO:0002307': {
    'name': 'DNA_loop',
    'def': 'A region of DNA between two loop anchor positions that are held in close physical proximity. [GREEKC:cl, PMID:32782014]'
  },
  'SO:0002308': {
    'name': 'DNA_loop_anchor',
    'def': 'The ends of a DNA loop where the two strands of DNA are held in close physical proximity. During interphase the anchors of DNA loops are convergently oriented CTCF binding sites. [GREEKC:cl, PMID:32782014]'
  },
  'SO:0002309': {
    'name': 'core_promoter_element',
    'def': 'An element that always exists within the promoter region of a gene. When multiple transcripts exist for a gene, the separate transcripts may have separate core_promoter_elements. [GREEKC:rl]'
  },
  'SO:0002310': {
    'name': 'cryptic_promoter',
    'def': 'The promoter of a cryptic gene. [GREEKC:cl]'
  },
  'SO:0002311': {
    'name': 'viral_promoter',
    'def': 'A regulatory_region including the Transcription Start Site (TSS) of a gene found in genes of viruses. [GREEKC:cl]'
  },
  'SO:0002312': {
    'name': 'core_prokaryotic_promoter_element',
    'def': 'An element that always exists within the promoter region of a prokaryotic gene. [GREEKC:rl]'
  },
  'SO:0002313': {
    'name': 'core_viral_promoter_element',
    'def': 'An element that always exists within the promoter region of a viral gene. [GREEKC:rl]'
  },
  'SO:0002314': {
    'name': 'altered_gene_product_level',
    'def': 'A sequence variant that alters the level or amount of gene product produced. This high level term can be applied where the direction of level change (increased vs decreased gene product level) is unknown or not confirmed. [GenCC:AR]'
  },
  'SO:0002315': {
    'name': 'increased_gene_product_level',
    'def': 'A variant that increases the level or amount of gene product produced. [GenCC:AR]'
  },
  'SO:0002316': {
    'name': 'decreased_gene_product_level',
    'def': 'A sequence variant that decreases the level or amount of gene product produced. [GenCC:AR]'
  },
  'SO:0002317': {
    'name': 'absent_gene_product',
    'def': 'A sequence variant that results in no gene product. [GenCC:AR]'
  },
  'SO:0002318': {
    'name': 'altered_gene_product_sequence',
    'def': 'A sequence variant that alters the sequence of a gene product. [GenCC:AR]'
  },
  'SO:0002319': {
    'name': 'NMD_triggering_variant',
    'def': 'A sequence variant that leads to a change in the location of a termination codon in a transcript that leads to nonsense-mediated decay (NMD). The change in location of a termination codon can be caused by several different types of sequence variants, including stop_gained (SO:0001587), frameshift_variant (SO:0001589), splice_donor_variant (SO:0001575), and splice_acceptor_variant (SO:0001574) types of variants. [GenCC:AR]'
  },
  'SO:0002320': {
    'name': 'NMD_escaping_variant',
    'def': 'A sequence variant that leads to a change in the location of a termination codon in a transcript but allows the transcript to escape nonsense-mediated decay (NMD). The change in location of a termination codon can be caused by several different types of sequence variants, including stop_gained (SO:0001587), frameshift_variant (SO:0001589), splice_donor_variant (SO:0001575), and splice_acceptor_variant (SO:0001574) types of variants. [GenCC:AR]'
  },
  'SO:0002321': {
    'name': 'stop_gained_NMD_triggering',
    'def': 'A stop_gained (SO:0001587) variant that is degraded by nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002322': {
    'name': 'stop_gained_NMD_escaping',
    'def': 'A stop_gained (SO:0001587) variant that allows the transcript to escape nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002323': {
    'name': 'frameshift_variant_NMD_triggering',
    'def': 'A frameshift_variant (SO:0001589) that is degraded by nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002324': {
    'name': 'frameshift_variant_NMD_escaping',
    'def': 'A frameshift_variant (SO:0001589) that allows the transcript to escape nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002325': {
    'name': 'splice_donor_variant_NMD_triggering',
    'def': 'A splice_donor_variant (SO:0001575) that is degraded by nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002326': {
    'name': 'splice_donor_variant_NMD_escaping',
    'def': 'A splice_donor_variant (SO:0001575) that allows the transcript to escape nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002327': {
    'name': 'splice_acceptor_variant_NMD_triggering',
    'def': 'A splice_acceptor_variant (SO:0001574) that is degraded by nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002328': {
    'name': 'splice_acceptor_variant_NMD_escaping',
    'def': 'A splice_acceptor_variant (SO:0001574) that allows the transcript to escape nonsense-mediated decay (NMD). [GenCC:AR]'
  },
  'SO:0002329': {
    'name': 'minus_1_translational_frameshift',
    'def': 'The region of mRNA 1 base long that is included as part of two separate codons during the process of translational frameshifting (GO:0006452), causing the reading frame to be different. [SO:ds]'
  },
  'SO:0002330': {
    'name': 'minus_2_translational_frameshift',
    'def': 'The region of mRNA 2 bases long that is included as part of two separate codons during the process of translational frameshifting (GO:0006452), causing the reading frame to be different. [SO:ds]'
  },
  'SO:0002331': {
    'name': 'accessible_DNA_region',
    'def': 'A region of DNA that is depleted of nucleosomes and accessible to DNA-binding proteins including transcription factors and nucleases. [PMID:25903461, SO:ds]'
  },
  'SO:0002332': {
    'name': 'epigenomically_modified_region',
    'def': 'A biological region implicated in inherited changes caused by mechanisms other than changes in the underlying DNA sequence. [http://en.wikipedia.org/wiki/Epigenetics, SO:ds]'
  },
  'SO:0002333': {
    'name': 'amber_stop_codon',
    'def': 'A stop codon with the DNA sequence TAG. [https://en.wikipedia.org/wiki/Stop_codon]'
  },
  'SO:0002334': {
    'name': 'ochre_stop_codon',
    'def': 'A stop codon with the DNA sequence TAA. [https://en.wikipedia.org/wiki/Stop_codon]'
  },
  'SO:0002335': {
    'name': 'opal_stop_codon',
    'def': 'A stop codon with the DNA sequence TGA. [https://en.wikipedia.org/wiki/Stop_codon]'
  },
  'SO:0002336': {
    'name': 'cytosolic_rRNA_2S_gene',
    'def': 'A gene that encodes for 2S ribosomal RNA, which functions as a component of the large subunit of the ribosome in Drosophila and at least some other Diptera. [PMID: 118436, PMID: 29474379, PMID: 3136294, PMID:10788608, PMID:407103, PMID:4847940, PMID:768488]'
  },
  'SO:0002337': {
    'name': 'cytosolic_2S_rRNA',
    'def': "Cytosolic 2S rRNA is a 30 nucleotide RNA component of the large subunit of cytosolic ribosomes in Drosophila and at least some other Diptera. It is homologous to the 3' part of other 5.8S rRNA molecules. The 3' end of the 5.8S molecule is able to base-pair with the 5' end of the 2S rRNA to generate a helical region equivalent in position to the 'GC-rich hairpin' found in all previously sequenced 5.8S molecules. [PMID: 118436, PMID: 29474379, PMID: 3136294, PMID:10788608, PMID:407103, PMID:4847940, PMID:768488]"
  },
  'SO:0002338': {
    'name': 'U7_snRNA',
    'def': 'A 57 to 71 nucleotide RNA that is a component of the U7 small nuclear ribonucleoprotein complex (U7 snRNP). The U7 snRNP is required for histone pre-mRNA processing. [PMID:15526162]'
  },
  'SO:0002339': {
    'name': 'scaRNA_gene',
    'def': 'A gene that encodes for a scaRNA (small Cajal body-specific RNA). [PMID:27775477, PMID:28869095]'
  },
  'SO:0002340': {
    'name': 'RNA_7SK',
    'def': 'An abundant small nuclear RNA that, together with associated cellular proteins, regulates the activity of the positive transcription elongation factor b (P-TEFb). It is often described in literature as similar to a snRNA, except of longer length. [PMID:19246988, PMID:21853533, PMID:27369380]'
  },
  'SO:0002341': {
    'name': 'RNA_7SK_gene',
    'def': 'A gene encoding a 7SK RNA (SO:0002340). [PMID:19246988, PMID:21853533, PMID:27369380]'
  },
  'SO:0002342': {
    'name': 'sncRNA_gene',
    'def': 'A ncRNA_gene that encodes an ncRNA less than 200 nucleotides in length. [PMID:28449079, PMID:30069443, PMID:30937442]'
  },
  'SO:0002343': {
    'name': 'cytosolic_rRNA',
    'def': 'Cytosolic rRNA is an RNA component of the small or large subunits of cytosolic ribosomes. [PMID:3044395]'
  },
  'SO:0002344': {
    'name': 'mt_SSU_rRNA',
    'def': 'Mitochondrial SSU rRNA is an RNA component of the small subunit of mitochondrial ribosomes. [PMID: 24572720, PMID:3044395]'
  },
  'SO:0002345': {
    'name': 'mt_LSU_rRNA',
    'def': 'Mitochondrial LSU rRNA is an RNA component of the large subunit of mitochondrial ribosomes. [PMID: 24572720, PMID:3044395]'
  },
  'SO:0002346': {
    'name': 'plastid_rRNA',
    'def': 'Plastid rRNA is an RNA component of the small or large subunits of plastid (such as chloroplast) ribosomes. [PMID: 24572720, PMID:3044395]'
  },
  'SO:0002347': {
    'name': 'plastid_SSU_rRNA',
    'def': 'Plastid SSU rRNA is an RNA component of the small subunit of plastid (such as chloroplast) ribosomes. [PMID: 24572720, PMID:3044395]'
  },
  'SO:0002348': {
    'name': 'plastid_LSU_rRNA',
    'def': 'Plastid LSU rRNA is an RNA component of the large subunit of plastid (such as chloroplast) ribosomes. [PMID: 24572720, PMID:3044395]'
  },
  'SO:0002349': {
    'name': 'fragile_site',
    'def': 'A heritable locus on a chromosome that is prone to DNA breakage. []'
  },
  'SO:0002350': {
    'name': 'common_fragile_site',
    'def': 'A fragile site considered part of the normal chromosomal structure. [PMID: 16236432, PMID: 17608616]'
  },
  'SO:0002351': {
    'name': 'rare_fragile_site',
    'def': 'A fragile site found in the chromosomes of less than five percent of the human population. [PMID:16236432, PMID:17608616]'
  },
  'SO:0002352': {
    'name': 'sisRNA',
    'def': 'A non-coding RNA typically derived from intronic sequence of the sense strand of a cognate host gene, that is not rapidly degraded. It may contain exonic sequences, 5â² caps, and/or polyA tails. [PMID:27147469, PMID:29397203, PMID:30391089]'
  },
  'SO:0002353': {
    'name': 'sbRNA_gene',
    'def': 'A gene encoding a stem-bulge RNA. [PMID:25908866, PMID:30666901]'
  },
  'SO:0002354': {
    'name': 'sbRNA',
    'def': 'A small non-coding stem-loop RNA present in nematodes and insects, functionally and structurally related to vertebrate Y RNA. [PMID:25908866, PMID:30666901]'
  },
  'SO:0002355': {
    'name': 'hpRNA_gene',
    'def': 'A gene encoding a hpRNA. [PMID:18463630, PMID:18719707, PMID:25544562]'
  },
  'SO:0002356': {
    'name': 'hpRNA',
    'def': 'An RNA comprising an extended inverted repeat, the stem of which is typically much longer than that of miRNA precursors and can be up to 400 base pairs in length. hpRNAs are processed by Dicer-2 to generate endogenous short interfering RNAs (siRNAs). [PMID:18463630, PMID:18719707, PMID:25544562]'
  },
  'SO:0002357': {
    'name': 'biosynthetic_gene_cluster',
    'def': 'A physically clustered group of two or more genes in a particular genome that together encode a biosynthetic pathway for the production of a specialized metabolite (including its chemical variants). [PMID:26284661]'
  },
  'SO:0002358': {
    'name': 'vault_RNA_gene',
    'def': 'A gene that encodes a vault RNA. [PMID:19298825, PMID:19491402, PMID:22058117, PMID:22926522, PMID:30773316, PMID:9535882]'
  },
  'SO:0002359': {
    'name': 'Y_RNA_gene',
    'def': 'A gene that encodes a Y RNA. [PMID:1698620, PMID:6187471, PMID:6816230, PMID:7520568, PMID:7539809, PMID:8836182]'
  },
  'SO:0002360': {
    'name': 'cytosolic_rRNA_gene',
    'def': 'A gene that codes for cytosolic rRNA. []'
  },
  'SO:0002361': {
    'name': 'cytosolic_LSU_rRNA_gene',
    'def': 'A gene that codes for cytosolic LSU rRNA. []'
  },
  'SO:0002362': {
    'name': 'cytosolic_SSU_rRNA_gene',
    'def': 'A gene that codes for cytosolic SSU rRNA. []'
  },
  'SO:0002363': {
    'name': 'mt_rRNA_gene',
    'def': 'A gene that codes for mitochondrial rRNA. []'
  },
  'SO:0002364': {
    'name': 'mt_LSU_rRNA_gene',
    'def': 'A gene that codes for mitochondrial LSU rRNA. []'
  },
  'SO:0002365': {
    'name': 'mt_SSU_rRNA_gene',
    'def': 'A gene that codes for mitochondrial SSU rRNA. []'
  },
  'SO:0002366': {
    'name': 'plastid_rRNA_gene',
    'def': 'A gene that codes for plastid rRNA. []'
  },
  'SO:0002367': {
    'name': 'plastid_LSU_rRNA_gene',
    'def': 'A gene that codes for plastid LSU rRNA. []'
  },
  'SO:0002368': {
    'name': 'plastid_SSU_rRNA_gene',
    'def': 'A gene that codes for plastid SSU rRNA. []'
  },
  'SO:0002369': {
    'name': 'C_D_box_scaRNA',
    'def': 'A scaRNA possessing a box C/D sequence motif, guiding the methylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002370': {
    'name': 'H_ACA_box_scaRNA',
    'def': 'A scaRNA possessing a box H/ACA sequence motif, guiding the pseudouridylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002371': {
    'name': 'C-D_H_ACA_box_scaRNA',
    'def': 'A scaRNA possessing both box C/D and box H/ACA sequence motifs, guiding both the methylation and pseudouridylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002372': {
    'name': 'C_D_box_scaRNA_gene',
    'def': 'A gene that codes for scaRNA possessing a box C/D sequence motif, guiding the methylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002373': {
    'name': 'H_ACA_box_scaRNA_gene',
    'def': 'A gene that codes for scaRNA possessing a box H/ACA sequence motif, guiding the pseudouridylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002374': {
    'name': 'C-D_H_ACA_box_scaRNA_gene',
    'def': 'A gene that codes for scaRNA possessing both box C/D and box H/ACA sequence motifs, guiding both the methylation and pseudouridylation of snRNAs. [PMID:17099227, PMID:24659245]'
  },
  'SO:0002375': {
    'name': 'C_D_box_snoRNA_gene',
    'def': "A gene that codes a C_D_box_snoRNA. Most box C/D snoRNAs also contain long (>10 nt) sequences complementary to rRNA. Boxes C and D, as well as boxes C' and D', are usually located in close proximity, and form a structure known as the box C/D motif. This motif is important for snoRNA stability, processing, nucleolar targeting and function. A small number of box C/D snoRNAs are involved in rRNA processing; most, however, are known or predicted to serve as guide RNAs in ribose methylation of rRNA. Targeting involves direct base pairing of the snoRNA at the rRNA site to be modified and selection of a rRNA nucleotide a fixed distance from box D or D'. [PMID:12457565, PMID:22065625]"
  },
  'SO:0002376': {
    'name': 'H_ACA_box_snoRNA_gene',
    'def': "A gene that codes for H_ACA_box_snoRNA. Members of the box H/ACA family contain an ACA triplet, exactly 3 nt upstream from the 3' end and an H-box in a hinge region that links two structurally similar functional domains of the molecule. Both boxes are important for snoRNA biosynthesis and function. A few box H/ACA snoRNAs are involved in rRNA processing; most others are known or predicted to participate in selection of uridine nucleosides in rRNA to be converted to pseudouridines. Site selection is mediated by direct base pairing of the snoRNA with rRNA through one or both targeting domains. [PMID:12457565, PMID:22065625]"
  },
  'SO:0002377': {
    'name': 'U14_snoRNA_gene',
    'def': 'A gene that codes for U14_snoRNA. U14 small nucleolar RNA (U14 snoRNA) is required for early cleavages of eukaryotic precursor rRNAs. In yeasts, this molecule possess a stem-loop region (known as the Y-domain) which is essential for function. A similar structure, but with a different consensus sequence, is found in plants, but is absent in vertebrates. []'
  },
  'SO:0002378': {
    'name': 'U3_snoRNA_gene',
    'def': "A gene that codes for U3_snoRNA. U3 snoRNA is a member of the box C/D class of small nucleolar RNAs. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localization, stability and hypermethylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA. []"
  },
  'SO:0002379': {
    'name': 'methylation_guide_snoRNA_gene',
    'def': "A gene that codes for methylation_guide_snoRNA. A snoRNA that specifies the site of 2'-O-ribose methylation in an RNA molecule by base pairing with a short sequence around the target residue. [PMID:12457565]"
  },
  'SO:0002380': {
    'name': 'pseudouridylation_guide_snoRNA_gene',
    'def': 'A gene that codes for pseudouridylation_guide_snoRNA. A snoRNA that specifies the site of pseudouridylation in an RNA molecule by base pairing with a short sequence around the target residue. [PMID:12457565]'
  },
  'SO:0002381': {
    'name': 'bidirectional_promoter_lncRNA',
    'def': 'A long non-coding RNA which is produced using the promoter of a protein-coding gene but with transcription occurring in the opposite direction. [PMID:30175284, PMID:34956340] {comment="PMID:26578749"}'
  },
  'SO:0002382': {
    'name': 'range_extender_element',
    'def': 'A conserved cis-acting element that confers extreme-distance regulatory activity to an enhancer." [https://doi.org/10.1101/2024.05.26.595809] {comment="On its own, a range extender element is not a classical enhancer, but its addition can extend the range of action of a heterologous short-range enhancer by more than 10-fold compared to its native range. In extreme cases, extended ranges may span approximately 840 kilobases of genomic space. The evolutionarily conserved homeodomain moti [C/T]AATA, required for long-range enhancer activity, is present within the range extender element. Range extender elements are distinct from and do not share sequence similarity with other cis-regulatory elements like tethering and remote control elements in Drosophila; or CTCF sites, CpG islands, and enhancer booster elements in mammals. Rather than conferring robustness of remote enhancer activity, a range extender element is both required and sufficient for long-range enhancer-promoter activation."}'
  },
  'SO:0002383': {
    'name': 'oncogenic_variant',
    'def': 'A variant that has been found to be pathogenic in the context of a neoplastic disease. [PMID:35101336]'
  },
  'SO:0002385': {
    'name': '5_prime_UTR_uORF_variant',
    'def': "A 5' UTR variant within an upstream open reading frame. [PMID:32461616, PMID:32926138]"
  },
  'SO:0002386': {
    'name': '5_prime_UTR_uORF_stop_codon_variant',
    'def': "A 5' UTR variant where a stop codon in an upstream open reading frame is introduced, moved or lost. [] {comment='PMID:32461616', comment='PMID:32926138'}"
  },
  'SO:0002387': {
    'name': '5_prime_UTR_uORF_frameshift_variant',
    'def': "A 5' UTR variant which disrupts the translation of an upstream open reading frame because the number of nucleotides inserted or deleted is not a multiple of three. [] {comment='PMID:32461616', comment='PMID:32926138'}"
  },
  'SO:0002388': {
    'name': '5_prime_UTR_uORF_stop_codon_gain_variant',
    'def': "A 5' UTR variant where a premature stop codon is gained in an upstream open reading frame. [PMID:32461616, PMID:32926138]"
  },
  'SO:0002389': {
    'name': '5_prime_UTR_uORF_stop_codon_loss_variant',
    'def': "A 5' UTR variant where the stop codon of an upstream open reading frame is lost. [PMID:32461616, PMID:32926138]"
  },
  'SO:0005836': {
    'name': 'regulatory_region',
    'def': 'A region of sequence that is involved in the control of a biological process. [SO:ke]'
  },
  'SO:0005837': {
    'name': 'U14_snoRNA_primary_transcript',
    'def': 'The primary transcript of an evolutionarily conserved eukaryotic low molecular weight RNA capable of intermolecular hybridization with both homologous and heterologous 18S rRNA. [PMID:2251119]'
  },
  'SO:0005841': {
    'name': 'methylation_guide_snoRNA',
    'def': "A snoRNA that specifies the site of 2'-O-ribose methylation in an RNA molecule by base pairing with a short sequence around the target residue. [GOC:mah, PMID:12457565]"
  },
  'SO:0005843': {
    'name': 'rRNA_cleavage_RNA',
    'def': 'An ncRNA that is part of a ribonucleoprotein that cleaves the primary pre-rRNA transcript in the process of producing mature rRNA molecules. [GOC:kgc]'
  },
  'SO:0005845': {
    'name': 'exon_of_single_exon_gene',
    'def': 'An exon that is the only exon in a gene. [RSC:cb]'
  },
  'SO:0005847': {
    'name': 'cassette_array_member',
    'def': 'A gene that is a member of a gene cassette, which is a mobile genetic element. []'
  },
  'SO:0005848': {
    'name': 'gene_cassette_member',
    'def': 'A gene that is a member of a gene cassette, which is a mobile genetic element. []'
  },
  'SO:0005849': {
    'name': 'gene_subarray_member',
    'def': 'A gene that is a member of a group of genes that are either regulated or transcribed together within a larger group of genes that are regulated or transcribed together. []'
  },
  'SO:0005850': {
    'name': 'primer_binding_site',
    'def': 'Non-covalent primer binding site for initiation of replication, transcription, or reverse transcription. [http://www.insdc.org/files/feature_table.html]'
  },
  'SO:0005851': {
    'name': 'gene_array',
    'def': 'An array includes two or more genes, or two or more gene subarrays, contiguously arranged where the individual genes, or subarrays, are either identical in sequence, or essentially so. [SO:ma]'
  },
  'SO:0005852': {
    'name': 'gene_subarray',
    'def': 'A subarray is, by defintition, a member of a gene array (SO:0005851); the members of a subarray may differ substantially in sequence, but are closely related in function. [SO:ma]'
  },
  'SO:0005853': {
    'name': 'gene_cassette',
    'def': 'A gene that can be substituted for a related gene at a different site in the genome. [SGD:se]'
  },
  'SO:0005854': {
    'name': 'gene_cassette_array',
    'def': 'An array of non-functional genes whose members, when captured by recombination form functional genes. [SO:ma]'
  },
  'SO:0005855': {
    'name': 'gene_group',
    'def': 'A collection of related genes. [SO:ma]'
  },
  'SO:0005856': {
    'name': 'selenocysteine_tRNA_primary_transcript',
    'def': 'A primary transcript encoding seryl tRNA (SO:000269). [SO:ke]'
  },
  'SO:0005857': {
    'name': 'selenocysteinyl_tRNA',
    'def': "A tRNA sequence that has a selenocysteine anticodon, and a 3' selenocysteine binding region. [SO:ke]"
  },
  'SO:0005858': {
    'name': 'syntenic_region',
    'def': 'A region in which two or more pairs of homologous markers occur on the same chromosome in two or more species. [http://www.informatics.jax.org/silverbook/glossary.shtml]'
  },
  'SO:0100001': {
    'name': 'biochemical_region_of_peptide',
    'def': 'A region of a peptide that is involved in a biochemical function. [EBIBS:GAR]'
  },
  'SO:0100002': {
    'name': 'molecular_contact_region',
    'def': 'A region that is involved a contact with another molecule. [EBIBS:GAR]'
  },
  'SO:0100003': {
    'name': 'intrinsically_unstructured_polypeptide_region',
    'def': 'A region of polypeptide chain with high conformational flexibility. [EBIBS:GAR]'
  },
  'SO:0100004': {
    'name': 'catmat_left_handed_three',
    'def': 'A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0100005': {
    'name': 'catmat_left_handed_four',
    'def': 'A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170. The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0100006': {
    'name': 'catmat_right_handed_three',
    'def': 'A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0100007': {
    'name': 'catmat_right_handed_four',
    'def': 'A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170. The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0100008': {
    'name': 'alpha_beta_motif',
    'def': 'A motif of five consecutive residues and two H-bonds in which: H-bond between CO of residue(i) and NH of residue(i+4), H-bond between CO of residue(i) and NH of residue(i+3),Phi angles of residues(i+1), (i+2) and (i+3) are negative. [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]'
  },
  'SO:0100009': {
    'name': 'lipoprotein_signal_peptide',
    'def': 'A peptide that acts as a signal for both membrane translocation and lipid attachment in prokaryotes. [EBIBS:GAR]'
  },
  'SO:0100010': {
    'name': 'no_output',
    'def': 'An experimental region wherean analysis has been run and not produced any annotation. [EBIBS:GAR]'
  },
  'SO:0100011': {
    'name': 'cleaved_peptide_region',
    'def': 'The cleaved_peptide_region is the region of a peptide sequence that is cleaved during maturation. [EBIBS:GAR]'
  },
  'SO:0100012': {
    'name': 'peptide_coil',
    'def': "Irregular, unstructured regions of a protein's backbone, as distinct from the regular region (namely alpha helix and beta strand - characterised by specific patterns of main-chain hydrogen bonds). [EBIBS:GAR]"
  },
  'SO:0100013': {
    'name': 'hydrophobic_region_of_peptide',
    'def': 'Hydrophobic regions are regions with a low affinity for water. [EBIBS:GAR]'
  },
  'SO:0100014': {
    'name': 'n_terminal_region',
    'def': 'The amino-terminal positively-charged region of a signal peptide (approx 1-5 aa). [EBIBS:GAR]'
  },
  'SO:0100015': {
    'name': 'c_terminal_region',
    'def': 'The more polar, carboxy-terminal region of the signal peptide (approx 3-7 aa). [EBIBS:GAR]'
  },
  'SO:0100016': {
    'name': 'central_hydrophobic_region_of_signal_peptide',
    'def': 'The central, hydrophobic region of the signal peptide (approx 7-15 aa). [EBIBS:GAR]'
  },
  'SO:0100017': {
    'name': 'polypeptide_conserved_motif',
    'def': 'A conserved motif is a short (up to 20 amino acids) region of biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found. [EBIBS:GAR]'
  },
  'SO:0100018': {
    'name': 'polypeptide_binding_motif',
    'def': 'A polypeptide binding motif is a short (up to 20 amino acids) polypeptide region of biological interest that contains one or more amino acids experimentally shown to bind to a ligand. [EBIBS:GAR]'
  },
  'SO:0100019': {
    'name': 'polypeptide_catalytic_motif',
    'def': 'A polypeptide catalytic motif is a short (up to 20 amino acids) polypeptide region that contains one or more active site residues. [EBIBS:GAR]'
  },
  'SO:0100020': {
    'name': 'polypeptide_DNA_contact',
    'def': 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with DNA. [EBIBS:GAR, SO:ke]'
  },
  'SO:0100021': {
    'name': 'polypeptide_conserved_region',
    'def': 'A subsection of sequence with biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found. [EBIBS:GAR]'
  },
  'SO:1000002': {
    'name': 'substitution',
    'def': 'A sequence alteration where the length of the change in the variant is the same as that of the reference. [SO:ke]'
  },
  'SO:1000005': {
    'name': 'complex_substitution',
    'def': 'When no simple or well defined DNA mutation event describes the observed DNA change, the keyword "complex" should be used. Usually there are multiple equally plausible explanations for the change. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000008': {
    'name': 'point_mutation',
    'def': 'A single nucleotide change which has occurred at the same position of a corresponding nucleotide in a reference sequence. [SO:immuno_workshop]'
  },
  'SO:1000009': {
    'name': 'transition',
    'def': 'Change of a pyrimidine nucleotide, C or T, into an other pyrimidine nucleotide, or change of a purine nucleotide, A or G, into an other purine nucleotide. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000010': {
    'name': 'pyrimidine_transition',
    'def': 'A substitution of a pyrimidine, C or T, for another pyrimidine. [SO:ke]'
  },
  'SO:1000011': {
    'name': 'C_to_T_transition',
    'def': 'A transition of a cytidine to a thymine. [SO:ke]'
  },
  'SO:1000012': {
    'name': 'C_to_T_transition_at_pCpG_site',
    'def': "The transition of cytidine to thymine occurring at a pCpG site as a consequence of the spontaneous deamination of 5'-methylcytidine. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]"
  },
  'SO:1000013': {
    'name': 'T_to_C_transition',
    'def': 'A transition of a thymine to a cytidine. []'
  },
  'SO:1000014': {
    'name': 'purine_transition',
    'def': 'A substitution of a purine, A or G, for another purine. [SO:ke]'
  },
  'SO:1000015': {
    'name': 'A_to_G_transition',
    'def': 'A transition of an adenine to a guanine. [SO:ke]'
  },
  'SO:1000016': {
    'name': 'G_to_A_transition',
    'def': 'A transition of a guanine to an adenine. [SO:ke]'
  },
  'SO:1000017': {
    'name': 'transversion',
    'def': 'Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G, or vice versa. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000018': {
    'name': 'pyrimidine_to_purine_transversion',
    'def': 'Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G. [SO:ke]'
  },
  'SO:1000019': {
    'name': 'C_to_A_transversion',
    'def': 'A transversion from cytidine to adenine. [SO:ke]'
  },
  'SO:1000020': {
    'name': 'C_to_G_transversion',
    'def': 'A transversion of a cytidine to a guanine. []'
  },
  'SO:1000021': {
    'name': 'T_to_A_transversion',
    'def': 'A transversion from T to A. [SO:ke]'
  },
  'SO:1000022': {
    'name': 'T_to_G_transversion',
    'def': 'A transversion from T to G. [SO:ke]'
  },
  'SO:1000023': {
    'name': 'purine_to_pyrimidine_transversion',
    'def': 'Change of a purine nucleotide, A or G , into a pyrimidine nucleotide C or T. [SO:ke]'
  },
  'SO:1000024': {
    'name': 'A_to_C_transversion',
    'def': 'A transversion from adenine to cytidine. [SO:ke]'
  },
  'SO:1000025': {
    'name': 'A_to_T_transversion',
    'def': 'A transversion from adenine to thymine. [SO:ke]'
  },
  'SO:1000026': {
    'name': 'G_to_C_transversion',
    'def': 'A transversion from guanine to cytidine. [SO:ke]'
  },
  'SO:1000027': {
    'name': 'G_to_T_transversion',
    'def': 'A transversion from guanine to thymine. [SO:ke]'
  },
  'SO:1000028': {
    'name': 'intrachromosomal_mutation',
    'def': 'A chromosomal structure variation within a single chromosome. [SO:ke]'
  },
  'SO:1000029': {
    'name': 'chromosomal_deletion',
    'def': 'An incomplete chromosome. [SO:ke]'
  },
  'SO:1000030': {
    'name': 'chromosomal_inversion',
    'def': 'An interchromosomal mutation where a region of the chromosome is inverted with respect to wild type. [SO:ke]'
  },
  'SO:1000031': {
    'name': 'interchromosomal_mutation',
    'def': 'A chromosomal structure variation whereby more than one chromosome is involved. [SO:ke]'
  },
  'SO:1000032': {
    'name': 'delins',
    'def': 'A sequence alteration which included an insertion and a deletion, affecting 2 or more bases. [http://varnomen.hgvs.org/recommendations/DNA/variant/delins/]'
  },
  'SO:1000035': {
    'name': 'duplication',
    'def': 'An insertion which derives from, or is identical in sequence to, nucleotides present at a known location in the genome. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html, NCBI:th]'
  },
  'SO:1000036': {
    'name': 'inversion',
    'def': 'A continuous nucleotide sequence is inverted in the same position. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000037': {
    'name': 'chromosomal_duplication',
    'def': 'An extra chromosome. [SO:ke]'
  },
  'SO:1000038': {
    'name': 'intrachromosomal_duplication',
    'def': 'A duplication that occurred within a chromosome. [SO:ke]'
  },
  'SO:1000039': {
    'name': 'direct_tandem_duplication',
    'def': 'A tandem duplication where the individual regions are in the same orientation. [SO:ke]'
  },
  'SO:1000040': {
    'name': 'inverted_tandem_duplication',
    'def': 'A tandem duplication where the individual regions are not in the same orientation. [SO:ke]'
  },
  'SO:1000041': {
    'name': 'intrachromosomal_transposition',
    'def': 'A chromosome structure variation whereby a transposition occurred within a chromosome. [SO:ke]'
  },
  'SO:1000042': {
    'name': 'compound_chromosome',
    'def': 'A chromosome structure variant where a monocentric element is caused by the fusion of two chromosome arms. [SO:ke]'
  },
  'SO:1000043': {
    'name': 'Robertsonian_fusion',
    'def': 'A non reciprocal translocation whereby the participating chromosomes break at their centromeres and the long arms fuse to form a single chromosome with a single centromere. [http://en.wikipedia.org/wiki/Robertsonian_translocation]'
  },
  'SO:1000044': {
    'name': 'chromosomal_translocation',
    'def': 'A chromosomal mutation. Rearrangements that alter the pairing of telomeres are classified as translocations. [FB:reference_manual]'
  },
  'SO:1000045': {
    'name': 'ring_chromosome',
    'def': 'A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome. [http://en.wikipedia.org/wiki/Ring_chromosome]'
  },
  'SO:1000046': {
    'name': 'pericentric_inversion',
    'def': 'A chromosomal inversion that includes the centromere. [FB:reference_manual]'
  },
  'SO:1000047': {
    'name': 'paracentric_inversion',
    'def': 'A chromosomal inversion that does not include the centromere. [FB:reference_manual]'
  },
  'SO:1000048': {
    'name': 'reciprocal_chromosomal_translocation',
    'def': 'A chromosomal translocation with two breaks; two chromosome segments have simply been exchanged. [FB:reference_manual]'
  },
  'SO:1000049': {
    'name': 'sequence_variation_affecting_transcript',
    'def': 'Any change in mature, spliced and processed, RNA that results from a change in the corresponding DNA sequence. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000050': {
    'name': 'sequence_variant_causing_no_change_in_transcript',
    'def': 'No effect on the state of the RNA. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000054': {
    'name': 'sequence_variation_affecting_coding_sequence',
    'def': 'Any of the amino acid coding triplets of a gene are affected by the DNA mutation. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000055': {
    'name': 'sequence_variant_causing_initiator_codon_change_in_transcript',
    'def': 'The DNA mutation changes, usually destroys, the first coding triplet of a gene. Usually prevents translation although another initiator codon may be used. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000056': {
    'name': 'sequence_variant_causing_amino_acid_coding_codon_change_in_transcript',
    'def': 'The DNA mutation affects the amino acid coding sequence of a gene; this region includes both the initiator and terminator codons. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000057': {
    'name': 'sequence_variant_causing_synonymous_codon_change_in_transcript',
    'def': 'The changed codon has the same translation product as the original codon. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000058': {
    'name': 'sequence_variant_causing_non_synonymous_codon_change_in_transcript',
    'def': 'A DNA point mutation that causes a substitution of an amino acid by an other. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000059': {
    'name': 'sequence_variant_causing_missense_codon_change_in_transcript',
    'def': 'The nucleotide change in the codon leads to a new codon coding for a new amino acid. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000060': {
    'name': 'sequence_variant_causing_conservative_missense_codon_change_in_transcript',
    'def': 'The amino acid change following from the codon change does not change the gross properties (size, charge, hydrophobicity) of the amino acid at that position. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000061': {
    'name': 'sequence_variant_causing_nonconservative_missense_codon_change_in_transcript',
    'def': 'The amino acid change following from the codon change changes the gross properties (size, charge, hydrophobicity) of the amino acid in that position. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000062': {
    'name': 'sequence_variant_causing_nonsense_codon_change_in_transcript',
    'def': 'The nucleotide change in the codon triplet creates a terminator codon. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000063': {
    'name': 'sequence_variant_causing_terminator_codon_change_in_transcript',
    'def': 'The nucleotide change in the codon triplet changes the stop codon, causing an elongated transcript sequence. [SO:ke]'
  },
  'SO:1000064': {
    'name': 'sequence_variation_affecting_reading_frame',
    'def': 'An umbrella term for terms describing an effect of a sequence variation on the frame of translation. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000065': {
    'name': 'frameshift_sequence_variation',
    'def': 'A mutation causing a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three. [SO:ke]'
  },
  'SO:1000066': {
    'name': 'sequence_variant_causing_plus_1_frameshift_mutation',
    'def': 'A mutation causing a disruption of the translational reading frame, due to the insertion of a nucleotide. [SO:ke]'
  },
  'SO:1000067': {
    'name': 'sequence_variant_causing_minus_1_frameshift',
    'def': 'A mutation causing a disruption of the translational reading frame, due to the deletion of a nucleotide. [SO:ke]'
  },
  'SO:1000068': {
    'name': 'sequence_variant_causing_plus_2_frameshift',
    'def': 'A mutation causing a disruption of the translational reading frame, due to the insertion of two nucleotides. [SO:ke]'
  },
  'SO:1000069': {
    'name': 'sequence_variant_causing_minus_2_frameshift',
    'def': 'A mutation causing a disruption of the translational reading frame, due to the deletion of two nucleotides. [SO:ke]'
  },
  'SO:1000070': {
    'name': 'sequence_variant_affecting_transcript_processing',
    'def': 'Sequence variant affects the way in which the primary transcriptional product is processed to form the mature transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000071': {
    'name': 'sequence_variant_affecting_splicing',
    'def': 'A sequence_variant_effect where the way in which the primary transcriptional product is processed to form the mature transcript, specifically by the removal (splicing) of intron sequences is changed. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000072': {
    'name': 'sequence_variant_affecting_splice_donor',
    'def': 'A sequence_variant_effect that changes the splice donor sequence. [SO:ke]'
  },
  'SO:1000073': {
    'name': 'sequence_variant_affecting_splice_acceptor',
    'def': 'A sequence_variant_effect that changes the splice acceptor sequence. [SO:ke]'
  },
  'SO:1000074': {
    'name': 'sequence_variant_causing_cryptic_splice_activation',
    'def': 'A sequence variant causing a new (functional) splice site. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000075': {
    'name': 'sequence_variant_affecting_editing',
    'def': 'Sequence variant affects the editing of the transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000076': {
    'name': 'sequence_variant_affecting_transcription',
    'def': 'Mutation affects the process of transcription, its initiation, progression or termination. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000078': {
    'name': 'sequence_variant_decreasing_rate_of_transcription',
    'def': 'A sequence variation that decreases the rate a which transcription of the sequence occurs. [SO:ke]'
  },
  'SO:1000079': {
    'name': 'sequence_variation_affecting_transcript_sequence'
  },
  'SO:1000080': {
    'name': 'sequence_variant_increasing_rate_of_transcription'
  },
  'SO:1000081': {
    'name': 'sequence_variant_affecting_rate_of_transcription',
    'def': 'A mutation that alters the rate a which transcription of the sequence occurs. [SO:ke]'
  },
  'SO:1000082': {
    'name': 'sequence variant_affecting_transcript_stability',
    'def': 'Sequence variant affects the stability of the transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000083': {
    'name': 'sequence_variant_increasing_transcript_stability',
    'def': 'Sequence variant increases the stability (half-life) of the transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000084': {
    'name': 'sequence_variant_decreasing_transcript_stability',
    'def': 'Sequence variant decreases the stability (half-life) of the transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000085': {
    'name': 'sequence_variation_affecting_level_of_transcript',
    'def': 'A sequence variation that causes a change in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence. [SO:ke]'
  },
  'SO:1000086': {
    'name': 'sequence_variation_decreasing_level_of_transcript',
    'def': 'A sequence variation that causes a decrease in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence. [SO:ke]'
  },
  'SO:1000087': {
    'name': 'sequence_variation_increasing_level_of_transcript',
    'def': 'A sequence_variation that causes an increase in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence. [SO:ke]'
  },
  'SO:1000088': {
    'name': 'sequence_variant_affecting_translational_product',
    'def': 'A sequence variant causing a change in primary translation product of a transcript. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000089': {
    'name': 'sequence_variant_causing_no_change_of_translational_product',
    'def': 'The sequence variant at RNA level does not lead to any change in polypeptide. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000092': {
    'name': 'sequence_variant_causing_complex_change_of_translational_product',
    'def': 'Any sequence variant effect that is known at nucleotide level but cannot be explained by using other key terms. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000093': {
    'name': 'sequence_variant_causing_amino_acid_substitution',
    'def': 'The replacement of a single amino acid by another. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000094': {
    'name': 'sequence_variant_causing_conservative_amino_acid_substitution'
  },
  'SO:1000095': {
    'name': 'sequence_variant_causing_nonconservative_amino_acid_substitution'
  },
  'SO:1000096': {
    'name': 'sequence_variant_causing_amino_acid_insertion',
    'def': 'The insertion of one or more amino acids from the polypeptide, without affecting the surrounding sequence. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000097': {
    'name': 'sequence_variant_causing_amino_acid_deletion',
    'def': 'The deletion of one or more amino acids from the polypeptide, without affecting the surrounding sequence. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000098': {
    'name': 'sequence_variant_causing_polypeptide_truncation',
    'def': 'The translational product is truncated at its C-terminus, usually a result of a nonsense codon change in transcript (SO:1000062). [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000099': {
    'name': 'sequence_variant_causing_polypeptide_elongation',
    'def': 'The extension of the translational product at either (or both) the N-terminus and/or the C-terminus. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000100': {
    'name': 'mutation_causing_polypeptide_N_terminal_elongation',
    'def': '. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000101': {
    'name': 'mutation_causing_polypeptide_C_terminal_elongation',
    'def': '. [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000102': {
    'name': 'sequence_variant_affecting_level_of_translational_product'
  },
  'SO:1000103': {
    'name': 'sequence_variant_decreasing_level_of_translation_product'
  },
  'SO:1000104': {
    'name': 'sequence_variant_increasing_level_of_translation_product'
  },
  'SO:1000105': {
    'name': 'sequence_variant_affecting_polypeptide_amino_acid_sequence'
  },
  'SO:1000106': {
    'name': 'mutation_causing_inframe_polypeptide_N_terminal_elongation'
  },
  'SO:1000107': {
    'name': 'mutation_causing_out_of_frame_polypeptide_N_terminal_elongation'
  },
  'SO:1000108': {
    'name': 'mutaton_causing_inframe_polypeptide_C_terminal_elongation'
  },
  'SO:1000109': {
    'name': 'mutation_causing_out_of_frame_polypeptide_C_terminal_elongation'
  },
  'SO:1000110': {
    'name': 'frame_restoring_sequence_variant',
    'def': 'A mutation that reverts the sequence of a previous frameshift mutation back to the initial frame. [SO:ke]'
  },
  'SO:1000111': {
    'name': 'sequence_variant_affecting_3D_structure_of_polypeptide',
    'def': 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule. [SO:ke]'
  },
  'SO:1000112': {
    'name': 'sequence_variant_causing_no_3D_structural_change'
  },
  'SO:1000115': {
    'name': 'sequence_variant_causing_complex_3D_structural_change'
  },
  'SO:1000116': {
    'name': 'sequence_variant_causing_conformational_change'
  },
  'SO:1000117': {
    'name': 'sequence_variant_affecting_polypeptide_function'
  },
  'SO:1000118': {
    'name': 'sequence_variant_causing_loss_of_function_of_polypeptide'
  },
  'SO:1000119': {
    'name': 'sequence_variant_causing_inactive_ligand_binding_site'
  },
  'SO:1000120': {
    'name': 'sequence_variant_causing_inactive_catalytic_site'
  },
  'SO:1000121': {
    'name': 'sequence_variant_causing_polypeptide_localization_change'
  },
  'SO:1000122': {
    'name': 'sequence_variant_causing_polypeptide_post_translational_processing_change'
  },
  'SO:1000123': {
    'name': 'polypeptide_post_translational_processing_affected'
  },
  'SO:1000124': {
    'name': 'sequence_variant_causing_partial_loss_of_function_of_polypeptide'
  },
  'SO:1000125': {
    'name': 'sequence_variant_causing_gain_of_function_of_polypeptide'
  },
  'SO:1000126': {
    'name': 'sequence_variant_affecting_transcript_secondary_structure',
    'def': 'A sequence variant that affects the secondary structure (folding) of the RNA transcript molecule. [SO:ke]'
  },
  'SO:1000127': {
    'name': 'sequence_variant_causing_compensatory_transcript_secondary_structure_mutation'
  },
  'SO:1000132': {
    'name': 'sequence_variant_effect',
    'def': 'The effect of a change in nucleotide sequence. [SO:ke]'
  },
  'SO:1000134': {
    'name': 'sequence_variant_causing_polypeptide_fusion'
  },
  'SO:1000136': {
    'name': 'autosynaptic_chromosome',
    'def': 'An autosynaptic chromosome is the aneuploid product of recombination between a pericentric inversion and a cytologically wild-type chromosome. [PMID:6804304]'
  },
  'SO:1000138': {
    'name': 'homo_compound_chromosome',
    'def': 'A compound chromosome whereby two copies of the same chromosomal arm attached to a common centromere. The chromosome is diploid for the arm involved. [SO:ke]'
  },
  'SO:1000140': {
    'name': 'hetero_compound_chromosome',
    'def': 'A compound chromosome whereby two arms from different chromosomes are connected through the centromere of one of them. [FB:reference_manual, SO:ke]'
  },
  'SO:1000141': {
    'name': 'chromosome_fission',
    'def': 'A chromosome that occurred by the division of a larger chromosome. [SO:ke]'
  },
  'SO:1000142': {
    'name': 'dextrosynaptic_chromosome',
    'def': 'An autosynaptic chromosome carrying the two right (D = dextro) telomeres. [FB:manual]'
  },
  'SO:1000143': {
    'name': 'laevosynaptic_chromosome',
    'def': 'LS is an autosynaptic chromosome carrying the two left (L = levo) telomeres. [FB:manual]'
  },
  'SO:1000144': {
    'name': 'free_duplication',
    'def': 'A chromosome structure variation whereby the duplicated sequences are carried as a free centric element. [FB:reference_manual]'
  },
  'SO:1000145': {
    'name': 'free_ring_duplication',
    'def': 'A ring chromosome which is a copy of another chromosome. [SO:ke]'
  },
  'SO:1000147': {
    'name': 'deficient_translocation',
    'def': 'A chromosomal deletion whereby a translocation occurs in which one of the four broken ends loses a segment before re-joining. [FB:reference_manual]'
  },
  'SO:1000148': {
    'name': 'inversion_cum_translocation',
    'def': 'A chromosomal translocation whereby the first two breaks are in the same chromosome, and the region between them is rejoined in inverted order to the other side of the first break, such that both sides of break one are present on the same chromosome. The remaining free ends are joined as a translocation with those resulting from the third break. [FB:reference_manual]'
  },
  'SO:1000149': {
    'name': 'bipartite_duplication',
    'def': 'An interchromosomal mutation whereby the (large) region between the first two breaks listed is lost, and the two flanking segments (one of them centric) are joined as a translocation to the free ends resulting from the third break. [FB:reference_manual]'
  },
  'SO:1000150': {
    'name': 'cyclic_translocation',
    'def': 'A chromosomal translocation whereby three breaks occurred in three different chromosomes. The centric segment resulting from the first break listed is joined to the acentric segment resulting from the second, rather than the third. [FB:reference_manual]'
  },
  'SO:1000151': {
    'name': 'bipartite_inversion',
    'def': 'A chromosomal inversion caused by three breaks in the same chromosome; both central segments are inverted in place (i.e., they are not transposed). [FB:reference_manual]'
  },
  'SO:1000152': {
    'name': 'uninverted_insertional_duplication',
    'def': 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments. [FB:reference_manual]'
  },
  'SO:1000153': {
    'name': 'inverted_insertional_duplication',
    'def': 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments. [FB:reference_manual]'
  },
  'SO:1000154': {
    'name': 'insertional_duplication',
    'def': 'A chromosome duplication involving the insertion of a duplicated region (as opposed to a free duplication). [SO:ke]'
  },
  'SO:1000155': {
    'name': 'interchromosomal_transposition',
    'def': 'A chromosome structure variation whereby a transposition occurred between chromosomes. [SO:ke]'
  },
  'SO:1000156': {
    'name': 'inverted_interchromosomal_transposition',
    'def': 'An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segment. [FB:reference_manual]'
  },
  'SO:1000157': {
    'name': 'uninverted_interchromosomal_transposition',
    'def': 'An interchromosomal transition where the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments. [FB:reference_manual]'
  },
  'SO:1000158': {
    'name': 'inverted_intrachromosomal_transposition',
    'def': 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments. [FB:reference_manual]'
  },
  'SO:1000159': {
    'name': 'uninverted_intrachromosomal_transposition',
    'def': 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments. [FB:reference_manual]'
  },
  'SO:1000160': {
    'name': 'unoriented_insertional_duplication',
    'def': 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded. [FB:reference_manual]'
  },
  'SO:1000161': {
    'name': 'unoriented_interchromosomal_transposition',
    'def': 'An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded. [FB:reference_manual]'
  },
  'SO:1000162': {
    'name': 'unoriented_intrachromosomal_transposition',
    'def': 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded. [FB:reference_manual]'
  },
  'SO:1000170': {
    'name': 'uncharacterized_chromosomal_mutation',
    'def': 'A chromosome structure variant that has not been characterized. []'
  },
  'SO:1000171': {
    'name': 'deficient_inversion',
    'def': 'A chromosomal deletion whereby three breaks occur in the same chromosome; one central region is lost, and the other is inverted. [FB:reference_manual, SO:ke]'
  },
  'SO:1000173': {
    'name': 'tandem_duplication',
    'def': 'A duplication consisting of 2 identical adjacent regions. [SO:ke]'
  },
  'SO:1000175': {
    'name': 'partially_characterized_chromosomal_mutation',
    'def': 'A chromosome structure variant that has not been characterized fully. []'
  },
  'SO:1000180': {
    'name': 'sequence_variant_affecting_gene_structure',
    'def': 'A sequence_variant_effect that changes the gene structure. [SO:ke]'
  },
  'SO:1000181': {
    'name': 'sequence_variant_causing_gene_fusion',
    'def': 'A sequence_variant_effect that changes the gene structure by causing a fusion to another gene. [SO:ke]'
  },
  'SO:1000182': {
    'name': 'chromosome_number_variation',
    'def': 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number. [SO:ke]'
  },
  'SO:1000183': {
    'name': 'chromosome_structure_variation',
    'def': 'An alteration of the genome that leads to a change in the structure or number of one or more chromosomes. []'
  },
  'SO:1000184': {
    'name': 'sequence_variant_causes_exon_loss',
    'def': 'A sequence variant affecting splicing and causes an exon loss. [SO:ke]'
  },
  'SO:1000185': {
    'name': 'sequence_variant_causes_intron_gain',
    'def': 'A sequence variant effect, causing an intron to be gained by the processed transcript; usually a result of a donor acceptor mutation (SO:1000072). [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]'
  },
  'SO:1000186': {
    'name': 'sequence_variant_causing_cryptic_splice_donor_activation'
  },
  'SO:1001186': {
    'name': 'sequence_variant_causing_cryptic_splice_acceptor_activation'
  },
  'SO:1001187': {
    'name': 'alternatively_spliced_transcript',
    'def': 'A transcript that is alternatively spliced. [SO:xp]'
  },
  'SO:1001188': {
    'name': 'encodes_1_polypeptide',
    'def': 'A gene that is alternately spliced, but encodes only one polypeptide. [SO:ke]'
  },
  'SO:1001189': {
    'name': 'encodes_greater_than_1_polypeptide',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide. [SO:ke]'
  },
  'SO:1001190': {
    'name': 'encodes_different_polypeptides_different_stop',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different stop codons. [SO:ke]'
  },
  'SO:1001191': {
    'name': 'encodes_overlapping_peptides_different_start',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start codons. [SO:ke]'
  },
  'SO:1001192': {
    'name': 'encodes_disjoint_polypeptides',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide, that do not have overlapping peptide sequences. [SO:ke]'
  },
  'SO:1001193': {
    'name': 'encodes_overlapping_polypeptides_different_start_and_stop',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start and stop codons. [SO:ke]'
  },
  'SO:1001194': {
    'name': 'alternatively_spliced_gene_encoding_greater_than_1_polypeptide_coding_regions_overlapping'
  },
  'SO:1001195': {
    'name': 'encodes_overlapping_peptides',
    'def': 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences. [SO:ke]'
  },
  'SO:1001196': {
    'name': 'cryptogene',
    'def': 'A maxicircle gene so extensively edited that it cannot be matched to its edited mRNA sequence. [SO:ma]'
  },
  'SO:1001197': {
    'name': 'dicistronic_primary_transcript',
    'def': 'A primary transcript that has the quality dicistronic. [SO:xp]'
  },
  'SO:1001217': {
    'name': 'member_of_regulon',
    'def': 'A gene that is a member of a group of genes that are either regulated or transcribed together. []'
  },
  'SO:1001244': {
    'name': 'alternatively_spliced_transcript_encoding_greater_than_1_polypeptide_different_start_codon_different_stop_codon_coding_regions_non_overlapping'
  },
  'SO:1001246': {
    'name': 'CDS_independently_known',
    'def': 'A CDS with the evidence status of being independently known. [SO:xp]'
  },
  'SO:1001247': {
    'name': 'orphan_CDS',
    'def': 'A CDS whose predicted amino acid sequence is unsupported by any experimental evidence or by any match with any other known sequence. [SO:ma]'
  },
  'SO:1001249': {
    'name': 'CDS_supported_by_domain_match_data',
    'def': 'A CDS that is supported by domain similarity. [SO:xp]'
  },
  'SO:1001251': {
    'name': 'CDS_supported_by_sequence_similarity_data',
    'def': 'A CDS that is supported by sequence similarity data. [SO:xp]'
  },
  'SO:1001254': {
    'name': 'CDS_predicted',
    'def': 'A CDS that is predicted. [SO:ke]'
  },
  'SO:1001255': {
    'name': 'status_of_coding_sequence'
  },
  'SO:1001259': {
    'name': 'CDS_supported_by_EST_or_cDNA_data',
    'def': 'A CDS that is supported by similarity to EST or cDNA data. [SO:xp]'
  },
  'SO:1001260': {
    'name': 'internal_Shine_Dalgarno_sequence',
    'def': 'A Shine-Dalgarno sequence that stimulates recoding through interactions with the anti-Shine-Dalgarno in the RNA of small ribosomal subunits of translating ribosomes. The signal is only operative in Bacteria. [PMID:12519954, SO:ke]'
  },
  'SO:1001261': {
    'name': 'recoded_mRNA',
    'def': 'The sequence of a mature mRNA transcript, modified before translation or during translation, usually by special cis-acting signals. [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8811194&dopt=Abstract]'
  },
  'SO:1001262': {
    'name': 'minus_1_translationally_frameshifted',
    'def': 'An attribute describing a translational frameshift of -1. [SO:ke]'
  },
  'SO:1001263': {
    'name': 'plus_1_translationally_frameshifted',
    'def': 'An attribute describing a translational frameshift of +1. [SO:ke]'
  },
  'SO:1001264': {
    'name': 'mRNA_recoded_by_translational_bypass',
    'def': 'A recoded_mRNA where translation was suspended at a particular codon and resumed at a particular non-overlapping downstream codon. [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8811194&dopt=Abstract]'
  },
  'SO:1001265': {
    'name': 'mRNA_recoded_by_codon_redefinition',
    'def': 'A recoded_mRNA that was modified by an alteration of codon meaning. [SO:ma]'
  },
  'SO:1001266': {
    'name': 'stop_codon_redefinition_as_selenocysteine'
  },
  'SO:1001267': {
    'name': 'stop_codon_readthrough'
  },
  'SO:1001268': {
    'name': 'recoding_stimulatory_region',
    'def': 'A site in an mRNA sequence that stimulates the recoding of a region in the same mRNA. [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12519954&dopt=Abstract]'
  },
  'SO:1001269': {
    'name': 'four_bp_start_codon',
    'def': 'A non-canonical start codon with 4 base pairs. [SO:ke]'
  },
  'SO:1001270': {
    'name': 'stop_codon_redefinition_as_pyrrolysine'
  },
  'SO:1001271': {
    'name': 'archaeal_intron',
    'def': 'An intron characteristic of Archaeal tRNA and rRNA genes, where intron transcript generates a bulge-helix-bulge motif that is recognised by a splicing endoribonuclease. [PMID:9301331, SO:ma]'
  },
  'SO:1001272': {
    'name': 'tRNA_intron',
    'def': 'An intron found in tRNA that is spliced via endonucleolytic cleavage and ligation rather than transesterification. [SO:ke]'
  },
  'SO:1001273': {
    'name': 'CTG_start_codon',
    'def': 'A non-canonical start codon of sequence CTG. [SO:ke]'
  },
  'SO:1001274': {
    'name': 'SECIS_element',
    'def': "The incorporation of selenocysteine into a protein sequence is directed by an in-frame UGA codon (usually a stop codon) within the coding region of the mRNA. Selenoprotein mRNAs contain a conserved secondary structure in the 3' UTR that is required for the distinction of UGA stop from UGA selenocysteine. The selenocysteine insertion sequence (SECIS) is around 60 nt in length and adopts a hairpin structure which is sufficiently well-defined and conserved to act as a computational screen for selenoprotein genes. [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00031]"
  },
  'SO:1001275': {
    'name': 'retron',
    'def': 'Sequence coding for a short, single-stranded, DNA sequence via a retrotransposed RNA intermediate; characteristic of some microbial genomes. [SO:ma]'
  },
  'SO:1001277': {
    'name': 'three_prime_recoding_site',
    'def': 'The recoding stimulatory signal located downstream of the recoding site. [SO:ke]'
  },
  'SO:1001279': {
    'name': 'three_prime_stem_loop_structure',
    'def': 'A recoding stimulatory region, the stem-loop secondary structural element is downstream of the redefined region. [PMID:12519954, SO:ke]'
  },
  'SO:1001280': {
    'name': 'five_prime_recoding_site',
    'def': 'The recoding stimulatory signal located upstream of the recoding site. [SO:ke]'
  },
  'SO:1001281': {
    'name': 'flanking_three_prime_quadruplet_recoding_signal',
    'def': 'Four base pair sequence immediately downstream of the redefined region. The redefined region is a frameshift site. The quadruplet is 2 overlapping codons. [PMID:12519954, SO:ke]'
  },
  'SO:1001282': {
    'name': 'UAG_stop_codon_signal',
    'def': 'A stop codon signal for a UAG stop codon redefinition. [SO:ke]'
  },
  'SO:1001283': {
    'name': 'UAA_stop_codon_signal',
    'def': 'A stop codon signal for a UAA stop codon redefinition. [SO:ke]'
  },
  'SO:1001284': {
    'name': 'regulon',
    'def': 'A set of units of gene expression directly regulated by a common set of one or more common regulatory gene products. [ISBN:0198506732, PMID:32665585]'
  },
  'SO:1001285': {
    'name': 'UGA_stop_codon_signal',
    'def': 'A stop codon signal for a UGA stop codon redefinition. [SO:ke]'
  },
  'SO:1001286': {
    'name': 'three_prime_repeat_recoding_signal',
    'def': 'A recoding stimulatory signal, downstream sequence important for recoding that contains repetitive elements. [PMID:12519954, SO:ke]'
  },
  'SO:1001287': {
    'name': 'distant_three_prime_recoding_signal',
    'def': "A recoding signal that is found many hundreds of nucleotides 3' of a redefined stop codon. [http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8709208&dopt=Abstract]"
  },
  'SO:1001288': {
    'name': 'stop_codon_signal',
    'def': 'A recoding stimulatory signal that is a stop codon and has effect on efficiency of recoding. [PMID:12519954, SO:ke]'
  },
  'SO:2000061': {
    'name': 'databank_entry',
    'def': 'The sequence referred to by an entry in a databank such as GenBank or SwissProt. [SO:ke]'
  },
  'variant_of': {
    'name': 'variant_of',
    'def': "A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A. [SO:immuno_workshop]"
  }
}
